# A List of ontologies and biological resources relevant for sample and assay description

The ELIXIR implementation study ‘Next level of reproducible, comparable and integrable Metabolomics’ (WP5) is part of the ELIXIR Cellular and Molecular Research” commissioned service (ELIXIR CMR-COS) roadmap for 2024-2028. WP5 addresses several challenges faced by the metabolomics community by developing interoperable solutions, promoting FAIR principles for data and enabling broad integration of metabolic profiles with other omics data in different applicative contexts. The metabolomics community within ELIXIR has been able to benefit from key standards and existing open source tools, laying the foundations for further use of harmonised data, the evolution of key infrastructures, and the establishment of robust means to move towards more inter-study comparisons.

Within this framework, Activity 3 is dedicated to improving the interpretation and meta-analysis of metabolomics data. It pushes for systematic annotation of compounds, using stable identifiers and automated reporting of confidence levels to make data more reliable and reusable—exemplified by enhancements to platforms like PeakForest. By enriching sample-level metadata and aligning it with external molecular biological repositories and ontologies, the activity enables powerful cross-study meta-analyses. Furthermore, Activity 3 develops species-specific resources and semantic knowledge graphs built with Linked Open Data methods.

As a result of Milestone 5.4, the RCI-M project team has compiled a list of useful European resources to help biologists, bioinformaticians and chemists semantically describe a sample or test description, for example to facilitate the submission of studies to the MetaboLights repository or to make new information systems interoperable. This intermediate step made it possible to draw up a list of the main resources maintained and an inventory of relevant ontologies or controlled vocabularies, selected and grouped by category from the NCBO BioPortal. To select your terms from the ontologies proposed, beyond the meaning of the concept itself, the team recommends adding the following criteria: 

* The selected term has a definition.
* The selected term is part of an extensive hierarchy in the selected thesaurus/ontology.
* The selected term establishes equivalence links to other thesauri.
* The thesaurus/ontology is recognized within the scientific community of the dataset.

If you want to participate to this initiative, don't hesitate to contact the Elixir RCI-M project: [cmr-wp5@elixir-europe.org](mailto:cmr-wp5@elixir-europe.org)

## Main biological resources relevant for sample and assay description

  + OLS (Ontology Lookup Service) - EU - [Homepage](https://www.ebi.ac.uk/ols/index)
>The Ontology Lookup Service (OLS) is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions. You can browse the ontologies through the website as well as programmatically via the OLS API. In 2023 OLS was updated to scale better and with a new user interface. OLS is used within life sciences but also in the fields of chemistry and engineering. Code is available under an Apache 2.0 licence.
  
  + NCBO BioPortal - EU - [Homepage](https://bioportal.bioontology.org/)
>BioPortal is a library of biomedical ontologies and terminologies developed in Web Ontology Language (OWL), Resource Description Framework (RDF)(S), Open Biological and Biomedical Ontologies (OBO) format, and Protégé frames and Rich Release Format. BioPortal groups ontologies by domain to aid discoverability and allows users to browse, search and visualize the content of ontologies. BioPortal also provides ontology recommendation, text annotation, ontology mapping, and a reference index to resources like PubMed.
  
  + FAIRsharing - EU - [Homepage](https://fairsharing.org/)
>FAIRsharing is a FAIR-supporting resource that provides an informative and educational registry on data standards, databases, repositories and policy, alongside search and visualization tools and services that interoperate with other FAIR-enabling resources. FAIRsharing guides consumers to discover, select and use standards, databases, repositories and policy with confidence, and producers to make their resources more discoverable, more widely adopted and cited. Each record in FAIRsharing is curated in collaboration with the maintainers of the resource themselves, ensuring that the metadata in the FAIRsharing registry is accurate and timely. Every record is manually reviewed at least once a year. Records can be collated into collections, based on a project, society, organisation or other community. 
  
  + AgroPortal - FR - [Homepage](https://agroportal.lirmm.fr/)
>AgroPortal is an ontology repository (or semantic artefact catalogue) for agri-food. It provides ontology hosting, search, versioning, visualization, comment, and recommendation; enables semantic annotation; stores and exploits ontology alignments; and enables interoperation with the semantic web. To align with the needs of the agronomy community, AgroPortal uses SKOS vocabularies and trait dictionaries) and supported features (offering detailed metadata and advanced annotation capabilities).
  
  + OntoBee - US - [Homepage](https://ontobee.org/)
>A linked data server designed for ontologies. Ontobee is aimed to facilitate ontology data sharing, visualization, query, integration, and analysis. Ontobee dynamically dereferences and presents individual ontology term URIs to (i) HTML web pages for user-friendly web browsing and navigation, and to (ii) RDF source code for Semantic Web applications. Ontobee is the default linked data server for most OBO Foundry library ontologies. Ontobee has also been used for many non-OBO ontologies. 
  
  

  More information are available on the [FAIR COOK BOOK web portal](https://faircookbook.elixir-europe.org/content/recipes/infrastructure/vocabularies/Selecting_and_using_ontology_lookup_services.html)

## EBI MetaboLights' ‘favourite’ ontologies

Based on all public studies available in MTBLS, the following table compiles a list of the most commonly used ontologies to describe information relating to `organism part`, `organism` and `sample type`.

| Name | ID | Description | Version | Status | MTBLS part |
|------|----|-------------|---------|--------|------------|
| The BRENDA Tissue Ontology | BTO | A structured controlled vocabulary for the source of an enzyme comprising tissues, cell lines, cell types and cell cultures. | 2021-10-26 | prod | organism part, sample type |
| NCI Thesaurus OBO Edition | NCIT | NCI Thesaurus (NCIt)is a reference terminology that includes broad coverage of the cancer domain, including cancer related diseases, findings and abnormalities. The NCIt OBO Edition aims to increase integration of the NCIt with OBO Library ontologies. NCIt OBO Edition releases should be considered experimental. | 25.06e | prod | organism part, organism, sample type |
|MetaboLigths| MTBLS ||||organism part, organism, sample type|
|Chemical Entities of Biological Interest| CHEBI |A structured classification of molecular entities of biological interest focusing on 'small' chemical compounds.|245|prod|organism part, organism, |
|Experimental Factor Ontology| EFO |The Experimental Factor Ontology (EFO) provides a systematic description of many experimental variables available in EBI databases, and for external projects such as the NHGRI GWAS catalogue. It combines parts of several biological ontologies, such as anatomy, disease and chemical compounds. The scope of EFO is to support the annotation, analysis and visualization of data handled by many groups at the EBI and as the core ontology for OpenTargets.org|3.83.0|prod|organism part, organism, sample type |
|Uber-anatomy ontology| UBERON |Uberon is an integrated cross-species anatomy ontology representing a variety of entities classified according to traditional anatomical criteria such as structure, function and developmental lineage. The ontology includes comprehensive relationships to taxon-specific anatomical ontologies, allowing integration of functional, phenotype and expression data.||prod|organism part, sample type|
|Ontology for MIRNA Target| OMIT |Ontology to establish data exchange standards and common data elements in the microRNA (miR) domain||prod|organism part, organism, |
|Plant Ontology| PO |The Plant Ontology is a structured vocabulary and database resource that links plant anatomy, morphology and growth and development to plant genomics data.||prod|organism part, |
|BioAssay Ontology| BAO |The BioAssay Ontology (BAO) describes biological screening assays and their results including high-throughput screening (HTS) data for the purpose of categorizing assays and data analysis. BAO is an extensible, knowledge-based, highly expressive (currently SHOIQ(D)) description of biological assays making use of descriptive logic based features of the Web Ontology Language (OWL). BAO currently has over 700 classes and also makes use of several other ontologies. It describes several concepts related to biological screening, including Perturbagen, Format, Meta Target, Design, Detection Technology, and Endpoint. Perturbagens are perturbing agents that are screened in an assay; they are mostly small molecules. Assay Meta Target describes what is known about the biological system and / or its components interrogated in the assay (and influenced by the Perturbagen). Meta target can be directly described as a molecular entity (e.g. a purified protein or a protein complex), or indirectly by a biological process or event (e.g. phosphorylation). Format describes the biological or chemical features common to each test condition in the assay and includes biochemical, cell-based, organism-based, and variations thereof. The assay Design describes the assay methodology and implementation of how the perturbation of the biological system is translated into a detectable signal. Detection Technology relates to the physical method and technical details to detect and record a signal. Endpoints are the final HTS results as they are usually published (such as IC50, percent inhibition, etc). BAO has been designed to accommodate multiplexed assays. All main BAO components include multiple levels of sub-categories and specification classes, which are linked via object property relationships forming an expressive knowledge-based representation.|2.8.18a|prod|organism part, organism, sample type |
|Metagenome and Microbes Environmental Ontology| MEO |An ontology for describing organismal habitats, with a particular focus on microbial environments.|2025-08-27|beta|organism part, |
|Chemical Methods Ontology| CHMO |CHMO, the chemical methods ontology, describes methods used to collect data in chemical experiments, such as mass spectrometry and electron microscopy prepare and separate material for further analysis, such as sample ionisation, chromatography, and electrophoresis synthesise materials, such as epitaxy and continuous vapour deposition It also describes the instruments used in these experiments, such as mass spectrometers and chromatography columns. It is intended to be complementary to the Ontology for Biomedical Investigations (OBI).|2025-01-22|prod|organism part, sample type |
|The Environment Ontology| ENVO | ENVO is an ontology which represents knowledge about environments,environmental processes, ecosystems, habitats, and related entities|2025-08-19|prod|organism part, organism, |
|Cell Line Ontology| CLO |The Cell Line Ontology (CLO) is a community-based ontology of cell lines. The CLO is developed to unify publicly available cell line entry data from multiple sources to a standardized logically defined format based on consensus design patterns.|2.1.188|prod|organism part, |
|NCBI organismal classification| NCBITAXON |An ontology representation of the NCBI organismal taxonomy|2025-09-11|prod|organism part, organism, sample type |
|Interlinking Ontology for Biological Concepts| IOBC |Interlinking Ontology for Biological Concepts (IOBC) contains approximately 80,000 biological concepts such as biological phenomena, diseases, molecular functions, gene products, chemicals, and medical cares, and approximately 20,000 related concepts such as earth science and environmental science. The concepts are structured by thirty-five relations such as has function, has role, has quality, and precedes. Each concept has both of English and Japanese labels|2021-09-25|beta|organism part, |
|Metabolomics Standards Initiative Ontology (MSIO)| MSIO |n application ontology for supporting description and annotation of mass-spectrometry and nmr-spectroscopy based metabolomics experiments and fluxomics studies.|1.0.1|prod|organism part, organism, sample type |
|Gene Ontology| GO |The Gene Ontology (GO) provides a framework and set of concepts for describing the functions of gene products from all organisms.|2025-10-10|prod|organism part, |
|Food Ontology| FOODON |A broadly scoped ontology representing entities which bear a “food role”. It encompasses materials in natural ecosystems and agriculture that are consumed by humans and domesticated animals. This includes any generic (unbranded) raw or processed food material found in processing plants, markets, stores or food distribution points. FoodOn also imports nutritional component and dietary pattern terms from other OBO Foundry ontologies to support interoperability in diet and nutrition research|2025-08-01|prod|organism part, |
|Medical Subject Headings| MESH |MeSH (Medical Subject Headings) is the National Library of Medicine's controlled vocabulary thesaurus. It consists of sets of terms naming descriptors in a hierarchical structure that permits searching at various levels of specificity. This thesaurus is used by NLM for indexing articles from biomedical journals, cataloguing of books, documents, etc.|2025|prod|organism part, organism |
|Porifera (sponge) ontology| PORO |An ontology describing the anatomical structures and characteristics of Porifera (sponges)||prod|organism part, |
|World Register of Marine Species| WORMS |||| organism, |
|Ontology for Biomedical Investigations| OBI |An ontology for representing biomedical investigations, including study designs, the collection and preparation of the targets of investigation, assays, instrumentation and reagents used, as well as the data generated and the types of analysis performed on the data to reach conclusions, and their documentation.|2025-07-28|prod| organism, |
|Population and Community Ontology| PCO |The Population and Community Ontology (PCO) describes material entities, qualities, and processes related to collections of interacting organisms such as populations and communities. It is taxon neutral, and can be used for any species, including humans. The classes in the PCO are useful for describing evolutionary processes, organismal interactions, and ecological experiments. Practical applications of the PCO include community health care, plant pathology, behavioral studies, sociology, and ecology.||prod| organism, |
|Mass spectrometry ontology| MS |A structured controlled vocabulary for the annotation of experiments concerned with proteomics mass spectrometry.||prod| organism, sample type |
|Antibiotic Resistance Ontology| ARO |Antibiotic resistance genes and mutations||prod| sample type |

The ‘favourite’ tag is linked to a series of computer studies on the frequency and origin of terms used to define the properties of the `organism`, `organism part`, and `sample type` in all public MTBLS studies available in October 2025. The onotology name, description, version and status were extracted from the NCBO BioPortal and the OLS.

<a name="bioportal-categories-list"></a>
## BioPortal Categories list

The RCI-M project team has also compiled a list of useful ontological resources, grouped by category.

  + [BioPortal Category Anatomy](#bioportal-category-anatomy)
  + [BioPortal Category Animal Development](#bioportal-category-animal-development)
  + [BioPortal Category Animal Gross Anatomy](#bioportal-category-animal-gross-anatomy)
  + [BioPortal Category Arabidopsis](#bioportal-category-arabidopsis)
  + [BioPortal Category Biological Process](#bioportal-category-biological-process)
  + [BioPortal Category Biomedical Resources](#bioportal-category-biomedical-resources)
  + [BioPortal Category Cell](#bioportal-category-cell)
  + [BioPortal Category Cellular Anatomy](#bioportal-category-cellular-anatomy)
  + [BioPortal Category Chemical](#bioportal-category-chemical)
  + [BioPortal Category Development](#bioportal-category-development)
  + [BioPortal Category Dysfunction](#bioportal-category-dysfunction)
  + [BioPortal Category Experimental Conditions](#bioportal-category-experimental-conditions)
  + [BioPortal Category Fish Anatomy](#bioportal-category-fish-anatomy)
  + [BioPortal Category Gene Product](#bioportal-category-gene-product)
  + [BioPortal Category Genomic and Proteomic](#bioportal-category-genomic-and-proteomic)
  + [BioPortal Category Gross Anatomy](#bioportal-category-gross-anatomy)
  + [BioPortal Category Health](#bioportal-category-health)
  + [BioPortal Category Human](#bioportal-category-human)
  + [BioPortal Category Human Developmental Anatomy](#bioportal-category-human-developmental-anatomy)
  + [BioPortal Category Imaging](#bioportal-category-imaging)
  + [BioPortal Category Immunology](#bioportal-category-immunology)
  + [BioPortal Category Microbial Anatomy](#bioportal-category-microbial-anatomy)
  + [BioPortal Category Molecule](#bioportal-category-molecule)
  + [BioPortal Category Mouse Anatomy](#bioportal-category-mousse-anatomy)
  + [BioPortal Category Neurologic Disease](#bioportal-category-neurologic-disease)
  + [BioPortal Category Neurological Disorder](#bioportal-category-neurological-disorder)
  + [BioPortal Category Phenotype](#bioportal-category-phenotype)
  + [BioPortal Category Physicochemical](#bioportal-category-physicochemical)
  + [BioPortal Category Plant](#bioportal-category-plant)
  + [BioPortal Category Plant Anatomy](#bioportal-category-plant-anatomy)
  + [BioPortal Category Plant Development](#bioportal-category-plant-development)
  + [BioPortal Category Protein](#bioportal-category-protein)
  + [BioPortal Category Subcellular](#bioportal-category-subcellular)
  + [BioPortal Category Subcellular anatomy](#bioportal-category-subcellular-anatomy)
  + [BioPortal Category Taxonomic Classification](#bioportal-category-taxonomic-classification)
  + [BioPortal Category Yeast](#bioportal-category-yeast)

## BioPortal Category "Anatomy"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Animal Natural History and Life History Ontology | ADW | An ontology for animal life history and natural history characteristics suitable for populations and higher taxonomic entities. | 10/11/04 | production | - |
| Coleoptera Anatomy Ontology | COLAO | The Coleoptera Anatomy Ontology contains terms used for describing the anatomy and phenotype of beetles in biodiversity research. It has been built using the Ontology Develoment Kit with the Ontology for the Anatomy of the Insect SkeletoMuscular system (AISM) as a backbone. The project is open for collaboration. | 2024-06-21 | alpha | beetles (7041) |
| Neuroscience Domain Insight Graph | NEUDIGS | Neuroscience Domain Insight Graphs provide a framework for expressing linked open data about neuronal populations expressed as statements about (A) distributions of subcellular components, (B) measurable physiological changes under various experimental conditions. The goal of this representation is to provide a computable language for reasoning over experimental neuroscientific findings.    | 0.0.1 | alpha | neuronal (CELL) |
| Cell Line Ontology | CLO | The Cell Line Ontology (CLO) is a community-based ontology in domain of biological cell lines with a focus on permanent cell lines from culture collections. Upper ontology structures that frame the skeleton of CLO include Basic Formal Ontology and Relation Ontology. Cell lines contained in CLO are associated with terms from other ontologies such as Cell Type Ontology, NCBI Taxonomy, and Ontology for Biomedical Investigation. A common design pattern for the cell line is used to model cell lines and their attributes, the Jurkat cell line provides ane xample.  Currently CLO contains over 36,000 cell line entries obtained from ATCC, HyperCLDB, Coriell, and bymanual curation. The cell lines are derived from 194 cell types, 656 anatomical entries, and 217 organisms. The OWL-based CLO is machine-readable and can be used in various applications. | 2.1.188 | production | Cell Line (CELL), cell lines (CELL), Cell lines (CELL), Cell Type (CELL), cell line (CELL), Jurkat cell line (CELL), ATCC (CELL), Coriell (CELL), cell (CELL) |
| In Vivo Methodologies Ontology for MAATrica metric | VVMM | MAATrica is a novel metric for assessing the coherence of methodologies in research papers within the fields of medicinal and nutraceutical chemistry. MAATrica employs user-controlled and customizable ontologies (in silico, in vitro and in vivo), enabling personalized analysis of research papers in the fields of medicinal and nutraceutical chemistry. Here we present the in vivo ontology we developed for the scope. |  | alpha | - |
| Ontology for the Anatomy of the Insect SkeletoMuscular system | AISM | The AISM contains terms used in insect biodiversity research for describing structures of the exoskeleton and the skeletomuscular system. It aims to serve as the basic backbone of generalized terms to be expanded with order-specific terminology. | 2023-04-14 | alpha | exoskeleton (TISSUE), skeletomuscular (MULTI_TISSUE_STRUCTURE) |
| Biomedical Informatics Research Network Project Lexicon | BIRNLEX | The BIRN Project lexicon will provide entities for data and database annotation for the BIRN project, covering anatomy, disease, data collection, project management and experimental design. | 1.3.1 | production | BIRN (CANCER), collection (32644) |
| Cephalopod Ontology | CEPH | An anatomical and developmental ontology for cephalopods |  | production | cephalopods (6605) |
| Common Diabetes Medications Ontology | CDMONTO | The Common Diabetes Medications Ontology (CDMOnto) is an ontology-based decision support system that has been developed with the objective of providing appropriate treatment suggestions for diabetic patients. The CDMOnto Ontology serves as a foundation for shared semantics, domain-specific, standard, and interoperable knowledge pertinent to common diabetes medications. It encompasses the knowledge of diabetes medications and offers a unified terminology that can be disseminated and processed by medical professionals, pharmacists, and diabetic patients. |  | beta | - |
| Xenopus Anatomy Ontology | XAO | A structured, controlled vocabulary of the anatomy and development of the African clawed frogs (Xenopus laevis and Xenopus tropicalis). | releases/2024-09-03 | production | frogs (8342), Xenopus laevis (8355), Xenopus tropicalis (GENE_OR_GENE_PRODUCT) |
| Halocynthia roretzi Anatomy and Development Ontology | HAROREADO |  "The first ontology describing the anatomy and the development of Halocynthia roretzi. " |  | alpha | Halocynthia roretzi (7729) |
| Neomark Oral Cancer Ontology, version 4 | NEOMARK4 | Ontology that describes the medical information necessary for early detection of the oral cancer reoccurrence extracted from the NeoMark Project. | 4.1 | production | cancer (6754) |
| Human Reference Atlas Common Coordinate Framework Ontology | CCF | The Common Coordinate Framework (CCF) Ontology is an application ontology built to support the development of the Human Reference Atlas (HRA).  It unifies vocabulary for HRA construction and usage—making it possible to ingest external data sources; supporting uniform tissue sample registration that includes the spatial positioning and semantic annotations within 3D reference organs; and supporting user-formulated cross-domain queries over tissue donor properties, anatomical structures, cell types, biomarkers, and 3D space. The CCF Ontology consists of three major ontologies. The Biological Structure Ontology records anatomical structures, cell types, and biomarkers (ASCT+B) and the relationships between them.  The ASCT+B tables are authored by human experts using templated Google Sheets. The biomarkers, cell types, and anatomical structures are mapped to existing ontologies (Uberon/FMA, CL, HGNC) whenever possible.  All relationships between anatomical structures and from cell types to anatomical structures are valid Uberon and CL relationships. The Spatial Ontology defines the shape, size, location, and rotation of experimental tissue and data major anatomical structures in the 3D Reference Object Library. The Specimen Ontology captures the sex, age, and other information on donors that provided tissue data used in the construction of the HRA. | 3.0.0 | production | CCF (MULTI_TISSUE_STRUCTURE), Human (9606), HRA (CANCER), tissue (TISSUE), organs (ORGAN), cell (CELL), 3D (GENE_OR_GENE_PRODUCT), human (9606), CL (MULTI_TISSUE_STRUCTURE) |
| Tick Gross Anatomy Ontology | TADS | The anatomy of the Tick, Families: Ixodidae, Argassidae | 1.21 | beta | - |
| Ciona savignyi Anatomy and Development Ontology  | CISAVIADO | The first ontology describing the anatomy and the development of Ciona savignyi, based on the Hotta developmental table.  |  | alpha | Ciona (7718), Hotta developmental (GENE_OR_GENE_PRODUCT) |
| Graphic Descriptor Ontology Anatomy Extension | GDOA | For the best browsing experience, see our illustrated browser at https://gdo.endlessforms.info that provides visual depictions of many classes and individuals. The GitHub repository is at https://github.com/endlessforms-info/GDO-GDOA. The version of the GDO/A here is the GDOA and the GDO merged. The version number applies to the merged variant. For separate GDOA and GDO artifacts, please visit the GitHub repo above. The Graphic Descriptor Ontology (GDO) is intended for use in describing graphics that represent the form of objects. The Graphic Descriptor Ontology Anatomy extension (GDOA) extends the GDO to the domain of anatomical graphics by providing classes for anatomical views and anatomically-relevant region mark roles, such as representing a type of body tissue or body fluid. | 1.0.4 | production | GDO/A (CANCER), body tissue (TISSUE) |
| Dictyostelium Discoideum Anatomy Ontology | DDANAT | A structured controlled vocabulary of the anatomy of the slime-mould Dictyostelium discoideum. | 1.19 | production | slime-mould Dictyostelium discoideum (CELLULAR_COMPONENT) |
| Teleost Anatomy Ontology | TAO | TAO is a multispecies anatomy ontology for teleost fishes. Originally seeded from ZFA, but intended to cover terms relevant to other taxa. | 2012-08-10 | production | TAO (GENE_OR_GENE_PRODUCT), teleost fishes (32443) |
| Anatomical Entity Ontology | AEO | AEO is an ontology of anatomical structures that expands CARO, the Common Anatomy Reference Ontology, to about 200 classes using the is_a relationship; it thus provides a detailed type classification for tissues. The new classes were chosen for their use in categorizing the major vertebrate and invertebrate anatomy ontologies at a granularity adequate for tissues of a single cell type. The ontology should be useful in increasing the amount of knowledge in anatomy ontologies, facilitating annotation and enabling interoperability across anatomy ontologies |  | production | AEO (GENE_OR_GENE_PRODUCT), tissues (TISSUE), cell (CELL) |
| Computational Ontology Framework for Brain Profiles | BRAIN-COF | BRAIN-COF is an ontology for representing neuroanatomical concepts aligned with the Desikan–Killiany–Tourville (DKT) brain atlas. Using hierarchical relationships, our ontology explicitly encodes a priori neuroanatomical relationships amongst brain regions. Brain-COF implements computationally efficient calculation of volumetric deviation indicative of disease pathology. | 3 | production | DKT (CANCER), brain (ORGAN), disease (CANCER) |
| Ctenophore Ontology | CTENO | An anatomical and developmental ontology for ctenophores (Comb Jellies) |  | production | ctenophores (10197), Comb (ANATOMICAL_SYSTEM) |
| Planarian Anatomy and Developmental Stage Ontolgoy | PLANA | Anatomy ontology for planaria and terms specific to the developmental stages of the planarian Schmidtea mediterranea. | 2023-03-13 | production | planarian Schmidtea mediterranea (CANCER) |
| Porifera Ontology | PORO | An ontology covering the anatomy of Porifera (sponges) | unknown | beta | sponges (6040) |
| Ontology of Craniofacial Development and Malformation | OCDM | The Ontology of Craniofacial Development and Malformation (OCDM) is a mechanism for representing knowledge about craniofacial development and malformation, and for using that knowledge to facilitate integrating craniofacial data obtained via multiple techniques from multiple labs and at multiple levels of granularity. The OCDM is a project of the NIDCR-sponsored FaceBase Consortium, whose goal is to promote and enable research into the genetic and epigenetic causes of specific craniofacial abnormalities through the provision of publicly accessible, integrated craniofacial data.  | 1.10.0 | production | Craniofacial (CANCER), craniofacial (PATHOLOGICAL_FORMATION), FaceBase (GENE_OR_GENE_PRODUCT) |
| A comprehensive ontology of flaps in reconstructive surgery | OFL | Introduction: The diversity of flap types in plastic surgery has led to multiple classification systems based on factors such as blood supply, tissue composition and distance between donor and recipient site. However, no comprehensive framework exists that systematically integrates all relevant aspects. Capturing the full range of surgical flaps in a structured manner could facilitate future research, improve interoperability, and even optimize billing processes. Ontologies could address this gap. They offer a structured approach to organizing and formalizing knowledge by defining a hierarchical tree of classes and individuals linked logically to other entities. Materials and Methods: We applied principles of ontology engineering, integrating knowledge from surgical literature and clinical expertise to model a wide variety of flaps. These are linked to the Foundational Model of Anatomy (FMA) ontology to describe their tissue components. The reasoner Hermit was employed to ensure logical consistency. Results: Structured hierarchically, the ontology links flaps to relevant anatomical structures and applies logical constraints to define necessary and sufficient conditions for each flap type. This enables precise representation of flap variations, each uniquely identified by an Ontology of Flaps in Reconstructive Surgery Identifier (OFLID). The developed ontology encompasses a wide range of local, pedicled, and free flaps. Simple local random-pattern V-Y flaps to complex chimeric and supercharged free flaps can be modelled and automatically classified. Conclusions: This ontology of surgical flaps could facilitate interoperability, multicentric research and artificial intelligence applications. Future work will focus on validation with larger datasets and expanding the ontology’s functionality.  | 1.0.0 | alpha | flap types (PATHOLOGICAL_FORMATION), blood (ORGANISM_SUBSTANCE), tissue (TISSUE), all (1), flaps (ORGAN), flap type (PATHOLOGICAL_FORMATION), flap (PATHOLOGICAL_FORMATION), Flaps (ORGAN), pedicled (PATHOLOGICAL_FORMATION), random-pattern V-Y flaps (PATHOLOGICAL_FORMATION), artificial (32630) |
| physiotherapy | PHY | This ontology contains structured data for five physiotherapy exercises tailored for lower limb rehabilitation. Each exercise is aimed at supporting patients post-surgery, with a particular focus on preventing complications like deep vein thrombosis (DVT) and improving strength, mobility, and blood circulation in the legs. Included Exercises: Ankle Pump Exercise: Enhances blood flow in the lower limbs by flexing and extending the ankle, helping to prevent DVT. Quad Set Exercise: Strengthens the quadriceps by engaging the knee in gentle extension, aiding recovery and muscle endurance. Heel Slide Exercise: Improves knee mobility by sliding the heel toward the buttocks, promoting flexibility. Straight Leg Raise Exercise: Strengthens hip flexors and quadriceps by lifting a straight leg, enhancing lower limb stability. Glute Bridge Exercise: Activates the glutes and stabilizes the hips through lifting movements, improving strength and control. Each exercise entry includes details such as targeted body parts, primary muscles, joint movements, instructions, precautions, progression criteria, and patient response metrics. The file leverages biomedical ontologies like OBI, ICF, and SNOMED CT for interoperability, making it suitable for integration into healthcare or rehabilitation applications. | 0.0.2 | alpha | lower limb (ORGANISM_SUBDIVISION), blood (ORGANISM_SUBSTANCE), legs (ORGANISM_SUBDIVISION), lower limbs (ORGANISM_SUBDIVISION), DVT (PATHOLOGICAL_FORMATION), Set (GENE_OR_GENE_PRODUCT), quadriceps (TISSUE), muscle (ORGAN), buttocks (PATHOLOGICAL_FORMATION), Leg (ANATOMICAL_SYSTEM), Strengthens hip flexors (CANCER), leg (ORGAN), muscles (TISSUE), joint (MULTI_TISSUE_STRUCTURE), ICF (GENE_OR_GENE_PRODUCT) |
| Mammalian Feeding Muscle Ontology | MFMO | An antomy ontology for the muscles of the head and neck that participate in feeding, swallowing, and other oral-pharyngeal behaviors. |  | production | muscles (ORGAN), head and neck (CANCER), oral-pharyngeal (CANCER) |
| Human Developmental Anatomy Ontology, abstract version 2 | EHDAA2 | A structured controlled vocabulary of stage-specific anatomical structures of the human. It has been designed to mesh with the mouse anatomy and incorporates each Carnegie stage of development (CS1-20). The abstract version of the human developmental anatomy ontology compresses all the tissues present over Carnegie stages 1-20 into a single hierarchy. The heart, for example, is present from Carnegie Stage 9 onwards and is thus represented by 12 EHDA IDs (one for each stage). In the abstract mouse, it has a single ID so that the abstract term given as just heart really means heart (CS 9-20). Timing details will be added to the abstract version of the ontology in a future release. | releases/2024-01-11 | beta | stage-specific (GENE_OR_GENE_PRODUCT), human (9606), mouse (10090), all (1), tissues (TISSUE), heart (ORGAN), CS 9 (GENE_OR_GENE_PRODUCT) |
| Spatial Ontology | BSPO | An ontology for respresenting spatial concepts, anatomical axes, gradients, regions, planes, sides, and surfaces. These concepts can be used at multiple biological scales and in a diversity of taxa, including plants, animals and fungi. The BSPO is used to provide a source of anatomical location descriptors for logically defining anatomical entity classes in anatomy ontologies. | 2023-05-27 | production | plants (3193), animals (33208), fungi (4751) |
| Foundational Model of Anatomy | FMA | FMA is a domain ontology that represents a coherent body of explicit declarative knowledge about human anatomy. For a description of how this OWL version is generated, see: "Pushing the Envelope: Challenges in a Frame-Based Representation of Human Anatomy" by N. F. Noy, M. A. Musen, J. L. Mejino Jr., C. Rosse (https://www.sciencedirect.com/science/article/pii/S0169023X03001253). | 5.0.0 | production | human (9606), OWL (GENE_OR_GENE_PRODUCT), Human (9606), F. (PATHOLOGICAL_FORMATION) |
| Phallusia fumigata Anatomy and Development Ontology | PHFUMIADO | The first ontology describing the anatomy and the development of Phallusia fumigata.  |  | alpha | Phallusia fumigata (395376) |
| Neuroscience Information Framework (NIF) Standard Ontology | NIFSTD | NIF Standard ontology (NIFSTD) is a core component of Neuroscience Information Framework (NIF) project (http://neuinfo.org), a semantically enhanced portal for accessing and integrating neuroscience data, tools and information. NIFSTD includes a set of modular ontologies that provide a comprehensive collection of terminologies to describe neuroscience data and resources. | 3.1 - January 15, 2018 | production | portal (TISSUE), collection (32644) |
| The Amphioxus Development and Anatomy Ontology | AMPHX | An ontology for the development and anatomy of Amphioxus (Branchiostoma lanceolatum). |  | production | Amphioxus (GENE_OR_GENE_PRODUCT), Branchiostoma lanceolatum (7740) |
| MultiCaRe Taxonomy | MCR_TX | The MultiCaRe Taxonomy contains the class structure used for medical image classification in the MultiCaRe dataset (v2). | 1.0 | production | - |
| MultiCaRe Taxonomy (ML Labels) | ML_MCR_TX | This is a reduced version of the MultiCaRe Taxonomy (MCR_TX), the class structure used for medical image classification in the MultiCaRe dataset. This reduced version includes only the labels used in Machine Learning (ML) model training. |  | beta | - |
| Radiation Oncology Structures Ontology | ROS | This ontology describes commonly contoured (anatomical and treatment planning) structures for radiation treatment planning. 22000 structures labels (created over a 16 years period in our radiation department) were extracted, classified and categorized to produce this ontology. Lymph nodes delineation international guidelines are provided. This ontology was created to ease and standardize the integration of radiation oncology data into clinical datawarehouses for multicentric studies. The high granularity of this ontology will allow precise dosimetric evaluations. | 1.2.2 | production | Lymph nodes (MULTI_TISSUE_STRUCTURE) |
| Mouse gross anatomy and development, timed | EMAPA | Abstract (i.e. non-stage specific) stage-specific anatomical structures of the mouse. |  | production | non-stage (CANCER), stage-specific (GENE_OR_GENE_PRODUCT), mouse (10090) |
| Combined Ontology for Inflammatory Diseases | COID | A domain ontology for inflammatory diseases, with ICD-10 used as a foundation. Ontology is expanded to include medical therapies, anatomy, and symptoms. With the addition of synonyms and lay terms derived from embedding models. | 1.0.0 | alpha | - |
| The Echinoderm Anatomy and Development Ontology | ECAO | An ontology for the development and anatomy of the different species of the phylum Echinodermata (NCBITaxon:7586). |  | production | - |
| Vertebrate Homologous Organ Group Ontology | VHOG | vHOG is a multi-species anatomical ontology for the vertebrate lineage, developed as part of the Bgee project. This mapping represents homology-strict relationships, in the sense of HOM:0000007 "historical homology". Each mapping has been manually reviewed, and we provide confidence codes and references when available. | unknown | production | - |
| Medical Imaging and Diagnostic Ontology | MIDO | MIDO is an OWL 2 ontology for creating a standardized format for modeling data in medical imaging tasks such as Disease Classification/ Diagnosis as well as Disease localization/segmentation | 2015-11-20 | alpha | Disease (CANCER) |
| Uber Anatomy Ontology | UBERON | Uberon is an integrated cross-species anatomy ontology representing a variety of entities classified according to traditional anatomical criteria such as structure, function and developmental lineage. The ontology includes comprehensive relationships to taxon-specific anatomical ontologies, allowing integration of functional, phenotype and expression data. | 2023-07-25 | production | cross-species (GENE_OR_GENE_PRODUCT) |
| Anatomic Pathology Lexicon | PATHLEX | PathLex, a comprehensive lexicon—a unified language of anatomic pathology terms—for standardized indexing and retrieval of anatomic pathology information resources.  | 1.2 | beta | Pathology (CANCER) |
| Ontology of Homology and Related Concepts in Biology | HOM | This ontology represents concepts related to homology, as well as other concepts used to describe similarity and non-homology. | releases/2015-01-07 | production | - |
| Ciona intestinalis Anatomy and Development Ontology | CIINTEADO |  The first ontology describing the anatomy and the development of Ciona intestinalis, based on the Hotta developmental table.  | Version 1 | production | Ciona (7718), Hotta developmental (GENE_OR_GENE_PRODUCT) |
| Hymenoptera Anatomy Ontology | HAO | A structured controlled vocabulary of the anatomy of the Hymenoptera (bees, wasps, sawflies and ants) | 2023-06-01 | production | wasps (7438), ants (2153479) |
| Anatomic Ontology for Human Lung Maturation | LUNGMAP-HUMAN | Anatomic Ontology for Human Lung Maturation (LungMAP Human Anatomy (LMHA) | 1.4 | production | - |
| Neural ElectroMagnetic Ontology | NEMO | Neural ElectroMagnetic Ontology (NEMO) describes classes of event-related brain potentials (ERP) and their properties, including spatial, temporal, and functional (cognitive/behavioral) attributes, and data-level attributes (acquisition and analysis parameters). Working (unstable), Release (stable), and Old (archival) versions of NEMO can be found on sourceforge: http://sourceforge.net/projects/nemoontologies/. | v3.10 | production | Neural ElectroMagnetic (MULTI_TISSUE_STRUCTURE), brain (ORGAN), ERP (CANCER) |
| Drosophila Gross Anatomy Ontology | FB-BT | An ontology representing the gross anatomy of Drosophila melanogaster | fbbt/releases/2025-05-29 | production | Drosophila melanogaster (7227) |
| Zebrafish Developmental Stages | ZFS | An ontology of developmental stages of the Zebrafish (Danio rerio). Note that ZFA includes the leaf nodes of this ontology. |  | production | Zebrafish (7955), Danio rerio (7955), leaf nodes (MULTI_TISSUE_STRUCTURE) |
| Cell Line Ontology [by Mahadevan] | MCCL | Ontologies provide a common platform of controlled vocabulary for researchers who need to share information across a particular domain, inclusive of machine-interpretable definitions of basic concepts in the domain and relations among them. The need for ontologies that provide a systematic arrangement of available data and allow cross talk across various related domains is gaining momentum. In this backdrop, we have developed a comprehensive ontology on primary and established cell lines-both normal and pathologic. It covers around 400 cell lines. This ontology has been built to include the major domains in the field of biology like anatomy, bio-molecules, chemicals and drugs, pathological conditions and genetic variations around the cell lines. An extensive network of relations has been built across these concepts to enable different combinations of queries. The ontology covers all cell lines from major sources like ATCC, DSMZ, ECACC, ICLC etc. and is built in OWL format. | 2 | beta | cell lines-both (CANCER), cell lines (CELL), bio-molecules (CANCER), all (1), DSMZ (CELL), ECACC (CELL) |
| Cell Ontology | CL | The Cell Ontology is a structured controlled vocabulary for cell types in animals. The Cell Ontology is designed as a structured controlled vocabulary for cell types. This ontology was constructed for use by the model organism and other bioinformatics databases, where there is a need for a controlled vocabulary of cell types. This ontology is not organism specific. It covers cell types from prokaryotes to mammals. However, it excludes plant cell types, which are covered by PO. | 2025-06-20 | production | Cell (CELL), cell (CELL), animals (33208), prokaryotes (2157), mammals (40674) |
| Fungal Gross Anatomy Ontology | FAO | A structured controlled vocabulary for the anatomy of fungi. |  | production | fungi (4751) |
| Spider Anatomy Ontology | SPD | An ontology for spider anatomy. | 1.1 | beta | - |
| Dictyostelium Discoideum Phenotype Ontology | DDPHENO | A structured controlled vocabulary of phenotypes of the slime-mould Dictyostelium discoideum. |  | production | slime-mould Dictyostelium discoideum (CELLULAR_COMPONENT) |
| Molgula occulta Anatomy and Development Ontology | MOOCCUADO | The first ontology describing the anatomy and the development of Molgula occulta.  |  | alpha | - |
| Collembola Anatomy Ontology | CLAO | Collembola Anatomy Ontology (CLAO) includes anatomical terms to refer to sclerites, anatomical systems, cell components, and chaetotaxy. CLAO is directed towards building semantic annotations or the production of morphological data employing a standardized language. CLAO is intended as a complementary tool to morhological descriptions during taxonomic work, before obtaining phylogenetic characters. | 2021-09-27 | alpha | cell (CELL) |
| Mosquito Gross Anatomy Ontology | TGMA | A structured controlled vocabulary of the anatomy of mosquitoes. | releases/2013-06-03 | production | - |
| Mouse Adult Gross Anatomy Ontology | MA | A structured controlled vocabulary of the adult anatomy of the mouse (Mus). | releases/2017-02-07 | alpha | mouse (10090), Mus (862507) |
| C. elegans Gross Anatomy Vocabulary | WB-BT | A structured controlled vocabulary of the anatomy of Caenorhabditis elegans. | releases/2025-03-26 | production | - |
| Phallusia mammillata Anatomy and Development Ontology | PHMAMMADO | The first ontology describing the anatomy and the development of Phallusia mammillata.  |  | alpha | Phallusia mammillata (59560) |
| Atlas Ontology Model | ATOM | An ontology model for anatomical atlases." | 2023-04-27 | beta | - |
| Ontology of Arthropod Circulatory Systems | OARCS | OArCS is an ontology describing the Arthropod ciruclatory system. | 2019-04-18 | production | - |
| Zebrafish Anatomy and Development Ontology | ZFA | A structured controlled vocabulary of the anatomy and development of the Zebrafish (Danio rerio). | releases/2025-01-28 | production | Zebrafish (7955), Danio rerio (7955) |
| Glioblastoma | GBM | Ontology of Glioblastoma |  | alpha | Glioblastoma (CANCER) |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |
| Clytia hemisphaerica Development and Anatomy Ontology | CLYH | The Clytia hemisphaerica Development and Anatomy Ontology (CLYH) describes the anatomical and developmental features of the Clytia hemisphaerica life cycle. |  | production | Clytia hemisphaerica (252671) |
| Ontology of Physical Exercises | OPE | The Ontology of Physical Exercises (OPE) provides a reference for describing an exercise in terms of functional movements, engaged musculoskeletal system parts, related equipment or monitoring devices, intended health outcomes, as well as target ailments for which the exercise might be employed as a treatment or preventative measure.  | 0.0.1 | alpha | - |
| Molgula occidentalis Anatomy and Development Ontology | MOOCCIADO |  "The first ontology describing the anatomy and the development of Molgula occidentalis. " |  | alpha | Molgula occidentalis (30282) |
| Molgula oculata Anatomy and Development Ontology | MOOCULADO | The first ontology describing the anatomy and the development of Molgula oculata. |  | alpha | Molgula oculata (27575) |
| insectH | INSECTH | NeuroMorpho.Org insect brain regions ontology is compiled to create data annotation standard for depositing insect data. Currently at v6.2, NeuroMorpho.Org hosts over 17k neuronal reconstructions mostly from the fly. The insect ontology follows the standard nomenclature of the insect brain (Ito et al., 2013) and the virtualfly.org.   | 0.1 | alpha | brain (ORGAN), neuronal (CELL) |
| Vertebrate Skeletal Anatomy Ontology | VSAO | VSAO is an anatomy ontology covering the vertebrate skeletal system. VSAO integrates terms for skeletal cells, tissues, biological processes, organs (skeletal elements such as bones and cartilages), and subdivisions of the skeletal system. | 2012-11-06 | production | skeletal (ORGAN), skeletal cells (CELL), tissues (TISSUE), organs (ORGAN), bones (TISSUE) |

<a name="bioportal-category-animal-development"></a>
## BioPortal Category "Animal Development"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Halocynthia roretzi Anatomy and Development Ontology | HAROREADO |  "The first ontology describing the anatomy and the development of Halocynthia roretzi. " |  | alpha | Halocynthia roretzi (7729) |
| Ciona savignyi Anatomy and Development Ontology  | CISAVIADO | The first ontology describing the anatomy and the development of Ciona savignyi, based on the Hotta developmental table.  |  | alpha | Ciona (7718), Hotta developmental (GENE_OR_GENE_PRODUCT) |
| Planarian Anatomy and Developmental Stage Ontolgoy | PLANA | Anatomy ontology for planaria and terms specific to the developmental stages of the planarian Schmidtea mediterranea. | 2023-03-13 | production | planarian Schmidtea mediterranea (CANCER) |
| Drosophila Development Ontology | FB-DV | A structured controlled vocabulary of the development of Drosophila melanogaster. | fbdv/releases/2025-05-29 | production | Drosophila melanogaster (7227) |
| Phallusia fumigata Anatomy and Development Ontology | PHFUMIADO | The first ontology describing the anatomy and the development of Phallusia fumigata.  |  | alpha | Phallusia fumigata (395376) |
| Vertebrate Homologous Organ Group Ontology | VHOG | vHOG is a multi-species anatomical ontology for the vertebrate lineage, developed as part of the Bgee project. This mapping represents homology-strict relationships, in the sense of HOM:0000007 "historical homology". Each mapping has been manually reviewed, and we provide confidence codes and references when available. | unknown | production | - |
| Ciona intestinalis Anatomy and Development Ontology | CIINTEADO |  The first ontology describing the anatomy and the development of Ciona intestinalis, based on the Hotta developmental table.  | Version 1 | production | Ciona (7718), Hotta developmental (GENE_OR_GENE_PRODUCT) |
| C. elegans Development Vocabulary | WB-LS | A structured controlled vocabulary of the development of Caenorhabditis elegans. | unknown | production | - |
| Molgula occulta Anatomy and Development Ontology | MOOCCUADO | The first ontology describing the anatomy and the development of Molgula occulta.  |  | alpha | - |
| Phallusia mammillata Anatomy and Development Ontology | PHMAMMADO | The first ontology describing the anatomy and the development of Phallusia mammillata.  |  | alpha | Phallusia mammillata (59560) |
| Molgula occidentalis Anatomy and Development Ontology | MOOCCIADO |  "The first ontology describing the anatomy and the development of Molgula occidentalis. " |  | alpha | Molgula occidentalis (30282) |
| Molgula oculata Anatomy and Development Ontology | MOOCULADO | The first ontology describing the anatomy and the development of Molgula oculata. |  | alpha | Molgula oculata (27575) |

<a name="bioportal-category-animal-gross-anatomy"></a>
## BioPortal Category "Animal Gross Anatomy"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Coleoptera Anatomy Ontology | COLAO | The Coleoptera Anatomy Ontology contains terms used for describing the anatomy and phenotype of beetles in biodiversity research. It has been built using the Ontology Develoment Kit with the Ontology for the Anatomy of the Insect SkeletoMuscular system (AISM) as a backbone. The project is open for collaboration. | 2024-06-21 | alpha | beetles (7041) |
| Tick Gross Anatomy Ontology | TADS | The anatomy of the Tick, Families: Ixodidae, Argassidae | 1.21 | beta | - |
| Teleost Anatomy Ontology | TAO | TAO is a multispecies anatomy ontology for teleost fishes. Originally seeded from ZFA, but intended to cover terms relevant to other taxa. | 2012-08-10 | production | TAO (GENE_OR_GENE_PRODUCT), teleost fishes (32443) |
| Planarian Anatomy and Developmental Stage Ontolgoy | PLANA | Anatomy ontology for planaria and terms specific to the developmental stages of the planarian Schmidtea mediterranea. | 2023-03-13 | production | planarian Schmidtea mediterranea (CANCER) |
| Human Developmental Anatomy Ontology, abstract version 2 | EHDAA2 | A structured controlled vocabulary of stage-specific anatomical structures of the human. It has been designed to mesh with the mouse anatomy and incorporates each Carnegie stage of development (CS1-20). The abstract version of the human developmental anatomy ontology compresses all the tissues present over Carnegie stages 1-20 into a single hierarchy. The heart, for example, is present from Carnegie Stage 9 onwards and is thus represented by 12 EHDA IDs (one for each stage). In the abstract mouse, it has a single ID so that the abstract term given as just heart really means heart (CS 9-20). Timing details will be added to the abstract version of the ontology in a future release. | releases/2024-01-11 | beta | stage-specific (GENE_OR_GENE_PRODUCT), human (9606), mouse (10090), all (1), tissues (TISSUE), heart (ORGAN), CS 9 (GENE_OR_GENE_PRODUCT) |
| Vertebrate Homologous Organ Group Ontology | VHOG | vHOG is a multi-species anatomical ontology for the vertebrate lineage, developed as part of the Bgee project. This mapping represents homology-strict relationships, in the sense of HOM:0000007 "historical homology". Each mapping has been manually reviewed, and we provide confidence codes and references when available. | unknown | production | - |
| Hymenoptera Anatomy Ontology | HAO | A structured controlled vocabulary of the anatomy of the Hymenoptera (bees, wasps, sawflies and ants) | 2023-06-01 | production | wasps (7438), ants (2153479) |
| Drosophila Gross Anatomy Ontology | FB-BT | An ontology representing the gross anatomy of Drosophila melanogaster | fbbt/releases/2025-05-29 | production | Drosophila melanogaster (7227) |
| Spider Anatomy Ontology | SPD | An ontology for spider anatomy. | 1.1 | beta | - |
| Collembola Anatomy Ontology | CLAO | Collembola Anatomy Ontology (CLAO) includes anatomical terms to refer to sclerites, anatomical systems, cell components, and chaetotaxy. CLAO is directed towards building semantic annotations or the production of morphological data employing a standardized language. CLAO is intended as a complementary tool to morhological descriptions during taxonomic work, before obtaining phylogenetic characters. | 2021-09-27 | alpha | cell (CELL) |
| Mosquito Gross Anatomy Ontology | TGMA | A structured controlled vocabulary of the anatomy of mosquitoes. | releases/2013-06-03 | production | - |
| Mouse Adult Gross Anatomy Ontology | MA | A structured controlled vocabulary of the adult anatomy of the mouse (Mus). | releases/2017-02-07 | alpha | mouse (10090), Mus (862507) |
| C. elegans Gross Anatomy Vocabulary | WB-BT | A structured controlled vocabulary of the anatomy of Caenorhabditis elegans. | releases/2025-03-26 | production | - |
| Zebrafish Anatomy and Development Ontology | ZFA | A structured controlled vocabulary of the anatomy and development of the Zebrafish (Danio rerio). | releases/2025-01-28 | production | Zebrafish (7955), Danio rerio (7955) |
| Vertebrate Skeletal Anatomy Ontology | VSAO | VSAO is an anatomy ontology covering the vertebrate skeletal system. VSAO integrates terms for skeletal cells, tissues, biological processes, organs (skeletal elements such as bones and cartilages), and subdivisions of the skeletal system. | 2012-11-06 | production | skeletal (ORGAN), skeletal cells (CELL), tissues (TISSUE), organs (ORGAN), bones (TISSUE) |

<a name="bioportal-category-arabidopsisy"></a>
## BioPortal Category "Arabidopsis"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| GoMapMan | GMM | GoMapMan is an open web-accessible resource for gene functional annotations in the plant sciences. It was developed to facilitate improvement, consolidation and visualization of gene annotations across several plant species. | 1.2 | production | - |
| Cell Cycle Ontology | CCO | An application ontology integrating knowledge about the eukaryotic cell cycle. | 2.10 | beta | cell (CELL) |

<a name="bioportal-category-bioportal-category-biological-process"></a>
## BioPortal Category "Biological Process"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| The Biological and Environmental Research Variable Ontology | BERVO | The Biological and Environmental Research Variable Ontology (BERVO) is a model of the experimental variables, conditions, and concepts used in the study of environmental research, including studies in earth science, plant science, and geochemistry. | 2025-08-05 | alpha | - |
| Genomic Feature and Variation Ontology | GFVO | The Genomic Feature and Variation Ontology (GFVO) is modeled to represent genomic data using the Resource Description Format (RDF) or Linked Data using JSON (JSON-LD). It is captures the contents of data files that adhere to the Generic Feature Format Version 3 (GFF3, http://www.sequenceontology.org/resources/gff3.html), the General Transfer Format (GTF, http://mblab.wustl.edu/GTF22.html), the Genome Variation Format Version 1 (GVF, http://www.sequenceontology.org/resources/gvf.html), and the Variant Call Format (VCF, http://vcftools.sourceforge.net/specs.html), as well as the FASTA format (http://en.wikipedia.org/wiki/FASTA_format). The creation of the ontology was inspired by previous work of Robert Hoehndorf on RDF2OWL (http://code.google.com/p/rdf2owl). NOTE: GFVO in BioPortal is a simplified version without ontology imports and having class/property equivalences removed. The full ontology is available at: http://www.biointerchange.org/gfvo | 1.0.6 | production | GFVO (CANCER) |
| Space Life Sciences Ontology | SLSO | The Space Life Sciences Ontology is an application ontology and is intended to support the operation of NASA's Life Sciences Data Archive (located at https://nlsp.nasa.gov) and other systems that contain space life science research data. Many kinds of scientific research in space involve specialized equipment, experimental procedures, specimens and specimen collection apparatus, supporting operational structures, and references to specific space environments and their characteristics. Often such research is extremely costly compared with planetary/terrestrial investigations, and the community supporting space life sciences is accustomed to the use of various specialized concepts and terminologies when dealing with this kind of research and data. However, the underlying conceptual models (and especially the specific labeled relationships in these models) have never been made explicit. The SLSO attempts to provide definition and organization of these models for the space research community. The SLSO was developed ab inicio using the Ontology Development Kit and imports an extends many concepts from the Basic Formal Ontology (BFO), the Ontology of Biomedical Investigations (OBI), the Environmental Ontology (ENVO), and other OBO Foundry ontologies. Projects at NASA such as the Open Science Data Repository (https://osdr.nasa.gov/) are already using many OBO ontologies, including the Radiation Biology Ontology (https://github.com/Radiobiology-Informatics-Consortium/RBO) and OBI, to index space biology investigation data. With the development of the SLSO, this practice can be extended to include all life science research in space or addressing space effects. Furthermore, the SLSO has a component that imports concepts from the Science Data Discovery Ontology, which was developed to support NASA's Science Discovery Engine (https://sciencediscoveryengine.nasa.gov). These links in the imported SDDO to concepts underlying a broad spectrum of space research (astrophysics, heliophysics, etc.) can ultimately be used to provide key capabilities for discovering and analyzing space life science data and how they relate to other kinds of scientific data regarding space environments., The Space Life Sciences Ontology is an application ontology and is intended to support the operation of NASA's Life Sciences Data Archive and other systems that contain space life science research data. | 2025-08-18 | production | specimens (CELL), collection (32644), planetary/terrestrial (MULTI_TISSUE_STRUCTURE), SLSO (CANCER), all (1) |
| Cell Line Ontology | CLO | The Cell Line Ontology (CLO) is a community-based ontology in domain of biological cell lines with a focus on permanent cell lines from culture collections. Upper ontology structures that frame the skeleton of CLO include Basic Formal Ontology and Relation Ontology. Cell lines contained in CLO are associated with terms from other ontologies such as Cell Type Ontology, NCBI Taxonomy, and Ontology for Biomedical Investigation. A common design pattern for the cell line is used to model cell lines and their attributes, the Jurkat cell line provides ane xample.  Currently CLO contains over 36,000 cell line entries obtained from ATCC, HyperCLDB, Coriell, and bymanual curation. The cell lines are derived from 194 cell types, 656 anatomical entries, and 217 organisms. The OWL-based CLO is machine-readable and can be used in various applications. | 2.1.188 | production | Cell Line (CELL), cell lines (CELL), Cell lines (CELL), Cell Type (CELL), cell line (CELL), Jurkat cell line (CELL), ATCC (CELL), Coriell (CELL), cell (CELL) |
| WikiPathways | WIKIPATHWAYS | The WP ontology for WikiPathways. | 2013a | production | - |
| Pharmacogenomic Relationships Ontology | PHARE | The PHArmacogenomic RElationships Ontology (or PHARE) proposes concepts and roles to represent relationships of pharmacogenomics interest. | version id: 110114 | alpha | - |
| In Vivo Methodologies Ontology for MAATrica metric | VVMM | MAATrica is a novel metric for assessing the coherence of methodologies in research papers within the fields of medicinal and nutraceutical chemistry. MAATrica employs user-controlled and customizable ontologies (in silico, in vitro and in vivo), enabling personalized analysis of research papers in the fields of medicinal and nutraceutical chemistry. Here we present the in vivo ontology we developed for the scope. |  | alpha | - |
| Epigenetic Process | EPIP | Biological processes associated with changes in the regulation of gene activity without alteration of genetic structure. Development of this terminology was funded by the National Institute of Child Health and Human Development in support of the Placental Atlas Tool (PAT) with final verification by the PAT External Scientific Panel. This terminology was developed through a combination of resources including existing biomedical ontology resources, such as the Gene Ontology and the National Cancer Institute Thesaurus, web-based content, and epigenetic literature. | 1.3 | production | Human (9606), Cancer (CANCER) |
| OntoFood | OF | This is an ontology with SWRL rules of nutrition for diabetic patient. We can import the value of Blood Sugar and Weight by .xls and use that like instance for generate the rules with rule engine Jess. | 1.0 | alpha | SWRL (CANCER), Blood Sugar (ORGANISM_SUBSTANCE) |
| Ontology of Host-Pathogen Interactions | OHPI | The Ontology of Host-Pathogen Interactions (OHPI) is a biomedical ontology of the host-pathogen interactions. Its development follows the OBO Foundry Principles. | 1.0.26 | production | OHPI (CANCER) |
| Graphical Pathway Markup Language | GPML | GPML is the file format used to store pathway content at WikiPathways. GPML is short for Graphical Pathway Markup Language, and it is a custom XML format compatible with pathway visualization and analysis tools such as Cytoscape and PathVisio. | 2013a | beta | GPML (GENE_OR_GENE_PRODUCT) |
| BioAssay Ontology | BAO | The BioAssay Ontology (BAO) describes chemical biology screening assays and their results including high-throughput screening (HTS) data for the purpose of categorizing assays and data analysis.  BAO is an extensible, knowledge-based, highly expressive (currently SHOIQ(D)) description of biological assays making use of descriptive logic based features of the Web Ontology Language (OWL).  BAO currently has over 1000 classes and also makes use of several other ontologies.  It describes several concepts related to biological screening, including Perturbagen, Format, Meta Target, Design,  Detection Technology, and Endpoint.  Perturbagens are perturbing agents that are screened in an assay; they are mostly small molecules. Assay Meta Target describes what is known about the biological system and / or its components interrogated in the assay (and influenced by the Perturbagen).  Meta target can be directly described as a molecular entity (e.g. a purified protein or a protein complex), or indirectly by a biological process or event (e.g. phosphorylation).  Format describes the biological or chemical features common to each test condition in the assay and includes biochemical, cell-based, organism-based, and variations thereof.  The assay Design describes the assay methodology and implementation of how the perturbation of the biological system is translated into a detectable signal.  Detection Technology relates to the physical method and technical details to detect and record a signal.  Endpoints are the final HTS results as they are usually published (such as IC50, percent inhibition, etc.).  BAO has been designed to accommodate multiplexed assays.  All main BAO components include multiple levels of sub-categories and specification classes, which are linked via object property relationships forming an expressive knowledge-based representation. | 2.8.16 | production | SHOIQ(D (GENE_OR_GENE_PRODUCT) |
| Biological Pathway Taxonomy | BPT | Biological Pathway Taxonomy (BPT) - hierarchical taxonomy of cellular and molecular mechanisms, metabolic reactions, and diseases that can be shown as signaling pathways. BPT was created manually based on the Elsevier Pathway Collection available for public usage. There are 3 types of entities in BPT – pathway, group, and folder of pathways and groups. Group is the list of genes and proteins associated with biological function and diseases. Pathway is an interactive model of molecular interactions in human and mammalian cells described in the literature. Each model was reconstructed based on facts published in peer-reviewed journals, with manually quality control. BPT pathway entity includes several properties such as link to the model in Pathway Studio software which allows reading supported sentences from published articles and additional information about members. A list of members for each model can be provided by request. Property “network” describes maps without logical causal relationships between members to distinguish them from actual models of molecular mechanisms (“signaling”). The index folder contains all pathways and groups sorted alphabetically. | synonymes2022 | alpha | cellular (CELL), human (9606), mammalian cells (CELL), all (1) |
| Molecular Interaction Map | MIM | The Molecular Interaction Map (MIM) notation seeks to provide a standardized method to draw diagrams for bioregulatory networks with features similar to technical diagrams in other fields. This vocabulary uses the term hierarchy from the Formal MIM Notation Specification. Where possible, Systems Biology Ontology (SBO) and Biological Pathway Exchange (BioPAX) terms and IRIs are used. If no SBO or BioPAX term is available, a new term and IRI are created. This is currently only a partial implementation. Many elements still need to be added to cover the full the specification. | 10:30:2014 16:04 | alpha | SBO (CANCER), IRI (GENE_OR_GENE_PRODUCT) |
| BioModels Ontology | BIOMODELS | OWL Representation of the models in the BioModels repository. | 21 | production | - |
| Pregnancy Ontology | PREGONTO | The pregnancy ontology is a biomedical ontology used for identifying pregnancies in routine health databases. The ontology was primarily developed for conducting surveillance of vaccine uptake in pregnant women in the UK. |  | production | UK (GENE_OR_GENE_PRODUCT) |
| Pathway Ontology | PW | The goal of the Pathway Ontology is to cover all types of biological pathways, including altered and disease pathways, and to capture the relationships between them within the hierarchical structure of a Directed Acyclic Graph (DAG). The five nodes of the ontology are: classic metabolic, regulatory, signaling, drug and disease pathways. An extensive survey of the review literature along with searches of existing pathway databases have been used to choose terms and their position within the tree. The ontology is continually expanding along with the development of Pathway and Disease Portals at RGD. Mapping of pathways in other databases to terms in PW as synonyms further increased the content of the ontology. It also provided the means to bring in annotations made by these databases and link to their sites via automatic pipelines built at RGD. The ontology allows for the standardized annotation of rat as well as human and mouse genes to pathway terms. It also serves as a vehicle to connect between genes and ontology reports, between reports and interactive pathway diagrams, between pathways that directly connect to one another within a diagram or between pathways that in some fashion are globally related in pathway suites and suite networks. Metabolic, regulatory or signaling pathway terms have associated terms for the altered version of the pathway and the level at which the alteration(s) may occur. Proteins with mutations known to affect a particular pathway can be annotated to the normal and altered version(s) of the pathway and to a disease pathway, if known. Both the provision of terms for the altered versions of pathways and the representation of disease pathway diagrams as a collection of altered pathways are unique to the PW ontology and RGD's pathway data. The Pathway ontology is distributed under the terms of the Creative Commons Attribution license version 3.0 Unported (CC BY 3.0), see http://creativecommons.org/licenses/by/3.0 for details.  | 7.93 | production | all (1), disease (CANCER), rat (10116), human (9606), mouse (10090), collection (32644) |
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Experimental Factor Ontology | EFO | The Experimental Factor Ontology (EFO) is an application focused ontology modelling the experimental variables in multiple resources at the EBI and Open Targets. The ontology has been developed to increase the richness of the annotations that are currently made in resources and to promote consistent annotation, to facilitate automatic annotation and to integrate external data. The ontology pulls together classes from reference ontologies such as disease, cell line, cell type and anatomy and adds axiomatisation as necessary to connect areas such as disease to phenotype. Contact EFO users list for information: efo-users@ebi.ac.uk. | 3.79.0 | production | cell line (CELL), cell (CELL) |
| GoMapMan | GMM | GoMapMan is an open web-accessible resource for gene functional annotations in the plant sciences. It was developed to facilitate improvement, consolidation and visualization of gene annotations across several plant species. | 1.2 | production | - |
| Evidence Graph Ontology | EVI | The Evidence Graph ontology extends core concepts from the W3C Provenance Ontology PROV-O and Bioschemas' Profiles to describe evidence for correctness of findings in biomedical publications. The semantic data model in EVI is expressed using OWL2 Web Ontology Language. | 1.1 | alpha | - |
| Regulation of Transcription Ontology | RETO | An application ontology for the domain of gene transcription regulation. The ontology integrates fragments of GO and MI with data from GOA, IntAct, UniProt, NCBI and orthological properties  from RO and SIO. | 2015 | beta | RO (CELL) |
| Home Cage Monitoring definition | HCM-D | From COST-TEATIME (https://www.cost-teatime.org/) | v0.1 | alpha | - |
| Neuroscience Information Framework (NIF) Standard Ontology | NIFSTD | NIF Standard ontology (NIFSTD) is a core component of Neuroscience Information Framework (NIF) project (http://neuinfo.org), a semantically enhanced portal for accessing and integrating neuroscience data, tools and information. NIFSTD includes a set of modular ontologies that provide a comprehensive collection of terminologies to describe neuroscience data and resources. | 3.1 - January 15, 2018 | production | portal (TISSUE), collection (32644) |
| Ontology for Newborn Screening Follow-up and Translational Research | ONSTR | Ontology for Newborn Screening Follow-up and Translational Research (ONSTR) is an application ontology covering the domain of newborn screening, follow-up and translational research pertaining to patients diagnosed with inheritable and congenital diseases mainly identified through newborn dried blood spot screening. ONSTR is a central component of the project Newborn Screening Follow-up Data Integration Collaborative (NBSDC). ONSTR uses the Basic Formal Ontology v2 (BFO2, v2012-07-20) as top-level ontology and extends the classes imported from OBO Foundry ontologies and candidate ontologies. | 0.4 | alpha | blood (ORGANISM_SUBSTANCE), NBSDC (CANCER), BFO2 (GENE_OR_GENE_PRODUCT), v2012-07 (GENE_OR_GENE_PRODUCT) |
| Imaging Biomarker Ontology | IBO | The Imaging Biomarker Ontology describes radiological measurements that characterize biological processes (such as anisotropic water diffusion in a particular tissue region) of imaged subjects (such as neuro-imaging observations) and help clinicians, particularly in the assessment of therapeutic responses and the early prediction of pathologies. IBO reuses several OBO ontologies namely BFO, IAO, PATO, GO, FMA, and OGMS.  |  | beta | tissue (TISSUE), PATO (GENE_OR_GENE_PRODUCT) |
| Just Enough Results Model Ontology | JERM | The JERM ontology is an application ontology that describes the relationships between the data, models and other research assets that can be shared through the FAIRDOM platform, for sharing Systems and Synthetic Biology experiments. JERM 2 includes a new model for describing Biosamples, developed during the ELIXIR-lead Biosamples club | 2 | beta | FAIRDOM (GENE_OR_GENE_PRODUCT) |
| G Protein-Coupled Receptor BioAssays Ontology | BAO-GPCR | The G protein-coupled receptors (GPCRs) ontology (http://www.bioassayontology.org/bao_gpcr) describes pharmacology, biochemistry and physiology of these important and therapeutically promising class of academic and pharmaceutical research targets. Incorporation and comparison of various small molecule screening data sets, such as those deposited in PubChem, ChEMBL, KEGG, PDSP, and/or IUPHAR databases, requires a formalized electronic organization system. In order to bridge the gap between the overflow of HTS data and the bottleneck of integrated analysis tools, herein, we provide the first comprehensive GPCR ontology. The development and utility of GPCR ontology was based on previously developed BioAssay Ontology (BAO). The GPCR ontology contains information about biochemical, pharmacological, and functional properties of individual GPCRs as well as GPCR-selective ligands inclusive of their HTS screening results and other records. This provides the first all-inclusive GPCR ontology with all available data to model the relationship between the GPCR binding sites and their physiologic and pharmacologic role in physiology via small molecule chemical structures. We developed this system using emerging semantic technologies, by leveraging existing and descriptive domain level ontologies. | 1.1 | beta | GPCRs (GENE_OR_GENE_PRODUCT), all (1) |
| Radiation Biology Ontology | RBO | RBO is an ontology for the effects of radiation on biota in terrestrial and space environments. | 2025-05-30 | production | - |
| Ontology of Host-Microbe Interactions | OHMI | OHMI is a biomedical ontology that represents the entities and relations in the domain of host-microbiome interactions. | 2019-09-17 | production | - |
| Gene Expression Ontology | GEXO | An application ontology for the domain of gene expression. The ontology integrates fragments of GO and MI with data from GOA, IntAct, UniProt, NCBI, and orthology relations using ontological properties from RO and ISO. | 2015 | beta | RO (CELL), ISO (CELL) |
| Medical Imaging and Diagnostic Ontology | MIDO | MIDO is an OWL 2 ontology for creating a standardized format for modeling data in medical imaging tasks such as Disease Classification/ Diagnosis as well as Disease localization/segmentation | 2015-11-20 | alpha | Disease (CANCER) |
| Biofilm Ontology | BIFO | BIFO is intended to provide a biofilm ontology capable of understanding fundamental terminologies the relations between common biofilm terminologies following the guidelines of the Basic Formal Ontology (BFO). |  | alpha | Biofilm (CELL) |
| Cell Cycle Ontology | CCO | An application ontology integrating knowledge about the eukaryotic cell cycle. | 2.10 | beta | cell (CELL) |
| SMART Protocols | SP | The SeMAntic RepresenTation for Protocols, SMART Protocols, provides a structured OWL vocabulary of experimental protocols.  SMART Protocols ontology includes two modules, SP-document and SP-workflow. SP-document, aims to provide a structured vocabulary of concepts to represent information necessary and sufficient for reporting an experimental protocol.  SP-Workflow represents: i) the executable  elements of a protocol, protocol instructions; ii) the experimental actions and material entities that participates in instructions (sample/specimen, organisms, reagents, instruments);  and iii) the order of execution of the instructions. | 4.0 | beta | - |
| Neural ElectroMagnetic Ontology | NEMO | Neural ElectroMagnetic Ontology (NEMO) describes classes of event-related brain potentials (ERP) and their properties, including spatial, temporal, and functional (cognitive/behavioral) attributes, and data-level attributes (acquisition and analysis parameters). Working (unstable), Release (stable), and Old (archival) versions of NEMO can be found on sourceforge: http://sourceforge.net/projects/nemoontologies/. | v3.10 | production | Neural ElectroMagnetic (MULTI_TISSUE_STRUCTURE), brain (ORGAN), ERP (CANCER) |
| Functional Electrophysiology Nomenclature for Ion Channels | FENICS | The Functional Epilepsy Nomenclature for Ion Channels (FENICS) is allowing the consistent evaluation of electrophysiology experiments on Ion Channels in the context of neurological disorders, in particular epilepsy.  | 0.0.5 | alpha | Ion (TISSUE) |
| Context and Measurement Ontology | COMO | Ontological terms to describe experimental data, probably useful for other environmental microbiology projects.  Used in several CORAL (Contextual Ontology-based Repository Analysis Library) deployments.  See more about this platfom at https://github.com/jmchandonia/CORAL |  | beta | - |
| HIVOntology | HIVO004 | A Project of Biomedical Ontology for Systems Biomedicine. The goals of the Project include: (1). To pilot the novel researches on AIDS and HIV or other diseases by viruses, by means of the ontology and other biomedical informatics technologies, connecting the insights of the Biomedical Knowledge Integration (BMKI), systems biomedicine, complexity sciences etc. (2). To Carry out the methodology explorations for the goals above. | 020 | alpha | HIV (57667), viruses (10239) |
| Cell Culture Ontology | CCONT | CCONT is an ontology for the formal representation of cell lines and their correspnding culture conditions. | 1.1 | beta | cell lines (CELL) |
| Alzheimer Disease Map Ontology | ADMO | Alzheimer’s disease (AD) pathophysiology is still imperfectly understood and current paradigms have not led to curative outcome. Omics technologies offer great promises for improving our understanding and generating new hypotheses. However, integration and interpretation of such data pose major challenges, calling for adequate knowledge models. AlzPathway is a disease map that gives a detailed and broad account of AD pathophysiology. However, AlzPathway lacks formalism, which can lead to ambiguity and misinterpretation. Ontologies are an adequate framework to overcome this limitation, through their axiomatic definitions and logical reasoning prop-erties. We introduce the AD Map Ontology (ADMO), an ontological upper model based on systems biology terms. We then propose to convert AlzPathway into an ontology and to integrate it into ADMO. | beta | beta | AD (GENE_OR_GENE_PRODUCT), disease (CANCER) |
| Toxic Process Ontology | TXPO | TOXic Process Ontology (TXPO) includes entities related to toxic mechanisms (toxic courses) and processes, mainly focusing on hepatotoxicity. | 2022/12/07 | beta | - |
| OntoRepliCov | ONTOREPLICOV | OntoRepliCov is an ontology designed to describe the different stages of the SARS-CoV-2 replication process. | 0 | alpha | - |
| AdaLab ontology | ADALAB | AdaLab ontology | 0.9.2 | alpha | - |
| Mapping of Epilepsy Ontologies | MEPO | The Mapping of Epilepsy Ontologies (MEPO) provides owl:sameAs relationships of similar concepts for the ontologies EpSO, ESSO, EPILONT, EPISEM, and FENICS. Two concepts are considered similar if they share the same concept name or the same synonym. Additional synonyms for each of the concepts are generated using the Snowball stemming algorithm. The software implementation is available on github within the Maven project snoke in the module snoke.ontology: https://github.com/bernd-mueller/snoke/tree/master/snoke.ontology | 0.6 | alpha | MEPO (CANCER) |
| Biolink Model | BIOLINK | A high level datamodel of biological entities (genes, diseases, phenotypes, pathways, individuals, substances, etc) and their associations | 2.2.13 | production | - |
| Ontology of Physics for Biology | OPB | The OPB is a reference ontology of classical physics and thermodynamics as applied to the dynamics of biological systems. It is designed to encompass the multiple structural scales (atoms to organisms) and multiple physical domains (fluid dynamics, chemical kinetics, particle diffusion, etc.) that are encountered in the study and analysis of biological organisms. | 1.2 | beta | OPB (GENE_OR_GENE_PRODUCT) |
| Cell Behavior Ontology | CBO |  "The Cell Behavior Ontology (CBO) describes multi-cell computational models. In particular to describe both the existential behaviors of cells (spatiality | Version 1.1.2 | production | Cell (CELL), CBO (CANCER), multi-cell (CELL), cells (CELL) |
| Bacterial interlocked Process ONtology | BIPON | BiPON is an ontology permitting a multi-scale systemic representation of bacterial cellular processes and the coupling to their mathematical models. BiPON is further composed of two sub-ontologies, bioBiPON and modelBiPON. bioBiPON aims at organizing the systemic description of biological information while modelBiPON aims at describing the mathematical models (including parameters) associated to each biological process. | 2017-11-08-bis | production | cellular (CELL), bioBiPON (CELL) |
| Interaction Network Ontology | INO | The Interaction Network Ontology (INO) is an ontology in the domain of interactions and interaction networks. INO represents general and species-neutral types of interactions and interaction networks, and their related elements and relations. INO is a community-driven ontology, aligns with BFO, and is developed by following the OBO Foundry principles. | 1.1.13 | production | - |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |
| Ontology for MicroRNA Target | OMIT | The purpose of the OMIT ontology is to establish data exchange standards and common data elements in the microRNA (miR) domain. Biologists (cell biologists in particular) and bioinformaticians can make use of OMIT to leverage emerging semantic technologies in knowledge acquisition and discovery for more effective identification of important roles performed by miRs in humans' various diseases and biological processes (usually through miRs' respective target genes). OMIT has reused and extended a set of well-established concepts from existing bio-ontologies, e.g., Gene Ontology, Sequence Ontology, Protein Ontology, NCBI Organism Taxonomy, Human Disease Ontology, Foundational Model of Anatomy, and so forth. | OBO 4.0-12122013-2338 | alpha | cell (CELL), miRs (GENE_OR_GENE_PRODUCT), humans (9605), Human (9606) |
| Homeostasis imbalance process ontology | HOIP | Homeostasis imbalance process ontology (HoIP) is an ontology that systematizes a wide variety of terms involving homeostasis imbalance courses and processes. HoIP focuses on the course of COVID-19 infectious processes and cellular senescence.  | 2024/04/03 | alpha | cellular (CELL) |

<a name="bioportal-category-biomedical-resources"></a>
## BioPortal Category "Biomedical Resources"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| New Approach Methodology Ontology | NAMO | The New Approach Methodology Ontology and Schema (NAMO) is a comprehensive data framework designed to standardize and integrate information about alternative methods to animal testing in biomedical research. Built on LinkML, NAMO provides a common vocabulary and data structure for describing diverse model systems, from cellular cultures and organ-on-chip devices to computational models and digital twins. |  | production | Schema (GENE_OR_GENE_PRODUCT), cellular cultures (CELL), twins (CANCER) |
| Genomic Feature and Variation Ontology | GFVO | The Genomic Feature and Variation Ontology (GFVO) is modeled to represent genomic data using the Resource Description Format (RDF) or Linked Data using JSON (JSON-LD). It is captures the contents of data files that adhere to the Generic Feature Format Version 3 (GFF3, http://www.sequenceontology.org/resources/gff3.html), the General Transfer Format (GTF, http://mblab.wustl.edu/GTF22.html), the Genome Variation Format Version 1 (GVF, http://www.sequenceontology.org/resources/gvf.html), and the Variant Call Format (VCF, http://vcftools.sourceforge.net/specs.html), as well as the FASTA format (http://en.wikipedia.org/wiki/FASTA_format). The creation of the ontology was inspired by previous work of Robert Hoehndorf on RDF2OWL (http://code.google.com/p/rdf2owl). NOTE: GFVO in BioPortal is a simplified version without ontology imports and having class/property equivalences removed. The full ontology is available at: http://www.biointerchange.org/gfvo | 1.0.6 | production | GFVO (CANCER) |
| Common Terminology Criteria for Adverse Events | CTCAE | Common Terminology Criteria for Adverse Events (CTCAE) is widely accepted throughout the oncology community as the standard classification and severity grading scale for adverse events in cancer therapy clinical trials and other oncology settings. Version 5 was published by the NCI Cancer Therapy Evaluation Program (CTEP) in November 2017. It is organized by MedDRA System Organ Class and mapped to MedDRA LLTs with corresponding MedDRA codes, and harmonized with MedDRA at the Adverse Event (AE) level including revised AE terms and severity indicators to reflect clinical effects identified with current oncology interventions.Severity grades are assigned and most are defined to clarify the meaning of the term. CTCAE is designed to integrate into information networks for safety data exchange, and is the primary standard for data management for AE data collection, analysis, and patient outcomes associated with cancer research and care. | 5.0 | production | cancer (6754), NCI Cancer (CANCER), CTEP (CANCER), MedDRA (GENE_OR_GENE_PRODUCT), networks (MULTI_TISSUE_STRUCTURE), collection (32644) |
| Animal Health Surveillance Ontology | AHSO | (Work in progress - please contact ahso@datadrivensurveillance.org before using this ontology) This is an ontology for health surveillance, assuming that surveillance data is recorded at three main levels: SAMPLE data - data about specific collected samples; OBSERVATIONS, which can aggregate several samples; and OBSERVATIONAL CONTEXT, which is the context in which the observations were made (for instance clinical observation, surveillance sampling, etc). HEALTH EVENT is the occurrence in the real World to be modeled. This occurrence is an abstract concept, as it generally cannot be determined what are its boundaries in time, space, or population units. We assume that several OBSERVATIONS are derived from each health event, and recorded in the same or different databases - for instance clinical journal, laboratory database, abattoir. At the same time or not. AHSO attempts to model these time specific observations. The ontology is focused on the modeling of the animal and health information connected to a single observation, made at a specific point on time. All the information recorded is relative to that specific point in time. For instance, an animal belongs to a herd, which belongs to an owner, AT THE TIME OF THE OBSERVATION. We do not attempt to model how that animal changed herds on time. Similarly, we do not model clinical progression. It is up to those using the data to create definitions of when two observations should be connected to the same event. | 0.1.0 | alpha | AT THE TIME (CELL) |
| Cell Line Ontology | CLO | The Cell Line Ontology (CLO) is a community-based ontology in domain of biological cell lines with a focus on permanent cell lines from culture collections. Upper ontology structures that frame the skeleton of CLO include Basic Formal Ontology and Relation Ontology. Cell lines contained in CLO are associated with terms from other ontologies such as Cell Type Ontology, NCBI Taxonomy, and Ontology for Biomedical Investigation. A common design pattern for the cell line is used to model cell lines and their attributes, the Jurkat cell line provides ane xample.  Currently CLO contains over 36,000 cell line entries obtained from ATCC, HyperCLDB, Coriell, and bymanual curation. The cell lines are derived from 194 cell types, 656 anatomical entries, and 217 organisms. The OWL-based CLO is machine-readable and can be used in various applications. | 2.1.188 | production | Cell Line (CELL), cell lines (CELL), Cell lines (CELL), Cell Type (CELL), cell line (CELL), Jurkat cell line (CELL), ATCC (CELL), Coriell (CELL), cell (CELL) |
| Intelligent Medicine oriented Biobank Ontology | IMOBO | Intelligent Medicine oriented Biobank Ontology (IMOBO) is an ontology developed to facilitate the annotation and modeling of biobank repositories, metadata, and biobanking administration, supporting the paradigm shift towards AI in medicine. IMOBO focuses on AI modeling in both clinical applications and healthcare, with a specific emphasis on engaging clinical doctors and patients in data collection. Its goal is to structure knowledge for the future patient-centered medical paradigm. As measurement technologies and medical artificial intelligence continue to evolve, IMOBO will continually enhance its content and structure. | 0.1 | alpha | IMOBO (CANCER), collection (32644), artificial (32630) |
| Rheumatoid Arthritis ontology | RAO | A controlled vocabulary of RA-relevant information. Currently, only treatment information is available.  | Version 1.0 | alpha | - |
| COKPME - COVID19 Ontology for analyzing the Karnataka Private Medical Establishments Data   | COKPME | The ontology will be used by analysts for finding the relative and precautionary measures that can be put in action for controlling the spread. Also, it should be able to handle the competence question like: Which properties of the patient should be considered for modeling,  Which properties of patient changes,  Which properties of patient confirm COVID positive, Do the symptom of the patient change over time,  Do the treatment provided depends on the category of patient, Do other comorbid conditions derive any interestingness, Which symptom is high in a particular district? (Or) Does the geographical location influence the patient category. These keep change over an iterative process of ontology development | 1.0 | production | - |
| Electrocardiography Ontology | ECG | The Electrocardiography (ECG) Ontology is a Driving Biological Project of the NCBO. The ECG Ontology will contain terms for describing electrocardiograms, their capture method(s) and their waveforms. | 1 | production | - |
| PGxO | PGXO | PGxO: a lite Pharmacogenomic Ontology. This ontology aims at providing a small set of concepts and roles that may type elements of pharmacogenomic relationships. Its concepts has been mapped to other existing ontologies. Mappings are not included in this version for lisibility purposes. Mapping are available on demand.  | 0.4 | alpha | - |
| Human Reference Atlas | HRA | The Human Reference Atlas (HRA) is a comprehensive, high-resolution, three-dimensional atlas of all the cells in the healthy human body. The Human Reference Atlas provides standard terminologies and data structures for describing specimens, biological structures, and spatial positions linked to existing ontologies. | 2.3.0 | production | Human (9606), HRA (CANCER), all (1), cells (CELL), human (9606), specimens (CANCER) |
| In Vivo Methodologies Ontology for MAATrica metric | VVMM | MAATrica is a novel metric for assessing the coherence of methodologies in research papers within the fields of medicinal and nutraceutical chemistry. MAATrica employs user-controlled and customizable ontologies (in silico, in vitro and in vivo), enabling personalized analysis of research papers in the fields of medicinal and nutraceutical chemistry. Here we present the in vivo ontology we developed for the scope. |  | alpha | - |
| Cancer Care: Treatment Outcome Ontology | CCTOO | The Cancer Care: Treatment Outcomes Ontology (CCTOO) catalogues various clinical endpoints used in evaluating outcomes of cancer treatment. In addition to a vocabulary list, CCTOO also curates the dependencies between individual concepts, in form of production rules, for translating clinical findings recorded in the electronic medical records (EMR) into literature-based concepts. An earlier version of this ontology was presented as a scientific abstract alongside the American Society of Clinical Oncology annual meeting 2017, Chicago, USA (J Clin Oncol 2017; 35 suppl; abstr e18137). Full paper on http://ascopubs.org/doi/abs/10.1200/CCI.18.00026 | releases/2018-05-11 | beta | Cancer (CANCER), cancer (6754) |
| Software Ontology | SWO | The Software Ontology (SWO) describes software used in research, primarily Bioinformatics. The SWO covers areas such as the software type, licence, manufacturer of the software, the input and output data types and the uses (i.e. objectives) to which the software can be put. | 2023-03-05 | production | - |
| Glycomics Ontology | GLYCO | The GlycO ontology provides an up-to-date knowledge base of experimentally verified glycan structures. Glycan (oligosaccharide or polysaccharide) structures are represented as trees of monosaccharide residues. Linkage to proteins and lipids is supported as well. Insertion of a new glycan is controlled by curation process that includes matching the new glycan against a canonical glyco-tree (a highly branched representation for a family of glycans). | 4.6.0.0 | production | - |
| Common Diabetes Medications Ontology | CDMONTO | The Common Diabetes Medications Ontology (CDMOnto) is an ontology-based decision support system that has been developed with the objective of providing appropriate treatment suggestions for diabetic patients. The CDMOnto Ontology serves as a foundation for shared semantics, domain-specific, standard, and interoperable knowledge pertinent to common diabetes medications. It encompasses the knowledge of diabetes medications and offers a unified terminology that can be disseminated and processed by medical professionals, pharmacists, and diabetic patients. |  | beta | - |
| Human Reference Atlas Common Coordinate Framework Ontology | CCF | The Common Coordinate Framework (CCF) Ontology is an application ontology built to support the development of the Human Reference Atlas (HRA).  It unifies vocabulary for HRA construction and usage—making it possible to ingest external data sources; supporting uniform tissue sample registration that includes the spatial positioning and semantic annotations within 3D reference organs; and supporting user-formulated cross-domain queries over tissue donor properties, anatomical structures, cell types, biomarkers, and 3D space. The CCF Ontology consists of three major ontologies. The Biological Structure Ontology records anatomical structures, cell types, and biomarkers (ASCT+B) and the relationships between them.  The ASCT+B tables are authored by human experts using templated Google Sheets. The biomarkers, cell types, and anatomical structures are mapped to existing ontologies (Uberon/FMA, CL, HGNC) whenever possible.  All relationships between anatomical structures and from cell types to anatomical structures are valid Uberon and CL relationships. The Spatial Ontology defines the shape, size, location, and rotation of experimental tissue and data major anatomical structures in the 3D Reference Object Library. The Specimen Ontology captures the sex, age, and other information on donors that provided tissue data used in the construction of the HRA. | 3.0.0 | production | CCF (MULTI_TISSUE_STRUCTURE), Human (9606), HRA (CANCER), tissue (TISSUE), organs (ORGAN), cell (CELL), 3D (GENE_OR_GENE_PRODUCT), human (9606), CL (MULTI_TISSUE_STRUCTURE) |
| Ontological Knowledge Base Model for Cystic Fibrosis | ONTOKBCF | OntoKBCF is an ontological knowledge base model for cystic fibrosis. There are molecular genetic information (i.e. gene mutations) and health information included in OntoKBCF. The purposes of OntoKBCF include management of molecular genetic information and health information and embedding OntoKBCF into EHR settings. | 1301 | production | cystic (PATHOLOGICAL_FORMATION) |
| COVID-19OntologyInPatternMedicine | COVID-19-ONT-PM | This ontology will focus on the COVID-19 fields in order to help the hard efforts against the outbreak of the pandemic. It is expected, through the ontology, to understand the medical events or conditions and make medical decisions in more integrated or systematic ways, based on more engineered or calculation methods. This ontology will also be involved in the explorations on the Pattern Medicines (PMs) and so called the Generalized Biology Mechanics (GBM), raised by the author in his Biomedical Knowledge Integration (BMKI), a pilot topic full of challenges.  | 007 | alpha | PMs (CANCER), GBM (CANCER) |
| Chemotherapy Toxicities Ontology | ONTOTOX | OntoTox is an ontology designed to represent chemotherapy toxicities extracted from various data in Clinical Data Warehouses. OntoTox can integrate toxicities and grading information extracted from heterogeneous sources. | 1.0 | alpha | OntoTox (GENE_OR_GENE_PRODUCT) |
| Head and Neck Cancer Ontology | HENECON | HeNeCOn (Head and Neck Cancer Ontology) is the first ontology specifically focused on the Head and Neck Cancer (HNC) domain. It provides a clinically reliable data model for HNC processes during all phases of the disease: prognosis, treatment, and follow-up. The ontology is established based on an annotated dataset to provide formal reference descriptions, contains  170 clinical variables organized into a taxonomy, and is expanded and mapped to formalize and integrate multiple databases into the HeNeCOn ontology. The outcomes of this iterative process were reviewed and validated by clinicians and statisticians. HeNeCOn, with roughly 500 classes, is a very specific ontology which includes a taxonomy with a hierarchical structure, semantic definitions of 283 medical terms and detailed relations between them, which makes it a reference data model.  | 2.0 | production | Head and Neck Cancer Ontology (CANCER), Head and Neck Cancer (CANCER), HNC (CANCER), all (1) |
| VEuPathDB Ontology | EUPATH | The VEuPathDB ontology is an application ontology developed to encode our understanding of what data is about in the public resources developed and maintained by the Eukaryotic Pathogen, Host & Vector Genomics Resource (VEuPathDB; https://veupathdb.org). The VEuPathDB ontology was previously named the EuPathDB ontology prior to EuPathDB joining with VectorBase.The ontology was built based on the Ontology of Biomedical Investigations (OBI) with integration of other OBO ontologies such as PATO, OGMS, DO, etc. as needed for coverage. Currently the VEuPath ontology is primarily intended to be used for support of the VEuPathDB sites. Terms with VEuPathDB ontology IDs that are not specific to VEuPathDB will be submitted to OBO Foundry ontologies for subsequent import and replacement of those terms when they are available. | 2023-05-30 | production | PATO (GENE_OR_GENE_PRODUCT), IDs (GENE_OR_GENE_PRODUCT) |
| Pain Ontology | PAINO | A lexicon of pain terms that have been reviewed by 2 clinicians and a patient and public involvement group of people with lived experiences of chronic pain. | 1.0.0 | alpha | - |
| BioMedical Study Lifecycle Management | BMS-LM | Biomedical research data reuse and sharing is essential for fostering research progress. To this aim, data producers need to master data management and reporting through standard and rich metadata, as encouraged by open data initiatives such as the FAIR (Findable, Accessible, Interoperable, Reusable) guidelines. This helps data re-users to understand and reuse the shared data with confidence. Therefore, dedicated frameworks are required. The provenance reporting throughout a biomedical study lifecycle has been proposed as a way to increase confidence in data while reusing it. The Biomedical Study - Lifecycle Management (BMS-LM) data model has implemented provenance and lifecycle traceability for several multimodal-imaging techniques but this is not enough for data understanding while reusing it. Actually, in the large scope of biomedical research, a multitude of metadata sources, also called Knowledge Organization Systems (KOSs), are available for data annotation. In addition, data producers uses local terminologies or KOSs, containing vernacular terms for data reporting. The result is a set of heterogeneous KOSs (local and published) with different formats and levels of granularity. To manage the inherent heterogeneity, semantic interoperability is encouraged by the Research Data Management (RDM) community. Ontologies, and more specifically top ontologies such as BFO and DOLCE, make explicit the metadata semantics and enhance semantic interoperability. Based on the BMS-LM data model and the BFO top ontology, the BioMedical Study - Lifecycle Management (BMS-LM) core ontology is proposed together with an associated framework for semantic interoperability between heterogeneous KOSs. It is made of four ontological levels: top/core/domain/local and aims to build bridges between local and published KOSs. In this paper, the conversion of the BMS-LM data model to a core ontology is detailed. The implementation of its semantic interoperability in a specific domain context is explained and illustrated with examples from small animal preclinical research. | Version 1.1 | production | - |
| Pathogen and Biomarker Shedding | SHEDDING-HUB | The Shedding Hub Vocabulary (SHV) is a SKOS-based controlled vocabulary designed to standardize terminology for studies involving pathogen and biomarker shedding. |  | alpha | SHV (GENE_OR_GENE_PRODUCT) |
| Food Matrix for Predictive Microbiology | FMPM | Food matrix ontology for predictive microbiology. | 0.9.2 | beta | Matrix (CELLULAR_COMPONENT) |
| EDAM - The data analysis and management ontology | EDAM | EDAM is a domain ontology of data analysis and data management in bio- and other sciences, and science-based applications. It comprises concepts related to analysis, modelling, optimisation, and data life cycle. Targetting usability by diverse users, the structure of EDAM is relatively simple, divided into 4 main sections: Topic, Operation, Data (incl. Identifier), and Format. EDAM is particularly suitable for semantic annotations and categorisation of diverse resources related to data analysis and management: e.g. tools, workflows, learning materials, or standards. EDAM is also useful in data management itself, for recording provenance metadata of processed data. | 1.25-20240924T0027Z-unstable(1.26) | production | EDAM (GENE_OR_GENE_PRODUCT) |
| ChemoOnto | CHO | ChemoOnto, an ontology to qualify the course of chemotherapies | Version: 1.0.1 | alpha | - |
| Devices, Experimental scaffolds and Biomaterials Ontology | DEB | An ontology developed to facilitate information curation in the area of medical devices, experimental scaffolds and biomaterials. The ontology was designed to complement existing medical vocabularies, with emphasis on material processing, structures and features associated with biomaterials and experimental scaffolds. It is used as part of an automated text mining pipeline which populates the database of Experimental Biomaterials and their Biological Effect (DEBBIE). DEBBIE is an EU Horizon 2020 funded project. For more information visit https://projectdebbie.github.io/  | 4.0 | beta | - |
| Computer Assisted Brain Injury Rehabilitation Ontology | CABRO | a web ontology for the semantic representation of the computer assisted brain trauma rehabilitation domain. This is a novel and emerging domain, since it employs the use of robotic devices, adaptation software and machine learning to facilitate interactive, adaptive and personalized rehabilitation care, patient monitoring and assisted living.  | 0.5 | alpha | brain (ORGAN) |
| HeLiFit-Ontology | HELIFIT | This an ontology for healthcare. | 1.6.4 | beta | - |
| Tissue Microarray Ontology | TMA | Tissue microarrays (TMA) are enormously useful tools for translational research, but incompatibilities in database systems between various researchers and institutions prevent the efficient sharing of data that could help realize their full potential.  The Resource Description Framework (RDF) provides a flexible method to represent knowledge in triples, which take the form Subject-Predicate-Object.  All data resources are described using Uniform Resource Identifiers (URI), which are global in scope.  We present an OWL (Web Ontology Language) schema that expands upon the TMA data exchange specification to address this issue and assist in data sharing and integration. | 1.0.0 | alpha | TMA (CANCER) |
| Named Entity Recognition Ontology  | NERO | Named Entity Recognition Ontology (NERO) developed specifically for describing textual entities in biomedical texts, which accounts for diverse levels of ambiguity, bridging the scientific sublanguages of molecular biology, genetics, biochemistry, and medicine. The annotation of natural science texts is more challenging than in other domains. Biomedical language is replete with ambiguity distinct from that observed in news articles or informal text online. In biomedical texts, alternative meanings are not always clearly separated. For example, in some biomedical contexts, the words for a named entity may refer to a gene or a protein with nearly equal probability; for example, “a mutant hemoglobin α_2” can refer to either a gene or a protein. If the author meant gene-or-protein A, and we force an annotator to choose either interpretation gene A or protein A, the resulting annotation is of limited utility because the choice between gene and protein is random if the meanings are equally likely based on context. NERO attempts to minimize unwarranted, arbitrary annotative semantic label assignments for textual entities. NERO defines ambiguous concepts, such as GeneOrProtein, which subsumes both Gene and Protein using the following axiom: EquivalentTo: ‘Gene’ or ‘Protein.’ There are no biological entities that are either a gene or a protein, but there are lexical entities that can correspond to either or both of these entities. | The main developer of NERO is Prof. Robert Stevens, the University of Manchester | alpha | hemoglobin (GENE_OR_GENE_PRODUCT) |
| BioModels Ontology | BIOMODELS | OWL Representation of the models in the BioModels repository. | 21 | production | - |
| A comprehensive ontology of flaps in reconstructive surgery | OFL | Introduction: The diversity of flap types in plastic surgery has led to multiple classification systems based on factors such as blood supply, tissue composition and distance between donor and recipient site. However, no comprehensive framework exists that systematically integrates all relevant aspects. Capturing the full range of surgical flaps in a structured manner could facilitate future research, improve interoperability, and even optimize billing processes. Ontologies could address this gap. They offer a structured approach to organizing and formalizing knowledge by defining a hierarchical tree of classes and individuals linked logically to other entities. Materials and Methods: We applied principles of ontology engineering, integrating knowledge from surgical literature and clinical expertise to model a wide variety of flaps. These are linked to the Foundational Model of Anatomy (FMA) ontology to describe their tissue components. The reasoner Hermit was employed to ensure logical consistency. Results: Structured hierarchically, the ontology links flaps to relevant anatomical structures and applies logical constraints to define necessary and sufficient conditions for each flap type. This enables precise representation of flap variations, each uniquely identified by an Ontology of Flaps in Reconstructive Surgery Identifier (OFLID). The developed ontology encompasses a wide range of local, pedicled, and free flaps. Simple local random-pattern V-Y flaps to complex chimeric and supercharged free flaps can be modelled and automatically classified. Conclusions: This ontology of surgical flaps could facilitate interoperability, multicentric research and artificial intelligence applications. Future work will focus on validation with larger datasets and expanding the ontology’s functionality.  | 1.0.0 | alpha | flap types (PATHOLOGICAL_FORMATION), blood (ORGANISM_SUBSTANCE), tissue (TISSUE), all (1), flaps (ORGAN), flap type (PATHOLOGICAL_FORMATION), flap (PATHOLOGICAL_FORMATION), Flaps (ORGAN), pedicled (PATHOLOGICAL_FORMATION), random-pattern V-Y flaps (PATHOLOGICAL_FORMATION), artificial (32630) |
| Human Dermatological Disease Ontology | DERMO | Ontology of human dermatologic disease | 1.19 | production | human (9606) |
| Ontology of Clinical Research | OCRE | OCRe is an ontology designed to support systematic description of, and interoperable queries on, human studies and study elements. | Revision 315 | alpha | human (9606) |
| Pulmonary Embolism Ontology | PE | A controlled vocabulary of PE-relevant information. Currently, only treatment information is available.  | version 1.0 | alpha | Pulmonary (ORGAN) |
| Next Generation Biobanking Ontology | NGBO | The Next Generation Biobanking Ontology (NGBO) is an open application ontology representing contextual data about omics digital assets in biobank. The ontology focuses on capturing the information about three main activities: wet bench analysis used to generate omics data, bioinformatics analysis used to analyze and interpret data, and data management. | 2024-08-27 | production | NGBO (CANCER) |
| Diabetes Pharmacology Ontology | DPCO | The Diabetes Pharmacology Ontology contains the classes needed to describe antihyperglycemic therapies, pharmacotherapy factors, and diagnostic factors. | 1.0 | beta | - |
| Clinical Trials Ontology - Neurodegenerative Diseases | CTO-NDD | The CTO-NDD stands for Clinical Trial Ontology-Neurodegenerative Diseases, which was built with the purpose of organizing the knowledge domain of clinical trials in neurodegeneration field. This resource has been created for use in the IMI-funded AETIONOMY project (www.aetionomy.org).  | 0.6 | alpha | - |
| Radiomics Ontology | RO | The Radiomics Ontology aims to cover the radiomics feature domain with a strong focus on first order, shape, textural radiomics features. In addition, in the original version. it includes classes about segmentation algorithms and imaging filters. Due to a recent collaboration with the IBSI (International Biomarkers Standardization Initiative), the ontology has been expanded (v 1.6) and it includes all the entities presented in the IBSI document. Therefore, a broad coverage of not only radiomics features, but also every entity (e.g. software properties, filter properties, features extraction parameters) involved into radiomics computation has been added. In the latest version (v2.0), the ontology URIs have been updated to reflect the codes avaialble in the IBSI latest manual | 2.1 | alpha | all (1) |
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| International Classification of Diseases for Oncology, 3rd edition | ICD-O-3 | Ontology used for oncology research derived from the European Cancer Information System (ECIS) | OCD-O Revision 3.2 (complete) | alpha | Cancer (CANCER) |
| Breast cancer screening recommendation ontology | BCSR-ONTO | BCSR-Onto represents knowledge about risk assessment and recommendations by diverse guidelines for breast cancer screening both for women at high and average risk of developing breast cancer. | 2.1 | alpha | BCSR-Onto (GENE_OR_GENE_PRODUCT), cancer (6754) |
| Association pour le Développement de l'Informatique en Cytologie et Anatomie Pathologique | ADICAP | Modes de prélèvement, types de technique, organes, lésions, topographies, grading | 10-2024 | beta | - |
| Mammography Screening Ontology | MAMO-SCR-ONTO | The Mammography Screening Ontology (MAMO-SCR-Onto) is designed to provide mammography application recommendations for the early detection of breast cancer in patients at average risk of the disease. | 1.0 | alpha | cancer (6754) |
| The FoodOn Food Ontology | FOODON | FoodOn is an ontology built to represent entities which bear a “food role” and is initially focused on categorizing and processing of food for humans. We aim to develop semantics for food safety, food security, the agricultural and animal husbandry practices linked to food production, culinary, nutritional and chemical ingredients and processes.  FoodOn belongs to the OBOFoundry.org family of ontologies. | 2025-06-07 | production | bear (9641), humans (9605) |
| Taxonomy for Rehabilitation of Knee Conditions | TRAK | TRAK (Taxonomy for RehAbilitation of Knee conditions) is an ontology that formally models information relevant for the rehabilitation of knee conditions. TRAK provides the framework that can be used to collect coded data in sufficient detail to support epidemiologic studies so that the most effective treatment components can be identified, new interventions developed and the quality of future randomized control trials improved to incorporate a control intervention that is well defined and reflects clinical practice.  | unknown | beta | Knee (CELL), knee (TISSUE) |
| Brain-Computer Interaction (BCI) Ontology | BCI-O | The BCI ontology specifies a foundational metadata model set for real-world multimodal Brain-Computer Interaction (BCI) data capture activities.  Its structure depicts a conceptual framework that BCI applications can extend and use in their implementations, to define core concepts that capture a relevant and interoperable metadata vocabulary. This ontology is aligned to the Semantic Sensor Network Ontology (SSN): a domain-independent and end-to-end model for sensor/actuator applications.  Hence, its structure has been normalized to assist its use in conjunction with other ontologies or linked data resources to specify any particular definitions (such as units of measurement, time and time series, and location and mobility), that specialized applications in the BCI domain might need. Its spec provides general alignment data modeling guidelines for core concepts, to help BCI applications in their design. | 0.9.6 | beta | - |
| Data Science Education Ontology | DSEO | Simple ISA hierarchical taxonomy denoting subfields of study in data science for use in educational applications.  | 1.1 | alpha | ISA (TISSUE), subfields (CELLULAR_COMPONENT) |
| Histological Ontology | HO | A Histological Ontology of the Human Cardiovascular System v. 1.1 | 1.1 | alpha | - |
| OpenCareOntology  | OCO | Open healthCare Ontology (OpenCare) is an integrated metadata vocabulary expressed as an ontological data model, to facilitate the effective reasoning of patient-specific electronic health record (EHR) data. The OpenCare model identifies the concepts and semantic types in the EHR data with the aim of enabling the representation of clinical data in a way that is accessible and usable by researchers and healthcare facility users. The development of OpenCare and its components is formalized using the Ontology Web Language (OWL). This framework aims to enhance the accessibility and user-friendliness of EHR clinical data. | 1.1 | alpha | - |
| Melanoma Ontology | MELO | Ontology of melanoma disease, composed of terms of disease, drug, symptoms, molecular abnormality，anatomical structure, diagnosis, treatment and relationships between them. | 1.0 | alpha | melanoma (CANCER), disease (CANCER) |
| Evidence Graph Ontology | EVI | The Evidence Graph ontology extends core concepts from the W3C Provenance Ontology PROV-O and Bioschemas' Profiles to describe evidence for correctness of findings in biomedical publications. The semantic data model in EVI is expressed using OWL2 Web Ontology Language. | 1.1 | alpha | - |
| Home Cage Monitoring definition | HCM-D | From COST-TEATIME (https://www.cost-teatime.org/) | v0.1 | alpha | - |
| Ontology of Risk Stratification in Mental Health | ONTRISCAL | This ontology organizes concepts and properties of risk stratification in mental health. Stratification aims to determine the patient's risk level by scoring signs and symptoms. Psychotherapeutic treatments aim to minimize patient risk by producing qualitative data reported in therapeutic sessions. Thus, the ONTRISCAL is applied in projects that organize the knowledge about quali-quanti data produced by risk stratification and mental health treatments. Quali-quanti data, semantically annotated by ONTRISCAL, can be ingested into knowledge graphs. Domain specialists can use knowledge graphs to navigate the ONTRISCAL concepts, analyze psychotherapeutic treatments, and control risks in mental health. This is a non-profit and open-source project. | 1.0 | alpha | ONTRISCAL (CANCER) |
| Databank Family Ontology | DFO | The Databank Family Ontology describes groups of data banks with similar record prompts and originators. | 0.1 | alpha | - |
| HIVMutation | HIVMT | This is an ontology about the mutations Of HIV1 and the relations between them and the clinical practices. And it has imported the main ontology: https://bioportal.bioontology.org/ontologies/HIVO004.  | 01 | alpha | HIV1 (11676) |
| Nanbyo Disease Ontology | NANDO | Creating a comprehensive hierarchical controlled vocabulary for intractable and rare disease (i.e., nanbyo) representation in Japan. | 2024-11-21 | production | - |
| HGeneCodonOntology | HCODONONT | This ontology contains the information of the frequency of usage and the relative frequency of usage of Human Gene Codons, to be used to the operations by The Project Of HIV Ontology. | 2 | alpha | Human (9606), HIV (57667) |
| MultiCaRe Taxonomy | MCR_TX | The MultiCaRe Taxonomy contains the class structure used for medical image classification in the MultiCaRe dataset (v2). | 1.0 | production | - |
| Sickle Cell Disease Ontology | SCDO | The Sickle Cell Disease Ontology (SCDO) project is a collaboration between H3ABioNet (Pan African Bioinformatics Network) and SPAN (Sickle Cell Disease Pan African Network). The SCDO is currently under development and its purpose is to 1) establish community standardized SCD terms and descriptions, 2) establish canonical and hierarchical representation of knowledge on SCD, 3) links to other ontologies and bodies of work such as DO, PhenX, MeSH, ICD, NCI’s thesaurus, SNOMED and OMIM. | 2021-04-15 | beta | Cell (CELL), Network (CANCER), NCI (CELL), SNOMED (CANCER) |
| Study Cohort Ontology | SCO | The Study Cohort Ontology (SCO) is developed to model overall structure and patterns of cohort variables and control/intervention groups defined within the structured population descriptions (commonly referred to as Table1's or Cohort Summary Tables)of observational case studies and clinical trials. We maintain only domain agnostic and commonly occuring subject variables and interventions in SCO, and we have a suite of ontologies (Diseases, Medications, LabResults etc) to encode Diabetes related terminology. We have adopted a bottom-up approach to modeling and we have revised the structure of SCO upon investigation of a few research studies (11 to be precise) from the Pharmacological Interventions (Chapter 8) and Cardiovascular Complications (Chapter 9) of the American Diabetes Association (ADA)’s Standards of Medical Care 2018 CPG. We enable deep drive visualizations driven off queries to knowledge graphs modeled on SCO, to help assist physician visualize the similarity of a patient to a study group, at a quick glance. We welcome contributions and suggestions to extend SCO to represent cohort descriptions of publications targeted to other diseases, and we plan to release set of tools to make this process seamless. | 1.1.0 | alpha | SCO (CANCER), LabResults (CELL), Cardiovascular Complications (PATHOLOGICAL_FORMATION), ADA (GENE_OR_GENE_PRODUCT) |
| Cancer Chemoprevention Ontology | CANCO | The Cancer Chemoprevention Ontology constitutes a vocabulary that is able to describe and semantically interconnect the different paradigms of the cancer chemoprevention domain. | 0.3 | production | Cancer (CANCER), cancer (6754) |
| MultiCaRe Taxonomy (ML Labels) | ML_MCR_TX | This is a reduced version of the MultiCaRe Taxonomy (MCR_TX), the class structure used for medical image classification in the MultiCaRe dataset. This reduced version includes only the labels used in Machine Learning (ML) model training. |  | beta | - |
| Homo sapiens Chromosomal Location Ontology for GRCh38 | HSCLO38 | HSCLO38's purpose is to simplify the integration of genomic experimental data at different resolution scales within biomedical knowledge graphs. It defines chromosomal locations within the GRCh38 release for chr 1-22, X,Y and M.  HSCLO38 provides 3,431,153 classes at five distinct resolution levels: whole chromosome, 1 megabase pair (Mbp), 100 kilobase pairs (kbp), 10 kbp, and 1 kbp.  Each node within these class levels is interconnected to its scale parent and to the immediate neighbors on either side to support mapping and association between genomic datasets and features. For example, the 1kbp element HSCLO38:chr1.20517001-20518000 is_a connected to its "scale parent" HSCLO38:chr1.20510001-20520000 as well as to the 5' neighbor HSCLO38:chr1.20516001-20517000 and 3' neighbor HSCLO38:chr1.20518001-20519000.  We provide knowledge-graph-ready files for HSCLO38 on the project home page. |  | alpha | HSCLO38 (CANCER), chromosomal (CELLULAR_COMPONENT), chr 1 (GENE_OR_GENE_PRODUCT), X (CELLULAR_COMPONENT), 3,431,153 (GENE_OR_GENE_PRODUCT), node (MULTI_TISSUE_STRUCTURE), 3' neighbor (CELLULAR_COMPONENT) |
| Knowledge Object Reference Ontology | KORO | An ontology to define the parts of a Knowledge Object and their relations built using the Basic Formal Ontology and Information Artifact Ontology | 1.3 | beta | - |
| Presence Ontology | PREO | The Presence Ontology is a systematic vocabulary of terms with defined relationships that models the encounters taking place every day among providers, patients, and family members or friends in environments such as hospitals and clinics. The Presence Ontology provides a conceptual model for the human experience in medicine. This is a preliminary approach, but further use of methods we have developed here may aid in providing clarity and consensus to topics in healthcare. The ontology is also a model for interdisciplinary collaboration, as it was developed in conjunction with experts from bioinformatics, medicine, anthropology, business, and communication sciences through multiple iterative stages. | 1.0.0 | alpha | human (9606) |
| Adherence and Integrated Care | ADHER_INTCARE_EN | This ontology includes concepts in English related to adherence to therapeutic plans in the context of integrated care. Includes maps with Snomed CT, bibliographic references and definitions from the Dictionary of the Royal Academy of Medicine of Spain. Ontology developed with the support of the project StandICT.eu. The equivalent ontology in Spanish is ADHER_INTCARE_SP. | English 051319 | alpha | ADHER_INTCARE_SP (CANCER) |
| Just Enough Results Model Ontology | JERM | The JERM ontology is an application ontology that describes the relationships between the data, models and other research assets that can be shared through the FAIRDOM platform, for sharing Systems and Synthetic Biology experiments. JERM 2 includes a new model for describing Biosamples, developed during the ELIXIR-lead Biosamples club | 2 | beta | FAIRDOM (GENE_OR_GENE_PRODUCT) |
| Orphanet Rare Disease Ontology | ORDO | The Orphanet Rare Disease ontology (ORDO) is jointly developed by Orphanet and the EBI to provide a structured vocabulary for rare diseases capturing relationships between diseases, genes and other relevant features which will form a useful resource for the computational analysis of rare diseases. It derived from the Orphanet database (www.orpha.net) , a multilingual database dedicated to rare diseases populated from literature and validated by international experts. It integrates a nosology (classification of rare diseases), relationships (gene-disease relations, epiemological data) and connections with other terminologies (MeSH, SNOMED CT, UMLS, MedDRA),databases (OMIM, UniProtKB, HGNC, ensembl, Reactome, IUPHAR, Geantlas) or classifications (ICD10). The ontology will be maintained by Orphanet and further populated with new data. Orphanet classifications can be browsed in the OLS view. The Orphanet Rare Disease Ontology is updated monthly and follows the OBO guidelines on deprecation of terms. It constitutes the official ontology of rare diseases produced and maintained by Orphanet (INSERM, US14).  | 4.7 | production | ORDO (CANCER), gene-disease (GENE_OR_GENE_PRODUCT), UMLS (CANCER) |
| NCCN EHR Oncology Categories | NCCNEHR | The NCCN EHR Oncology Advisory Group was formed to address the high variance in oncology history documentation within EHR systems. Specifically, there is a lack of standardization in the names of oncology history categories across institutions. Moreover, no synonym list or mapping currently exists between the varied category names. Furthermore, oncology history documentation is often inconsistent, incomplete, and not updated appropriately. The NCCN EHR Oncology Advisory Group has established best practices and is working with EHR vendors to implement and enhance tools for documentation of oncology history. This ontology is a result of the work of the group, and provides users with a list of oncology history categories and synonyms. It is our sincere hope that this work can form the basis of improving standardization and interoperability of oncology history documentation within EHR systems. | 1.0.0 | alpha | - |
| NeuroBridge Ontology | NEUROBRG | NeuroBridge ontology is being developed as part of the NeuroBridge project to implement FAIR principles for neuroimaging and neurophysiology data using computable provenance metadata. The NeuroBridge ontology extends the W3C PROV ontology and the ProvCaRe ontology with terms from the SchizConnect and Data Bridge ontology aligned and merged into the original class structure of the ProvCaRe ontology.  | 1.0 (Release 2023) | production | SchizConnect (GENE_OR_GENE_PRODUCT) |
| Alzheimer Disease Relevance Ontology by Process | AD-DROP | Alzheimer Disease Relevance Ontology by Process (AD-DROP) aimed at classifying Disease Relevant Process according to their specificity, frequency and pathogenic intensity properties toward AD | alpha | alpha | - |
| Drug Database for Inborn Errors of Metabolism (DDIEM) Ontology | DDIEM | DDIEM Ontology for the Drug Database for Inborn Errors of Metabolism. The DDIEM database is a database of therapeutic strategies and treatments for inborn errors of metabolism. These strategies are classified by mechanism and outcome using the DDIEM Ontology. The DDIEM database also includes descriptions of the phenotypes addressed by the treatment, together with drugs and procedures applied. | 1.2 | production | DDIEM (GENE_OR_GENE_PRODUCT) |
| Medical Imaging and Diagnostic Ontology | MIDO | MIDO is an OWL 2 ontology for creating a standardized format for modeling data in medical imaging tasks such as Disease Classification/ Diagnosis as well as Disease localization/segmentation | 2015-11-20 | alpha | Disease (CANCER) |
| NMR-Controlled Vocabulary | NMR | This is the new msi-sanctioned nmr CV, created within the COSMOS EU project, to support the nmrML data standard for nuclear magnetic resonance data in metabolomics with meaningful raw data descriptors. | 1.1.0 | beta | - |
| Biofilm Ontology | BIFO | BIFO is intended to provide a biofilm ontology capable of understanding fundamental terminologies the relations between common biofilm terminologies following the guidelines of the Basic Formal Ontology (BFO). |  | alpha | Biofilm (CELL) |
| Obstetric and Neonatal Ontology | ONTONEO | The Obstetric and Neonatal Ontology is a structured controlled vocabulary to provide a representation of the data from electronic health records (EHRs) involved in the care of the pregnant woman, and of her baby. | 2025-03-14 | alpha | EHRs (CANCER) |
| Bionutrition Ontology | BNO | The Bionutrition Ontology relates concepts and terminologies used for human nutrition in a clinical and biomedical setting. | 0.2 | alpha | human (9606) |
| Statistics Ontology | STATO | STATO is a general-purpose STATistics Ontology. Its aim is to provide coverage for processes such as statistical tests, their conditions of applications, and information needed or resulting from statistical methods, such as probability distributions, variable, spread and variation metrics. STATO also covers aspects of experimental design and description of plots and graphical representations commonly used to provide visual cues of data distribution or layout and to assist review of the results. | 2025-02-23 | production | STATO (CANCER), general-purpose (GENE_OR_GENE_PRODUCT) |
| BrainTeaser Ontology (BTO) | BT-ONTOLOGY | BrainTeaser Ontology (BTO) whose purpose is to jointly model both Amyotrophic Lateral Sclerosis (ALS) and Multiple Sclerosis (MS). The BTO serves multiple purposes: to provide a unified and comprehensive conceptual view about ALS and MS, which are typically dealt with separately, allowing us to coherently integrate the data coming from the different medical partners in the project; to seamlessly represent both retrospective data and prospective data, produced during the lifetime of BRAINTEASER; to allow for sharing and re-using the BRAINTEASER datasets according to Open Science and FAIR principles. | 2.0 | alpha | - |
| Epilepsy Ontology | EPILONT | Ontology about the epilepsy domain and epileptic seizures. Based on the diagnosis proposed by the International League Against Epilepsy (ILAE). | V1 | beta | - |
| Information Artifact Ontology | IAO | The Information Artifact Ontology (IAO) is an  ontology of information entities, originally driven by work by the OBI consortium. | 2022-11-07 | beta | - |
| Head and Neck Cancer - Clinical and Gene Ontology | HENEGEO | HeNeGeO (Head and Neck Cancer - Clinical and Gene Ontology) presents a data model that encompasses both clinical and genomic aspects of Head and Neck Cancer (HNC) across every stage of the disease, covering prognosis, treatment and follow-up. This ontology builds upon the foundation laid by HENECON (Head and Neck Cancer Ontology) and has been significantly expanded with a focus on information pertaining to cancer recurrence, alongside a comprehensive list of genes that offer valuable insights throughout the entire head and neck cancer progression. To enhance clarity, novel medical terms have been thoughtfully annotated with their respective semantic meanings. This comprehensive update includes the incorporation of 29,770 genes, featuring essential details such as NCBI GeneID (EntrezID), gene symbol, gene name, gene alias, and the HGNC ID. The development of this ontology underwent rigorous review and validation by experts in both clinical and genomic domains. | 1.0 | production | Head and Neck Cancer (CANCER), HNC (CANCER), Head and Neck Cancer Ontology (CANCER), cancer (6754), head and neck (CANCER) |
| HIVOntology | HIVO004 | A Project of Biomedical Ontology for Systems Biomedicine. The goals of the Project include: (1). To pilot the novel researches on AIDS and HIV or other diseases by viruses, by means of the ontology and other biomedical informatics technologies, connecting the insights of the Biomedical Knowledge Integration (BMKI), systems biomedicine, complexity sciences etc. (2). To Carry out the methodology explorations for the goals above. | 020 | alpha | HIV (57667), viruses (10239) |
| Brain Tumour Ontology | BTO_ONTOLOGY | The purpose of this work is to develop an ontology-based framework for developing an information retrieval system to cater to specific queries of users. For creating such an ontology, information was obtained from a wide range of information sources involved with brain tumour study and research. The information thus obtained was compiled and analysed to provide a standard, reliable and relevant information base to aid our proposed system. Facet-based methodology has been used for ontology formalization for quite some time. Ontology formalization involves different steps such as identification of the terminology, analysis, synthesis, standardization and ordering. A vast majority of the ontologies being developed nowadays lack flexibility. This becomes a formidable constraint when it comes to interoperability. We found that a facet-based method provides a distinct guideline for the development of a robust and flexible model concerning the domain of brain tumours. Our attempt has been to bridge library and information science and computer science, which itself involved an experimental approach. It was discovered that a faceted approach is really enduring, as it helps in the achievement of properties like navigation, exploration and faceted browsing. Computer-based brain tumour ontology supports the work of researchers towards gathering information on brain tumour research and allows users across the world to intelligently access new scientific information quickly and efficiently. | 1 | beta | brain tumour (CANCER), brain tumours (CANCER) |
| FAIR Genomes | FG | The FAIR Genomes semantic metadata schema to power reuse of NGS data in research and healthcare. | 1.2-Minor | production | - |
| Spinal Cord Injury Ontology | SCIO | Representation of pre-clinical studies in the domain of spinal cord injury therapies | 1 | alpha | spinal cord (ORGAN) |
| NeuralReprogrammingOntology | NRO | The neural reprogramming ontology encompasses the domain of neural regenerative cell reprogramming knowledge. As research progresses, researchers and specialists in the field of reprogramming could continue to expand the bounds of this foundation, so adding to its depth. | 1.0.0 | beta | neural (TISSUE), cell (CELL) |
| Adherence and Integrated Care in Spanish | ADHER_INTCARE_SP | This ontology includes concepts in Spanish related to adherence to therapeutic plans in the context of integrated care. Includes maps with Snomed CT, bibliographic references and definitions from the Dictionary of the Royal Academy of Medicine of Spain. Ontology developed with the support of the project StandICT.eu. | Spanish 052319 | alpha | - |
| Ontology of Genes and Genomes - Mouse | OGG-MM | OGG-Mm is the OGG Mus musculus (mouse) subset. The OGG (Ontology of Genes and Genomes) is a formal ontology of genes and genomes of biological organisms. OGG is developed by following OBO Foundry principles and aligning with the BFO top ontology.  | Vision Release: 1.0.66 | production | OGG-Mm (GENE_OR_GENE_PRODUCT), Mus musculus (10090), mouse (10090), OGG (GENE_OR_GENE_PRODUCT) |
| Mapping of Epilepsy Ontologies | MEPO | The Mapping of Epilepsy Ontologies (MEPO) provides owl:sameAs relationships of similar concepts for the ontologies EpSO, ESSO, EPILONT, EPISEM, and FENICS. Two concepts are considered similar if they share the same concept name or the same synonym. Additional synonyms for each of the concepts are generated using the Snowball stemming algorithm. The software implementation is available on github within the Maven project snoke in the module snoke.ontology: https://github.com/bernd-mueller/snoke/tree/master/snoke.ontology | 0.6 | alpha | MEPO (CANCER) |
| Ontology for Nutritional Studies | ONS | Ontology For Nutritional Studies (ONS) is the first systematic effort to provide a formal ontology framework for the description of nutritional studies. | 2024-07-03 | production | ONS (CANCER) |
| Unwarranted Clinical Variation Analysis With Contextual Factors | UCVA | The Unwarranted Clinical Variation Analysis ontology (UCVA) is an application ontology that standardizes the representation of local context factors of UCV analysis. It includes numerous instances designed to support the extraction of these factors as covariates in UCV analysis, making it practical and easy to use | 1.0 | alpha | - |
| Ontology for Nutritional Epidemiology | ONE | Nutritional epidemiology is a specific research area. The generic ontologies for food science, nutrition science or medical science failed to cover the specific characteristics of nutritional epidemiologic studies. As a result, we developed the ontology for nutritional epidemiology (ONE) in order to describe nutritional epidemiologic studies accurately. | V2.2 | alpha | - |
| PMO Precision Medicine Ontology | PMAPP-PMO | The Precision Medicine Ontology (PMO) has been developed as a standardized ontology for human precision medicine with consistent, reusable and sustainable descriptions of human disease terms, genomic molecular, phenotype characteristics and related medical vocabulary disease concepts through collaborative efforts of researchers at Institute of Medical Information, Chinese Academy of Medical Sciences. The Precision Medicine Ontology focuses on the development of a high level patients-centric ontology for the academic research, thus, it systematically integrates biomedicine vocabularies from GO, NCBI Gene, DrugBank, HPO, MeSH, SNOMED CT, etc. This work is supported by the National Key Research and Development Program of China (Precision Medicine Knowledge Base, Grant No. 2016YFC0901901). | 4.0 | alpha | human (9606), disease (CANCER), HPO (CANCER) |
| ICHOM Set Pregnancy and Childbirth | ICHOM-PROMS-PCB | The ICHOM Set of Patient-Centered Outcome Measures for Pregnancy And Childbirth is the result of hard work by a group of leading physicians, measurement experts and patients. It is a recommendation of the outcomes that matter most to persons experiencing Pregnancy And Childbirth.  |  | alpha | - |
| Base de Données Publique des Médicaments | BDPM | LA BDPM est un des référentiels du médicament gérée et distribuée par l’ANSM. Elle recense l’ensemble des spécialités et des présentations effectivement commercialisées durant les 5 dernières années. De ce fait, elle ne propose pas un historique exhaustif des codes de présentation et spécialités, celui-ci peut être obtenu grâce à la base RCP disponible sur le site de l’ANSM. | 2025-03-04 | production | - |
| Ontology of Physics for Biology | OPB | The OPB is a reference ontology of classical physics and thermodynamics as applied to the dynamics of biological systems. It is designed to encompass the multiple structural scales (atoms to organisms) and multiple physical domains (fluid dynamics, chemical kinetics, particle diffusion, etc.) that are encountered in the study and analysis of biological organisms. | 1.2 | beta | OPB (GENE_OR_GENE_PRODUCT) |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |
| MGB Drug List | MGBD | MGB Drug List | 0.6 | alpha | - |
| Randomized Controlled Trials Ontology | RCTONT | We developed and evaluated an ontology specifically for Randomized Controlled Trials in order to facilitate the production of systematic reviews and metaanalysis. | 2.0 | alpha | - |
| Human Disease Ontology | DOID | Creating a comprehensive classification of human diseases organized by etiology. | 2025-05-30 | production | human (9606) |
| Open Predictive Microbiology Ontology | OFSMR | Food-matrix ontology for Open Food Safety Model Repository | 0.4 | alpha | Food-matrix (GENE_OR_GENE_PRODUCT) |
| HPO - ORDO Ontological Module | HOOM | Orphanet provides phenotypic annotations of the rare diseases in the Orphanet nomenclature using the Human Phenotype Ontology (HPO). HOOM is a module that qualifies the annotation between a clinical entity and phenotypic abnormalities according to a frequency and by integrating the notion of diagnostic criterion. In ORDO a clinical entity is either a group of rare disorders, a rare disorder or a subtype of disorder. The phenomes branch of ORDO has been refactored as a logical import of HPO, and the HPO-ORDO phenotype disease-annotations have been provided in a series of triples in OBAN format in which associations, frequency and provenance are modeled. HOOM is provided as an OWL (Ontologies Web Languages) file, using OBAN, the Orphanet Rare Disease Ontology (ORDO), and HPO ontological models. HOOM provides extra possibilities for researchers, pharmaceutical companies and others wishing to co-analyse rare and common disease phenotype associations, or re-use the integrated ontologies in genomic variants repositories or match-making tools.  | 2.3 | beta | Human (9606), HPO (CANCER), ORDO (CANCER) |
| DSMZ Digital Diversity Ontology | D3O | The DSMZ Digital Diversity Ontology (D3O) is an ontology designed to standardize and integrate data related to microbial diversity and associated resources within the context of the DSMZ’s Digital Diversity Knowledge Graph. This ontology provides a structured framework to represent various aspects of microbial data, including taxonomic information, strain properties, cultivation conditions, and media links. D3O facilitates cross-referencing between internal DSMZ datasets and external biological resources, supporting enhanced data discovery, interoperability, and research in microbiology, genomics, and metabolomics. The ontology includes both object properties, which link entities such as bacterial strains to external resources (e.g., literature references or media descriptions), and data properties, which associate entities with literal values (e.g., taxonomic identifiers, growth conditions). Designed for use in RDF-based knowledge graphs, D3O supports flexible integration with other life science ontologies, promoting consistency and reuse in the broader scientific community. | 1.1 | alpha | DSMZ (CELL), D3O (GENE_OR_GENE_PRODUCT) |

<a name="bioportal-category-cell"></a>
## BioPortal Category "Cell"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| In Vitro Methodologies Ontology for MAATrica metric | VIMM | MAATrica is a novel metric for assessing the coherence of methodologies in research papers within the fields of medicinal and nutraceutical chemistry. MAATrica employs user-controlled and customizable ontologies (in silico, in vitro and in vivo), enabling personalized analysis of research papers in the fields of medicinal and nutraceutical chemistry. Here we present the in vitro ontology we developed for the scope. |  | alpha | - |
| Neuroscience Domain Insight Graph | NEUDIGS | Neuroscience Domain Insight Graphs provide a framework for expressing linked open data about neuronal populations expressed as statements about (A) distributions of subcellular components, (B) measurable physiological changes under various experimental conditions. The goal of this representation is to provide a computable language for reasoning over experimental neuroscientific findings.    | 0.0.1 | alpha | neuronal (CELL) |
| Cell Line Ontology | CLO | The Cell Line Ontology (CLO) is a community-based ontology in domain of biological cell lines with a focus on permanent cell lines from culture collections. Upper ontology structures that frame the skeleton of CLO include Basic Formal Ontology and Relation Ontology. Cell lines contained in CLO are associated with terms from other ontologies such as Cell Type Ontology, NCBI Taxonomy, and Ontology for Biomedical Investigation. A common design pattern for the cell line is used to model cell lines and their attributes, the Jurkat cell line provides ane xample.  Currently CLO contains over 36,000 cell line entries obtained from ATCC, HyperCLDB, Coriell, and bymanual curation. The cell lines are derived from 194 cell types, 656 anatomical entries, and 217 organisms. The OWL-based CLO is machine-readable and can be used in various applications. | 2.1.188 | production | Cell Line (CELL), cell lines (CELL), Cell lines (CELL), Cell Type (CELL), cell line (CELL), Jurkat cell line (CELL), ATCC (CELL), Coriell (CELL), cell (CELL) |
| Whole-cell modeling ontology | WC | Ontology for whole-cell (WC) modeling. See https://wholecell.org for more information about WC modeling | 0.0.1 | alpha | - |
| Human Reference Atlas Common Coordinate Framework Ontology | CCF | The Common Coordinate Framework (CCF) Ontology is an application ontology built to support the development of the Human Reference Atlas (HRA).  It unifies vocabulary for HRA construction and usage—making it possible to ingest external data sources; supporting uniform tissue sample registration that includes the spatial positioning and semantic annotations within 3D reference organs; and supporting user-formulated cross-domain queries over tissue donor properties, anatomical structures, cell types, biomarkers, and 3D space. The CCF Ontology consists of three major ontologies. The Biological Structure Ontology records anatomical structures, cell types, and biomarkers (ASCT+B) and the relationships between them.  The ASCT+B tables are authored by human experts using templated Google Sheets. The biomarkers, cell types, and anatomical structures are mapped to existing ontologies (Uberon/FMA, CL, HGNC) whenever possible.  All relationships between anatomical structures and from cell types to anatomical structures are valid Uberon and CL relationships. The Spatial Ontology defines the shape, size, location, and rotation of experimental tissue and data major anatomical structures in the 3D Reference Object Library. The Specimen Ontology captures the sex, age, and other information on donors that provided tissue data used in the construction of the HRA. | 3.0.0 | production | CCF (MULTI_TISSUE_STRUCTURE), Human (9606), HRA (CANCER), tissue (TISSUE), organs (ORGAN), cell (CELL), 3D (GENE_OR_GENE_PRODUCT), human (9606), CL (MULTI_TISSUE_STRUCTURE) |
| Neuron Phenotype Ontology | NPOKB | An ontology of neuron types based on the phenotypic dimensions of cells that can be measure experimentally. | 2025-05-02 | beta | neuron (CELL), cells (CELL) |
| Ontology of Host-Pathogen Interactions | OHPI | The Ontology of Host-Pathogen Interactions (OHPI) is a biomedical ontology of the host-pathogen interactions. Its development follows the OBO Foundry Principles. | 1.0.26 | production | OHPI (CANCER) |
| Placental Cell Type | PATCT | Cell types associated with placental structure and function. This Cell Type ontology builds on the Cell Ontology (CL) as well as the BRENDA Tissue and Enzyme Source Ontology (BTO) and was further developed by PAT External Scientific Panel Members and contributions from other subject matter experts. | 1.0 | production | Cell (CELL), Cell Type (CELL), CL (CANCER), BRENDA (GENE_OR_GENE_PRODUCT) |
| BioAssay Ontology | BAO | The BioAssay Ontology (BAO) describes chemical biology screening assays and their results including high-throughput screening (HTS) data for the purpose of categorizing assays and data analysis.  BAO is an extensible, knowledge-based, highly expressive (currently SHOIQ(D)) description of biological assays making use of descriptive logic based features of the Web Ontology Language (OWL).  BAO currently has over 1000 classes and also makes use of several other ontologies.  It describes several concepts related to biological screening, including Perturbagen, Format, Meta Target, Design,  Detection Technology, and Endpoint.  Perturbagens are perturbing agents that are screened in an assay; they are mostly small molecules. Assay Meta Target describes what is known about the biological system and / or its components interrogated in the assay (and influenced by the Perturbagen).  Meta target can be directly described as a molecular entity (e.g. a purified protein or a protein complex), or indirectly by a biological process or event (e.g. phosphorylation).  Format describes the biological or chemical features common to each test condition in the assay and includes biochemical, cell-based, organism-based, and variations thereof.  The assay Design describes the assay methodology and implementation of how the perturbation of the biological system is translated into a detectable signal.  Detection Technology relates to the physical method and technical details to detect and record a signal.  Endpoints are the final HTS results as they are usually published (such as IC50, percent inhibition, etc.).  BAO has been designed to accommodate multiplexed assays.  All main BAO components include multiple levels of sub-categories and specification classes, which are linked via object property relationships forming an expressive knowledge-based representation. | 2.8.16 | production | SHOIQ(D (GENE_OR_GENE_PRODUCT) |
| Biological Pathway Taxonomy | BPT | Biological Pathway Taxonomy (BPT) - hierarchical taxonomy of cellular and molecular mechanisms, metabolic reactions, and diseases that can be shown as signaling pathways. BPT was created manually based on the Elsevier Pathway Collection available for public usage. There are 3 types of entities in BPT – pathway, group, and folder of pathways and groups. Group is the list of genes and proteins associated with biological function and diseases. Pathway is an interactive model of molecular interactions in human and mammalian cells described in the literature. Each model was reconstructed based on facts published in peer-reviewed journals, with manually quality control. BPT pathway entity includes several properties such as link to the model in Pathway Studio software which allows reading supported sentences from published articles and additional information about members. A list of members for each model can be provided by request. Property “network” describes maps without logical causal relationships between members to distinguish them from actual models of molecular mechanisms (“signaling”). The index folder contains all pathways and groups sorted alphabetically. | synonymes2022 | alpha | cellular (CELL), human (9606), mammalian cells (CELL), all (1) |
| Cellular microscopy phenotype ontology | CMPO | The Cellular Microscopy Phenotype Ontology (CMPO) is a species neutral ontology for describing general phenotypic observations relating to the whole cell, cellular components, cellular processes and cell populations. CMPO is an application ontology developed in OWL that contains precomposed phenotypes descriptions that are defined using the Gene Ontology (GO) and the Phenotype Trait Ontology (PATO). | 1.3 | production | Cellular Microscopy (CELL), cell (CELL), cellular (CELL) |
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Provisional Cell Ontology | PCL | Cell types that are provisionally defined by experimental techniques such as single cell or single nucleus transcriptomics rather than a straightforward & coherent set of properties. | 2025-01-08 | production | Cell (CELL), cell (CELL), nucleus (CELLULAR_COMPONENT) |
| Neuroscience Information Framework (NIF) Standard Ontology | NIFSTD | NIF Standard ontology (NIFSTD) is a core component of Neuroscience Information Framework (NIF) project (http://neuinfo.org), a semantically enhanced portal for accessing and integrating neuroscience data, tools and information. NIFSTD includes a set of modular ontologies that provide a comprehensive collection of terminologies to describe neuroscience data and resources. | 3.1 - January 15, 2018 | production | portal (TISSUE), collection (32644) |
| Cell Ontology for Mouse Lung Maturation | LUNGMAP_M_CELL | Cell Ontology for Mouse Lung Maturation (LungMAP Mouse Anatomy (LMMA)) | 3.2 | production | Cell (CELL) |
| Context and Measurement Ontology | COMO | Ontological terms to describe experimental data, probably useful for other environmental microbiology projects.  Used in several CORAL (Contextual Ontology-based Repository Analysis Library) deployments.  See more about this platfom at https://github.com/jmchandonia/CORAL |  | beta | - |
| Cell Line Ontology [by Mahadevan] | MCCL | Ontologies provide a common platform of controlled vocabulary for researchers who need to share information across a particular domain, inclusive of machine-interpretable definitions of basic concepts in the domain and relations among them. The need for ontologies that provide a systematic arrangement of available data and allow cross talk across various related domains is gaining momentum. In this backdrop, we have developed a comprehensive ontology on primary and established cell lines-both normal and pathologic. It covers around 400 cell lines. This ontology has been built to include the major domains in the field of biology like anatomy, bio-molecules, chemicals and drugs, pathological conditions and genetic variations around the cell lines. An extensive network of relations has been built across these concepts to enable different combinations of queries. The ontology covers all cell lines from major sources like ATCC, DSMZ, ECACC, ICLC etc. and is built in OWL format. | 2 | beta | cell lines-both (CANCER), cell lines (CELL), bio-molecules (CANCER), all (1), DSMZ (CELL), ECACC (CELL) |
| Cell Ontology | CL | The Cell Ontology is a structured controlled vocabulary for cell types in animals. The Cell Ontology is designed as a structured controlled vocabulary for cell types. This ontology was constructed for use by the model organism and other bioinformatics databases, where there is a need for a controlled vocabulary of cell types. This ontology is not organism specific. It covers cell types from prokaryotes to mammals. However, it excludes plant cell types, which are covered by PO. | 2025-06-20 | production | Cell (CELL), cell (CELL), animals (33208), prokaryotes (2157), mammals (40674) |
| NeuralReprogrammingOntology | NRO | The neural reprogramming ontology encompasses the domain of neural regenerative cell reprogramming knowledge. As research progresses, researchers and specialists in the field of reprogramming could continue to expand the bounds of this foundation, so adding to its depth. | 1.0.0 | beta | neural (TISSUE), cell (CELL) |
| Biolink Model | BIOLINK | A high level datamodel of biological entities (genes, diseases, phenotypes, pathways, individuals, substances, etc) and their associations | 2.2.13 | production | - |
| Cell Behavior Ontology | CBO |  "The Cell Behavior Ontology (CBO) describes multi-cell computational models. In particular to describe both the existential behaviors of cells (spatiality | Version 1.1.2 | production | Cell (CELL), CBO (CANCER), multi-cell (CELL), cells (CELL) |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |
| Ontology for MicroRNA Target | OMIT | The purpose of the OMIT ontology is to establish data exchange standards and common data elements in the microRNA (miR) domain. Biologists (cell biologists in particular) and bioinformaticians can make use of OMIT to leverage emerging semantic technologies in knowledge acquisition and discovery for more effective identification of important roles performed by miRs in humans' various diseases and biological processes (usually through miRs' respective target genes). OMIT has reused and extended a set of well-established concepts from existing bio-ontologies, e.g., Gene Ontology, Sequence Ontology, Protein Ontology, NCBI Organism Taxonomy, Human Disease Ontology, Foundational Model of Anatomy, and so forth. | OBO 4.0-12122013-2338 | alpha | cell (CELL), miRs (GENE_OR_GENE_PRODUCT), humans (9605), Human (9606) |

<a name="bioportal-category-cellular-anatomy"></a>
## BioPortal Category "Cellular anatomy"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Neuroscience Domain Insight Graph | NEUDIGS | Neuroscience Domain Insight Graphs provide a framework for expressing linked open data about neuronal populations expressed as statements about (A) distributions of subcellular components, (B) measurable physiological changes under various experimental conditions. The goal of this representation is to provide a computable language for reasoning over experimental neuroscientific findings.    | 0.0.1 | alpha | neuronal (CELL) |
| Neuron Phenotype Ontology | NPOKB | An ontology of neuron types based on the phenotypic dimensions of cells that can be measure experimentally. | 2025-05-02 | beta | neuron (CELL), cells (CELL) |
| Cellular microscopy phenotype ontology | CMPO | The Cellular Microscopy Phenotype Ontology (CMPO) is a species neutral ontology for describing general phenotypic observations relating to the whole cell, cellular components, cellular processes and cell populations. CMPO is an application ontology developed in OWL that contains precomposed phenotypes descriptions that are defined using the Gene Ontology (GO) and the Phenotype Trait Ontology (PATO). | 1.3 | production | Cellular Microscopy (CELL), cell (CELL), cellular (CELL) |
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Association pour le Développement de l'Informatique en Cytologie et Anatomie Pathologique | ADICAP | Modes de prélèvement, types de technique, organes, lésions, topographies, grading | 10-2024 | beta | - |
| Neuroscience Information Framework (NIF) Standard Ontology | NIFSTD | NIF Standard ontology (NIFSTD) is a core component of Neuroscience Information Framework (NIF) project (http://neuinfo.org), a semantically enhanced portal for accessing and integrating neuroscience data, tools and information. NIFSTD includes a set of modular ontologies that provide a comprehensive collection of terminologies to describe neuroscience data and resources. | 3.1 - January 15, 2018 | production | portal (TISSUE), collection (32644) |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |
| NeuroMorpho.Org brain region ontologies | NMOBR | The species, brain regions and cell types are integrated into a single ontology for enabling OntoSearch functionality to mine the latest v7.0 release of NeuroMorpho.Org. | 2.0 | beta | brain (ORGAN), cell (CELL) |

<a name="bioportal-category-chemical"></a>
## BioPortal Category "Chemical"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| New Approach Methodology Ontology | NAMO | The New Approach Methodology Ontology and Schema (NAMO) is a comprehensive data framework designed to standardize and integrate information about alternative methods to animal testing in biomedical research. Built on LinkML, NAMO provides a common vocabulary and data structure for describing diverse model systems, from cellular cultures and organ-on-chip devices to computational models and digital twins. |  | production | Schema (GENE_OR_GENE_PRODUCT), cellular cultures (CELL), twins (CANCER) |
| The Biological and Environmental Research Variable Ontology | BERVO | The Biological and Environmental Research Variable Ontology (BERVO) is a model of the experimental variables, conditions, and concepts used in the study of environmental research, including studies in earth science, plant science, and geochemistry. | 2025-08-05 | alpha | - |
| AGROVOC | AGROVOC | AGROVOC is a multilingual and controlled vocabulary designed to cover concepts and terminology under FAO's areas of interest. It is a large Linked Open Data set about agriculture, available for public use, and its highest impact is through facilitating the access and visibility of data across domains and languages. |  | production | - |
| In Vitro Methodologies Ontology for MAATrica metric | VIMM | MAATrica is a novel metric for assessing the coherence of methodologies in research papers within the fields of medicinal and nutraceutical chemistry. MAATrica employs user-controlled and customizable ontologies (in silico, in vitro and in vivo), enabling personalized analysis of research papers in the fields of medicinal and nutraceutical chemistry. Here we present the in vitro ontology we developed for the scope. |  | alpha | - |
| Pharmacogenomic Relationships Ontology | PHARE | The PHArmacogenomic RElationships Ontology (or PHARE) proposes concepts and roles to represent relationships of pharmacogenomics interest. | version id: 110114 | alpha | - |
| Drug Target Ontology | DTO | Drug Target Ontology (DTO) is being developed at the University of Miami in the research group of Stephan Schürer. DTO is supported by grant U54CA189205 (Illuminating the Druggable Genome Knowledge Management Center, Tudor Oprea, PI) awarded by the NCI through the NIH Common Fund. It is a component of the Illuminating the Druggable Genome (IDG) project (https://commonfund.nih.gov/idg). The DTO project develops a novel semantic framework to formalize knowledge about drug targets with a focus on the current IDG protein families. The DTO is developed as a reference for drug targets with the longer-term goal to create a community standard that will facilitate the integration of diverse drug discovery information from numerous heterogeneous resources. The first version of the DTO consists of asserted class hierarchies of the four IDG protein families, GPCRs, kinases, ion channels, and nuclear hormone receptors. Protein classes are linked to tissue and disease via different levels of confidence. DTO also contains drug target development level classifications developed in the IDG project (http://targetcentral.ws/), and functional and qualitative annotations and classifications for kinase proteins, GPCR ligands and ion channels. DTO is modeled in OWL2-DL to enable further classification by inference reasoning and SPARQL queries. DTO is implemented following a modularization approach. DTO is used as the organizational framework for drug targets in the IDG PHAROS User Interface Portal (https://pharos.nih.gov) and also the Tin-X Target Importance and Novelty Explorer (http://newdrugtargets.org). | 1.1.1 | beta | DTO (TISSUE), U54CA189205 (GENE_OR_GENE_PRODUCT), NCI (CELL), GPCRs (GENE_OR_GENE_PRODUCT), nuclear hormone receptors (GENE_OR_GENE_PRODUCT), tissue (TISSUE), disease (CANCER) |
| Glycomics Ontology | GLYCO | The GlycO ontology provides an up-to-date knowledge base of experimentally verified glycan structures. Glycan (oligosaccharide or polysaccharide) structures are represented as trees of monosaccharide residues. Linkage to proteins and lipids is supported as well. Insertion of a new glycan is controlled by curation process that includes matching the new glycan against a canonical glyco-tree (a highly branched representation for a family of glycans). | 4.6.0.0 | production | - |
| Ontology of chemically concrete, composable DNA, RNA, and protein residue and crosslinks | BPFORMS | BpForms is an ontology of molecularly-concrete DNA, RNA, and protein residues and crosslinks which can be composed into molecularly-concrete forms of DNA, RNA, proteins, and complexes. The ontology can be used with the BpForms grammar and software tools for polymers and the BcForms grammar and software tools for complexes. See https://www.bpforms.org and https://www.bcforms.org for more information. | 0.0.9 | beta | DNA (CELLULAR_COMPONENT), https://www.bcforms.org (GENE_OR_GENE_PRODUCT) |
| Chemical Entities of Biological Interest Ontology | CHEBI | A structured classification of chemical compounds of biological relevance. | 241 | production | - |
| BioAssay Ontology | BAO | The BioAssay Ontology (BAO) describes chemical biology screening assays and their results including high-throughput screening (HTS) data for the purpose of categorizing assays and data analysis.  BAO is an extensible, knowledge-based, highly expressive (currently SHOIQ(D)) description of biological assays making use of descriptive logic based features of the Web Ontology Language (OWL).  BAO currently has over 1000 classes and also makes use of several other ontologies.  It describes several concepts related to biological screening, including Perturbagen, Format, Meta Target, Design,  Detection Technology, and Endpoint.  Perturbagens are perturbing agents that are screened in an assay; they are mostly small molecules. Assay Meta Target describes what is known about the biological system and / or its components interrogated in the assay (and influenced by the Perturbagen).  Meta target can be directly described as a molecular entity (e.g. a purified protein or a protein complex), or indirectly by a biological process or event (e.g. phosphorylation).  Format describes the biological or chemical features common to each test condition in the assay and includes biochemical, cell-based, organism-based, and variations thereof.  The assay Design describes the assay methodology and implementation of how the perturbation of the biological system is translated into a detectable signal.  Detection Technology relates to the physical method and technical details to detect and record a signal.  Endpoints are the final HTS results as they are usually published (such as IC50, percent inhibition, etc.).  BAO has been designed to accommodate multiplexed assays.  All main BAO components include multiple levels of sub-categories and specification classes, which are linked via object property relationships forming an expressive knowledge-based representation. | 2.8.16 | production | SHOIQ(D (GENE_OR_GENE_PRODUCT) |
| ChemoOnto | CHO | ChemoOnto, an ontology to qualify the course of chemotherapies | Version: 1.0.1 | alpha | - |
| Syrian Drugs and medicines | SYRIANDRUGS | This ontology is to represent Syrian drugs | 0.01 | alpha | - |
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Chemical Information Ontology | CHEMINF | Includes terms for the descriptors commonly used in cheminformatics software applications and the algorithms which generate them. | 2.1.0 | production | - |
| Exposure Value Set | EVS | A collection of curated and standardized values used in metadata specifications produced by ADAMS (Accelerating DAta and Metadata Standards in the Environmental Health Sciences Study of Emerging Water Contaminants) group. These values are designed to ensure uniformity in the description of the various metadata standards produced by the group.  | 0.0.6 | beta | collection (32644) |
| Chemical Methods Ontology | CHMO | CMO, the chemical methods ontology, describes methods used to collect data in chemical experiments, such as mass spectrometry and electron microscopy, prepare and separate material for further analysis, such as sample ionisation, chromatography, and electrophoresis, synthesise materials, such as epitaxy and continuous vapour deposition. It also describes the instruments used in these experiments, such as mass spectrometers and chromatography columns. It is intended to be complementary to the Ontology for Biomedical Investigations (OBI). | 2025-01-22 | beta | CMO (GENE_OR_GENE_PRODUCT) |
| G Protein-Coupled Receptor BioAssays Ontology | BAO-GPCR | The G protein-coupled receptors (GPCRs) ontology (http://www.bioassayontology.org/bao_gpcr) describes pharmacology, biochemistry and physiology of these important and therapeutically promising class of academic and pharmaceutical research targets. Incorporation and comparison of various small molecule screening data sets, such as those deposited in PubChem, ChEMBL, KEGG, PDSP, and/or IUPHAR databases, requires a formalized electronic organization system. In order to bridge the gap between the overflow of HTS data and the bottleneck of integrated analysis tools, herein, we provide the first comprehensive GPCR ontology. The development and utility of GPCR ontology was based on previously developed BioAssay Ontology (BAO). The GPCR ontology contains information about biochemical, pharmacological, and functional properties of individual GPCRs as well as GPCR-selective ligands inclusive of their HTS screening results and other records. This provides the first all-inclusive GPCR ontology with all available data to model the relationship between the GPCR binding sites and their physiologic and pharmacologic role in physiology via small molecule chemical structures. We developed this system using emerging semantic technologies, by leveraging existing and descriptive domain level ontologies. | 1.1 | beta | GPCRs (GENE_OR_GENE_PRODUCT), all (1) |
| Ontology of Nuclear Toxicity | ONTOTOXNUC | Ontology of nuclear toxicity developed during ToxNuc project. | 1.0 | alpha | nuclear (CELLULAR_COMPONENT) |
| NMR-Controlled Vocabulary | NMR | This is the new msi-sanctioned nmr CV, created within the COSMOS EU project, to support the nmrML data standard for nuclear magnetic resonance data in metabolomics with meaningful raw data descriptors. | 1.1.0 | beta | - |
| Context and Measurement Ontology | COMO | Ontological terms to describe experimental data, probably useful for other environmental microbiology projects.  Used in several CORAL (Contextual Ontology-based Repository Analysis Library) deployments.  See more about this platfom at https://github.com/jmchandonia/CORAL |  | beta | - |
| NanoParticle Ontology | NPO | An ontology that represents the basic knowledge of physical, chemical and functional characteristics of nanotechnology as used in cancer diagnosis and therapy. | 2011-12-08 (yyyy-mm-dd) | production | cancer (6754) |
| Venom Ontology | CU-VO | An ontology of venoms and venom-related compounds, designed to facilitate the discovery of novel venom-derived therapeutics. | 1.0 | production | venom-derived (MULTI_TISSUE_STRUCTURE) |
| OriGinal Chemical Ontology | OGCO | The Original Chemical Ontology was developed by Michael Ashburner & Pankaj Jaiswal in 2002, before development shifted to CHEBI. It was revived by Chris Mungall for purposes of comparison with the current CHEBI and CHEMESSENCE |  | alpha | - |
| eNanoMapper | ENM | The eNanoMapper ontology covers the full scope of terminology needed to support research into nanomaterial safety. It builds on multiple pre-existing external ontologies such as the NanoParticle Ontology.  | 10.0 | beta | - |
| Anatomical Therapeutic Chemical Classification | ATC | The Anatomical Therapeutic Chemical (ATC) Classification System is used for the classification of active ingredients of drugs according to the organ or system on which they act and their therapeutic, pharmacological and chemical properties. It is controlled by the World Health Organization Collaborating Centre for Drug Statistics Methodology (WHOCC), and was first published in 1976.  | ATC_2024_01_19_24_09_03 | production | organ (ORGAN) |
| Biolink Model | BIOLINK | A high level datamodel of biological entities (genes, diseases, phenotypes, pathways, individuals, substances, etc) and their associations | 2.2.13 | production | - |
| Functional Therapeutic Chemical Classification System | FTC | The Functional Therapeutic Chemical Classification System (FTC) defines over 20'000 mechanisms and modes of action for approuved drugs. The resource abstracts away from the traditional chemical structure-based approach and focuses solely on the mode of action of drugs. The classification sorts therapeutic agents based on their function in the human body. | 1 | production | human (9606) |
| In Silico Methodologies Ontology for MAATrica metric | SIMM | MAATrica is a novel metric for assessing the coherence of methodologies in research papers within the fields of medicinal and nutraceutical chemistry. MAATrica employs user-controlled and customizable ontologies (in silico, in vitro and in vivo), enabling personalized analysis of research papers in the fields of medicinal and nutraceutical chemistry. |  | alpha | - |
| Base de Données Publique des Médicaments | BDPM | LA BDPM est un des référentiels du médicament gérée et distribuée par l’ANSM. Elle recense l’ensemble des spécialités et des présentations effectivement commercialisées durant les 5 dernières années. De ce fait, elle ne propose pas un historique exhaustif des codes de présentation et spécialités, celui-ci peut être obtenu grâce à la base RCP disponible sur le site de l’ANSM. | 2025-03-04 | production | - |
| Ontology of Physics for Biology | OPB | The OPB is a reference ontology of classical physics and thermodynamics as applied to the dynamics of biological systems. It is designed to encompass the multiple structural scales (atoms to organisms) and multiple physical domains (fluid dynamics, chemical kinetics, particle diffusion, etc.) that are encountered in the study and analysis of biological organisms. | 1.2 | beta | OPB (GENE_OR_GENE_PRODUCT) |
| Name Reaction Ontology | RXNO | RXNO, the name reaction ontology, connects organic name reactions such as the Diels-Alder cyclization and the Cannizzaro reaction to their roles in an organic synthesis.  | releases/2021-12-16 | beta | - |
| A formal representation of the knowledge within the domain of Isotopes for Food Science | ISO-FOOD | To link and harmonize different knowledge repositories with respect to isotopic data, we propose an ISO-FOOD ontology as a domain ontology for describing isotopic data within Food Science. The ISO-FOOD ontology consists of metadata and provenance data that needs to be stored together with data elements in order to describe isotopic measurements with all necessary information required for future analysis. The new domain has been linked with existing ontologies, such as Units of Measurements Ontology, Food, Nutrient and the Bibliographic Ontology. So linked ontology consists of several superclasses, and for each of these the relevant subclasses, object properties and data properties are defined. To show how such an ontology can be used in practice, it was populated with 20 isotopic measurements of Slovenian food samples. Describing data in this way offers a powerful technique for organizing and sharing stable isotope data across Food Science.  |  | alpha | all (1) |
| CHEMROF | CHEMROF | CHEMROF is an ontological framework for Chemical Entities Mixtures Reactions Ontological Framework. |  | beta | - |
| Allotrope Merged Ontology Suite | AFO | The AFO is an ontology suite that provides a standard vocabulary and semantic model for the representation of laboratory analytical processes. The AFO suite is aligned at the upper layer to the Basic Formal Ontology (BFO). The core domains modeled include: Equipment, Material, Process, and Results. | REC/2025/06 | alpha | upper layer (ORGANISM_SUBDIVISION), Material (GENE_OR_GENE_PRODUCT) |

<a name="bioportal-category-development"></a>
## BioPortal Category "Development"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Data Collection Ontology | GDCO | The Data Collection Ontology (DCO) is an ontology designed for data collection providing classes and relations on top of the Basic Formal Ontology (BFO) to facilitate data collection processes, subjects, and data anaylsis through the use of classifiers. DCO is based on the notion of reasoning to validate incoming data through their definitions based on classifiers. | 1.0 | alpha | collection (32644) |
| LDbase | LDBASE | LDbase is an NIH-funded behavioral research data repository. The LDbase taxonomy includes project descriptors for projects in the fields of educational and developmental sciences. Developed as a collaboration between researchers and librarians, this first-of-its-kind project-oriented data repository contains decades of knowledge and includes data on individuals across the full range of abilities. The LDbase ontology is used to describe research projects in the developmental sciences.  | 0.0.1 | alpha | LDbase (CELL) |
| Xenopus Anatomy Ontology | XAO | A structured, controlled vocabulary of the anatomy and development of the African clawed frogs (Xenopus laevis and Xenopus tropicalis). | releases/2024-09-03 | production | frogs (8342), Xenopus laevis (8355), Xenopus tropicalis (GENE_OR_GENE_PRODUCT) |
| Radiation Oncology Ontology | ROO | The Radiation Oncology Ontology aims to cover the radiation oncology domain with a strong focus on re-using existing ontologies. | 0.5 | alpha | - |
| Planarian Anatomy and Developmental Stage Ontolgoy | PLANA | Anatomy ontology for planaria and terms specific to the developmental stages of the planarian Schmidtea mediterranea. | 2023-03-13 | production | planarian Schmidtea mediterranea (CANCER) |
| Drosophila Development Ontology | FB-DV | A structured controlled vocabulary of the development of Drosophila melanogaster. | fbdv/releases/2025-05-29 | production | Drosophila melanogaster (7227) |
| Ontology of Craniofacial Development and Malformation | OCDM | The Ontology of Craniofacial Development and Malformation (OCDM) is a mechanism for representing knowledge about craniofacial development and malformation, and for using that knowledge to facilitate integrating craniofacial data obtained via multiple techniques from multiple labs and at multiple levels of granularity. The OCDM is a project of the NIDCR-sponsored FaceBase Consortium, whose goal is to promote and enable research into the genetic and epigenetic causes of specific craniofacial abnormalities through the provision of publicly accessible, integrated craniofacial data.  | 1.10.0 | production | Craniofacial (CANCER), craniofacial (PATHOLOGICAL_FORMATION), FaceBase (GENE_OR_GENE_PRODUCT) |
| Ontology for Newborn Screening Follow-up and Translational Research | ONSTR | Ontology for Newborn Screening Follow-up and Translational Research (ONSTR) is an application ontology covering the domain of newborn screening, follow-up and translational research pertaining to patients diagnosed with inheritable and congenital diseases mainly identified through newborn dried blood spot screening. ONSTR is a central component of the project Newborn Screening Follow-up Data Integration Collaborative (NBSDC). ONSTR uses the Basic Formal Ontology v2 (BFO2, v2012-07-20) as top-level ontology and extends the classes imported from OBO Foundry ontologies and candidate ontologies. | 0.4 | alpha | blood (ORGANISM_SUBSTANCE), NBSDC (CANCER), BFO2 (GENE_OR_GENE_PRODUCT), v2012-07 (GENE_OR_GENE_PRODUCT) |
| Epilepsy Ontology | EPILONT | Ontology about the epilepsy domain and epileptic seizures. Based on the diagnosis proposed by the International League Against Epilepsy (ILAE). | V1 | beta | - |
| Ontology of Homology and Related Concepts in Biology | HOM | This ontology represents concepts related to homology, as well as other concepts used to describe similarity and non-homology. | releases/2015-01-07 | production | - |
| C. elegans Development Vocabulary | WB-LS | A structured controlled vocabulary of the development of Caenorhabditis elegans. | unknown | production | - |
| Mapping of Epilepsy Ontologies | MEPO | The Mapping of Epilepsy Ontologies (MEPO) provides owl:sameAs relationships of similar concepts for the ontologies EpSO, ESSO, EPILONT, EPISEM, and FENICS. Two concepts are considered similar if they share the same concept name or the same synonym. Additional synonyms for each of the concepts are generated using the Snowball stemming algorithm. The software implementation is available on github within the Maven project snoke in the module snoke.ontology: https://github.com/bernd-mueller/snoke/tree/master/snoke.ontology | 0.6 | alpha | MEPO (CANCER) |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |
| NeuroMorpho.Org brain region ontologies | NMOBR | The species, brain regions and cell types are integrated into a single ontology for enabling OntoSearch functionality to mine the latest v7.0 release of NeuroMorpho.Org. | 2.0 | beta | brain (ORGAN), cell (CELL) |

<a name="bioportal-category-dysfunction"></a>
## BioPortal Category "Dysfunction"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Bilingual Ontology of Alzheimer's Disease and Related Diseases | ONTOAD | OntoAD is a bilingual (English-French) domain ontology for modeling knowledge about Alzheimer's Disease and Related Syndromes. | 1.0 | beta | OntoAD (GENE_OR_GENE_PRODUCT) |
| Computer Assisted Brain Injury Rehabilitation Ontology | CABRO | a web ontology for the semantic representation of the computer assisted brain trauma rehabilitation domain. This is a novel and emerging domain, since it employs the use of robotic devices, adaptation software and machine learning to facilitate interactive, adaptive and personalized rehabilitation care, patient monitoring and assisted living.  | 0.5 | alpha | brain (ORGAN) |
| International Classification of Diseases for Oncology, 3rd edition | ICD-O-3 | Ontology used for oncology research derived from the European Cancer Information System (ECIS) | OCD-O Revision 3.2 (complete) | alpha | Cancer (CANCER) |
| Hearing Impairment Ontology | HIO | The Hearing Impairment Ontology (HIO) is a common controlled hearing impairment (HI) vocabulary, which is expected to enhance collaborative research. This ontology is currently the most comprehensive and standardized human- and machine-readable resource that unambiguously defines HI concepts and terminology for researchers, patients and clinicians in order to help process, reuse and re-apply existing HI knowledge in biomedical research and health-care systems. In the context of big data analytics, this ontology may facilitate a retrospective data harmonization and contribute to mapping HI datasets to functional knowledge to enable the subsequent HI research translation into clinical applications and policy guidelines. The HIO will allow researchers, clinicians and patients to readily access standardized HI-related knowledge in a single location and promote HI data integration, interoperability and sharing, including epidemiological, socio-environmental, biomedical, genetic and phenotypic datasets. | 1 | production | - |
| Neuroscience Information Framework (NIF) Standard Ontology | NIFSTD | NIF Standard ontology (NIFSTD) is a core component of Neuroscience Information Framework (NIF) project (http://neuinfo.org), a semantically enhanced portal for accessing and integrating neuroscience data, tools and information. NIFSTD includes a set of modular ontologies that provide a comprehensive collection of terminologies to describe neuroscience data and resources. | 3.1 - January 15, 2018 | production | portal (TISSUE), collection (32644) |
| Ontology for Newborn Screening Follow-up and Translational Research | ONSTR | Ontology for Newborn Screening Follow-up and Translational Research (ONSTR) is an application ontology covering the domain of newborn screening, follow-up and translational research pertaining to patients diagnosed with inheritable and congenital diseases mainly identified through newborn dried blood spot screening. ONSTR is a central component of the project Newborn Screening Follow-up Data Integration Collaborative (NBSDC). ONSTR uses the Basic Formal Ontology v2 (BFO2, v2012-07-20) as top-level ontology and extends the classes imported from OBO Foundry ontologies and candidate ontologies. | 0.4 | alpha | blood (ORGANISM_SUBSTANCE), NBSDC (CANCER), BFO2 (GENE_OR_GENE_PRODUCT), v2012-07 (GENE_OR_GENE_PRODUCT) |
| Allergy Detector II | ALLERGYDETECTOR | Allergy Detection Ontology. | 0.0.1 | alpha | Allergy Detector II (GENE_OR_GENE_PRODUCT) |
| HIVOntology | HIVO004 | A Project of Biomedical Ontology for Systems Biomedicine. The goals of the Project include: (1). To pilot the novel researches on AIDS and HIV or other diseases by viruses, by means of the ontology and other biomedical informatics technologies, connecting the insights of the Biomedical Knowledge Integration (BMKI), systems biomedicine, complexity sciences etc. (2). To Carry out the methodology explorations for the goals above. | 020 | alpha | HIV (57667), viruses (10239) |
| Toxic Process Ontology | TXPO | TOXic Process Ontology (TXPO) includes entities related to toxic mechanisms (toxic courses) and processes, mainly focusing on hepatotoxicity. | 2022/12/07 | beta | - |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |
| Homeostasis imbalance process ontology | HOIP | Homeostasis imbalance process ontology (HoIP) is an ontology that systematizes a wide variety of terms involving homeostasis imbalance courses and processes. HoIP focuses on the course of COVID-19 infectious processes and cellular senescence.  | 2024/04/03 | alpha | cellular (CELL) |

<a name="bioportal-category-experimental-conditions"></a>
## BioPortal Category "Experimental Conditions"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| New Approach Methodology Ontology | NAMO | The New Approach Methodology Ontology and Schema (NAMO) is a comprehensive data framework designed to standardize and integrate information about alternative methods to animal testing in biomedical research. Built on LinkML, NAMO provides a common vocabulary and data structure for describing diverse model systems, from cellular cultures and organ-on-chip devices to computational models and digital twins. |  | production | Schema (GENE_OR_GENE_PRODUCT), cellular cultures (CELL), twins (CANCER) |
| The Biological and Environmental Research Variable Ontology | BERVO | The Biological and Environmental Research Variable Ontology (BERVO) is a model of the experimental variables, conditions, and concepts used in the study of environmental research, including studies in earth science, plant science, and geochemistry. | 2025-08-05 | alpha | - |
| Cognitive Task Ontology | COGITO | CogiTO relates Tasks from Cognitive Atlas to Hierarchical Event Descriptor (HED) tags via logical relationships (as equivalent class axioms) between concepts. In other words, terms for cognitive tasks are defined by their relation to HED tags. The purpose of this ontology is to enable querying partially annotated datasets either by a Cognitive Task or by HED tags.  | 1.0.1 | beta | - |
| In Vitro Methodologies Ontology for MAATrica metric | VIMM | MAATrica is a novel metric for assessing the coherence of methodologies in research papers within the fields of medicinal and nutraceutical chemistry. MAATrica employs user-controlled and customizable ontologies (in silico, in vitro and in vivo), enabling personalized analysis of research papers in the fields of medicinal and nutraceutical chemistry. Here we present the in vitro ontology we developed for the scope. |  | alpha | - |
| Space Life Sciences Ontology | SLSO | The Space Life Sciences Ontology is an application ontology and is intended to support the operation of NASA's Life Sciences Data Archive (located at https://nlsp.nasa.gov) and other systems that contain space life science research data. Many kinds of scientific research in space involve specialized equipment, experimental procedures, specimens and specimen collection apparatus, supporting operational structures, and references to specific space environments and their characteristics. Often such research is extremely costly compared with planetary/terrestrial investigations, and the community supporting space life sciences is accustomed to the use of various specialized concepts and terminologies when dealing with this kind of research and data. However, the underlying conceptual models (and especially the specific labeled relationships in these models) have never been made explicit. The SLSO attempts to provide definition and organization of these models for the space research community. The SLSO was developed ab inicio using the Ontology Development Kit and imports an extends many concepts from the Basic Formal Ontology (BFO), the Ontology of Biomedical Investigations (OBI), the Environmental Ontology (ENVO), and other OBO Foundry ontologies. Projects at NASA such as the Open Science Data Repository (https://osdr.nasa.gov/) are already using many OBO ontologies, including the Radiation Biology Ontology (https://github.com/Radiobiology-Informatics-Consortium/RBO) and OBI, to index space biology investigation data. With the development of the SLSO, this practice can be extended to include all life science research in space or addressing space effects. Furthermore, the SLSO has a component that imports concepts from the Science Data Discovery Ontology, which was developed to support NASA's Science Discovery Engine (https://sciencediscoveryengine.nasa.gov). These links in the imported SDDO to concepts underlying a broad spectrum of space research (astrophysics, heliophysics, etc.) can ultimately be used to provide key capabilities for discovering and analyzing space life science data and how they relate to other kinds of scientific data regarding space environments., The Space Life Sciences Ontology is an application ontology and is intended to support the operation of NASA's Life Sciences Data Archive and other systems that contain space life science research data. | 2025-08-18 | production | specimens (CELL), collection (32644), planetary/terrestrial (MULTI_TISSUE_STRUCTURE), SLSO (CANCER), all (1) |
| Cell Line Ontology | CLO | The Cell Line Ontology (CLO) is a community-based ontology in domain of biological cell lines with a focus on permanent cell lines from culture collections. Upper ontology structures that frame the skeleton of CLO include Basic Formal Ontology and Relation Ontology. Cell lines contained in CLO are associated with terms from other ontologies such as Cell Type Ontology, NCBI Taxonomy, and Ontology for Biomedical Investigation. A common design pattern for the cell line is used to model cell lines and their attributes, the Jurkat cell line provides ane xample.  Currently CLO contains over 36,000 cell line entries obtained from ATCC, HyperCLDB, Coriell, and bymanual curation. The cell lines are derived from 194 cell types, 656 anatomical entries, and 217 organisms. The OWL-based CLO is machine-readable and can be used in various applications. | 2.1.188 | production | Cell Line (CELL), cell lines (CELL), Cell lines (CELL), Cell Type (CELL), cell line (CELL), Jurkat cell line (CELL), ATCC (CELL), Coriell (CELL), cell (CELL) |
| Software Ontology | SWO | The Software Ontology (SWO) describes software used in research, primarily Bioinformatics. The SWO covers areas such as the software type, licence, manufacturer of the software, the input and output data types and the uses (i.e. objectives) to which the software can be put. | 2023-03-05 | production | - |
| Zebrafish Experimental Conditions Ontology | ZECO | This ontology is designed to represent the experimental conditions applied to zebrafish and has been developed to facilitate experiment condition annotation at ZFIN. | 2022-02-14 | production | zebrafish (7955), ZFIN (CANCER) |
| Ontology for Genetic Disease Investigations | OGDI | It is the ontology used to model the scientific investigation, especially  Genome-Wide Association Study (GWAS),  to find out  genetic susceptibility factor to disease, such as Diabetes. It models the genetic varaints, polymorphisms, statistical measurement, populations and other elements that are essential to determine a genetic susceptibility factor in GWAS study. It must be used with other two ontologies, in the case of Diabetes, :Ontology of Geographical Region (OGR) and Ontology of Glucose Metabolism Disorder (OGMD) . | 1.0 | alpha | - |
| Ontology of RNA Sequencing | ORNASEQ | This is an application ontology used to annotate next-generation sequencing experiments performed on RNA (RNAseq). It uses terms from BFO, CHEBI, CL, GENEPIO, GO, IAO, NCBITaxon, NCIT, OBI, OBIws, OGMS, OMIABIS, SO, TAXRANK, and UBERON as well as EFO. |  | production | CL (MULTI_TISSUE_STRUCTURE), SO (GENE_OR_GENE_PRODUCT) |
| BioMedical Study Lifecycle Management | BMS-LM | Biomedical research data reuse and sharing is essential for fostering research progress. To this aim, data producers need to master data management and reporting through standard and rich metadata, as encouraged by open data initiatives such as the FAIR (Findable, Accessible, Interoperable, Reusable) guidelines. This helps data re-users to understand and reuse the shared data with confidence. Therefore, dedicated frameworks are required. The provenance reporting throughout a biomedical study lifecycle has been proposed as a way to increase confidence in data while reusing it. The Biomedical Study - Lifecycle Management (BMS-LM) data model has implemented provenance and lifecycle traceability for several multimodal-imaging techniques but this is not enough for data understanding while reusing it. Actually, in the large scope of biomedical research, a multitude of metadata sources, also called Knowledge Organization Systems (KOSs), are available for data annotation. In addition, data producers uses local terminologies or KOSs, containing vernacular terms for data reporting. The result is a set of heterogeneous KOSs (local and published) with different formats and levels of granularity. To manage the inherent heterogeneity, semantic interoperability is encouraged by the Research Data Management (RDM) community. Ontologies, and more specifically top ontologies such as BFO and DOLCE, make explicit the metadata semantics and enhance semantic interoperability. Based on the BMS-LM data model and the BFO top ontology, the BioMedical Study - Lifecycle Management (BMS-LM) core ontology is proposed together with an associated framework for semantic interoperability between heterogeneous KOSs. It is made of four ontological levels: top/core/domain/local and aims to build bridges between local and published KOSs. In this paper, the conversion of the BMS-LM data model to a core ontology is detailed. The implementation of its semantic interoperability in a specific domain context is explained and illustrated with examples from small animal preclinical research. | Version 1.1 | production | - |
| Microbial Isolation Source Ontology | MISO | This ontology aims to classify and tag microbial isolation sources, which are often only provided as free-text commentaries, to improve their accessibility and usability. |  | production | - |
| Tissue Microarray Ontology | TMA | Tissue microarrays (TMA) are enormously useful tools for translational research, but incompatibilities in database systems between various researchers and institutions prevent the efficient sharing of data that could help realize their full potential.  The Resource Description Framework (RDF) provides a flexible method to represent knowledge in triples, which take the form Subject-Predicate-Object.  All data resources are described using Uniform Resource Identifiers (URI), which are global in scope.  We present an OWL (Web Ontology Language) schema that expands upon the TMA data exchange specification to address this issue and assist in data sharing and integration. | 1.0.0 | alpha | TMA (CANCER) |
| Ontology for Biomedical Investigations | OBI | OBI is an ontology of investigations, the protocols and instrumentation used, the material used, the data generated and the types of analysis performed on it. | 2025-05-01 | production | - |
| LinkML Common Metadata Elements and Terms | COMET | A collection of common metadata elements and terms for constructing LinkML Schemas and Data Models |  | alpha | collection (32644) |
| Experimental Factor Ontology | EFO | The Experimental Factor Ontology (EFO) is an application focused ontology modelling the experimental variables in multiple resources at the EBI and Open Targets. The ontology has been developed to increase the richness of the annotations that are currently made in resources and to promote consistent annotation, to facilitate automatic annotation and to integrate external data. The ontology pulls together classes from reference ontologies such as disease, cell line, cell type and anatomy and adds axiomatisation as necessary to connect areas such as disease to phenotype. Contact EFO users list for information: efo-users@ebi.ac.uk. | 3.79.0 | production | cell line (CELL), cell (CELL) |
| Brain-Computer Interaction (BCI) Ontology | BCI-O | The BCI ontology specifies a foundational metadata model set for real-world multimodal Brain-Computer Interaction (BCI) data capture activities.  Its structure depicts a conceptual framework that BCI applications can extend and use in their implementations, to define core concepts that capture a relevant and interoperable metadata vocabulary. This ontology is aligned to the Semantic Sensor Network Ontology (SSN): a domain-independent and end-to-end model for sensor/actuator applications.  Hence, its structure has been normalized to assist its use in conjunction with other ontologies or linked data resources to specify any particular definitions (such as units of measurement, time and time series, and location and mobility), that specialized applications in the BCI domain might need. Its spec provides general alignment data modeling guidelines for core concepts, to help BCI applications in their design. | 0.9.6 | beta | - |
| Human Ancestry Ontology | HANCESTRO | The Human Ancestry Ontology (HANCESTRO) provides a systematic description of the ancestry concepts used in the NHGRI-EBI Catalog of published genome-wide association studies. | 2025-04-01 | beta | Human (9606) |
| Evidence Graph Ontology | EVI | The Evidence Graph ontology extends core concepts from the W3C Provenance Ontology PROV-O and Bioschemas' Profiles to describe evidence for correctness of findings in biomedical publications. The semantic data model in EVI is expressed using OWL2 Web Ontology Language. | 1.1 | alpha | - |
| Home Cage Monitoring definition | HCM-D | From COST-TEATIME (https://www.cost-teatime.org/) | v0.1 | alpha | - |
| Cancer Chemoprevention Ontology | CANCO | The Cancer Chemoprevention Ontology constitutes a vocabulary that is able to describe and semantically interconnect the different paradigms of the cancer chemoprevention domain. | 0.3 | production | Cancer (CANCER), cancer (6754) |
| Exposure Value Set | EVS | A collection of curated and standardized values used in metadata specifications produced by ADAMS (Accelerating DAta and Metadata Standards in the Environmental Health Sciences Study of Emerging Water Contaminants) group. These values are designed to ensure uniformity in the description of the various metadata standards produced by the group.  | 0.0.6 | beta | collection (32644) |
| Just Enough Results Model Ontology | JERM | The JERM ontology is an application ontology that describes the relationships between the data, models and other research assets that can be shared through the FAIRDOM platform, for sharing Systems and Synthetic Biology experiments. JERM 2 includes a new model for describing Biosamples, developed during the ELIXIR-lead Biosamples club | 2 | beta | FAIRDOM (GENE_OR_GENE_PRODUCT) |
| Plant Experimental Conditions Ontology | PECO | The Plant Experimental Conditions Ontology describes the treatments, growing conditions, and/or study types used in plant biology experiments. The subjects of the studies may also include in vitro plant structures (PO:0000004). A plant treatment is the most specific,and may be a component of a plant growing condition, and the plant study type. The broadest classes are the plant study types where the terms can be used to identify the growth study facility. Each growth facility such as field study, growth chamber, green house etc is a environment on its own and it may also involve instances of biotic and abiotic treatments and different growing conditions. | releases/2023-11-10 | production | - |
| Biological Imaging Methods Ontology | FBbi | A structured controlled vocabulary of sample preparation, visualization and imaging methods used in biomedical research. |  | production | - |
| NeuroBridge Ontology | NEUROBRG | NeuroBridge ontology is being developed as part of the NeuroBridge project to implement FAIR principles for neuroimaging and neurophysiology data using computable provenance metadata. The NeuroBridge ontology extends the W3C PROV ontology and the ProvCaRe ontology with terms from the SchizConnect and Data Bridge ontology aligned and merged into the original class structure of the ProvCaRe ontology.  | 1.0 (Release 2023) | production | SchizConnect (GENE_OR_GENE_PRODUCT) |
| Ontology of Host-Microbe Interactions | OHMI | OHMI is a biomedical ontology that represents the entities and relations in the domain of host-microbiome interactions. | 2019-09-17 | production | - |
| EDDA Publication Types Taxonomy | EDDA_PT | The EDDA Publication Types Taxonomy (v1.0) was developed by Tanja Bekhuis (Principal Scientist) and Eugene Tseytlin (Systems Developer) of the Evidence in Documents, Discovery, and Analytics (EDDA) Group. The EDDA Group is a division of TCB Research & Indexing LLC. Variants include synonyms for preferred terms, singular and plural forms, and American and British spellings. Definitions, if they exist, are mainly from MeSH, NCIT, Emtree, and dictionaries. | 1.0 | beta | Eugene Tseytlin (GENE_OR_GENE_PRODUCT) |
| Statistics Ontology | STATO | STATO is a general-purpose STATistics Ontology. Its aim is to provide coverage for processes such as statistical tests, their conditions of applications, and information needed or resulting from statistical methods, such as probability distributions, variable, spread and variation metrics. STATO also covers aspects of experimental design and description of plots and graphical representations commonly used to provide visual cues of data distribution or layout and to assist review of the results. | 2025-02-23 | production | STATO (CANCER), general-purpose (GENE_OR_GENE_PRODUCT) |
| Breast Cancer-Microbiome Ontology  | BCMO | The aim of BCMO is to construct a comprehensive and standardized data collection method to study the interaction between the microbiome and breast cancer, including clinical data, pathology, epidemiology, demographic characteristics and lifestyle of breast cancer. Through standardized methods, research results in related research fields can be structured, shareable and comparable. It will contribute to the study of how the microbiome interacts with breast cancer. |  | alpha | BCMO (GENE_OR_GENE_PRODUCT), collection (32644), cancer (6754) |
| Neural ElectroMagnetic Ontology | NEMO | Neural ElectroMagnetic Ontology (NEMO) describes classes of event-related brain potentials (ERP) and their properties, including spatial, temporal, and functional (cognitive/behavioral) attributes, and data-level attributes (acquisition and analysis parameters). Working (unstable), Release (stable), and Old (archival) versions of NEMO can be found on sourceforge: http://sourceforge.net/projects/nemoontologies/. | v3.10 | production | Neural ElectroMagnetic (MULTI_TISSUE_STRUCTURE), brain (ORGAN), ERP (CANCER) |
| Context and Measurement Ontology | COMO | Ontological terms to describe experimental data, probably useful for other environmental microbiology projects.  Used in several CORAL (Contextual Ontology-based Repository Analysis Library) deployments.  See more about this platfom at https://github.com/jmchandonia/CORAL |  | beta | - |
| Experimental Conditions Ontology | XCO | The Experimental Conditions Ontology is designed to represent the conditions under which physiological and morphological measurements are made both in the clinic and in studies involving humans or model organisms. | 4.205 | production | humans (9605) |
| Microbial Conditions Ontology | MCO | Microbial Conditions Ontology contains terms to describe growth conditions in microbiological experiments. The first version is based on gene regulation experiments in Escherichia coli K-12. It is being used in RegulonDB to link growth conditions to gene regulation data. |  | production | Escherichia coli K-12 (83333) |
| Cell Culture Ontology | CCONT | CCONT is an ontology for the formal representation of cell lines and their correspnding culture conditions. | 1.1 | beta | cell lines (CELL) |
| Environment Ontology | ENVO | EnvO is an OBO Foundry and Library ontology for the concise, controlled description of environmental entities such as ecosystems, environmental processes, and environmental qualities. It closely interoperates with a broad collection of other OBO ontologies and is used in a diverse range of projects.  | 2024-07-01 | production | EnvO (GENE_OR_GENE_PRODUCT), collection (32644) |
| Spinal Cord Injury Ontology | SCIO | Representation of pre-clinical studies in the domain of spinal cord injury therapies | 1 | alpha | spinal cord (ORGAN) |
| Comparative Data Analysis Ontology | CDAO | A formalization of concepts and relations relevant to evolutionary comparative analysis, such as phylogenetic trees, OTUs (operational taxonomic units) and compared characters (including molecular characters as well as other types). CDAO is being developed by scientists in biology, evolution, and computer science. | $Revision$ | production | OTUs (CELLULAR_COMPONENT) |
| EDDA Study Designs Taxonomy | EDDA | The EDDA Study Designs Taxonomy (v2.0) was developed by the Evidence in Documents, Discovery, and Analytics (EDDA) Group: Tanja Bekhuis (Principal Scientist); Eugene Tseytlin (Systems Developer); Ashleigh Faith (Taxonomist); Faina Linkov (Epidemiologist). The EDDA Group is a division of TCB Research & Indexing LLC. This work was made possible, in part, by the US National Library of Medicine, National Institutes of Health, grant no. R00LM010943. | 2.0 | beta | - |
| Minimal Information about any Sequence Ontology | MIXS | An OWL representation of the Minimum Information for any (x) Standard (MIxS), managed by the Genomic Standards Consortium.  | 6.2.0 | beta | OWL (GENE_OR_GENE_PRODUCT) |
| Ontology for Nutritional Studies | ONS | Ontology For Nutritional Studies (ONS) is the first systematic effort to provide a formal ontology framework for the description of nutritional studies. | 2024-07-03 | production | ONS (CANCER) |
| Mass Spectrometry Ontology | MS | A structured controlled vocabulary for the annotation of experiments concerned with proteomics mass spectrometry. Developed by the HUPO Proteomics Standards Initiative (PSI). | 4.1.195 | alpha | - |
| Microarray and Gene Expression Data Ontology | MO | Concepts, definitions, terms, and resources for standardized description of a microarray experiment in support of MAGE v.1. | 1.3.1.1 | alpha | MAGE (GENE_OR_GENE_PRODUCT) |
| Epoch Clinical Trial Ontology | CTONT |  | 0.97 | production | - |

<a name="bioportal-category-fish-anatomy"></a>
## BioPortal Category "Fish Anatomy"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Fish Ontology | FISHO | This Fish Ontology is an ontology created as how the author views the fish structure following the book "The Diversity of Fishes: Biology, Evolution and Ecology" as the main reference, which covers one part of view on ichthyology, with an emphasis on diversity and adaptation. This ontology is created with the mindset of categorizing fish automatically based on the attributes and terms mined from the fish specimen. Most of the basic organization from this ontology follows Nelson's research due to its synthetic and broad approach, while recognizing that Nelson's conclusions are one of many alternative interpretations of the literature, which is mostly agreed by our main references, and many fish and fisheries researchers. Most of the classification used in this ontology is in no means completed and just following the recent classification provided by the books. For now our work focused more on two group of fish which is early jawless fish, and advanced jawed fishes. The classification from both of this group are structured following the reference. | Version 1.0.3 | beta | Fish (118072), fish (118072), Fishes (118072), synthetic (32630), advanced jawed (ANATOMICAL_SYSTEM), fishes (118072) |
| Teleost Anatomy Ontology | TAO | TAO is a multispecies anatomy ontology for teleost fishes. Originally seeded from ZFA, but intended to cover terms relevant to other taxa. | 2012-08-10 | production | TAO (GENE_OR_GENE_PRODUCT), teleost fishes (32443) |
| Vertebrate Homologous Organ Group Ontology | VHOG | vHOG is a multi-species anatomical ontology for the vertebrate lineage, developed as part of the Bgee project. This mapping represents homology-strict relationships, in the sense of HOM:0000007 "historical homology". Each mapping has been manually reviewed, and we provide confidence codes and references when available. | unknown | production | - |
| Zebrafish Anatomy and Development Ontology | ZFA | A structured controlled vocabulary of the anatomy and development of the Zebrafish (Danio rerio). | releases/2025-01-28 | production | Zebrafish (7955), Danio rerio (7955) |
| Salmon Ontology | SALMON | An ontology representing knowledge about salmon, features of their habitats, salmon stakeholders, and related entities. This ontology is considered to be in the early stages of development, though it is based upon considerable previous work. Contributions of all kinds are welcome and encouraged, including alignments, updates to existing terms, and new terms. | 0.4.0 | beta | all (1) |
| Vertebrate Skeletal Anatomy Ontology | VSAO | VSAO is an anatomy ontology covering the vertebrate skeletal system. VSAO integrates terms for skeletal cells, tissues, biological processes, organs (skeletal elements such as bones and cartilages), and subdivisions of the skeletal system. | 2012-11-06 | production | skeletal (ORGAN), skeletal cells (CELL), tissues (TISSUE), organs (ORGAN), bones (TISSUE) |

<a name="bioportal-category-gene-product"></a>
## BioPortal Category "Gene Product"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| INRAE Thesaurus | INRAETHES | INRAE Thesaurus contains more than 16,000 concepts relevant to domains covered by the National Research Institute for Agriculture, Food and the Environment. Concepts are described with terms in French (100%) and English (81%) terms, some having textual definitions and mappings to other semantic artifacts. The concepts are organized in 12 domains subdivided in thematic sections called micro-thesauri., Thésaurus ouvert et partagé couvrant les domaines de recherche d’INRAE. Il sert de référentiel au sein de l’institut pour indexer et annoter des documents, pages web, descriptions d’activités, jeux de données, à des fins de recherche ou d’analyse de l’information. | 2.4 | production | INRAE (CANCER), fins (ORGAN) |
| Genomic Feature and Variation Ontology | GFVO | The Genomic Feature and Variation Ontology (GFVO) is modeled to represent genomic data using the Resource Description Format (RDF) or Linked Data using JSON (JSON-LD). It is captures the contents of data files that adhere to the Generic Feature Format Version 3 (GFF3, http://www.sequenceontology.org/resources/gff3.html), the General Transfer Format (GTF, http://mblab.wustl.edu/GTF22.html), the Genome Variation Format Version 1 (GVF, http://www.sequenceontology.org/resources/gvf.html), and the Variant Call Format (VCF, http://vcftools.sourceforge.net/specs.html), as well as the FASTA format (http://en.wikipedia.org/wiki/FASTA_format). The creation of the ontology was inspired by previous work of Robert Hoehndorf on RDF2OWL (http://code.google.com/p/rdf2owl). NOTE: GFVO in BioPortal is a simplified version without ontology imports and having class/property equivalences removed. The full ontology is available at: http://www.biointerchange.org/gfvo | 1.0.6 | production | GFVO (CANCER) |
| Orthologous Group Ontology | OGROUP | Ontology to help prossessing the information from orthologous groups in order to infer new functional annotations. The ontology integrates  definitions from diverse ontologies. | 1.0 | alpha | - |
| Pharmacogenomic Relationships Ontology | PHARE | The PHArmacogenomic RElationships Ontology (or PHARE) proposes concepts and roles to represent relationships of pharmacogenomics interest. | version id: 110114 | alpha | - |
| FlyGlycoDB | FLYGLYCODB | Ontology to describe glycogenes of Drosophila.  Includes mutants, activity, references and links to FlyBase and UniProt. | 1.0.0 | beta | Drosophila (2081351), FlyBase (GENE_OR_GENE_PRODUCT) |
| Drug Target Ontology | DTO | Drug Target Ontology (DTO) is being developed at the University of Miami in the research group of Stephan Schürer. DTO is supported by grant U54CA189205 (Illuminating the Druggable Genome Knowledge Management Center, Tudor Oprea, PI) awarded by the NCI through the NIH Common Fund. It is a component of the Illuminating the Druggable Genome (IDG) project (https://commonfund.nih.gov/idg). The DTO project develops a novel semantic framework to formalize knowledge about drug targets with a focus on the current IDG protein families. The DTO is developed as a reference for drug targets with the longer-term goal to create a community standard that will facilitate the integration of diverse drug discovery information from numerous heterogeneous resources. The first version of the DTO consists of asserted class hierarchies of the four IDG protein families, GPCRs, kinases, ion channels, and nuclear hormone receptors. Protein classes are linked to tissue and disease via different levels of confidence. DTO also contains drug target development level classifications developed in the IDG project (http://targetcentral.ws/), and functional and qualitative annotations and classifications for kinase proteins, GPCR ligands and ion channels. DTO is modeled in OWL2-DL to enable further classification by inference reasoning and SPARQL queries. DTO is implemented following a modularization approach. DTO is used as the organizational framework for drug targets in the IDG PHAROS User Interface Portal (https://pharos.nih.gov) and also the Tin-X Target Importance and Novelty Explorer (http://newdrugtargets.org). | 1.1.1 | beta | DTO (TISSUE), U54CA189205 (GENE_OR_GENE_PRODUCT), NCI (CELL), GPCRs (GENE_OR_GENE_PRODUCT), nuclear hormone receptors (GENE_OR_GENE_PRODUCT), tissue (TISSUE), disease (CANCER) |
| Ontological Knowledge Base Model for Cystic Fibrosis | ONTOKBCF | OntoKBCF is an ontological knowledge base model for cystic fibrosis. There are molecular genetic information (i.e. gene mutations) and health information included in OntoKBCF. The purposes of OntoKBCF include management of molecular genetic information and health information and embedding OntoKBCF into EHR settings. | 1301 | production | cystic (PATHOLOGICAL_FORMATION) |
| Therapeutic Antibody Ontology | TAON | TAON is a comprehensive ontology of the therapeutic antibody, which  included antibody terms and drug terms. | 1.1 | alpha | - |
| Ontology of chemically concrete, composable DNA, RNA, and protein residue and crosslinks | BPFORMS | BpForms is an ontology of molecularly-concrete DNA, RNA, and protein residues and crosslinks which can be composed into molecularly-concrete forms of DNA, RNA, proteins, and complexes. The ontology can be used with the BpForms grammar and software tools for polymers and the BcForms grammar and software tools for complexes. See https://www.bpforms.org and https://www.bcforms.org for more information. | 0.0.9 | beta | DNA (CELLULAR_COMPONENT), https://www.bcforms.org (GENE_OR_GENE_PRODUCT) |
| NIST Genome Editing Lexicon | NIST_GEL | Genome editing technology is a fast-growing and rapidly advancing global bioscience field with applications in many biotechnology sectors. Genome editing is used to modify the nucleic acids of a genetic code, which can be composed of DNA or RNA, in a site-specific manner. Modifications can include insertion, deletion or alteration of nucleic acids. The technology operates by biochemical principles generally applicable to every kind of cell. Examples of genome editing technology applications with global significance include human cell-based therapeutics, agriculture, microbial based therapeutics, synthetic biology and biomanufacturing. While genome editing technology is being actively utilized, there is a need for international standardization in terms and definitions for this field, so as to enhance interpretation and communication of concepts, data and results. This document has been developed to provide a unified standard set of terms and definitions that serve the needs of biotechnology stakeholders and act as a reference for genome editing technology. | 2.2 | production | DNA (CELLULAR_COMPONENT), insertion (2673), cell (CELL), genome (CELLULAR_COMPONENT), human (9606), synthetic (32630) |
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Ontology for SARS-CoV-2  lineages and mutations | SARSMUTONTO | The SARSMutOnto ontology provides a list of all SARS-CoV-2 Pango lineages while maintaining their hierarchy (lineage/sublineage) describing in detail all lineage mutations | 1.1 | alpha | all (1), Pango (CANCER) |
| Clusters of Orthologous Groups (COG) Analysis Ontology | CAO | CAO ontology is designed for supporting the COG enrichment study by using Fisher's exact test | 1.4 | production | Fisher (76720) |
| Functional Electrophysiology Nomenclature for Ion Channels | FENICS | The Functional Epilepsy Nomenclature for Ion Channels (FENICS) is allowing the consistent evaluation of electrophysiology experiments on Ion Channels in the context of neurological disorders, in particular epilepsy.  | 0.0.5 | alpha | Ion (TISSUE) |
| Chromatin Regulator Ontology | CHRO | Functional and biochemical description of covalent histone modifiers and chromatin remodeler proteins. | 2023-08-18 | alpha | histone (GENE_OR_GENE_PRODUCT), chromatin remodeler (GENE_OR_GENE_PRODUCT) |
| Genetic Feature Format Ontology | GFFO | An ontology to represent genome feature annotations in gff3 format. | 0.0.1 | alpha | - |
| Mapping of Epilepsy Ontologies | MEPO | The Mapping of Epilepsy Ontologies (MEPO) provides owl:sameAs relationships of similar concepts for the ontologies EpSO, ESSO, EPILONT, EPISEM, and FENICS. Two concepts are considered similar if they share the same concept name or the same synonym. Additional synonyms for each of the concepts are generated using the Snowball stemming algorithm. The software implementation is available on github within the Maven project snoke in the module snoke.ontology: https://github.com/bernd-mueller/snoke/tree/master/snoke.ontology | 0.6 | alpha | MEPO (CANCER) |
| Biolink Model | BIOLINK | A high level datamodel of biological entities (genes, diseases, phenotypes, pathways, individuals, substances, etc) and their associations | 2.2.13 | production | - |
| REGN_GO | REGN_GO | Provides structured controlled vocabularies for the annotation of gene products with respect to their molecular function, cellular component, and biological role. The Gene Ontology consists of three Vocabularies. | 2023-01-01 | production | cellular (CELL) |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |
| Ontology for MicroRNA Target | OMIT | The purpose of the OMIT ontology is to establish data exchange standards and common data elements in the microRNA (miR) domain. Biologists (cell biologists in particular) and bioinformaticians can make use of OMIT to leverage emerging semantic technologies in knowledge acquisition and discovery for more effective identification of important roles performed by miRs in humans' various diseases and biological processes (usually through miRs' respective target genes). OMIT has reused and extended a set of well-established concepts from existing bio-ontologies, e.g., Gene Ontology, Sequence Ontology, Protein Ontology, NCBI Organism Taxonomy, Human Disease Ontology, Foundational Model of Anatomy, and so forth. | OBO 4.0-12122013-2338 | alpha | cell (CELL), miRs (GENE_OR_GENE_PRODUCT), humans (9605), Human (9606) |
| Ontology of Genes and Genomes | OGG | OGG is a biological ontology in the area of genes and genomes. OGG uses the Basic Formal Ontology (BFO) as its upper level ontology. This OGG document contains the genes and genomes of a list of selected organisms, including human, two viruses (HIV and influenza virus), and bacteria (B. melitensis strain 16M, E. coli strain K-12 substrain MG1655, M. tuberculosis strain H37Rv, and P. aeruginosa strain PAO1). More OGG information for other organisms (e.g., mouse, zebrafish, fruit fly, yeast, etc.) may be found in other OGG subsets. | Vision Release: 1.0.59 | production | OGG (GENE_OR_GENE_PRODUCT), human (9606), viruses (10239), HIV (57667), influenza virus (11309), bacteria (629395), E. coli (562), mouse (10090), zebrafish (7955), fruit fly (103775) |

<a name="bioportal-category-genomic-and-proteomic"></a>
## BioPortal Category "Genomic and Proteomic"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Genomic Feature and Variation Ontology | GFVO | The Genomic Feature and Variation Ontology (GFVO) is modeled to represent genomic data using the Resource Description Format (RDF) or Linked Data using JSON (JSON-LD). It is captures the contents of data files that adhere to the Generic Feature Format Version 3 (GFF3, http://www.sequenceontology.org/resources/gff3.html), the General Transfer Format (GTF, http://mblab.wustl.edu/GTF22.html), the Genome Variation Format Version 1 (GVF, http://www.sequenceontology.org/resources/gvf.html), and the Variant Call Format (VCF, http://vcftools.sourceforge.net/specs.html), as well as the FASTA format (http://en.wikipedia.org/wiki/FASTA_format). The creation of the ontology was inspired by previous work of Robert Hoehndorf on RDF2OWL (http://code.google.com/p/rdf2owl). NOTE: GFVO in BioPortal is a simplified version without ontology imports and having class/property equivalences removed. The full ontology is available at: http://www.biointerchange.org/gfvo | 1.0.6 | production | GFVO (CANCER) |
| Ontology for Genetic Susceptibility Factor | OGSF | Ontology for Genetic Susceptibility Factor (OGSF) is an application ontology to model/represent the notion of genetic susceptibility to a specific disease or an adverse event or a pathological biological process. It is developed using BFO2.0's framwork. The ontology is under the domain of genetic epidemiology. | 2.0 | alpha | BFO2.0 (GENE_OR_GENE_PRODUCT) |
| Suggested Ontology for Pharmacogenomics | SOPHARM | SO-Pharm (Suggested Ontology for Pharmacogenomics) is a domain ontology implemented in OWL-DL, which proposes a formal description of pharmacogenomic knowledge. SO-Pharm articulates different ontologies that represent complementary sub-domains of pharmacogenomics, i.e. related to genotype, phenotype, drugs, and clinical trials. SO-Pharm enables to represent pharmacogenomic relationships between a drug, a genomic variation and a phenotype trait. In addition, it enables to represent patient and more largely panel included in trials, and populations. SO-Pharm enables the representation of measured items on this patients such as results from the observation of a phenotype trait or of genomic variations. SO-Pharm supports knowledge about pharmacogenomic hypothesis, case study, and investigations in pharmacogenomics. SO-Pharm is designed to facilitate data integration and knowledge discovery in pharmacogenomics. In addition it provides a consistent articulation of ontologies of pharmacogenomic sub-domains. | Version: SO-Pharm 2.2 Release: 02 Mar 2017 | beta | - |
| Neomark Oral Cancer Ontology, version 4 | NEOMARK4 | Ontology that describes the medical information necessary for early detection of the oral cancer reoccurrence extracted from the NeoMark Project. | 4.1 | production | cancer (6754) |
| Ontology of chemically concrete, composable DNA, RNA, and protein residue and crosslinks | BPFORMS | BpForms is an ontology of molecularly-concrete DNA, RNA, and protein residues and crosslinks which can be composed into molecularly-concrete forms of DNA, RNA, proteins, and complexes. The ontology can be used with the BpForms grammar and software tools for polymers and the BcForms grammar and software tools for complexes. See https://www.bpforms.org and https://www.bcforms.org for more information. | 0.0.9 | beta | DNA (CELLULAR_COMPONENT), https://www.bcforms.org (GENE_OR_GENE_PRODUCT) |
| EDAM - The data analysis and management ontology | EDAM | EDAM is a domain ontology of data analysis and data management in bio- and other sciences, and science-based applications. It comprises concepts related to analysis, modelling, optimisation, and data life cycle. Targetting usability by diverse users, the structure of EDAM is relatively simple, divided into 4 main sections: Topic, Operation, Data (incl. Identifier), and Format. EDAM is particularly suitable for semantic annotations and categorisation of diverse resources related to data analysis and management: e.g. tools, workflows, learning materials, or standards. EDAM is also useful in data management itself, for recording provenance metadata of processed data. | 1.25-20240924T0027Z-unstable(1.26) | production | EDAM (GENE_OR_GENE_PRODUCT) |
| HIV ontology | HIV | The HIV ontology encompasses all knowledge about HIV | 2.6 | production | all (1) |
| Genome Variation Ontology | GVO | The Genome Variation Ontology (GVO) is an ontology for the systematic description of various genomic variations, including complex structural variations in genomes. | 2021-11-18 | beta | - |
| NIST Genome Editing Lexicon | NIST_GEL | Genome editing technology is a fast-growing and rapidly advancing global bioscience field with applications in many biotechnology sectors. Genome editing is used to modify the nucleic acids of a genetic code, which can be composed of DNA or RNA, in a site-specific manner. Modifications can include insertion, deletion or alteration of nucleic acids. The technology operates by biochemical principles generally applicable to every kind of cell. Examples of genome editing technology applications with global significance include human cell-based therapeutics, agriculture, microbial based therapeutics, synthetic biology and biomanufacturing. While genome editing technology is being actively utilized, there is a need for international standardization in terms and definitions for this field, so as to enhance interpretation and communication of concepts, data and results. This document has been developed to provide a unified standard set of terms and definitions that serve the needs of biotechnology stakeholders and act as a reference for genome editing technology. | 2.2 | production | DNA (CELLULAR_COMPONENT), insertion (2673), cell (CELL), genome (CELLULAR_COMPONENT), human (9606), synthetic (32630) |
| Orthology Ontology | ORTH | Orthology Ontology | 2.0 | beta | - |
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Ontology for SARS-CoV-2  lineages and mutations | SARSMUTONTO | The SARSMutOnto ontology provides a list of all SARS-CoV-2 Pango lineages while maintaining their hierarchy (lineage/sublineage) describing in detail all lineage mutations | 1.1 | alpha | all (1), Pango (CANCER) |
| Homo sapiens Chromosomal Location Ontology for GRCh38 | HSCLO38 | HSCLO38's purpose is to simplify the integration of genomic experimental data at different resolution scales within biomedical knowledge graphs. It defines chromosomal locations within the GRCh38 release for chr 1-22, X,Y and M.  HSCLO38 provides 3,431,153 classes at five distinct resolution levels: whole chromosome, 1 megabase pair (Mbp), 100 kilobase pairs (kbp), 10 kbp, and 1 kbp.  Each node within these class levels is interconnected to its scale parent and to the immediate neighbors on either side to support mapping and association between genomic datasets and features. For example, the 1kbp element HSCLO38:chr1.20517001-20518000 is_a connected to its "scale parent" HSCLO38:chr1.20510001-20520000 as well as to the 5' neighbor HSCLO38:chr1.20516001-20517000 and 3' neighbor HSCLO38:chr1.20518001-20519000.  We provide knowledge-graph-ready files for HSCLO38 on the project home page. |  | alpha | HSCLO38 (CANCER), chromosomal (CELLULAR_COMPONENT), chr 1 (GENE_OR_GENE_PRODUCT), X (CELLULAR_COMPONENT), 3,431,153 (GENE_OR_GENE_PRODUCT), node (MULTI_TISSUE_STRUCTURE), 3' neighbor (CELLULAR_COMPONENT) |
| mass spectrometry cross-linking and derivatization reagents | XLMOD | XLMOD is an ontology for cross-linking mass spectrometry reagents and GC-MS derivatization reagents. | release/2019-10-28 | production | - |
| Gene Regulation Ontology | GRO | The Gene Regulation Ontology (GRO) is a conceptual model for the domain of gene regulation. It covers processes  that are linked to the regulation of gene expression as well as physical entities that are involved in these processes (such as genes and transcription factors) in terms of ontology classes and semantic relations between classes. | 0.5 | production | - |
| CPTAC Proteomics Pipeline Infrastructure Ontology | CPTAC | a basic ontology which describes the proteomics pipeline infrastructure for CPTAC project | alpha 1.1 | alpha | - |
| Single-Nucleotide Polymorphism (SNP) Ontology | SNPO | SNP-Ontology is a domain ontology that provides a formal representation (OWL-DL) of genomic variations. Despite its name, SNP-Ontology, is not limited to the representation of SNPs but it encompasses genomic variations in a broader meaning. SNP-Ontology is general enough to enable the representation of variations observed in genome of various species. Latest versions of SNP-Ontology include the representation of haplotype and of CNV. The unambiguous representation of genomic variations provided by SNP-Ontology enables to integrate heterogeneous data related to genomic variations. To achieve this goal SNP-Ontology enables (1) to represent one variation in accordance with various ways that exist for describing it, (2) to represent the equivalence between two distinct descriptions of one variation, and (3) to represent correspondence between a genomic variation and its outcome at the transcriptome and proteome levels. | version: full 1.6 release: 14/03/2017 | beta | - |
| CODO: an Ontology for collection and analysis of COviD-19 data | CODO | The initial version, CODO ontology v1.0 to v1.3, encompasses pivotal features such as classes like patients, clinical findings, symptoms, and properties like relationships between patients, travel history, and test results. It facilitates tracking specific pandemic cases, detailing how patients may have been infected, and identifying potential contacts at risk due to their connection with the infected individual. CODO also enables monitoring of clinical tests, travel history, available resources, and actual needs like ICU beds and invasive ventilators. With capabilities for advanced analytics, contact tracing, trend studies, and growth projections based on daily COVID-19 data, CODO supports the organization and representation of COVID-19 data on a daily basis. It allows semantic querying and data retrieval and aids in behavior analysis of the disease and transmission routes. | 1.5 | beta | CODO (CANCER), disease (CANCER) |
| Sequence Types and Features Ontology | SO | SO is a collaborative ontology project for the definition of sequence features used in biological sequence annotation. SO was initially developed by the Gene Ontology Consortium. Contributors to SO include the GMOD community, model organism database groups such as WormBase, FlyBase, Mouse Genome Informatics group, and institutes such as the Sanger Institute and the EBI. Input to SO is welcomed from the sequence annotation community. The OBO revision is available here: http://sourceforge.net/p/song/svn/HEAD/tree/ | unknown | production | WormBase (GENE_OR_GENE_PRODUCT), FlyBase (GENE_OR_GENE_PRODUCT), Mouse (10090) |
| Head and Neck Cancer - Clinical and Gene Ontology | HENEGEO | HeNeGeO (Head and Neck Cancer - Clinical and Gene Ontology) presents a data model that encompasses both clinical and genomic aspects of Head and Neck Cancer (HNC) across every stage of the disease, covering prognosis, treatment and follow-up. This ontology builds upon the foundation laid by HENECON (Head and Neck Cancer Ontology) and has been significantly expanded with a focus on information pertaining to cancer recurrence, alongside a comprehensive list of genes that offer valuable insights throughout the entire head and neck cancer progression. To enhance clarity, novel medical terms have been thoughtfully annotated with their respective semantic meanings. This comprehensive update includes the incorporation of 29,770 genes, featuring essential details such as NCBI GeneID (EntrezID), gene symbol, gene name, gene alias, and the HGNC ID. The development of this ontology underwent rigorous review and validation by experts in both clinical and genomic domains. | 1.0 | production | Head and Neck Cancer (CANCER), HNC (CANCER), Head and Neck Cancer Ontology (CANCER), cancer (6754), head and neck (CANCER) |
| Monochrom | CHR | Ontological representation of chromosomes in major species | 2025-02-15 | beta | chromosomes (CELLULAR_COMPONENT) |
| FAIR Genomes | FG | The FAIR Genomes semantic metadata schema to power reuse of NGS data in research and healthcare. | 1.2-Minor | production | - |
| Venom Ontology | CU-VO | An ontology of venoms and venom-related compounds, designed to facilitate the discovery of novel venom-derived therapeutics. | 1.0 | production | venom-derived (MULTI_TISSUE_STRUCTURE) |
| Genetic Feature Format Ontology | GFFO | An ontology to represent genome feature annotations in gff3 format. | 0.0.1 | alpha | - |
| Placental Genetic Variance | PATGV | Includes variations of DNA sequence, chromosomal structure and copy number, as well as RNA and translational variation. The Genetic Variation ontology expands on work done for Variation Ontology (VariO) and Sequence Types and Features Ontology (SO) which can be found on NCBO. This ontology was further developed by PAT External Scientific Panel Members and contributions from other subject matter experts. | 1.0 | production | DNA (CELLULAR_COMPONENT), chromosomal (CELLULAR_COMPONENT) |
| Ontology of Genes and Genomes - Mouse | OGG-MM | OGG-Mm is the OGG Mus musculus (mouse) subset. The OGG (Ontology of Genes and Genomes) is a formal ontology of genes and genomes of biological organisms. OGG is developed by following OBO Foundry principles and aligning with the BFO top ontology.  | Vision Release: 1.0.66 | production | OGG-Mm (GENE_OR_GENE_PRODUCT), Mus musculus (10090), mouse (10090), OGG (GENE_OR_GENE_PRODUCT) |
| Gene Ontology | GO | Provides structured controlled vocabularies for the annotation of gene products with respect to their molecular function, cellular component, and biological role. The Gene Ontology consists of three Vocabularies. | releases/2025-06-01 | production | cellular (CELL) |
| HGNC New Releases Ontology | HGNC-NR | HUGO Gene Nomenclature Committee Ontology with recurring uploads | 2024-03-04 | production | HUGO (GENE_OR_GENE_PRODUCT) |
| Epigenome Ontology | EGO | The Epigenome Ontology (EGO) a biomedical ontology for integrative epigenome knowledge representation and data analysis. | Arbor version; 1.0.1 | alpha | - |
| Genomic Clinical Decision Support Ontology | GENE-CDS | The Genomic CDS ontology aims to unify several functionalities in a single resource, being A knowledge base for clinical pharmacogenomics/pharmacogenetics that can be used for question-answering (e.g., which SNPs are associated with this drug) A rule base for clinical decision support (e.g., inferring that a patient with a specific set of SNPs requires a lowered dose of warfarin and generating a CDS message that can be viewed by clinicians) A tool for checking data consistency (e.g., highlighting which allele definitions in PharmGKB are overlapping, or which clinical decision support rules are matching the same group of patients) | 0.8.0 | alpha | - |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |
| Ontology of Genes and Genomes | OGG | OGG is a biological ontology in the area of genes and genomes. OGG uses the Basic Formal Ontology (BFO) as its upper level ontology. This OGG document contains the genes and genomes of a list of selected organisms, including human, two viruses (HIV and influenza virus), and bacteria (B. melitensis strain 16M, E. coli strain K-12 substrain MG1655, M. tuberculosis strain H37Rv, and P. aeruginosa strain PAO1). More OGG information for other organisms (e.g., mouse, zebrafish, fruit fly, yeast, etc.) may be found in other OGG subsets. | Vision Release: 1.0.59 | production | OGG (GENE_OR_GENE_PRODUCT), human (9606), viruses (10239), HIV (57667), influenza virus (11309), bacteria (629395), E. coli (562), mouse (10090), zebrafish (7955), fruit fly (103775) |

<a name="bioportal-category-gross-anatomy"></a>
## BioPortal Category "Gross Anatomy"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Tick Gross Anatomy Ontology | TADS | The anatomy of the Tick, Families: Ixodidae, Argassidae | 1.21 | beta | - |
| Graphic Descriptor Ontology Anatomy Extension | GDOA | For the best browsing experience, see our illustrated browser at https://gdo.endlessforms.info that provides visual depictions of many classes and individuals. The GitHub repository is at https://github.com/endlessforms-info/GDO-GDOA. The version of the GDO/A here is the GDOA and the GDO merged. The version number applies to the merged variant. For separate GDOA and GDO artifacts, please visit the GitHub repo above. The Graphic Descriptor Ontology (GDO) is intended for use in describing graphics that represent the form of objects. The Graphic Descriptor Ontology Anatomy extension (GDOA) extends the GDO to the domain of anatomical graphics by providing classes for anatomical views and anatomically-relevant region mark roles, such as representing a type of body tissue or body fluid. | 1.0.4 | production | GDO/A (CANCER), body tissue (TISSUE) |
| Dictyostelium Discoideum Anatomy Ontology | DDANAT | A structured controlled vocabulary of the anatomy of the slime-mould Dictyostelium discoideum. | 1.19 | production | slime-mould Dictyostelium discoideum (CELLULAR_COMPONENT) |
| Teleost Anatomy Ontology | TAO | TAO is a multispecies anatomy ontology for teleost fishes. Originally seeded from ZFA, but intended to cover terms relevant to other taxa. | 2012-08-10 | production | TAO (GENE_OR_GENE_PRODUCT), teleost fishes (32443) |
| Human Developmental Anatomy Ontology, abstract version 2 | EHDAA2 | A structured controlled vocabulary of stage-specific anatomical structures of the human. It has been designed to mesh with the mouse anatomy and incorporates each Carnegie stage of development (CS1-20). The abstract version of the human developmental anatomy ontology compresses all the tissues present over Carnegie stages 1-20 into a single hierarchy. The heart, for example, is present from Carnegie Stage 9 onwards and is thus represented by 12 EHDA IDs (one for each stage). In the abstract mouse, it has a single ID so that the abstract term given as just heart really means heart (CS 9-20). Timing details will be added to the abstract version of the ontology in a future release. | releases/2024-01-11 | beta | stage-specific (GENE_OR_GENE_PRODUCT), human (9606), mouse (10090), all (1), tissues (TISSUE), heart (ORGAN), CS 9 (GENE_OR_GENE_PRODUCT) |
| Vertebrate Homologous Organ Group Ontology | VHOG | vHOG is a multi-species anatomical ontology for the vertebrate lineage, developed as part of the Bgee project. This mapping represents homology-strict relationships, in the sense of HOM:0000007 "historical homology". Each mapping has been manually reviewed, and we provide confidence codes and references when available. | unknown | production | - |
| Uber Anatomy Ontology | UBERON | Uberon is an integrated cross-species anatomy ontology representing a variety of entities classified according to traditional anatomical criteria such as structure, function and developmental lineage. The ontology includes comprehensive relationships to taxon-specific anatomical ontologies, allowing integration of functional, phenotype and expression data. | 2023-07-25 | production | cross-species (GENE_OR_GENE_PRODUCT) |
| Hymenoptera Anatomy Ontology | HAO | A structured controlled vocabulary of the anatomy of the Hymenoptera (bees, wasps, sawflies and ants) | 2023-06-01 | production | wasps (7438), ants (2153479) |
| Drosophila Gross Anatomy Ontology | FB-BT | An ontology representing the gross anatomy of Drosophila melanogaster | fbbt/releases/2025-05-29 | production | Drosophila melanogaster (7227) |
| Fungal Gross Anatomy Ontology | FAO | A structured controlled vocabulary for the anatomy of fungi. |  | production | fungi (4751) |
| Spider Anatomy Ontology | SPD | An ontology for spider anatomy. | 1.1 | beta | - |
| Mosquito Gross Anatomy Ontology | TGMA | A structured controlled vocabulary of the anatomy of mosquitoes. | releases/2013-06-03 | production | - |
| Mouse Adult Gross Anatomy Ontology | MA | A structured controlled vocabulary of the adult anatomy of the mouse (Mus). | releases/2017-02-07 | alpha | mouse (10090), Mus (862507) |
| C. elegans Gross Anatomy Vocabulary | WB-BT | A structured controlled vocabulary of the anatomy of Caenorhabditis elegans. | releases/2025-03-26 | production | - |
| Zebrafish Anatomy and Development Ontology | ZFA | A structured controlled vocabulary of the anatomy and development of the Zebrafish (Danio rerio). | releases/2025-01-28 | production | Zebrafish (7955), Danio rerio (7955) |
| NeuroMorpho.Org brain region ontologies | NMOBR | The species, brain regions and cell types are integrated into a single ontology for enabling OntoSearch functionality to mine the latest v7.0 release of NeuroMorpho.Org. | 2.0 | beta | brain (ORGAN), cell (CELL) |
| Vertebrate Skeletal Anatomy Ontology | VSAO | VSAO is an anatomy ontology covering the vertebrate skeletal system. VSAO integrates terms for skeletal cells, tissues, biological processes, organs (skeletal elements such as bones and cartilages), and subdivisions of the skeletal system. | 2012-11-06 | production | skeletal (ORGAN), skeletal cells (CELL), tissues (TISSUE), organs (ORGAN), bones (TISSUE) |

<a name="bioportal-category-health"></a>
## BioPortal Category "Health"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| New Approach Methodology Ontology | NAMO | The New Approach Methodology Ontology and Schema (NAMO) is a comprehensive data framework designed to standardize and integrate information about alternative methods to animal testing in biomedical research. Built on LinkML, NAMO provides a common vocabulary and data structure for describing diverse model systems, from cellular cultures and organ-on-chip devices to computational models and digital twins. |  | production | Schema (GENE_OR_GENE_PRODUCT), cellular cultures (CELL), twins (CANCER) |
| INRAE Thesaurus | INRAETHES | INRAE Thesaurus contains more than 16,000 concepts relevant to domains covered by the National Research Institute for Agriculture, Food and the Environment. Concepts are described with terms in French (100%) and English (81%) terms, some having textual definitions and mappings to other semantic artifacts. The concepts are organized in 12 domains subdivided in thematic sections called micro-thesauri., Thésaurus ouvert et partagé couvrant les domaines de recherche d’INRAE. Il sert de référentiel au sein de l’institut pour indexer et annoter des documents, pages web, descriptions d’activités, jeux de données, à des fins de recherche ou d’analyse de l’information. | 2.4 | production | INRAE (CANCER), fins (ORGAN) |
| Promoting Health Aging through Semantic Enrichment of Solitude Research | PHASES | PHASES provides an ontology for integrating research in solitude and gerotranscendence. By integrating these areas of study, we aim to deepen our understanding of each and how they promote healthy aging. | 2025-05-16 | production | - |
| Ontology of Dental Care-Related Fear and Anxiety | ODFA | The Ontology of Dental care-related Fear, Anxiety, and/or Phobia (ODFA) contains terms for defining and classifying dental-care related fear, anxiety, and/or phobia (DFA). The ODFA’s concepts and relations facilitate the development of tools and resources capable of enhancing our understanding of the impacts of DFA, and when applied to study data, the ODFA enables the integration and interoperability of data from multiple studies. The ODFA is developed as a subset of the Oral Health and Disease Ontology (http://purl.obolibrary.org/obo/ohd.owl). Issues and new term requests can be submitted at https://github.com/oral-health-and-disease-ontologies/ohd-ontology/issues., the ODFA enables the integration and interoperability of data from multiple studies. The ODFA is developed as a subset of the Oral Health and Disease Ontology (http://purl.obolibrary.org/obo/ohd.owl). Issues and new term requests can be submitted at https://github.com/oral-health-and-disease-ontologies/ohd-ontology/issues. | 2025-08-04 | production | ODFA (ORGANISM_SUBSTANCE), Oral Health (PATHOLOGICAL_FORMATION) |
| Behaviour Change Intervention Ontology | BCIO | The Human Behaviour Change Project (HBCP) funded by the Wellcome Trust, and the Advancing Prevention Research in Cancer through Ontology Tools (APRICOT) Project funded by the US National Institutes of Health, are developing the Behaviour Change Intervention Ontology and associated tools and resources to be used in reporting research; linking datasets and synthesising evidence; and AI/machine learning algorithms to predict intervention outcomes in novel scenarios. The overall aim of these projects is to automate evidence searching, synthesis and interpretation to rapidly address questions from policy-makers, practitioners and others to answer ‘What works, compared with what, how well, with what exposure, with what behaviours (for how long), for whom, in what settings and why?’. To achieve this, ontologies can help structure evidence, i.e., link evidence with a shared formal description of classes and relationships capturing domain knowledge in order to enable aggregation and semantic querying. We are developing and maintaining the Behaviour Change Intervention Ontology (BCIO) to do this. |  | production | Cancer (CANCER) |
| Rare Diseases Awareness Ontology | RDAO | The Rare Diseases Awareness Ontology was developed using the Basic Formal Ontology (BFO) framework to support education and awareness processes about rare diseases. It is a semantic data structure that organizes and categorizes specific information about these processes, facilitating access and understanding of the data. This ontology includes key concepts from this domain. Furthermore, it incorporates information to train health professionals, family members, and patients, promoting awareness and more comprehensive education. Through data modeling and relationships between these concepts, the ontology allows the creation of interactive and personalized educational resources, in addition to facilitating interoperability between health information systems. With accessible language and visual tools, the system supports disseminating knowledge about rare diseases. This study is supported by the São Paulo Research Foundation (FAPESP) as part of the Public Policy Research Program (grant number 2023/10203-8, managed by Domingos Alves). Diego Yamada is also supported by FAPESP (grant number 2024/06751-2). | 3.1 | beta | - |
| Heart Failure Ontology | HFO | Heart failure ontology is an attempt to define heart-failure-relevant information including the causes and risk factors, signs and symptoms, diagnostic tests and results, and treatment.  | version 1.0 | alpha | Heart (ORGAN) |
| Medical Dictionary for Regulatory Activities Terminology (MedDRA) | MEDDRA | MedDRA is an international medical terminology with an emphasis on use for data entry, retrieval, analysis, and display. It applies to all phases of drug development, excluding animal toxicology, and to the health effects and malfunction of devices. An appendix includes concept descriptions which describe how a medical concept is interpreted, used, and classified within the MedDRA terminology. It is not intended as a medical definition. The concept descriptions are intended to aid the consistent and accurate use of MedDRA in coding, retrieval, and analysis. | MDR27_0 | production | all (1), appendix (PATHOLOGICAL_FORMATION), MedDRA (GENE_OR_GENE_PRODUCT) |
| Ontology for Genetic Susceptibility Factor | OGSF | Ontology for Genetic Susceptibility Factor (OGSF) is an application ontology to model/represent the notion of genetic susceptibility to a specific disease or an adverse event or a pathological biological process. It is developed using BFO2.0's framwork. The ontology is under the domain of genetic epidemiology. | 2.0 | alpha | BFO2.0 (GENE_OR_GENE_PRODUCT) |
| Space Life Sciences Ontology | SLSO | The Space Life Sciences Ontology is an application ontology and is intended to support the operation of NASA's Life Sciences Data Archive (located at https://nlsp.nasa.gov) and other systems that contain space life science research data. Many kinds of scientific research in space involve specialized equipment, experimental procedures, specimens and specimen collection apparatus, supporting operational structures, and references to specific space environments and their characteristics. Often such research is extremely costly compared with planetary/terrestrial investigations, and the community supporting space life sciences is accustomed to the use of various specialized concepts and terminologies when dealing with this kind of research and data. However, the underlying conceptual models (and especially the specific labeled relationships in these models) have never been made explicit. The SLSO attempts to provide definition and organization of these models for the space research community. The SLSO was developed ab inicio using the Ontology Development Kit and imports an extends many concepts from the Basic Formal Ontology (BFO), the Ontology of Biomedical Investigations (OBI), the Environmental Ontology (ENVO), and other OBO Foundry ontologies. Projects at NASA such as the Open Science Data Repository (https://osdr.nasa.gov/) are already using many OBO ontologies, including the Radiation Biology Ontology (https://github.com/Radiobiology-Informatics-Consortium/RBO) and OBI, to index space biology investigation data. With the development of the SLSO, this practice can be extended to include all life science research in space or addressing space effects. Furthermore, the SLSO has a component that imports concepts from the Science Data Discovery Ontology, which was developed to support NASA's Science Discovery Engine (https://sciencediscoveryengine.nasa.gov). These links in the imported SDDO to concepts underlying a broad spectrum of space research (astrophysics, heliophysics, etc.) can ultimately be used to provide key capabilities for discovering and analyzing space life science data and how they relate to other kinds of scientific data regarding space environments., The Space Life Sciences Ontology is an application ontology and is intended to support the operation of NASA's Life Sciences Data Archive and other systems that contain space life science research data. | 2025-08-18 | production | specimens (CELL), collection (32644), planetary/terrestrial (MULTI_TISSUE_STRUCTURE), SLSO (CANCER), all (1) |
| Coding Symbols for a Thesaurus of Adverse Reaction Terms | COSTART | Coding Symbols for Thesaurus of Adverse Reaction Terms (COSTART). 5th ed. Rockville (MD). COSTART is used for coding, filing, and retrieving post-marketing adverse drug and biologic experience reports. COSTART is organized in body system and pathophysiology hierarchies, as well as a separate fetal/neonatal category of less than 20 terms. COSTART has been superseded by the Medical Dictionary for Regulatory Activities (MedDRA) Terminology. For more information about MedDRA in the Metathesaurus, see the MedDRA source synopsis. COSTART was last updated in the Metathesaurus in 1999. | CSP2006 | production | - |
| Animal Health Surveillance Ontology | AHSO | (Work in progress - please contact ahso@datadrivensurveillance.org before using this ontology) This is an ontology for health surveillance, assuming that surveillance data is recorded at three main levels: SAMPLE data - data about specific collected samples; OBSERVATIONS, which can aggregate several samples; and OBSERVATIONAL CONTEXT, which is the context in which the observations were made (for instance clinical observation, surveillance sampling, etc). HEALTH EVENT is the occurrence in the real World to be modeled. This occurrence is an abstract concept, as it generally cannot be determined what are its boundaries in time, space, or population units. We assume that several OBSERVATIONS are derived from each health event, and recorded in the same or different databases - for instance clinical journal, laboratory database, abattoir. At the same time or not. AHSO attempts to model these time specific observations. The ontology is focused on the modeling of the animal and health information connected to a single observation, made at a specific point on time. All the information recorded is relative to that specific point in time. For instance, an animal belongs to a herd, which belongs to an owner, AT THE TIME OF THE OBSERVATION. We do not attempt to model how that animal changed herds on time. Similarly, we do not model clinical progression. It is up to those using the data to create definitions of when two observations should be connected to the same event. | 0.1.0 | alpha | AT THE TIME (CELL) |
| Alzheimer's Disease Ontology | ADO | Alzheimer's Disease Ontology is a knowledge-based ontology that encompasses varieties of concepts related to Alzheimer's Disease, structured by upper level Basic Formal Ontology (BFO). This ontology is enriched by the interrelated entities that demonstrate the network of the understanding on Alzheimer's disease and can be readily applied for text mining. | 2.0.1 | production | - |
| Rheumatoid Arthritis ontology | RAO | A controlled vocabulary of RA-relevant information. Currently, only treatment information is available.  | Version 1.0 | alpha | - |
| WHO COVID-19 Rapid Version CRF semantic data model | COVIDCRFRAPID | This is a semantic data model for the WHO's COVID-19 case record form RAPID version from April 8 2020. It aims at providing semantic references to the questions and answers of the form. | 1.1.4 | production | CRF (GENE_OR_GENE_PRODUCT) |
| COKPME - COVID19 Ontology for analyzing the Karnataka Private Medical Establishments Data   | COKPME | The ontology will be used by analysts for finding the relative and precautionary measures that can be put in action for controlling the spread. Also, it should be able to handle the competence question like: Which properties of the patient should be considered for modeling,  Which properties of patient changes,  Which properties of patient confirm COVID positive, Do the symptom of the patient change over time,  Do the treatment provided depends on the category of patient, Do other comorbid conditions derive any interestingness, Which symptom is high in a particular district? (Or) Does the geographical location influence the patient category. These keep change over an iterative process of ontology development | 1.0 | production | - |
| Digital Health Technologies | ODHT | This is an ontology of digital health technologies (DHTs) with a specific focus on DHT use in clinical trials. It was created as part of a collaborative project between Prism.Bio and Janssen Clinical Innovation on digital sustainability for the clinical trials enterprise. | 0.2 | alpha | - |
| International Classification of Functioning, Disability and Health | ICF | The International Classification of Functioning, Disability and Health, known more commonly as ICF, is a classification of health and health-related domains. These domains are classified from body, individual and societal perspectives by means of two lists: a list of body functions and structure, and a list of domains of activity and participation. Since an individual's functioning and disability occurs in a context, the ICF also includes a list of environmental factors. | 1.0.2 | production | ICF (GENE_OR_GENE_PRODUCT), body (ORGANISM_SUBDIVISION) |
| Pharmacogenomic Relationships Ontology | PHARE | The PHArmacogenomic RElationships Ontology (or PHARE) proposes concepts and roles to represent relationships of pharmacogenomics interest. | version id: 110114 | alpha | - |
| Radiology Gamuts Ontology | GAMUTS | In radiology, a "gamut" is the set of conditions that can cause a specified imaging finding.  The Radiology Gamuts Ontology (RGO) links more than 16,000 diseases and imaging findings.  The causal relation (may_cause; inverse, may_be_caused_by) connects findings and possible causes. This work is made available under provisions of the Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0) license. | 2.0 | production | - |
| Symptom Ontology | SYMP | The symptom ontology was designed around the guiding concept of a symptom being: "A perceived change in function, sensation or appearance reported by a patient indicative of a disease". Understanding the close relationship of Signs and Symptoms, where Signs are the objective observation of an illness, the Symptom Ontology will work to broaden it's scope to capture and document in a more robust manor these two sets of terms. Understanding that at times, the same term may be both a Sign and a Symptom | 2024-05-17 | production | Sign (CANCER) |
| Ontology of Adverse Events | OAE | The Ontology of Adverse Events (OAE) is a community-based biomedical ontology in the domain of adverse events. OAE is developed to standardize adverse event annotation, integrate various adverse event data, and support computer-assisted reasoning. Its development follows the OBO Foundry principles.  | 1.2.47 | production | - |
| Cancer Care: Treatment Outcome Ontology | CCTOO | The Cancer Care: Treatment Outcomes Ontology (CCTOO) catalogues various clinical endpoints used in evaluating outcomes of cancer treatment. In addition to a vocabulary list, CCTOO also curates the dependencies between individual concepts, in form of production rules, for translating clinical findings recorded in the electronic medical records (EMR) into literature-based concepts. An earlier version of this ontology was presented as a scientific abstract alongside the American Society of Clinical Oncology annual meeting 2017, Chicago, USA (J Clin Oncol 2017; 35 suppl; abstr e18137). Full paper on http://ascopubs.org/doi/abs/10.1200/CCI.18.00026 | releases/2018-05-11 | beta | Cancer (CANCER), cancer (6754) |
| OntoFood | OF | This is an ontology with SWRL rules of nutrition for diabetic patient. We can import the value of Blood Sugar and Weight by .xls and use that like instance for generate the rules with rule engine Jess. | 1.0 | alpha | SWRL (CANCER), Blood Sugar (ORGANISM_SUBSTANCE) |
| Translational Medicine Ontology | TMO | This project focuses on the development of a high level patient-centric ontology for the pharmaceutical industry. The ontology should enable silos in discovery research, hypothesis management, experimental studies, compounds, formulation, drug development, market size, competitive data, population data, etc. to be brought together. This would enable scientists to answer new questions, and to answer existing scientific questions more quickly. This will help pharmaceutical companies to model patient-centric information, which is essential for the tailoring of drugs, and for early detection of compounds that may have sub-optimal safety profiles. The ontology should link to existing publicly available domain ontologies. | 1.0.2 | production | - |
| Context and Provenance Based Standardized Patient Generated Health Data Shared with an Electronic Health Record | PGHDPROV | This ontology seeks to capture the provenance and data quality associated with PGHD shared with an EHR. The objective is to have an ontological formal representation of the provenance and contextual data related to PGHD shared with EHR, in order to offer clinicians a means to use PGHD data for decision making and treatment support. | 1.0.0 | alpha | - |
| Common Diabetes Medications Ontology | CDMONTO | The Common Diabetes Medications Ontology (CDMOnto) is an ontology-based decision support system that has been developed with the objective of providing appropriate treatment suggestions for diabetic patients. The CDMOnto Ontology serves as a foundation for shared semantics, domain-specific, standard, and interoperable knowledge pertinent to common diabetes medications. It encompasses the knowledge of diabetes medications and offers a unified terminology that can be disseminated and processed by medical professionals, pharmacists, and diabetic patients. |  | beta | - |
| Veterans Health Administration National Drug File | VANDF | NDF includes information on clinical drugs, drug classes, ingredients and National Drug Code (NDC) Directory codes. | VANDF_2024_07_31 | production | NDF (GENE_OR_GENE_PRODUCT), NDC (CANCER) |
| Allergy Ontology | ALLERGIDOC | L'ontologie ALLERGIDOC représente la terminologie et les connaissances scientifiques, cliniques et généralistes du domaine de l'allergologie. Elle a été élaborée par Marcin Trzmielewski pour l'Unité d'Exploration des Allergies du CHU de Montpellier (référents : Docteur Davide Caimmi et Prof. Pascal Demoly), dans le cadre d'une thèse en sciences de l'information, de la communication et de la documentation à l'Université Paul Valéry Montpellier 3, financée par la Région Occitanie (France) entre 2019 et 2023 et encadrée par Prof. Céline Paganelli. L'ontologie a été élaborée dans une approche constructiviste, à partir d'une analyse des pratiques informationnelles des employés de l'unité d'allergologie et d'une analyse des documents que ces employés utilisent dans leur quotidien. La version 3.0, élaborée après la thèse, comprend des modifications des versions 1.1 et 2.0 effectuées en février-mars 2025 par Marcin Trzmielewski (Laboratoire d'Etudes et de Recherches Appliquées en Sciences Sociales - LERASS), avec la contribution des étudiantes de la licence professionnelle en documentation : Anne Guerre, Nada Bouhsina, Sabrina Duc et Claire Casasso (Université de Montpellier Paul-Valéry). La version 3.0 contient également une nouvelle structuration au niveau des classes principales (facettes) et d'autres niveaux de profondeur classificatoire. L'analyse à facettes a été basée sur les catégories représentant les cibles informationnelles des professionnels d'allergologie et celles de la catégorie Descriptive Ontology for Linguistic and Cognitive Engineering (DOLCE). Les concepts, les termes et les relations qui composent l'ontologie sont issus des versions précédentes 1.1 et 2.0, des extractions terminologiques effectuées lors de l'élaboration de la version 1.0, comme des réunions de validation des versions 1.0 et 2.0. Ils proviennent également des systèmes d'organisation des connaissances existants : Ontology for General Medical Science, Information Artifact Ontology, MeSH, SNOMED, LOINC, Allergen Nomenclature, catalogues des allergènes ThermoFisher et Eurofins, site IMAIOS, taxonomies biologiques consultées sur Wikipédia, VIDAL, Gender, Sex, and Sexual Orientation Ontology, et classification d'Eckman. La version 3.0 est actuellement en cours de validation par les professionnels d'allergologie. | 3.0 | alpha | encadrée (PATHOLOGICAL_FORMATION), facettes (TISSUE), Eurofins (CELLULAR_COMPONENT), IMAIOS (CANCER), Sex (GENE_OR_GENE_PRODUCT) |
| Cardiovascular Magnetic Resonance Quality Assessment Ontology | CMR-QA | CMR-QA ontology aims at defining the semantics of free-text quality assessment annotations associated to Cardiac Magnetic Resonance (CMR) Imaging. | 0.4.5 | beta | Cardiac (ORGAN) |
| Neomark Oral Cancer Ontology, version 4 | NEOMARK4 | Ontology that describes the medical information necessary for early detection of the oral cancer reoccurrence extracted from the NeoMark Project. | 4.1 | production | cancer (6754) |
| Persian Medicine Diseases Ontology | PMD | PMD is an ontology that represents classification and related information of Persian medicine diseases. Persian medicine also known as Iranian traditional medicine is one of famous and most ancient forms of traditional medicines. This ontology is based on the top-level ontology GFO (General Formal Ontology). | 1.0 | alpha | - |
| Ontological Knowledge Base Model for Cystic Fibrosis | ONTOKBCF | OntoKBCF is an ontological knowledge base model for cystic fibrosis. There are molecular genetic information (i.e. gene mutations) and health information included in OntoKBCF. The purposes of OntoKBCF include management of molecular genetic information and health information and embedding OntoKBCF into EHR settings. | 1301 | production | cystic (PATHOLOGICAL_FORMATION) |
| Coronavirus Infectious Disease Ontology | CIDO | The Ontology of Coronavirus Infectious Disease (CIDO) is a community-driven open-source biomedical ontology in the area of coronavirus infectious disease. The CIDO is developed to provide standardized human- and computer-interpretable annotation and representation of various coronavirus infectious diseases, including their etiology, transmission, pathogenesis, diagnosis, prevention, and treatment. | 2024-02-16 | production | Coronavirus (694013), CIDO (CANCER), coronavirus (694013) |
| Read Clinical Terminology Version 2 | RCTV2 | NHS UK Read Codes Version 2 | 2015 | beta | NHS UK (CANCER) |
| Caroli's Disease and Syndrome Ontology | CAROLIO | We present CaroliO, an ontology for Caroli's disease and syndrome, aimed at enhancing the diagnosis and treatment of these rare liver conditions. Our work addresses the pressing need for structured knowledge resources in the field of rare diseases, which often present significant diagnostic and treatment challenges due to limited information. We developed our ontology in two versions: CaroliO VB, which is the Basic ontology; and CaroliO VA, the Applied Version, which includes more complex axioms i.e. closure axioms and axioms that indicate combinations of scenarios among symptoms, signs, treatments, and outcomes. The Basic Version follows the underlying Open World Assumption and is suitable for ontologists and future researches, while the Applied Version includes more complex axioms to better reflect real world decision making in diagnosis and choosing treatments for Caroli’s disease and syndrome based on our current knowledge. | 2.2 | alpha | Caroli (CANCER), disease (CANCER), liver (ORGAN) |
| Ontology of Precision Medicine and Investigation | OPMI | The Ontology of Precision Medicine and Investigation (OPMI) is an application ontology to support precision medicine and its related investigations. OPMI is community-driven and developed by following the OBO Foundry ontology development principles. OPMI reuses, aligns, and integrates related terms from existing OBO ontologies.   | Vision Release: 1.0.166 | production | - |
| Ontology of Information System on Live Births | ONTOSINASC | The SINASC Ontology was developed in the context of DIASUS project. This ontology intent to rescue the data structure presented in Live Birth Certificate form, from Information System on Live Births - SINASC, from Brazilian Public Health System - SUS.  | 1.0.0 | alpha | SINASC (TISSUE) |
| Radiation Oncology Ontology | ROO | The Radiation Oncology Ontology aims to cover the radiation oncology domain with a strong focus on re-using existing ontologies. | 0.5 | alpha | - |
| Physical Medicine and Rehabilitation | PMR | Knowledge representation related to computer-based decision support in rehabilitation; concepts and relationships in the rehabilitation domain, integrating clinical practice, the ICD (specifically its 11th revision), the clinical investigator record ontology, the ICF and SNOMED CT. | 1.6 | alpha | ICF (GENE_OR_GENE_PRODUCT) |
| Ontology of Host-Pathogen Interactions | OHPI | The Ontology of Host-Pathogen Interactions (OHPI) is a biomedical ontology of the host-pathogen interactions. Its development follows the OBO Foundry Principles. | 1.0.26 | production | OHPI (CANCER) |
| Digital medicine Outcomes Value Set (DOVeS) Ontology | DOVES | The Digital medicine Outcomes Value Set (DOVeS) ontology aims to provide users with a set of ontologically structured healthcare outcomes (clinical, operational, economic, technical, and others) tailored to digital medicine and digital therapeutic applications. | releases/2023-03-01 | alpha | Set (GENE_OR_GENE_PRODUCT) |
| Head and Neck Cancer Ontology | HENECON | HeNeCOn (Head and Neck Cancer Ontology) is the first ontology specifically focused on the Head and Neck Cancer (HNC) domain. It provides a clinically reliable data model for HNC processes during all phases of the disease: prognosis, treatment, and follow-up. The ontology is established based on an annotated dataset to provide formal reference descriptions, contains  170 clinical variables organized into a taxonomy, and is expanded and mapped to formalize and integrate multiple databases into the HeNeCOn ontology. The outcomes of this iterative process were reviewed and validated by clinicians and statisticians. HeNeCOn, with roughly 500 classes, is a very specific ontology which includes a taxonomy with a hierarchical structure, semantic definitions of 283 medical terms and detailed relations between them, which makes it a reference data model.  | 2.0 | production | Head and Neck Cancer Ontology (CANCER), Head and Neck Cancer (CANCER), HNC (CANCER), all (1) |
| Biologie Hors Nomenclature | BHN | L'activité innovante de biologie et d'anatomo-pathologie réalisée notamment dans les Centres Hospitalo-Universitaires est habituellement appelée "activité hors nomenclature" (BHN pour la biologie hors nomenclature et PHN pour l'anatomo-pathologie hors nomenclature). Ce caractère "hors nomenclature" signifie que l'assurance maladie n'a pas encore intégré ces actes dans la Nomenclature des Actes de Biologie Médicale (NABM) ou la Nomenclature Générale des Actes Professionnels (NGAP). | 3.2 | production | - |
| Pneumonia Diagnosis Ontology | PNADO | Pneumonia Diagnosis Ontology (PNADO) provides the necessary knowledge to diagnose infective pneumonia. It covers symptoms and clinical signs, clinical history, pathogens, laboratory tests and results, imaging, differential diagnosis, complications and pneumonia types. | Version 2.0, 2024/07/10 | beta | PNADO (CANCER) |
| Bilingual Ontology of Alzheimer's Disease and Related Diseases | ONTOAD | OntoAD is a bilingual (English-French) domain ontology for modeling knowledge about Alzheimer's Disease and Related Syndromes. | 1.0 | beta | OntoAD (GENE_OR_GENE_PRODUCT) |
| Disease core ontology applied to Rare Diseases | HRDO | This resource was designed during a PhD in medical informatics (funded by INSERM, 2010-2012). Its components are (i) a core ontology consistent with a metamodel (disorders and groups of disorders, genes, clinical signs and their relations) and (ii) an instantiation of this metamodel with Orphanet Data (available on http://orphadata.org). Research experiments demonstrated (i) efficient classifications generation based on SPARQL Construct, (ii) perspectives in semantic audit of a knowledge base, (iii) semantic comparison with OMIM (www.omim.org) using proximity measurements and (iv) opened perspectives in knowledge sharing (LORD, http://lord.bndmr.fr). Current production services of Orphanet developed ORDO, released in 2014, an ontology synchronized with their production database. This ontology is now available on Bioportal. | V2 | alpha | - |
| Chronic Disease Patient Education Ontology | CDPEO | This ontology is used to recommend personalized education materials to chronic disease patients. The constructed ontology mainly consists of two levels. Level 1 includes 5 terms: demographic, disease, vital sign, lifestyle and medication, which describe the characteristics contained in the patient data, as well as the topics of education materials. Leve 2 contains the detailed elements for each Level 1 class. The patient profile class is used to generate the instance of patient data, using object properties to connect to the specific characteristic class. The patient data will be mapped to a vector space generated from the Level 2 classes. SWRL rules are added to implement the semantic logic of mapping from patient original data to the ontology vector space. The pulblication of the ontology is in preperation. | 1.0.0 | alpha | - |
| PatientSafetyOntology | PSO | It is the ontology embedding PRIME (Prospective Risk Analysis in Medical Environments) in related issues of patient safety research and activities. It classifies terms and express relations between them. | 008:2017-08-04 | alpha | PRIME (CANCER) |
| BioMedical Study Lifecycle Management | BMS-LM | Biomedical research data reuse and sharing is essential for fostering research progress. To this aim, data producers need to master data management and reporting through standard and rich metadata, as encouraged by open data initiatives such as the FAIR (Findable, Accessible, Interoperable, Reusable) guidelines. This helps data re-users to understand and reuse the shared data with confidence. Therefore, dedicated frameworks are required. The provenance reporting throughout a biomedical study lifecycle has been proposed as a way to increase confidence in data while reusing it. The Biomedical Study - Lifecycle Management (BMS-LM) data model has implemented provenance and lifecycle traceability for several multimodal-imaging techniques but this is not enough for data understanding while reusing it. Actually, in the large scope of biomedical research, a multitude of metadata sources, also called Knowledge Organization Systems (KOSs), are available for data annotation. In addition, data producers uses local terminologies or KOSs, containing vernacular terms for data reporting. The result is a set of heterogeneous KOSs (local and published) with different formats and levels of granularity. To manage the inherent heterogeneity, semantic interoperability is encouraged by the Research Data Management (RDM) community. Ontologies, and more specifically top ontologies such as BFO and DOLCE, make explicit the metadata semantics and enhance semantic interoperability. Based on the BMS-LM data model and the BFO top ontology, the BioMedical Study - Lifecycle Management (BMS-LM) core ontology is proposed together with an associated framework for semantic interoperability between heterogeneous KOSs. It is made of four ontological levels: top/core/domain/local and aims to build bridges between local and published KOSs. In this paper, the conversion of the BMS-LM data model to a core ontology is detailed. The implementation of its semantic interoperability in a specific domain context is explained and illustrated with examples from small animal preclinical research. | Version 1.1 | production | - |
| Dengue Fever Ontology | IDODEN | An ontology for dengue fever. | 0.15b | beta | - |
| BirthOnto: Birth Data Analysis | BIRTHONTO | US Birth ontology is a formal representation of the concepts, properties, and relationships that are relevant to the domain of US birth data. US Birth ontology can be used to facilitate data integration, analysis, and sharing across different data sources and systems. It can help to ensure that data is consistent, interoperable, and reusable, and can facilitate data discovery and reuse. The ontology can be used by researchers, healthcare professionals, and policymakers to ask and answer questions about birth outcomes, risk factors, and trends in the US birth data. | 1.1 | beta | - |
| Dietary lifestyle ontology | DILON | DILON was created to represent dietary lifestyle data.  Concepts were pulled from Korean dietary assessment scales and English assessment scales. Concepts are labeled in English and Korean translations are also provided. | 2 | alpha | - |
| SMASH Ontology | SMASH | The SMASH (Semantic Mining of Activity, Social, and Health data) system ontology describes concepts used in describing the semantic features of healthcare data and social networks. The SMASH system primarily addresses the research topic of sustained weight loss with continued intervention with frequent social contacts. |  | alpha | - |
| Logical Observation Identifier Names and Codes | LOINC | Logical Observation Identifier Names and Codes (LOINC) provides formal names and standardized codes for laboratory and other clinical observations. The data cover laboratory terminology, vital signs, hemodynamics intake/output, EKG, obstetric ultrasound, cardiac echo, urologic, imaging, gastroendoscopic procedures, pulmonary ventilator management, and selected survey instruments. | LNC278 | production | cardiac (ORGAN), pulmonary (ORGAN) |
| Multiplex Classification Ontology for the MultiCaRe Experiment | MCO4MCRE | Ontology used for one of the experiments from the Multiplex Classification Framework (paper under review). This ontology is based on the set of classes from the MultiCaRe dataset. The MultiCaRe dataset is a multimodal case report dataset containing medical images that were programmatically labeled based on the content of their captions (Nievas Offidani & Delrieux, 2024). Images may have more than one label, making this a multi-label classification task. For this experiment, we selected a subset of 31 classes, including “endoscopy”, “colonoscopy”, and “mri”. The original ontology is flat, and it does not contain any logical constraints among the classes. Data Article: Nievas Offidani, M. A., & Delrieux, C. A. (2024). Dataset of clinical cases, images, image labels and captions from open access case reports from PubMed Central (1990–2023). Data in Brief, 52, 110008. https://doi.org/10.1016/j.dib.2023.110008 |  | alpha | - |
| Nurse Surgical | SURGICAL | This is an ontology of a surgical nurse's process domain |  | alpha | - |
| Biological Pathway Taxonomy | BPT | Biological Pathway Taxonomy (BPT) - hierarchical taxonomy of cellular and molecular mechanisms, metabolic reactions, and diseases that can be shown as signaling pathways. BPT was created manually based on the Elsevier Pathway Collection available for public usage. There are 3 types of entities in BPT – pathway, group, and folder of pathways and groups. Group is the list of genes and proteins associated with biological function and diseases. Pathway is an interactive model of molecular interactions in human and mammalian cells described in the literature. Each model was reconstructed based on facts published in peer-reviewed journals, with manually quality control. BPT pathway entity includes several properties such as link to the model in Pathway Studio software which allows reading supported sentences from published articles and additional information about members. A list of members for each model can be provided by request. Property “network” describes maps without logical causal relationships between members to distinguish them from actual models of molecular mechanisms (“signaling”). The index folder contains all pathways and groups sorted alphabetically. | synonymes2022 | alpha | cellular (CELL), human (9606), mammalian cells (CELL), all (1) |
| ChemoOnto | CHO | ChemoOnto, an ontology to qualify the course of chemotherapies | Version: 1.0.1 | alpha | - |
| Computer Assisted Brain Injury Rehabilitation Ontology | CABRO | a web ontology for the semantic representation of the computer assisted brain trauma rehabilitation domain. This is a novel and emerging domain, since it employs the use of robotic devices, adaptation software and machine learning to facilitate interactive, adaptive and personalized rehabilitation care, patient monitoring and assisted living.  | 0.5 | alpha | brain (ORGAN) |
| HeLiFit-Ontology | HELIFIT | This an ontology for healthcare. | 1.6.4 | beta | - |
| Sepsis Ontology | SEPON | The Sepsis Ontology (SEPON) is a disease-specific ontology that defines the concepts and relationships between concepts in the field of sepsis and related diseases, contributing to data standardization and providing precise and personalized diagnosis, treatment and prognosis recommandations for sepsis. | V1.0.0 | alpha | SEPON (CANCER) |
| Non Clinical Determinants of Health | N-CDOH | Supporting ontology reuse by integrating CDoH with an existing SDoH ontology. We are presenting an overarching conception for all non-clinical determinants of health (N-CDoH). | v1 | alpha | all (1) |
| OCD ontology | OCD | The Obsessive Compulsive Disorder Ontology (OCD) is a domain-specific ontology for representing the knowledge of OCD. |  | alpha | - |
| Cardiovascular Disease Ontology | CVDO | CVDO is an ontology based on the OGMS model of disease, designed to describe entities related to cardiovascular diseases (including the diseases themselves, the underlying disorders, and the related pathological processes). It is being developed at Sherbrooke University (Canada) and the INSERM research institute (Institut National de la Santé et de la Recherche Médicale). | 2024-05-17 | production | disease (CANCER), cardiovascular (ANATOMICAL_SYSTEM) |
| Emergency care ontology | ONTOLURGENCES | Emergency care ontology build during LERUDI project. http://www.ncbi.nlm.nih.gov/pubmed/25160343 (v4.0) | 3.0.4 | production | - |
| Ontology of Drug Neuropathy Adverse Events | ODNAE | ODNAE is an extension of OAE with a focus on drug-associated neuropathy adverse events. | Vision Release; 1.0.13 | production | ODNAE (GENE_OR_GENE_PRODUCT) |
| PatientSafetyOntologyRevisted | PSO_2 | This is a revised version of the Patient Safety Ontology | 005 | alpha | - |
| Pulmonary Embolism Ontology | PE | A controlled vocabulary of PE-relevant information. Currently, only treatment information is available.  | version 1.0 | alpha | Pulmonary (ORGAN) |
| Ontology of Drug Adverse Events | ODAE | The Ontology of Drug Adverse Events (ODAE) is a biomedical ontology in the area of drug adverse events, developed by following OBO Foundry principles (e.g., openness, collaboration).  | Vision Release: 1.0.08 | beta | - |
| Diabetes Pharmacology Ontology | DPCO | The Diabetes Pharmacology Ontology contains the classes needed to describe antihyperglycemic therapies, pharmacotherapy factors, and diagnostic factors. | 1.0 | beta | - |
| Ontology for Biobanking | OBIB | The ontology for Biobanking (OBIB) is an ontology built for annotation and modeling of biobank repository and biobanking administration. It is developed based on subset of Ontology for Biomedical Investigations (OBI) using Basic Formal Ontology (BFO) as top ontology and following OBO Foundry principles. The first version of the ontology is merged of two existing biobank related ontologies, OMIABIS and biobank ontology. | 2023-04-05 | production | OBIB (CANCER), OMIABIS (CELL) |
| Clinical Trials Ontology - Neurodegenerative Diseases | CTO-NDD | The CTO-NDD stands for Clinical Trial Ontology-Neurodegenerative Diseases, which was built with the purpose of organizing the knowledge domain of clinical trials in neurodegeneration field. This resource has been created for use in the IMI-funded AETIONOMY project (www.aetionomy.org).  | 0.6 | alpha | - |
| RegenBase ontology | RB | The RegenBase ontology describes biological entities and processes studied in spinal cord injury (SCI) research. | 1.0 | alpha | spinal cord (ORGAN), SCI (CANCER) |
| Social Determinants of Health Ontology | SOHO | SOHO has terminologies covering societal factors, such as where a person was born, grew up, works, lives, etc., along with socio-economic and community factors that affect an individual’s health. | 4 | production | - |
| Pre-eclampsia Ontology | PE-O | The PEO incorporates a wide range of key concepts and terms of PE from clinical and biomedical research in structuring the knowledge base that is specific to PE. | v15 | production | Pre-eclampsia (CANCER) |
| HIVHPVSchistosomiasis | HIVHPVSCHISTOSOM | This is the ontology containing the controlled terms and semantic properties used in the HIV, HPV and Schistosomiasis dataset obtained from the MOH Uganda | 1.1 | alpha | HIV (57667) |
| International Classification of Diseases for Oncology, 3rd edition | ICD-O-3 | Ontology used for oncology research derived from the European Cancer Information System (ECIS) | OCD-O Revision 3.2 (complete) | alpha | Cancer (CANCER) |
| Dementia-Related Emotional And Mood Disturbance Non-Pharmacological Treatment Ontology | DREAMDNPTO | DREAMDNPTO represents the domain knowledge specific to the non-pharmacological treatment of emotional and mood disturbances (e.g., depression and anxiety in dementia) in people with dementia living in the long-term care setting. | v1.0 | alpha | - |
| Non-Pharmacological Interventions (NPIs) | NPI | Terminologies of Non-Pharmacological Interventions | Version 8 | alpha | - |
| Human Health Exposure Analysis Resource | HHEAR | The Human Health Exposure Analysis Resource (HHEAR) measures exposures that cover the breadth of the “exposome,” which encompasses all environmental exposures including chemical, physical, and biological stressors, as well as lifestyle and social environments, from conception through adulthood. | 1.11 | production | HHEAR (CANCER), all (1) |
| Electronic Care Plan | ECP | Our focus in this work is to define in a formal way (ontology-based) and platform independent representative metadata and relationships of care plans. This approach has as goal to work as an umbrella over all these systems and provides some advantages regarding the management, the organization and the searching of formal care plans. | 0.6 | beta | all (1) |
| Breast cancer screening recommendation ontology | BCSR-ONTO | BCSR-Onto represents knowledge about risk assessment and recommendations by diverse guidelines for breast cancer screening both for women at high and average risk of developing breast cancer. | 2.1 | alpha | BCSR-Onto (GENE_OR_GENE_PRODUCT), cancer (6754) |
| Ontology for Representing CDM Semantics | ORCS | The Ontology for Representing CDM Semantics (ORCS) is an application ontology that was developed to transform common-data-model-based data into ontologically-based data such as RDF, RDF*, OWL2, or Referent Tracking tuples.  Although initially developed for such transformations of PCORnet Common Data Model (CDM) based data, ongoing and future work involves other CDMs such as OMOP. | 2021-11-13 | beta | common-data-model-based (CELLULAR_COMPONENT), RDF (CELLULAR_COMPONENT), CDMs (CELL) |
| Ontology for Modeling and Representation of Social Entities | OMRSE | The Ontology for Modeling and Representation of Social Entities (OMRSE) is an OBO Foundry ontology that represents the various entities that arise from human social interactions, such as social acts, social roles, social groups, and organizations. | 2025-03-26 | production | OMRSE (CANCER), human (9606) |
| Chinese Diabetes Mellitus Ontology | CDO | Note: The abbreviation of Chinese Diabetes Mellitus Ontology was changed to CDMO. CDMO is an OWL 2 ontology designed to cover the diabetes mellitus knowledge represented in Chinese language. The ontology is based on BFO and OGMS. In addition, the top hierarchical structure was adapted from SNOMED-CT, and strive for mapping with other medical ontologies in English, such as SNOMED-CT, HPO, DOID, DRON. | 3.0 | production | HPO (CANCER), DRON (GENE_OR_GENE_PRODUCT) |
| Association pour le Développement de l'Informatique en Cytologie et Anatomie Pathologique | ADICAP | Modes de prélèvement, types de technique, organes, lésions, topographies, grading | 10-2024 | beta | - |
| Diabetes Mellitus Treatment Ontology | DMTO | DMTO is an OWL 2 ontology for creating customized treatment plans for diabetic patients. The ontology is based on BFO and OGMS. In addition, it extends the DDO ontology by adding the treatment classes and axioms to the existing diagnosis part.  | 2015-11-20 | alpha | - |
| Mammography Screening Ontology | MAMO-SCR-ONTO | The Mammography Screening Ontology (MAMO-SCR-Onto) is designed to provide mammography application recommendations for the early detection of breast cancer in patients at average risk of the disease. | 1.0 | alpha | cancer (6754) |
| Mental Health Management Ontology | MHMO | This ontology can assist in the management of mental healthcare networks. It allows the integration and semantic interoperability between independent health databases, promotes the ability to infer knowledge in this domain, enables the generation of relevant indicators for mental health to improve decision-making, and increases mental health management and planning efficiency and effectiveness. This is a non-profit, open-source project. | 3.2 | alpha | - |
| LinkML Common Metadata Elements and Terms | COMET | A collection of common metadata elements and terms for constructing LinkML Schemas and Data Models |  | alpha | collection (32644) |
| Dental Restorative Material Ontology | DRMO | The DrMO ontology is a domain ontology that represents knowledge underlying the composition, characterization and standardization of different materials involved in the dental restoration procedure. It will assist dentists in selecting appropriate materials based on up-to-date scientific knowledge to satisfy a patient’s specific requirements, without jeopardizing their clinical time. It reuses several ontologies from the OBO foundry, especially the Oral Health and Disease (OHD) Ontology. However, the dental restoration domain is complex and also requires concepts from materials science and engineering. Thus, DrMO also incorporates knowledge from the Devices, Experimental scaffolds, and Biomaterials (DEB) and Functionally Graded Materials (FGM) ontologies to provide more comprehensive knowledge of this area of dental material than previous ontologies. However, much of the terminology from FGM is different than that used in clinical dentistry. Thus, DrMO has changed the appropriate classes to make them consistent with terminology common in dentistry.  DrMO also follows ontology design best practices by reusing meta-data properties from the Dublin Core vocabulary. It captures knowledge from a large set of the most recent and influential papers in Dental Materials and related fields. Links to these papers are included in the ontology as meta-data defined with Dublin Core. It is implemented in OWL2 and was developed with the Protégé 5.5 ontology editor.  The ontology was created using the Ontology Development 101 methodology by Natalya et.al. Several domain experts in addition to Dr. Dutta also provided their expertise. The ontology is available on GitHub and licensed via an open source license. The GitHub project includes a corresponding file of SPARQL queries that answer the competency questions defined as part of the ontology development methodology. |  | alpha | Oral Health (PATHOLOGICAL_FORMATION) |
| Continuity of care | CONTSONTO | The Ontology of Continuity of care (ContSOnto) is an emerging research area consisting of the extension of a healthcare ontology to inform the continuity of care domain. This field is positioned at the confluence of health informatics, nursing informatics, process modelling, and artificial intelligence. This research relating to ContSOnto is underpinned by health informatics standard ISO 13940 Systems of Concepts for Continuity of Care ContSOnto focuses on how information flows from different information systems across and between services for clinical applications and health care professionals to use. Figure 14.2 provides an illustration (see https://contsys.org/pages/Guest%20blog/FormalOntology)of how the different classes and relationships in the standard are represented. The subject of care refers to the individual service user and health care actors can be human or non-human, for example, a health care professional or an organization. We have added concepts related to social determinants of health (SDH) in version 1.6.   | 1.6 | alpha | artificial (32630), human (9606), SDH (GENE_OR_GENE_PRODUCT) |
| Ontology of Organizational Structures of Trauma centers and Trauma systems | OOSTT | The Ontology of Organizational Structures of Trauma centers and Trauma systems (OOSTT) is a representation of the components of trauma centers and trauma systems coded in Web Ontology Language (OWL2). It is developed collaboratively by domain and ontology experts in the NIH-funded CAFE (Comparative Assessment Framework for Environments of trauma care) project (1R01GM111324-01A1). It will be used as the ontology backbone of a graphical user interface comparing graphical representations of organizational structures. | release version - 2025-01-24 | production | - |
| The FoodOn Food Ontology | FOODON | FoodOn is an ontology built to represent entities which bear a “food role” and is initially focused on categorizing and processing of food for humans. We aim to develop semantics for food safety, food security, the agricultural and animal husbandry practices linked to food production, culinary, nutritional and chemical ingredients and processes.  FoodOn belongs to the OBOFoundry.org family of ontologies. | 2025-06-07 | production | bear (9641), humans (9605) |
| Brain-Computer Interaction (BCI) Ontology | BCI-O | The BCI ontology specifies a foundational metadata model set for real-world multimodal Brain-Computer Interaction (BCI) data capture activities.  Its structure depicts a conceptual framework that BCI applications can extend and use in their implementations, to define core concepts that capture a relevant and interoperable metadata vocabulary. This ontology is aligned to the Semantic Sensor Network Ontology (SSN): a domain-independent and end-to-end model for sensor/actuator applications.  Hence, its structure has been normalized to assist its use in conjunction with other ontologies or linked data resources to specify any particular definitions (such as units of measurement, time and time series, and location and mobility), that specialized applications in the BCI domain might need. Its spec provides general alignment data modeling guidelines for core concepts, to help BCI applications in their design. | 0.9.6 | beta | - |
| International Classification of Primary Care - 2 PLUS | ICPC2P | Australian Modification of ICPC-2, known as ICPC-2 Plus. ICPC-2 Plus is primarily used in Australia and provides greater specificity for data input than ICPC-2. | ICPC2P_2005 | production | Care - 2 (CELL) |
| Human Phenotype Ontology | HP | The Human Phenotype Ontology is being developed to provide a structured and controlled vocabulary for the phenotypic features encountered in human hereditary and other disease. Our goal is to provide resource for the computational analysis of the human phenome, with a current focus on monogenic diseases listed in the Online Mendelian Inheritance in Man (OMIM) and Orphanet databases, for which annotations are also provided. Please see documentation and license information at http://www.human-phenotype-ontology.org | hp/releases/2025-05-06 | production | Human (9606), human (9606) |
| OpenCareOntology  | OCO | Open healthCare Ontology (OpenCare) is an integrated metadata vocabulary expressed as an ontological data model, to facilitate the effective reasoning of patient-specific electronic health record (EHR) data. The OpenCare model identifies the concepts and semantic types in the EHR data with the aim of enabling the representation of clinical data in a way that is accessible and usable by researchers and healthcare facility users. The development of OpenCare and its components is formalized using the Ontology Web Language (OWL). This framework aims to enhance the accessibility and user-friendliness of EHR clinical data. | 1.1 | alpha | - |
| Precision Medicine Ontology | PREMEDONTO | Precision MedicinePreMedOnto is an application ontology built for the precision medicine domain on top of gold standard biomedical ontologies. The ontology learning process involves mining the PubMed repository to extract domain specific texts, abstracts and vocabulary as sources of data. The information gathered is clustered and outlined to determine main modules. It reuses terms and concepts from NCIT, MESH and IOBC to construct the ontology hierarchy. |  | alpha | - |
| The Ontology of Information System on Mortality | ONTOSIM | The SIM Ontology was developed in the context of DIASUS project. This ontology intent to rescue the data structure presented in Death Certificate form, from Information System on Mortality - SIM, from Brazilian Public Health System - SUS. | 1.0.0 | alpha | SIM (CANCER) |
| Melanoma Ontology | MELO | Ontology of melanoma disease, composed of terms of disease, drug, symptoms, molecular abnormality，anatomical structure, diagnosis, treatment and relationships between them. | 1.0 | alpha | melanoma (CANCER), disease (CANCER) |
| National Drug File - Reference Terminology | NDFRT | National Drug File - Reference Terminology Public Inferred Edition, 2008_03_11 | NDFRT_2018_02_05_18_03_05 | production | - |
| International Society for the Study of Vascular Anomalies (ISSVA) Ontology | ISSVA | The International Society for the Study of Vascular Anomalies (ISSVA) is a multidisciplinary international society of physicians, scientists, and health care providers united by an interest in vascular anomalies. The Society aims to promote the highest standards of care for patients with vascular anomalies by advancing clinical and scientific knowledge concerning causes, diagnosis and treatment, and by education of physicians, health care providers, patients and the community. The Society encourages the free flow of information between its members and interested groups, through workshop meetings and teaching programs, and by the dissemination of a classification scheme and pertinent scientific data. The ISSVA ontology has been created to represent a sharable semantic data model of the ISSVA classification. This ontological representation of the ISSVA classification is machine-readable and facilitates interoperability between systems that capture data about vascular anomalies. It is structured hierarchically and contains all concepts of the ISSVA classification and the relationships between them. The ISSVA ontology does not substitute the ISSVA classification, instead, it should be used as an additional component to facilitate interoperability. Every concept in the ISSVA ontology has a unique code that should be attached to the corresponding data element. In the context of databases, the ISSVA ontology codes could be added as an extra column to the existing data. The ISSVA ontology contains machine-readable mappings to equivalent concepts in other ontologies, namely: the Orphanet Rare Disease Ontology (ORDO), SNOMED Clinical Terms (SNOMED CT), National Cancer Institute thesaurus (NCIT), and the Human Phenotype Ontology (HPO). This means that data annotated with codes from the ISSVA ontology are interoperable with data annotated with the aforementioned ontologies, without any extra effort, as long as there exists a mapping between the concepts. | 1.1.3 | production | Vascular Anomalies (MULTI_TISSUE_STRUCTURE), ISSVA (CANCER), vascular (MULTI_TISSUE_STRUCTURE), all (1), ORDO (CANCER), Cancer (CANCER), Human (9606), HPO (CANCER) |
| Medical Subject Headings | MESH | Medical Subject Headings (MeSH)National Library of Medicine; 2011 | MSH2025_2024_08_19 | production | - |
| OPTImAL: An ontology for patient adherence modeling in physical activity domain | OPTIMAL | OPTImAL is a reusable formal model of factors affecting CVD patient adherence to physical activity and exercise. The basis for this model relies on the analysis of data/evidence published in the scientific literature and enables identification of adherence based on the patient profile. In the model, the patient profile is seen from the perspective of 60 multidimensional aspects (320 factors). Each factor is associated with defined adherence to a particular patient activity behavior. | 1 | alpha | - |
| COVID-19 Ontology | COVID-19 | The COVID-19 ontology covers the role of molecular and cellular entities in virus-host-interactions, in the virus life cycle,  as well as a wide spectrum of medical and epidemiological concepts linked to COVID-19. | Version Release: 1.0.0 | alpha | cellular (CELL) |
| Artificial Intelligence Rheumatology Consultant System Ontology | AI-RHEUM | AI/RHEUM is used for the diagnosis of rheumatologic diseases. AI/RHEUM contains findings, such as clinical signs, symptoms, laboratory test results, radiologic observations, tissue biopsy results, and intermediate diagnosis hypotheses. Findings and hypotheses, which include definitions, are used to reach diagnostic conclusions with definite, probable, or possible certainty. AI/RHEUM is used by clinicians and informatics researchers. | AIR93 | production | tissue biopsy (MULTI_TISSUE_STRUCTURE) |
| Ontology of Risk Stratification in Mental Health | ONTRISCAL | This ontology organizes concepts and properties of risk stratification in mental health. Stratification aims to determine the patient's risk level by scoring signs and symptoms. Psychotherapeutic treatments aim to minimize patient risk by producing qualitative data reported in therapeutic sessions. Thus, the ONTRISCAL is applied in projects that organize the knowledge about quali-quanti data produced by risk stratification and mental health treatments. Quali-quanti data, semantically annotated by ONTRISCAL, can be ingested into knowledge graphs. Domain specialists can use knowledge graphs to navigate the ONTRISCAL concepts, analyze psychotherapeutic treatments, and control risks in mental health. This is a non-profit and open-source project. | 1.0 | alpha | ONTRISCAL (CANCER) |
| International Classification of Diseases, Version 10 - Clinical Modification | ICD10CM | International Classification of Diseases, 10th Edition, Clinical Modification | CD10CM_2025 | production | - |
| Nanbyo Disease Ontology | NANDO | Creating a comprehensive hierarchical controlled vocabulary for intractable and rare disease (i.e., nanbyo) representation in Japan. | 2024-11-21 | production | - |
| Ontology for Newborn Screening Follow-up and Translational Research | ONSTR | Ontology for Newborn Screening Follow-up and Translational Research (ONSTR) is an application ontology covering the domain of newborn screening, follow-up and translational research pertaining to patients diagnosed with inheritable and congenital diseases mainly identified through newborn dried blood spot screening. ONSTR is a central component of the project Newborn Screening Follow-up Data Integration Collaborative (NBSDC). ONSTR uses the Basic Formal Ontology v2 (BFO2, v2012-07-20) as top-level ontology and extends the classes imported from OBO Foundry ontologies and candidate ontologies. | 0.4 | alpha | blood (ORGANISM_SUBSTANCE), NBSDC (CANCER), BFO2 (GENE_OR_GENE_PRODUCT), v2012-07 (GENE_OR_GENE_PRODUCT) |
| MInimal Requirements for Automated Provenance Information Enrichment in biomedical research | MIRAPIE |  | 1.0.0 | beta | - |
| Prostate Cancer Lifestyle Ontology | PCALION | PCLiON:Prostate Cancer Lifestyle Ontology | 3.0 | production | Prostate Cancer (CANCER) |
| Healthcare Common Procedure Coding System | HCPCS | Healthcare Common Procedure Coding System | HCPCS2024 | production | - |
| Nurse triage | TRIAGE | An ontology showing the process domain of a triage nurse. |  | alpha | - |
| MultiCaRe Taxonomy | MCR_TX | The MultiCaRe Taxonomy contains the class structure used for medical image classification in the MultiCaRe dataset (v2). | 1.0 | production | - |
| SmartPITeS Proyect Ontology for Complex Chronic Patients | SP_ONTOLOGY | The ontology developed for the SmartPITeS project contains decision rules established in clinical practice on which a software developed for clinical decision support for the pharmacological prescription and de-prescription of complex chronic patients is based. Specifically patients with atrial fibrillation under treatment with anticoagulants.  |  | alpha | - |
| National Drug Data File | NDDF | FDB MedKnowledge, formerly National Drug Data File (NDDF) Plus Source Vocabulary. FDB MedKnowledge encompasses medications approved by the U.S. Food and Drug Administration, and information on commonly-used over-the-counter and alternative therapy agents such as herbals, nutraceuticals and dietary supplements. | NDDF_2024_08_14 | production | FDB (ORGAN), FDB MedKnowledge (CANCER) |
| SELECTED OTHER PRIORITY DISEASES UNDER IDSR (INTEGRATED DISEASE SURVEILLANCE AND RESPONSES) | OPD-NAMBOBI | Objective: To	report cases of notifiable diseases, other diseases of	public health importance. |  | alpha | - |
| Study Cohort Ontology | SCO | The Study Cohort Ontology (SCO) is developed to model overall structure and patterns of cohort variables and control/intervention groups defined within the structured population descriptions (commonly referred to as Table1's or Cohort Summary Tables)of observational case studies and clinical trials. We maintain only domain agnostic and commonly occuring subject variables and interventions in SCO, and we have a suite of ontologies (Diseases, Medications, LabResults etc) to encode Diabetes related terminology. We have adopted a bottom-up approach to modeling and we have revised the structure of SCO upon investigation of a few research studies (11 to be precise) from the Pharmacological Interventions (Chapter 8) and Cardiovascular Complications (Chapter 9) of the American Diabetes Association (ADA)’s Standards of Medical Care 2018 CPG. We enable deep drive visualizations driven off queries to knowledge graphs modeled on SCO, to help assist physician visualize the similarity of a patient to a study group, at a quick glance. We welcome contributions and suggestions to extend SCO to represent cohort descriptions of publications targeted to other diseases, and we plan to release set of tools to make this process seamless. | 1.1.0 | alpha | SCO (CANCER), LabResults (CELL), Cardiovascular Complications (PATHOLOGICAL_FORMATION), ADA (GENE_OR_GENE_PRODUCT) |
| Bone Dysplasia Ontology | BDO | The Bone Dysplasia ontology provides a comprehensive and formal representation of the different domain concepts involved in documenting the full complexity of the skeletal dysplasia domain. It captures and combines the genetic features that discriminate the bone dysplasias with the multitude of phenotypic characteristics manifested by patients and required to be taken into account in order to support the diagnosis process. | 1.5 | production | Bone Dysplasia (PATHOLOGICAL_FORMATION), skeletal dysplasia (PATHOLOGICAL_FORMATION), bone (ORGAN) |
| Cancer Chemoprevention Ontology | CANCO | The Cancer Chemoprevention Ontology constitutes a vocabulary that is able to describe and semantically interconnect the different paradigms of the cancer chemoprevention domain. | 0.3 | production | Cancer (CANCER), cancer (6754) |
| MultiCaRe Taxonomy (ML Labels) | ML_MCR_TX | This is a reduced version of the MultiCaRe Taxonomy (MCR_TX), the class structure used for medical image classification in the MultiCaRe dataset. This reduced version includes only the labels used in Machine Learning (ML) model training. |  | beta | - |
| Dispedia Core Ontology | DCO | Dispedia Core Ontology is a schema for information brokering and knowledge management in the complex field of rare diseases. On the one hand, the Dispedia schema uses semantic technologies to describe patients affected by rare diseases, and on the other hand, to record expertise concerning these particular diseases in machine-processable form. In addition to the differentiated attribution of facts to the individual need for information, Dispedia enables information brokering which adjusts to the type of recipient, its character as well as its personal attitude. The schema was initially geared to an exemplified disease, Amyotrophe Lateralsclerose (ALS). | 0.7.0.0002 | beta | disease (CANCER) |
| NHS Quality Indicators | NHSQI2009 | N/A | Unknown | Unknown | NHS (CANCER) |
| Holistic Ontology of Rare Diseases | HORD | This ontology describes the bio-psico-social state of a person of rare diseses in a holistic way. It considers several contents such as disease, psychological, social, environmental state of the person. | 4.0 | alpha | - |
| Pathogen-related Informed Consent Ontology | PICO | PICO is an application ontology designed for the standardization of pathogen-related informed consent data and taxonomic logic relationship classification. The current development priority focuses on constructing its morbidity data representation framework in a manner that ensures consistent structural mapping with medical questionnaire architectures, thereby enabling interoperable data schema across epidemiological research systems. | 1.0.0 | production | - |
| Ontology of Vaccine Adverse Events | OVAE | The Ontology of Vaccine Adverse Events (OVAE) is a biomedical ontology in the area of vaccine adverse events. OVAE is an extension of the Ontology of Adverse Events (OAE) and the Vaccine Ontology (VO). OVAE is aimed to represent and analyze various vaccine-specific adverse events.  | Vision Release: 1.0.34 | production | VO (CANCER) |
| COVID-19 Common Datamodel | GECCO | To ensure the interoperability and compatibility among COVID-19 datasets, we present here a Common Data Model (CDM) which has been built from 11 different COVID-19 datasets from various geographical locations. The current version of the CDM holds 4639 data variables related to COVID-19 such as basic patient information (age, biological sex, and diagnosis) as well as disease-specific data variables, for example, Anosmia and Dispnea. Each of the data variables in the data model is associated with specific data types, variable mappings, value ranges, data units, and data encodings that could be used for standardizing any dataset. | 1.0.0 | production | - |
| Lifestyle Factor Ontology | LSFO | In this ontology, we defined nine distinct categories that encompass various aspects of lifestyle factors (LSFs), creating a unified Lifestyle Factors Ontology (LSFO) across these categories. Within the context of lifestyle, we have identified nine categories that can collectively describe all LSFs, namely: 1-Nutrition, a category that covers different branches such as dietary habits, food groups, food processing and preparation, macronutrients, and micronu trients, among others. 	2-Socioeconomic factors, which includes social and economic conditions such as income, wealth, education and socioeconomic status. 	3-Environmental exposures, includes exposure to various environmental factors, such as air pollution, water quality, and workplace hazards, that can impact an individual’s health. 4-Substance use, covers concepts such as smoking, as well as illicit drug use. 	5-Physical activities, includes regular exercise and physically demanding activities like leisure time, occupational, and household physical activities. 6-Non-physical leisure time activities, describes any activity an individual might engage in during their free time that is not a physical activity. 7-Personal care products and cosmetic procedures, covers activities related to hygiene, use of cosmetic and cleaning products as well as invasive procedures that people undergo to improve their appearance, such as cosmetic surgery. 8-Sleep, covers sleep quality, stages, and habits. 9-Mental health practices, includes the behaviors and habits related to maintaining good mental health and emotional well-being such as meditation, and psychotherapy. | 1.01 | alpha | all (1) |
| Pregnancy Information Needs Ontology | PINO | An ontology about pregnancy information needs across all stages of pregnancy | 1.1 | production | all (1) |
| Myocardial Infarction Ontology | MIO | The myocardial infarction ontology (MIO) is a disease-specific ontology that characterizes the knowledge field of myocardial infarction. | V1.0.0 | production | myocardial (TISSUE) |
| Human Physiology Simulation Ontology | HUPSON | Large biomedical simulation initiatives, such as the Virtual Physiological Human (VPH), are substantially dependent on controlled vocabularies to facilitate the exchange of information, of data and of models. Hindering these initiatives is a lack of a comprehensive ontology that covers the essential concepts of the simulation domain. We propose a first version of a newly constructed ontology, HuPSON, as a basis for shared semantics and interoperability of simulations, of models, of algorithms and of other resources in this domain. The ontology is based on the Basic Formal Ontology, and adheres to the MIREOT principles. | 1.1.1 | alpha | Human (9606) |
| Asthma Ontology | AO | Asthma ontology | 1.3 | production | - |
| Surgical Secondary Events | SSE | Memorial Sloan-Kettering Cancer Center Surgical Secondary Events Ontology (Adverse Events) | 1.1 | production | Cancer (CANCER) |
| Presence Ontology | PREO | The Presence Ontology is a systematic vocabulary of terms with defined relationships that models the encounters taking place every day among providers, patients, and family members or friends in environments such as hospitals and clinics. The Presence Ontology provides a conceptual model for the human experience in medicine. This is a preliminary approach, but further use of methods we have developed here may aid in providing clarity and consensus to topics in healthcare. The ontology is also a model for interdisciplinary collaboration, as it was developed in conjunction with experts from bioinformatics, medicine, anthropology, business, and communication sciences through multiple iterative stages. | 1.0.0 | alpha | human (9606) |
| International Classification of Diseases Ontology | ICDO | A biomedical ontology for logical representation of the terms and relations related to the International Classification of Diseases (ICD) | 1.0.83 | beta | - |
| Infectious Disease Ontology | IDO | The IDO ontologies are designed as a set of interoperable ontologies that will together provide coverage of the infectious disease domain. At the core of the set is a general Infectious Disease Ontology (IDO-Core) of entities relevant to both biomedical and clinical aspects of most infectious diseases. Sub-domain specific extensions of IDO-Core complete the set providing ontology coverage of entities relevant to specific pathogens or diseases. | 8-3-20 | production | IDO (GENE_OR_GENE_PRODUCT) |
| Catalog Themes | CATALOG-THEMES | Controlled Vocabulary for Health Data Themes. A custom vocabulary for classifying health-related datasets in our organization. |  | production | - |
| Apollo Structured Vocabulary | APOLLO-SV | Apollo-SV, developed as an OWL specification, defines the terms and relations necessary for interoperation between epidemic simulators and public health application software that interface with these simulators. | 2024-12-24 | production | OWL (GENE_OR_GENE_PRODUCT) |
| Imaging Biomarker Ontology | IBO | The Imaging Biomarker Ontology describes radiological measurements that characterize biological processes (such as anisotropic water diffusion in a particular tissue region) of imaged subjects (such as neuro-imaging observations) and help clinicians, particularly in the assessment of therapeutic responses and the early prediction of pathologies. IBO reuses several OBO ontologies namely BFO, IAO, PATO, GO, FMA, and OGMS.  |  | beta | tissue (TISSUE), PATO (GENE_OR_GENE_PRODUCT) |
| Mental Stress Ontology | STRONG | This ontology is a module of the human stress ontology, developed in the context of the PGHD(person-generated health data) project, supported by the National Research Foundation of Korea. This ontology included such extension for 11 relatively standard instruments.  |  | production | human (9606) |
| Adherence and Integrated Care | ADHER_INTCARE_EN | This ontology includes concepts in English related to adherence to therapeutic plans in the context of integrated care. Includes maps with Snomed CT, bibliographic references and definitions from the Dictionary of the Royal Academy of Medicine of Spain. Ontology developed with the support of the project StandICT.eu. The equivalent ontology in Spanish is ADHER_INTCARE_SP. | English 051319 | alpha | ADHER_INTCARE_SP (CANCER) |
| Ontology of Pneumology | ONTOPNEUMO | Ontology of pneumology (french version). | 0.1 | beta | - |
| Children's Health Exposure Analysis Resource | CHEAR | Children's health and wellbeing are influenced by interactions between environmental and genetic factors. NIEHS is establishing an infrastructure, the Children's Health Exposure Analysis Resource (CHEAR), to provide the extramural research community access to laboratory and data analyses that add or expand the inclusion of environmental exposures in children's health research. The goal of CHEAR is to provide tools so researchers can assess the full range of environmental exposures which may affect children's health. We anticipate that CHEAR will be used by children's health researchers conducting epidemiological or clinical studies that currently have limited consideration of environmental exposures, or those who have collected exposure data but seek more extensive analyses. | 2.8 | production | CHEAR (CANCER), extramural (ANATOMICAL_SYSTEM) |
| Orphanet Rare Disease Ontology | ORDO | The Orphanet Rare Disease ontology (ORDO) is jointly developed by Orphanet and the EBI to provide a structured vocabulary for rare diseases capturing relationships between diseases, genes and other relevant features which will form a useful resource for the computational analysis of rare diseases. It derived from the Orphanet database (www.orpha.net) , a multilingual database dedicated to rare diseases populated from literature and validated by international experts. It integrates a nosology (classification of rare diseases), relationships (gene-disease relations, epiemological data) and connections with other terminologies (MeSH, SNOMED CT, UMLS, MedDRA),databases (OMIM, UniProtKB, HGNC, ensembl, Reactome, IUPHAR, Geantlas) or classifications (ICD10). The ontology will be maintained by Orphanet and further populated with new data. Orphanet classifications can be browsed in the OLS view. The Orphanet Rare Disease Ontology is updated monthly and follows the OBO guidelines on deprecation of terms. It constitutes the official ontology of rare diseases produced and maintained by Orphanet (INSERM, US14).  | 4.7 | production | ORDO (CANCER), gene-disease (GENE_OR_GENE_PRODUCT), UMLS (CANCER) |
| COntextualised and Personalised Physical activity and Exercise Recommendations | COPPER | The Contextualised and Personalised Physical activity and Exercise Recommendations (COPPER) ontology was created in order to support action and coping planning in the context of physical activity promotion by providing personalised recommendations for activities, activity context, barriers and coping strategies. | 2025-10-07 | production | - |
| Physical Activity Ontology | PACO | Physical Activity Concept Ontology (PACO) contains various concepts that are required to describe one's physical activity.  | 0.2 | alpha | - |
| Anthology of Biosurveillance Diseases | ABD | Our disease ontology provides information on infectious diseases, disease synonyms, transmission pathways, disease agents, affected populations and disease properties. Diseases have been grouped into syndromic disease categories, such that programmers can look through relevant categories, as well as at specific diseases. Organisms, linked to both agents and populations, are structured hierarchically, to provide multiple levels of organism resolution. In addition, both disease transmission and relevant disease properties are available to search. Disease properties include tags like 'notifiable diseases' and 'economic importance' to flag particular disease characteristics that may be of interest, but are not captured elsewhere. |  | alpha | disease (CANCER), Organisms (CANCER), flag (34205) |
| NCCN EHR Oncology Categories | NCCNEHR | The NCCN EHR Oncology Advisory Group was formed to address the high variance in oncology history documentation within EHR systems. Specifically, there is a lack of standardization in the names of oncology history categories across institutions. Moreover, no synonym list or mapping currently exists between the varied category names. Furthermore, oncology history documentation is often inconsistent, incomplete, and not updated appropriately. The NCCN EHR Oncology Advisory Group has established best practices and is working with EHR vendors to implement and enhance tools for documentation of oncology history. This ontology is a result of the work of the group, and provides users with a list of oncology history categories and synonyms. It is our sincere hope that this work can form the basis of improving standardization and interoperability of oncology history documentation within EHR systems. | 1.0.0 | alpha | - |
| Pathogenic Disease Ontology | PDO | An ontology for describing both human infectious diseases caused by microbes and the diseases that is related to microbial infection.  | Version 0.7 | beta | human (9606) |
| Thyroid Cancer Ontology | TCO | Thyroid cancer ontology (TCO) contains 578 concepts organized under 6 biological viewpoints, i.e., Anatomy and Histoembryology, Basic Information of Patient, Cellular or Molecular Interactions, Clinical Aspects, Etiology, Pathology | 1.0 | beta | cancer (6754), TCO (CANCER), Cellular (CELL) |
| Genomic Epidemiology Application Ontology  | GENEPIO | The Genomic Epidemiology Application Ontology describes the genomics, laboratory, clinical and epidemiological contextual information required to support data sharing and integration for foodborne infectious disease surveillance and outbreak investigations. |  | production | - |
| Placental Maternal Health Conditions | PATMHC | Physical and psychological disorders in the mother that occur during pregnancy, childbirth, and the postpartum period. These include both pre-existing and gestational maternal health conditions that may result in adverse pregnancy outcomes. The Maternal Conditions Ontology builds on the work done for a handful of existing ontologies, including the Human Phenotype Ontology (HPO) and NCI Thesaurus (NCIT), and was further developed by PAT External Scientific Panel Members and contributions from other subject matter experts. | 1.0 | production | Human (9606), HPO (CANCER), NCI Thesaurus (CELL) |
| Biomedical Research Integrated Domain Group Model | BRIDG | The Biomedical Research Integrated Domain Group (BRIDG) Model is a collaborative effort engaging stakeholders from the Clinical Data Interchange Standards Consortium (CDISC), the HL7 Regulated Clinical Research Information Management Technical Committee (RCRIM TC), the National Cancer Institute (NCI) and its Cancer Biomedical Informatics Grid (caBIG®), and the US Food and Drug Administration (FDA). The BRIDG model is an instance of a Domain Analysis Model (DAM). The goal of the BRIDG Model is to produce a shared view of the dynamic and static semantics for the domain of protocol-driven research and its associated regulatory artifacts. This domain of interest is further defined as: Protocol-driven research and its associated regulatory artifacts: i.e. the data, organization, resources, rules, and processes involved in the formal assessment of the utility, impact, or other pharmacological, physiological, or psychological effects of a drug, procedure, process, or device on a human, animal, or other subject or substance plus all associated regulatory artifacts required for or derived from this effort, including data specifically associated with post-marketing adverse event reporting. | 3.2 | production | CDISC (CANCER), TC (GENE_OR_GENE_PRODUCT), Cancer (CANCER), NCI (CELL), DAM (GENE_OR_GENE_PRODUCT), human (9606), all (1) |
| Nursing Care Coordination Ontology | NCCO | Nursing Care Coordination Ontology contains activities that nurses use while coordinating care among patients. | 1.0 | beta | - |
| Autism DSM-ADI-R ontology | ADAR | The autism DSM-ADI-R (ADAR) ontology extends an ontology by Tu et al. with additional SWRL rules to infer phenotypes from ADI-R items. It includes OWL class definitions representing DSM IV diagnostic criteria for autistic disorder and ASD criteria for Autism Spectrum Disorder. OWL Reasoners can thus infer autism-related phenotypes from ADI-R questionnaire results. | 1.1 | production | - |
| Brucellosis Ontology | IDOBRU | Brucellosis Ontology is a biomedical ontology in the domain of zoonotic disease brucellosis that is caused by Brucella, a facultative intracellular baterium. | Arbor version; 1.2.92 | production | intracellular (IMMATERIAL_ANATOMICAL_ENTITY) |
| COVID-19 Surveillance Ontology | COVID19 | The COVID-19 Surveillance Ontology is an application ontology used to support SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2)  surveillance in primary care. The ontology facilitates monitoring of COVID-19 cases and related respiratory conditions using data from multiple brands of computerised medical record systems. | V1.0 | beta | - |
| Minimum PDDI Information Ontology | MPIO | MPIO (Minimum PDDI Information Ontology) is an OWL representation of minimum information regarding potential drug-drug interaction information. It is based on and meant to be use in alignment with DIDEO or DINTO. | 2023-10-17 | production | MPIO (CELL) |
| Long Covid Phenotype Ontology | LONGCOVID | Oxford-RCGP RSC Long COVID-19 (LC) Phenotype Ontology | 20220106 | alpha | Oxford-RCGP RSC (GENE_OR_GENE_PRODUCT) |
| Viral Disease Ontology Trunk | VDOT | VDOT is a middle-layer ontology for the health care domain with special emphasis on viral infections and transplantation medicine. | Initial Version release date 28/02/2017 | production | - |
| Medical Imaging and Diagnostic Ontology | MIDO | MIDO is an OWL 2 ontology for creating a standardized format for modeling data in medical imaging tasks such as Disease Classification/ Diagnosis as well as Disease localization/segmentation | 2015-11-20 | alpha | Disease (CANCER) |
| CODO: an Ontology for collection and analysis of COviD-19 data | CODO | The initial version, CODO ontology v1.0 to v1.3, encompasses pivotal features such as classes like patients, clinical findings, symptoms, and properties like relationships between patients, travel history, and test results. It facilitates tracking specific pandemic cases, detailing how patients may have been infected, and identifying potential contacts at risk due to their connection with the infected individual. CODO also enables monitoring of clinical tests, travel history, available resources, and actual needs like ICU beds and invasive ventilators. With capabilities for advanced analytics, contact tracing, trend studies, and growth projections based on daily COVID-19 data, CODO supports the organization and representation of COVID-19 data on a daily basis. It allows semantic querying and data retrieval and aids in behavior analysis of the disease and transmission routes. | 1.5 | beta | CODO (CANCER), disease (CANCER) |
| Health and Social Person-centric Ontology | HSPO | The HSPO is an ontology that aims to represent and link together information about an individual from multiple domains. This work is motivated by the need to better understand the drivers of health outcomes that may arise from other domains including, but not limited to, the social determinants of health (e.g. ethnicity, transportation, education). This ontology can also describe a group of individuals that share similar characteristics (i.e. a cohort). Through this ontology we attempt to define a number of social contexts or social domains which can be useful to organise information coming from different terminologies such as ICD and SNOMED. | 0.1.0 | production | - |
| Diagnosis Ontology of Clinical Care Classification | DOCCC | Diagnosis Ontology of Clinical Care Classification (CCC) | 20131004 | alpha | - |
| Pathogen Transmission Ontology | PTRANS | The Pathogen Transmission Ontology describes the transmission methods of human disease pathogens from one host, reservoir, or source to another host. The pathogen transmission may occur either directly or indirectly and may involve animate vectors or inanimate vehicles. | releases/2022-10-10 | production | human (9606), vectors (29278) |
| HECON: Health Condition Evolution Ontology | HECON | Health Condition Evolution Ontology (HECON), is a formal model representation of the evolution of medical events over time. | 0.1 | alpha | HECON (CANCER) |
| Obesity Tracking Ontology | OTO | N/A | Unknown | Unknown | - |
| Data harmonization ontology for cross border malaria surveillance  | IRDG | Ontology used to describe the data system in the cross-border surveillance system of malaria between French Guiana and Brazil. Developed by the IRD and the Fiocruz and funding by the Bill and Melinda Gates Foundation (GCE Round 18, Grant ID OPP1171795) |  | alpha | border (TISSUE) |
| Obstetric and Neonatal Ontology | ONTONEO | The Obstetric and Neonatal Ontology is a structured controlled vocabulary to provide a representation of the data from electronic health records (EHRs) involved in the care of the pregnant woman, and of her baby. | 2025-03-14 | alpha | EHRs (CANCER) |
| Elderly Care Dementia Ontology | ECDO | This is an ontology for elderly dementia care in the Netherlands. |  | alpha | - |
| Cognitive Reserves Assessment Tasks | ONL-TASKS | This ontology is an extension of the OntoNeuroLOG ontology, developed in the context of a project supported by Russian Foundation for Basic Research (19-29-01017 grant). The ontology organizes tests that are used for assessing cognitive reserves with the focus on keywords that can be employed in mining of PubMed publications. |  | alpha | - |
| Bionutrition Ontology | BNO | The Bionutrition Ontology relates concepts and terminologies used for human nutrition in a clinical and biomedical setting. | 0.2 | alpha | human (9606) |
| Consumer Wearable Device | CWD | Consumer Wearable Devices (CWD) enable passive, low-burden, longitudinal collection of data relevant to health behaviors, human performance, and biometrics.   Such data are increasingly used to tailor clinical and public health interventions and as outcome measures in clinical trials.  The uptake of CWD in research applications warrants an examination of the data collected from these products.  This ontology was developed based on a review of publicly available documentation and data sets from three CWD vendors and is open for researchers to modify + extend as new features become available. |  | alpha | collection (32644), human (9606) |
| Anatomic Pathology Lexicon | PATHLEX | PathLex, a comprehensive lexicon—a unified language of anatomic pathology terms—for standardized indexing and retrieval of anatomic pathology information resources.  | 1.2 | beta | Pathology (CANCER) |
| Breast Cancer Ontology | BCON | The Breast Cancer Ontology (BCON) aims to capture the diverse aspects of breast cancer, including its molecular biology, pathology, epidemiology, clinical data, and lifestyle factors. By representing this knowledge in a standardized and machine-readable format, the ontology enables researchers, clinicians, and other stakeholders to access, share, and analyze breast cancer-related information more effectively. It serves as a valuable resource for organizing existing knowledge, discovering new insights, and supporting decision-making processes in the field of breast cancer research and healthcare. | 1.0 | alpha | Breast Cancer (CANCER), cancer (6754), breast (CANCER) |
| Vaccination Informed Consent Ontology | VICO | The Vaccination Informed Consent Ontology (VICO) is a community-driven ontology in the domain of vaccination/immunization informed consent. | Arbor version; 1.0.52 | production | VICO (CANCER) |
| BrainTeaser Ontology (BTO) | BT-ONTOLOGY | BrainTeaser Ontology (BTO) whose purpose is to jointly model both Amyotrophic Lateral Sclerosis (ALS) and Multiple Sclerosis (MS). The BTO serves multiple purposes: to provide a unified and comprehensive conceptual view about ALS and MS, which are typically dealt with separately, allowing us to coherently integrate the data coming from the different medical partners in the project; to seamlessly represent both retrospective data and prospective data, produced during the lifetime of BRAINTEASER; to allow for sharing and re-using the BRAINTEASER datasets according to Open Science and FAIR principles. | 2.0 | alpha | - |
| Epilepsy Ontology | EPILONT | Ontology about the epilepsy domain and epileptic seizures. Based on the diagnosis proposed by the International League Against Epilepsy (ILAE). | V1 | beta | - |
| Healthcare Monitoring and Emergency Response Ontology | HERO | This ontology is designed to support a healthcare monitoring system that leverages wearable devices and Semantic Web technologies to continuously monitor patient vital signs, assess their medical conditions, and provide timely alerts and interventions in case of abnormalities or emergencies. It aims to enhance patient care, safety, and decision-making by integrating ontology-based reasoning, SWRL rules, and real-time data from various sources. Main Features: Integration of Diverse Data Sources: The ontology integrates different data models like FHIR (for electronic health records and medical data), SOSA (for sensor data and vital signs), and FOAF (for patient profile management). This integration allows for a comprehensive representation of both static and dynamic patient data, supporting more accurate monitoring and decision-making. Vital Sign Monitoring: It includes classes and properties for representing vital signs such as Temperature, Blood Pressure, Heart Rate, Respiratory Rate, and Oxygen Saturation. Each vital sign is associated with corresponding observations and results, following the SOSA ontology pattern. Health Condition Assessment: The ontology defines multiple classes to represent different health conditions. Emergency and Critical Emergency Handling: In abnormal situations, SWRL rules are used to infer whether the situation is an emergency or a critical emergency. Emergency Response Workflow: The ontology supports a detailed emergency response workflow, which involves identifying the nearest hospital, dispatching an ambulance, informing the ambulance crew, alerting the receiving hospital, and continuously updating the patient with the ambulance's status. Range and Classification Definitions: The ontology defines ranges for normal, abnormal, and wrong values for each vital sign across various age groups, ensuring that the system provides age-appropriate assessments. Data Provenance and History Tracking: The ontology captures data provenance, tracking the source and history of observations, which is crucial for clinical decision-making and system transparency. | FHIR W5 categorization (Preliminary) | alpha | Blood (ORGANISM_SUBSTANCE), Heart (ORGAN) |
| Health Ontology for Minority Equity | HOME | An ontology to record the physical, emotional and psychological effects resulting from differences in treatment that citizens receive based on their identity when entering healthcare institutions. We present the first version of such a Health Ontology for Minority Equity (HOME) | 1 | alpha | - |
| CARRE Risk Factor ontology | CARRE | Clinical risk factors, evidence and observables | 0.2 | beta | - |
| Vaccine Ontology | VO | A biomedical ontology in the vaccine domain | 2025-05-24 | production | - |
| Informed Consent Ontology | ICO | Informed Consent Ontology (ICO) is a community-based ontology in the domain of informed consent. It is an OBO library ontology and developed by following the OBO Foundry principles. | 2025-03-15 | production | - |
| NEET_Ontology_ | NEET_ | The data about health conditions of NEETs and EETs |  | alpha | - |
| Mental State Assessment | ONL-MSA | This ontology is a module of the OntoNeuroLOG ontology, developed in the context of the NeuroLOG project, a french project supported by ANR (ANR-06-TLOG-024 grant) aiming at integrating distributed heterogeous resources in neuroimaging. This MSA ontology module covers the field of mental state assessments, i.e. instruments, instrument variables, assessments, and resulting scores. It includes a generic domain core ontology, that provides a general model of such entities and a general taxonomy of behavioural, neurosychological and neuroclinical instruments, that can be easily extended to model any particular kind of instrument. It also includes such extensions for 8 relatively standard instruments. | v1 | production | MSA (CANCER) |
| EmpowerBP | EBP | Clinical practice guidelines, behaviour change theories, and associated behaviour change strategies for the management of hypertension. | 2.5 | alpha | - |
| MFO Mental Disease Ontology | MFOMD | An ontology to describe and classify mental diseases such as schizophrenia, annotated with DSM-IV and ICD codes where applicable. | 2020-04-26 | production | MFO (GENE_OR_GENE_PRODUCT) |
| The Oral Health and Disease Ontology | OHD | The Oral Health and Disease Ontology is intended as a BFO and OBO Foundry compliant ontology for the oral health domain. It is currently used to represent the content of dental practice health records and is intended to be further developed for use in translation medicine. It demonstrates a principled split between billing codes as information entities and what the codes are about, such as dental procedures, materials, and patients. OHD is structured using BFO uses terms from OGMS, OBI, IAO and FMA. In addition it uses terms from CARO, OMRSE, NCBITaxon, and a subset of terms from the Current Dental Terminology (CDT) | 2024-06-21 | alpha | Oral Health (PATHOLOGICAL_FORMATION), oral health (PATHOLOGICAL_FORMATION), OMRSE (GENE_OR_GENE_PRODUCT) |
| SLEEP ontology | SLEEP | The SLEEP ontology aims to facilitate the collection of PGHD related to sleep patterns and behaviors. It was created by organizing sleep-related terms gathered from sleep questionnaires and specialized literature. | 1.0 | alpha | collection (32644) |
| Head and Neck Cancer - Clinical and Gene Ontology | HENEGEO | HeNeGeO (Head and Neck Cancer - Clinical and Gene Ontology) presents a data model that encompasses both clinical and genomic aspects of Head and Neck Cancer (HNC) across every stage of the disease, covering prognosis, treatment and follow-up. This ontology builds upon the foundation laid by HENECON (Head and Neck Cancer Ontology) and has been significantly expanded with a focus on information pertaining to cancer recurrence, alongside a comprehensive list of genes that offer valuable insights throughout the entire head and neck cancer progression. To enhance clarity, novel medical terms have been thoughtfully annotated with their respective semantic meanings. This comprehensive update includes the incorporation of 29,770 genes, featuring essential details such as NCBI GeneID (EntrezID), gene symbol, gene name, gene alias, and the HGNC ID. The development of this ontology underwent rigorous review and validation by experts in both clinical and genomic domains. | 1.0 | production | Head and Neck Cancer (CANCER), HNC (CANCER), Head and Neck Cancer Ontology (CANCER), cancer (6754), head and neck (CANCER) |
| Diabetes Mellitus Diagnosis and Support Ontology  | DMDSONT | Diabetes Mellitus Diagnosis and Support Ontology (DMDSOnt) is created using OWL 2.0 according to the directions given in the IDF’s guidelines (2022). As required and directed in the IDF’s guidelines, DMDSOnt contains concepts and relations about the signs and symptoms observed in DM, active lifestyle expected from patients, genetic conditions of patients, physical activities to be followed, laboratory tests, other factors/criteria in the diagnosis of DM, and supportive treatment recommendations for the proper management of DM risk. | ver 1.0 | alpha | Mellitus (CANCER), OWL (GENE_OR_GENE_PRODUCT) |
| Multiplex Classification Ontology for the HyperKvasir Experiment | MCO4HKE | Ontology used for one of the experiments from the Multiplex Classification Framework (paper under review). This ontology is based on the set of classes from the HyperKvasir dataset. The HyperKvasir dataset is a manually annotated, multi-class image and video dataset for gastrointestinal endoscopy (Borgli et al., 2020). For the Multiplex experiment, only pathological finding labels were used (such as “Esophagitis grade A,” “Esophagitis grade B-D,” and “Hemorrhoids"). |  | beta | gastrointestinal endoscopy (CANCER) |
| COPD Ontology | COPDO | The COPD Ontology  is a biomedical ontology used for modelling concepts associated to chronic obstructive pulmonary disease in routine clinical databases. | 1 | production | COPD (ORGAN), pulmonary (ORGAN) |
| HIVOntology | HIVO004 | A Project of Biomedical Ontology for Systems Biomedicine. The goals of the Project include: (1). To pilot the novel researches on AIDS and HIV or other diseases by viruses, by means of the ontology and other biomedical informatics technologies, connecting the insights of the Biomedical Knowledge Integration (BMKI), systems biomedicine, complexity sciences etc. (2). To Carry out the methodology explorations for the goals above. | 020 | alpha | HIV (57667), viruses (10239) |
| HUGO Gene Nomenclature | HGNC | Human Genome Organisation (HUGO) Gene Nomenclature Committee at the European Bioinformatics Institute approves a gene name and symbol (short-form abbreviation) for each known human gene. All approved symbols are stored in the HGNC database. | dev | production | Human (9606), human (9606) |
| The Prescription of Drugs Ontology | PDRO | PDRO is a realist ontology that aims to represent the domain of drug prescriptions. Such an ontology is currently missing in the OBOFoundry and is highly relevant to the domains of existing ontologies like DRON, OMRSE and OAE. PDRO’s central focus is the structure of a drug prescription, which is represented as a mereology of informational entities. Our current use cases are (1) refining this structure (e.g., adding closure axioms, cardinality, datatype bindings, etc) for prospectively standardizing local electronic prescriptions and (2) annotating prescription data of differing EHRs for detecting inappropriate prescriptions using a central semantic framework. Future ontological work will include aligning PDRO more closely with the Document Acts Ontology. |  | production | PDRO (GENE_OR_GENE_PRODUCT), DRON (GENE_OR_GENE_PRODUCT), EHRs (CANCER) |
| Brain Tumour Ontology | BTO_ONTOLOGY | The purpose of this work is to develop an ontology-based framework for developing an information retrieval system to cater to specific queries of users. For creating such an ontology, information was obtained from a wide range of information sources involved with brain tumour study and research. The information thus obtained was compiled and analysed to provide a standard, reliable and relevant information base to aid our proposed system. Facet-based methodology has been used for ontology formalization for quite some time. Ontology formalization involves different steps such as identification of the terminology, analysis, synthesis, standardization and ordering. A vast majority of the ontologies being developed nowadays lack flexibility. This becomes a formidable constraint when it comes to interoperability. We found that a facet-based method provides a distinct guideline for the development of a robust and flexible model concerning the domain of brain tumours. Our attempt has been to bridge library and information science and computer science, which itself involved an experimental approach. It was discovered that a faceted approach is really enduring, as it helps in the achievement of properties like navigation, exploration and faceted browsing. Computer-based brain tumour ontology supports the work of researchers towards gathering information on brain tumour research and allows users across the world to intelligently access new scientific information quickly and efficiently. | 1 | beta | brain tumour (CANCER), brain tumours (CANCER) |
| Cell Line Ontology [by Mahadevan] | MCCL | Ontologies provide a common platform of controlled vocabulary for researchers who need to share information across a particular domain, inclusive of machine-interpretable definitions of basic concepts in the domain and relations among them. The need for ontologies that provide a systematic arrangement of available data and allow cross talk across various related domains is gaining momentum. In this backdrop, we have developed a comprehensive ontology on primary and established cell lines-both normal and pathologic. It covers around 400 cell lines. This ontology has been built to include the major domains in the field of biology like anatomy, bio-molecules, chemicals and drugs, pathological conditions and genetic variations around the cell lines. An extensive network of relations has been built across these concepts to enable different combinations of queries. The ontology covers all cell lines from major sources like ATCC, DSMZ, ECACC, ICLC etc. and is built in OWL format. | 2 | beta | cell lines-both (CANCER), cell lines (CELL), bio-molecules (CANCER), all (1), DSMZ (CELL), ECACC (CELL) |
| Dementia-Related Psychotic Symptoms Non-Pharmacological Treatment Ontology | DRPSNPTO | The DRPSNPTO ontology represents the domain knowledge specific to non-pharmacological treatment of psychotic symptoms in dementia in the long-term care setting. | v1.0 | alpha | - |
| Fernald Community Cohort N | FCC1 | Fernald Community Cohort meta data | 1.0 | alpha | - |
| FAIR Genomes | FG | The FAIR Genomes semantic metadata schema to power reuse of NGS data in research and healthcare. | 1.2-Minor | production | - |
| SATO (IDEAS expAnded wiTh BCIO): workflow for designers of patient-centered mobile health behaviour change intervention applications | SATO | Designing effective theory-driven digital behaviour change interventions (DBCI) is a challenging task. To ease the design process, and assist with knowledge sharing and evaluation of the DBCI, we propose the SATO (IDEASexpAnded wiTh BCIO) design workflow based on the IDEAS (Integrate, Design, Assess, and Share) framework and aligned with the Behaviour Change Intervention Ontology (BCIO). BCIO is a structural representation of the knowledge in behaviour change domain supporting evaluation of behaviour change interventions (BCIs) but it is not straightforward to utilise it during DBCI design. IDEAS (Integrate, Design, Assess, and Share) framework guides multi-disciplinary teams through the mobile health (mHealth) application development life-cycle but it is not aligned with BCIO entities. SATO couples BCIO entities with workflow steps and extends IDEAS Integrate stage with consideration of customisation and personalisation. The SATO ontology provides the extensions of BCIO along with examples of a BCI Scenario of Fatigue Reduction. | v2 | alpha | Integrate (CANCER), BCIO (GENE_OR_GENE_PRODUCT) |
| World Health Organization (WHO) Adverse Reaction Terminology | WHO-ART | WHO Adverse Reaction Terminology, 1997 | WHO97 | production | - |
| eDEM-Connect: Ontology of Dementia-related Agitation and Relationship between Informal Caregivers and Persons with Dementia | EDEM-CONNECTONTO | The EDEM-CONNECTONTO has been created to incorporate domain knowledge about the various types of agitation observed in people with dementia (PwD), as well as the dyadic relationship existing between PwD and their informal caregivers. Furthermore, it offers a structured framework for non-pharmacological interventions, enabling caregivers to effectively manage and mitigate the bidirectional effects of agitation in PwD. | Version 1.0 | production | EDEM-CONNECTONTO (CELLULAR_COMPONENT), PwD (CANCER) |
| Physician Data Query | PDQ | Physician Data Query (PDQ) Terminology is part of NCI's comprehensive cancer information database, which contains expert summaries on a wide range of cancer topics, a listing of some 30,000 cancer clinical trials from around the world, a directory of genetics services professionals, the NCI Dictionary of Cancer Terms, and the NCI Drug Dictionary. | PDQ_2018_10_27 | production | NCI (CELL), cancer (6754), Cancer (CANCER) |
| Exercise Medicine Ontology | EXMO | A core reference ontology built upon BFO about exercise medicine. This ontology contains the related terms for healthy people, people with chronic conditions and people living with diability to exercise. | 2025-06-05 | beta | - |
| MedlinePlus Health Topics | MEDLINEPLUS | MedlinePlus Health Topics;National Library of Medicine;June 14, 2008;Bethesda, MD | MEDLINEPLUS_20231101 | production | - |
| Liver Case Ontology (LiCO) | LICO | Liver Case Ontology (LiCO) is a case-centric ontology for liver patient cases called lico (Liver Case Ontology) that integrates several ontologies and lexicons developed for sub-disciplines of medicine such as RadLex (Radiology Lexicon), SNOMED CT (Systematized Nomenclature of Medicine, Clinical Terms), LOINC (The Logical Observation Identifier Names and Codes). It is an extension of ONLIRA (Ontology of the Liver for Radiology) that represents the liver and lesions from a radiological perspective. |  | production | Liver Case (CANCER), liver patient (CANCER), ONLIRA (CELL), Liver (ORGAN), liver (ORGAN), lesions (PATHOLOGICAL_FORMATION) |
| SNOMED CT | SNOMEDCT | The US Edition of SNOMED CT combines the International Release of SNOMED CT and the US Extension of SNOMED CT. SNOMED CT provides the core general terminology for the electronic health record (EHR). The concepts have unique meanings and formal logic-based definitions organized into hierarchies. Each International Release includes the core of the terminology (concepts, descriptions, and relationships), together with works to support the implementation and use of SNOMED CT, including subsets, cross maps to existing classifications and coding schemes, and an extensive set of guidelines. The US Extension to SNOMED CT included the concepts, descriptions, and relationships accepted by NLM as a formal extension to the SNOMED CT International Release. NLM no longer releases the standalone US Extension. The US Edition of SNOMED CT includes more than 300,000 unique concepts. There are over 1,000,000 descriptions in SNOMED CT, including synonyms that can be used to refer to a concept. In addition, there are approximately 903,000 links or semantic relationships between the SNOMED CT concepts | SNOMEDCT_US_2024_09_01 | production | - |
| International Classification of Wellness | ICW | The International Classification of Wellness (ICW) is Wellness taxonomy. ICW provides a scheme for classifying Wellness events (activities) in which persons (users) can participate. ICW furthers the goals of preventative medicine and population health by supporting goal setting, quantifying user engagement in wellness activities, and correlating user engagement in wellness activities with health outcomes. | 1.2 | production | - |
| Adherence and Integrated Care in Spanish | ADHER_INTCARE_SP | This ontology includes concepts in Spanish related to adherence to therapeutic plans in the context of integrated care. Includes maps with Snomed CT, bibliographic references and definitions from the Dictionary of the Royal Academy of Medicine of Spain. Ontology developed with the support of the project StandICT.eu. | Spanish 052319 | alpha | - |
| eNanoMapper | ENM | The eNanoMapper ontology covers the full scope of terminology needed to support research into nanomaterial safety. It builds on multiple pre-existing external ontologies such as the NanoParticle Ontology.  | 10.0 | beta | - |
| HNS_Ontolgoy | HNS | The purpose of this research is to develop an ontology with subsequent testing and evaluation, for identifying utility and value. The domain that has been chosen is human nervous system (HNS) disorders. It is hypothesized here that an ontology-based patient records management system is more effective in meeting and addressing complex information needs of health-care personnel. Therefore, this study has been based on the premise that developing an ontology and using it as a component of the search interface in hospital records management systems will lead to more efficient and effective management of health-care. It is proposed here to develop an ontology of the domain of HNS disorders using a standard vocabulary such as MeSH or SNOMED CT. The principal classes of an ontology include facet analysis for arranging concepts based on their common characteristics to build mutually exclusive classes. We combine faceted theory with description logic, which helps us to better query and retrieve data by implementing an ontological model. Protégé 5.2.0 was used as ontology editor. The use of ontologies for domain modelling will be of acute help to doctors for searching patient records. In this paper we show how the faceted approach helps us to build a flexible model and retrieve better information. We use the medical domain as a case study to show examples and implementation. | 4.0 | beta | human (9606), HNS (CANCER) |
| Anatomical Therapeutic Chemical Classification | ATC | The Anatomical Therapeutic Chemical (ATC) Classification System is used for the classification of active ingredients of drugs according to the organ or system on which they act and their therapeutic, pharmacological and chemical properties. It is controlled by the World Health Organization Collaborating Centre for Drug Statistics Methodology (WHOCC), and was first published in 1976.  | ATC_2024_01_19_24_09_03 | production | organ (ORGAN) |
| Cigarette Smoke Exposure Ontology | CSEO | The CSEO has potential to become a widely used standard within the academic and industrial community. Mainly because of the emerging need of systems toxicology to controlled vocabularies and also the lack of suitable ontologies for this domain, the CSEO prepares the ground for integrative systems-based research in the exposure science. | 1.0 | beta | CSEO (CANCER) |
| Hypertension-Ontology | HPO | The Hypertension Ontology, developed using the Protege tool, serves as a comprehensive knowledge framework for representing and organizing information related to hypertensive disorders. This ontology is designed to capture various aspects of hypertension, including its types, causes, and associated risks, with the aim of providing valuable insights for healthcare professionals, researchers, and policymakers. Leveraging the power of Protege, the ontology employs the OWL (Web Ontology Language) format to define classes, properties, and relationships, ensuring semantic clarity and interoperability. The taxonomy within the ontology encompasses a wide range of hypertensive conditions, allowing for detailed classification and representation. The ontology facilitates queries and exploration of hypertensive data through the use of the DL Query module in Protege, offering a practical tool for evaluating its usefulness. To enhance accessibility, the ontology is made available for download, enabling seamless integration into existing healthcare information systems. Its development involved a rigorous review of scientific literature and consideration of established methodologies, ensuring its alignment with best practices in ontology engineering. This Hypertension Ontology aims to contribute to the advancement of hypertension research, clinical practice, and healthcare decision-making. |  | alpha | - |
| Living Environment Ontology | LEO | The Living Environment Ontology (LEO) is a structured framework designed to classify and standardize concepts related to human living conditions. It integrates keywords and concepts from diverse sources, such as questionnaires, references, and abstracts, to create a comprehensive ontology for assessing human living environments. | 1.0 | alpha | human (9606) |
| International Classification of Diseases, Version 10 - Procedure Coding System | ICD10PCS | ICD-10-PCS is a medical classification coding system for procedural codes. | ICD10PCS_2025 | production | - |
| Mapping of Epilepsy Ontologies | MEPO | The Mapping of Epilepsy Ontologies (MEPO) provides owl:sameAs relationships of similar concepts for the ontologies EpSO, ESSO, EPILONT, EPISEM, and FENICS. Two concepts are considered similar if they share the same concept name or the same synonym. Additional synonyms for each of the concepts are generated using the Snowball stemming algorithm. The software implementation is available on github within the Maven project snoke in the module snoke.ontology: https://github.com/bernd-mueller/snoke/tree/master/snoke.ontology | 0.6 | alpha | MEPO (CANCER) |
| Ontology for Nutritional Studies | ONS | Ontology For Nutritional Studies (ONS) is the first systematic effort to provide a formal ontology framework for the description of nutritional studies. | 2024-07-03 | production | ONS (CANCER) |
| Clinical Signs and Symptoms Ontology | CSSO | An ontology for describing clinical signs and symptoms. | Version 0.6 | beta | - |
| Ontology for Nutritional Epidemiology | ONE | Nutritional epidemiology is a specific research area. The generic ontologies for food science, nutrition science or medical science failed to cover the specific characteristics of nutritional epidemiologic studies. As a result, we developed the ontology for nutritional epidemiology (ONE) in order to describe nutritional epidemiologic studies accurately. | V2.2 | alpha | - |
| Computer-Based Patient Record Ontology | CPRO | A uniform core set of data elements (whose formal semantics are captured in OWL) for use in a Computer-Based Patient Record (CPR) | Last modification: July 31th, 2011 Chimezie Ogbuji | production | - |
| Prostate Cancer Ontology | PCAO | Here, we introduce the Prostate cancer ontology (PCaO) as a comprehensive semantic framework covering the whole breadth of the PCa’s knowledge domain. This ontology with a subclass-based taxonomic hierarchy not only covers the broad spectrum of major biomedical concepts from molecular to clinical features of the disease, but also the different views on disease features held by molecular biologists, clinicians and drug developers.  PCaO organized under three biological viewpoints, i.e., epidemiology, diagnosis, and treatment. | 2.0 | beta | cancer (6754), PCa (CANCER), disease (CANCER) |
| Drug Interaction and Evidence Ontology | DIDEO | The Potential Drug-drug Interaction and Potential Drug-drug Interaction Evidence Ontology. DIDEO is created as part of an NIH-funded project aiming towards developing a new knowledge base for potential drug drug interaction information. In DIDEO we represent PDDIs as information content entities and we aim to also provide the evidence for those entities using nano- and micro publication approaches. | release version 2017-11-17 | alpha | - |
| Health Level Seven Reference Implementation Model, Version 3 | HL7 | This version is the first update to Normative RIM, Release 3. It is based on changes approved in Harmonization in November 2010. This release of the RIM is bound to HL7 Abstract Data Types Release 2. | HL7V3.0_2024_07 | production | RIM (GENE_OR_GENE_PRODUCT) |
| Breast Cancer Lifestyle Ontology | BCLION | The Breast Cancer Lifestyle Ontology (BCLION) captures various factors related to lifestyle choices and behaviors that may influence the risk of developing breast cancer. It provides a structured representation of these factors in a hierarchical manner, allowing for a comprehensive understanding of the different aspects of breast cancer lifestyle. | 1.0 | alpha | Breast Cancer (CANCER), cancer (6754) |
| Dementia-Related Agitation Non-Pharmacological Treatment Ontology | DRANPTO | DRANPTO is developed for representing the domain knowledge specific to non-pharmacological intervention for agitation in dementia, particularly for the long-term care setting. | Version 1.0 | alpha | - |
| EMIF-AD ontology | EMIF-AD | Ontology developed as part of European Medical Information Framework' s (EMIF) Alzheimer’s disease (AD) project.  | 0.0.2 | alpha | AD (CANCER) |
| Neural Electronic Interface Ontology | NEIO | The Neural Electronic Interface Ontology represents the types of devices used to interface with the central nervous system to enhance (or augment) stimuli from a subject's external environment, the conditions under which these devices should be employed, the assessment of a subject's sensory ability, and the metrics for evaluating the performance. | 2024-06-25 | alpha | central nervous system (ANATOMICAL_SYSTEM) |
| Pediatric Consultation and Monitoring Ontology  | PCMO | Pediatric Consultation and Monitoring Ontology consists of 10 main classes, 10 sub-classes, 39 properties (object-type and sub-object-type properties,  data type), numerous OWL individuals and almost 90 SWRL rules (medical cases based on ARTI based diseases for pediatric patients). Almost all the rules of the knowledge base consider: Body temperature, Currently observed number of the local, major, and minor symptoms, Medications used and time of use in recent period (has drug and has drug time properties), Presence of medical cold treatment applied in recent period ⇒ only one of medical case will be inferencing result. | v.4.0 | alpha | OWL (GENE_OR_GENE_PRODUCT), SWRL (CANCER), all (1), Body (MULTI_TISSUE_STRUCTURE) |
| Epigenome Ontology | EGO | The Epigenome Ontology (EGO) a biomedical ontology for integrative epigenome knowledge representation and data analysis. | Arbor version; 1.0.1 | alpha | - |
| Placental Pathology Lesion Classification | PPLC | Placental pathology: current terminology with definitions, short descriptions, and representative images pertaining to the cause, origin and nature of placental disease. The Placental Pathology Lesion Classification was developed by a PAT External Scientific Panel Member, placental pathologist Dr. Raymond Redline. This terminology is a continuation of the effort of the Amsterdam Placental Lesion Consensus Group. | 1.0 | production | placental disease (PATHOLOGICAL_FORMATION), Placental Pathology (CANCER), placental (ORGAN) |
| Ontology of Chinese Medicine for Rheumatism | OCMR | OCMR is a biomedical ontology that represents anti-rheumatism traditional Chinese medicines and related information. | Vision Release; 1.0.11 | production | - |
| ICHOM Set Pregnancy and Childbirth | ICHOM-PROMS-PCB | The ICHOM Set of Patient-Centered Outcome Measures for Pregnancy And Childbirth is the result of hard work by a group of leading physicians, measurement experts and patients. It is a recommendation of the outcomes that matter most to persons experiencing Pregnancy And Childbirth.  |  | alpha | - |
| Base de Données Publique des Médicaments | BDPM | LA BDPM est un des référentiels du médicament gérée et distribuée par l’ANSM. Elle recense l’ensemble des spécialités et des présentations effectivement commercialisées durant les 5 dernières années. De ce fait, elle ne propose pas un historique exhaustif des codes de présentation et spécialités, celui-ci peut être obtenu grâce à la base RCP disponible sur le site de l’ANSM. | 2025-03-04 | production | - |
| Glioblastoma | GBM | Ontology of Glioblastoma |  | alpha | Glioblastoma (CANCER) |
| Illness and Injury | ILLNESSINJURY | This list of Illness and Injury (Morbidity and Mortality) classifications is an abbreviated version of the World Health Organization's (WHO) Startup Mortality List (ICD-10-SMoL).  ICD-10-SMoL is the WHO's application of ICD-10 for low-resource settings initial cause of death collection. | 1.0.2 | beta | collection (32644) |
| Liver Cancer Ontology Diagnosis Treatment | CONDOR | The CONDOR is a comprehensive framework designed to encapsulate the multifaceted aspects of liver cancer patient care. It encompasses essential elements ranging from patient profiles to disease classifications and treatment options. This ontology aims to streamline the process of liver cancer analysis and treatment, enabling more personalized and effective patient care strategies. | 0.1 | alpha | CONDOR (GENE_OR_GENE_PRODUCT), cancer (6754), disease (CANCER) |
| Genomic Clinical Decision Support Ontology | GENE-CDS | The Genomic CDS ontology aims to unify several functionalities in a single resource, being A knowledge base for clinical pharmacogenomics/pharmacogenetics that can be used for question-answering (e.g., which SNPs are associated with this drug) A rule base for clinical decision support (e.g., inferring that a patient with a specific set of SNPs requires a lowered dose of warfarin and generating a CDS message that can be viewed by clinicians) A tool for checking data consistency (e.g., highlighting which allele definitions in PharmGKB are overlapping, or which clinical decision support rules are matching the same group of patients) | 0.8.0 | alpha | - |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |
| Integrated Ontology for Symptom Phenotype Terminologies | ISPO | ISPO delivers an integrated controlled vocabulary for symptom phenotypes, with both clinical and biomedical literature synonyms in Chinese and English languages, it would enhance the semantic interoperability among heterogeneous medical data sources and clinical decision support systems. Furthermore, symptom ontology would propose a standard vocabulary for teasing the symptom-gene associations and thus become a vital requirement for symptom science and TCM biological discovery. | 2023V1 | production | - |
| Online Mendelian Inheritance in Man | OMIM | Online Mendelian Inheritance in Man, OMIM (TM). McKusick-Nathans Institute for Genetic Medicine, Johns Hopkins University (Baltimore, MD) and National Center for Biotechnology Information, National Library of Medicine | OMIM2024_08_09 | production | Man, OMIM ( (GENE_OR_GENE_PRODUCT) |
| Human Disease Ontology | DOID | Creating a comprehensive classification of human diseases organized by etiology. | 2025-05-30 | production | human (9606) |
| Sleep Domain Ontology | SDO | An application ontology for the domain of Sleep Medicine. | Last modification: Feb. 11, 2013 Sivaram Arabandi | beta | - |
| social determinants of health ontology-TaoLab | SDOHO | SDoHO aims to represent fundamental SDoH factors and their relationships in a standardized and measurable way. It has well-designed hierarchies, practical objective properties, and versatile functionalities, and the comprehensive semantic and coverage evaluation achieved promising performance compared to the existing ontologies relevant to SDoH. | SDoHO_1216 | alpha | - |
| Ontology of Physical Exercises | OPE | The Ontology of Physical Exercises (OPE) provides a reference for describing an exercise in terms of functional movements, engaged musculoskeletal system parts, related equipment or monitoring devices, intended health outcomes, as well as target ailments for which the exercise might be employed as a treatment or preventative measure.  | 0.0.1 | alpha | - |
| Neurologic Examination Ontology | NEO | An ontology to capture findings from the neurological examination.  The ontology is based on concepts from UMLS Metathesaurus and SNOMED CT.  Only abnormal findings are included.  The purpose of the ontology is to act as a limited vocabulary for coding the neurological examination for big data projects or machine learning. | 49.0 | production | - |
| The Drug Ontology | DRON | We built this ontology primarily to support comparative effectiveness researchers studying claims data. They need to be able to query U.S. National Drug Codes (NDCs) by ingredient, mechanism of action (beta-adrenergic blockade), physiological effect (diuresis), and therapeutic intent (anti-hypertensive). | 2025-04-18 | production | - |
| HPO - ORDO Ontological Module | HOOM | Orphanet provides phenotypic annotations of the rare diseases in the Orphanet nomenclature using the Human Phenotype Ontology (HPO). HOOM is a module that qualifies the annotation between a clinical entity and phenotypic abnormalities according to a frequency and by integrating the notion of diagnostic criterion. In ORDO a clinical entity is either a group of rare disorders, a rare disorder or a subtype of disorder. The phenomes branch of ORDO has been refactored as a logical import of HPO, and the HPO-ORDO phenotype disease-annotations have been provided in a series of triples in OBAN format in which associations, frequency and provenance are modeled. HOOM is provided as an OWL (Ontologies Web Languages) file, using OBAN, the Orphanet Rare Disease Ontology (ORDO), and HPO ontological models. HOOM provides extra possibilities for researchers, pharmaceutical companies and others wishing to co-analyse rare and common disease phenotype associations, or re-use the integrated ontologies in genomic variants repositories or match-making tools.  | 2.3 | beta | Human (9606), HPO (CANCER), ORDO (CANCER) |
| Environmental Determinants of Health | ENDOH | Clean air, stable climate, adequate water, sanitation and hygiene, safe use of chemicals, protection from radiation, healthy and safe workplaces, sound agricultural practices, health-supportive cities and built environments, and a preserved nature are all prerequisites for good health. | BETA | beta | all (1) |
| RxNORM | RXNORM | RxNorm is intended to cover all prescription medications approved for human use in the United States. Prescription medications from other countries may be included as opportunities allow, a principal consideration being that there be an authoritative source of information about these drugs. Drugs are added to the vocabulary as new products are put on the market. Over-the-counter (OTC) medications will be added and covered, as well, when reliable information about the medications can be found. Drugs are added to the vocabulary as new products are put on the market. An RxNorm clinical drug name reflects the active ingredients, strengths, and dose form composing that drug. When any of these elements vary, a new RxNorm drug name is created as a separate concept. Thus, an RxNorm name should exist for every strength and dose of every available combination of clinically significant ingredients. National Drug Codes (NDCs) for specific drug products are linked to that product in RxNorm. RxNorm links its names to many of the drug vocabularies commonly used in pharmacy management and drug interaction software, including those of First Databank, Micromedex®, Gold Standard Alchemy, and Multum. By providing links between these vocabularies, RxNorm can mediate messages between systems not using the same software and vocabulary. | RXNORM_20AA_240903F | production | all (1), human (9606) |

<a name="bioportal-category-human"></a>
## BioPortal Category "Human"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Promoting Health Aging through Semantic Enrichment of Solitude Research | PHASES | PHASES provides an ontology for integrating research in solitude and gerotranscendence. By integrating these areas of study, we aim to deepen our understanding of each and how they promote healthy aging. | 2025-05-16 | production | - |
| Behaviour Change Intervention Ontology | BCIO | The Human Behaviour Change Project (HBCP) funded by the Wellcome Trust, and the Advancing Prevention Research in Cancer through Ontology Tools (APRICOT) Project funded by the US National Institutes of Health, are developing the Behaviour Change Intervention Ontology and associated tools and resources to be used in reporting research; linking datasets and synthesising evidence; and AI/machine learning algorithms to predict intervention outcomes in novel scenarios. The overall aim of these projects is to automate evidence searching, synthesis and interpretation to rapidly address questions from policy-makers, practitioners and others to answer ‘What works, compared with what, how well, with what exposure, with what behaviours (for how long), for whom, in what settings and why?’. To achieve this, ontologies can help structure evidence, i.e., link evidence with a shared formal description of classes and relationships capturing domain knowledge in order to enable aggregation and semantic querying. We are developing and maintaining the Behaviour Change Intervention Ontology (BCIO) to do this. |  | production | Cancer (CANCER) |
| Heart Failure Ontology | HFO | Heart failure ontology is an attempt to define heart-failure-relevant information including the causes and risk factors, signs and symptoms, diagnostic tests and results, and treatment.  | version 1.0 | alpha | Heart (ORGAN) |
| Genomic Feature and Variation Ontology | GFVO | The Genomic Feature and Variation Ontology (GFVO) is modeled to represent genomic data using the Resource Description Format (RDF) or Linked Data using JSON (JSON-LD). It is captures the contents of data files that adhere to the Generic Feature Format Version 3 (GFF3, http://www.sequenceontology.org/resources/gff3.html), the General Transfer Format (GTF, http://mblab.wustl.edu/GTF22.html), the Genome Variation Format Version 1 (GVF, http://www.sequenceontology.org/resources/gvf.html), and the Variant Call Format (VCF, http://vcftools.sourceforge.net/specs.html), as well as the FASTA format (http://en.wikipedia.org/wiki/FASTA_format). The creation of the ontology was inspired by previous work of Robert Hoehndorf on RDF2OWL (http://code.google.com/p/rdf2owl). NOTE: GFVO in BioPortal is a simplified version without ontology imports and having class/property equivalences removed. The full ontology is available at: http://www.biointerchange.org/gfvo | 1.0.6 | production | GFVO (CANCER) |
| Ontology for Genetic Susceptibility Factor | OGSF | Ontology for Genetic Susceptibility Factor (OGSF) is an application ontology to model/represent the notion of genetic susceptibility to a specific disease or an adverse event or a pathological biological process. It is developed using BFO2.0's framwork. The ontology is under the domain of genetic epidemiology. | 2.0 | alpha | BFO2.0 (GENE_OR_GENE_PRODUCT) |
| Space Life Sciences Ontology | SLSO | The Space Life Sciences Ontology is an application ontology and is intended to support the operation of NASA's Life Sciences Data Archive (located at https://nlsp.nasa.gov) and other systems that contain space life science research data. Many kinds of scientific research in space involve specialized equipment, experimental procedures, specimens and specimen collection apparatus, supporting operational structures, and references to specific space environments and their characteristics. Often such research is extremely costly compared with planetary/terrestrial investigations, and the community supporting space life sciences is accustomed to the use of various specialized concepts and terminologies when dealing with this kind of research and data. However, the underlying conceptual models (and especially the specific labeled relationships in these models) have never been made explicit. The SLSO attempts to provide definition and organization of these models for the space research community. The SLSO was developed ab inicio using the Ontology Development Kit and imports an extends many concepts from the Basic Formal Ontology (BFO), the Ontology of Biomedical Investigations (OBI), the Environmental Ontology (ENVO), and other OBO Foundry ontologies. Projects at NASA such as the Open Science Data Repository (https://osdr.nasa.gov/) are already using many OBO ontologies, including the Radiation Biology Ontology (https://github.com/Radiobiology-Informatics-Consortium/RBO) and OBI, to index space biology investigation data. With the development of the SLSO, this practice can be extended to include all life science research in space or addressing space effects. Furthermore, the SLSO has a component that imports concepts from the Science Data Discovery Ontology, which was developed to support NASA's Science Discovery Engine (https://sciencediscoveryengine.nasa.gov). These links in the imported SDDO to concepts underlying a broad spectrum of space research (astrophysics, heliophysics, etc.) can ultimately be used to provide key capabilities for discovering and analyzing space life science data and how they relate to other kinds of scientific data regarding space environments., The Space Life Sciences Ontology is an application ontology and is intended to support the operation of NASA's Life Sciences Data Archive and other systems that contain space life science research data. | 2025-08-18 | production | specimens (CELL), collection (32644), planetary/terrestrial (MULTI_TISSUE_STRUCTURE), SLSO (CANCER), all (1) |
| Alzheimer's Disease Ontology | ADO | Alzheimer's Disease Ontology is a knowledge-based ontology that encompasses varieties of concepts related to Alzheimer's Disease, structured by upper level Basic Formal Ontology (BFO). This ontology is enriched by the interrelated entities that demonstrate the network of the understanding on Alzheimer's disease and can be readily applied for text mining. | 2.0.1 | production | - |
| Intelligent Medicine oriented Biobank Ontology | IMOBO | Intelligent Medicine oriented Biobank Ontology (IMOBO) is an ontology developed to facilitate the annotation and modeling of biobank repositories, metadata, and biobanking administration, supporting the paradigm shift towards AI in medicine. IMOBO focuses on AI modeling in both clinical applications and healthcare, with a specific emphasis on engaging clinical doctors and patients in data collection. Its goal is to structure knowledge for the future patient-centered medical paradigm. As measurement technologies and medical artificial intelligence continue to evolve, IMOBO will continually enhance its content and structure. | 0.1 | alpha | IMOBO (CANCER), collection (32644), artificial (32630) |
| Rheumatoid Arthritis ontology | RAO | A controlled vocabulary of RA-relevant information. Currently, only treatment information is available.  | Version 1.0 | alpha | - |
| International Classification of Functioning, Disability and Health | ICF | The International Classification of Functioning, Disability and Health, known more commonly as ICF, is a classification of health and health-related domains. These domains are classified from body, individual and societal perspectives by means of two lists: a list of body functions and structure, and a list of domains of activity and participation. Since an individual's functioning and disability occurs in a context, the ICF also includes a list of environmental factors. | 1.0.2 | production | ICF (GENE_OR_GENE_PRODUCT), body (ORGANISM_SUBDIVISION) |
| Pharmacogenomic Relationships Ontology | PHARE | The PHArmacogenomic RElationships Ontology (or PHARE) proposes concepts and roles to represent relationships of pharmacogenomics interest. | version id: 110114 | alpha | - |
| Human Reference Atlas | HRA | The Human Reference Atlas (HRA) is a comprehensive, high-resolution, three-dimensional atlas of all the cells in the healthy human body. The Human Reference Atlas provides standard terminologies and data structures for describing specimens, biological structures, and spatial positions linked to existing ontologies. | 2.3.0 | production | Human (9606), HRA (CANCER), all (1), cells (CELL), human (9606), specimens (CANCER) |
| Radiology Gamuts Ontology | GAMUTS | In radiology, a "gamut" is the set of conditions that can cause a specified imaging finding.  The Radiology Gamuts Ontology (RGO) links more than 16,000 diseases and imaging findings.  The causal relation (may_cause; inverse, may_be_caused_by) connects findings and possible causes. This work is made available under provisions of the Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0) license. | 2.0 | production | - |
| LDbase | LDBASE | LDbase is an NIH-funded behavioral research data repository. The LDbase taxonomy includes project descriptors for projects in the fields of educational and developmental sciences. Developed as a collaboration between researchers and librarians, this first-of-its-kind project-oriented data repository contains decades of knowledge and includes data on individuals across the full range of abilities. The LDbase ontology is used to describe research projects in the developmental sciences.  | 0.0.1 | alpha | LDbase (CELL) |
| Cancer Care: Treatment Outcome Ontology | CCTOO | The Cancer Care: Treatment Outcomes Ontology (CCTOO) catalogues various clinical endpoints used in evaluating outcomes of cancer treatment. In addition to a vocabulary list, CCTOO also curates the dependencies between individual concepts, in form of production rules, for translating clinical findings recorded in the electronic medical records (EMR) into literature-based concepts. An earlier version of this ontology was presented as a scientific abstract alongside the American Society of Clinical Oncology annual meeting 2017, Chicago, USA (J Clin Oncol 2017; 35 suppl; abstr e18137). Full paper on http://ascopubs.org/doi/abs/10.1200/CCI.18.00026 | releases/2018-05-11 | beta | Cancer (CANCER), cancer (6754) |
| Epilepsy and Seizure Ontology | EPSO | The Epilepsy Ontology models seizure types, syndromes, etiology, electrophysiology, medication, neuropathology, including  immunohistochemistry, microscopy, and anatomical description, together with terms describing preclinical model systems. The Epilepsy Ontology has been integrated with generative artificial intelligence (AI)/LLMs and machine learning models for natural language processing (NLP), drug efficacy analysis, and translational research. | 3.1 | production | artificial (32630) |
| Drug Target Ontology | DTO | Drug Target Ontology (DTO) is being developed at the University of Miami in the research group of Stephan Schürer. DTO is supported by grant U54CA189205 (Illuminating the Druggable Genome Knowledge Management Center, Tudor Oprea, PI) awarded by the NCI through the NIH Common Fund. It is a component of the Illuminating the Druggable Genome (IDG) project (https://commonfund.nih.gov/idg). The DTO project develops a novel semantic framework to formalize knowledge about drug targets with a focus on the current IDG protein families. The DTO is developed as a reference for drug targets with the longer-term goal to create a community standard that will facilitate the integration of diverse drug discovery information from numerous heterogeneous resources. The first version of the DTO consists of asserted class hierarchies of the four IDG protein families, GPCRs, kinases, ion channels, and nuclear hormone receptors. Protein classes are linked to tissue and disease via different levels of confidence. DTO also contains drug target development level classifications developed in the IDG project (http://targetcentral.ws/), and functional and qualitative annotations and classifications for kinase proteins, GPCR ligands and ion channels. DTO is modeled in OWL2-DL to enable further classification by inference reasoning and SPARQL queries. DTO is implemented following a modularization approach. DTO is used as the organizational framework for drug targets in the IDG PHAROS User Interface Portal (https://pharos.nih.gov) and also the Tin-X Target Importance and Novelty Explorer (http://newdrugtargets.org). | 1.1.1 | beta | DTO (TISSUE), U54CA189205 (GENE_OR_GENE_PRODUCT), NCI (CELL), GPCRs (GENE_OR_GENE_PRODUCT), nuclear hormone receptors (GENE_OR_GENE_PRODUCT), tissue (TISSUE), disease (CANCER) |
| Common Diabetes Medications Ontology | CDMONTO | The Common Diabetes Medications Ontology (CDMOnto) is an ontology-based decision support system that has been developed with the objective of providing appropriate treatment suggestions for diabetic patients. The CDMOnto Ontology serves as a foundation for shared semantics, domain-specific, standard, and interoperable knowledge pertinent to common diabetes medications. It encompasses the knowledge of diabetes medications and offers a unified terminology that can be disseminated and processed by medical professionals, pharmacists, and diabetic patients. |  | beta | - |
| Neomark Oral Cancer Ontology, version 4 | NEOMARK4 | Ontology that describes the medical information necessary for early detection of the oral cancer reoccurrence extracted from the NeoMark Project. | 4.1 | production | cancer (6754) |
| Persian Medicine Diseases Ontology | PMD | PMD is an ontology that represents classification and related information of Persian medicine diseases. Persian medicine also known as Iranian traditional medicine is one of famous and most ancient forms of traditional medicines. This ontology is based on the top-level ontology GFO (General Formal Ontology). | 1.0 | alpha | - |
| Parkinson and Movement Disorder Ontology | PMDO | The Parkinson Movement Disorder Ontology (PMDO) uses a systematic analysis of movement disorder instruments, extensive review of literature led by movement disorder specialists, and the AMP PD knowledge portal with an iterative ontology engineering process. The current version of PMDO is focused on parkinsonian disorders with classes representing three broad categories of: (1) neurological findings, (2) treatment plans, and (3) instruments used to evaluate various traits of PD. | 1.1 | beta | Parkinson Movement (PATHOLOGICAL_FORMATION), PMDO (GENE_OR_GENE_PRODUCT), portal (TISSUE) |
| Enhanced Personal Profile Ontology | E-PPO | The Enhanced Personal Profile Ontology (e-PPO) provides personal profile of personnel that are required for software requirement engineering task selection. Properties of the ontology includes static and dynamic features. Name, gender, date of birth, etc. are static in nature and may not change. Other static properties such as educational qualification and skills acquired can be updated while others including area of specialization and profession are dynamic and can be changed. Also, areas of specializations cannot be out of the scope of one’s profession, although one personnel can have multiple profession and skills, more than one profession can share things in common such as the same professional qualification may be required. |  | alpha | - |
| Congenital Heart Defects Ontology | CHD | An ontology that describes the Congenital Heart Defects data. | 1.1 | beta | Heart (ORGAN) |
| Human Reference Atlas Common Coordinate Framework Ontology | CCF | The Common Coordinate Framework (CCF) Ontology is an application ontology built to support the development of the Human Reference Atlas (HRA).  It unifies vocabulary for HRA construction and usage—making it possible to ingest external data sources; supporting uniform tissue sample registration that includes the spatial positioning and semantic annotations within 3D reference organs; and supporting user-formulated cross-domain queries over tissue donor properties, anatomical structures, cell types, biomarkers, and 3D space. The CCF Ontology consists of three major ontologies. The Biological Structure Ontology records anatomical structures, cell types, and biomarkers (ASCT+B) and the relationships between them.  The ASCT+B tables are authored by human experts using templated Google Sheets. The biomarkers, cell types, and anatomical structures are mapped to existing ontologies (Uberon/FMA, CL, HGNC) whenever possible.  All relationships between anatomical structures and from cell types to anatomical structures are valid Uberon and CL relationships. The Spatial Ontology defines the shape, size, location, and rotation of experimental tissue and data major anatomical structures in the 3D Reference Object Library. The Specimen Ontology captures the sex, age, and other information on donors that provided tissue data used in the construction of the HRA. | 3.0.0 | production | CCF (MULTI_TISSUE_STRUCTURE), Human (9606), HRA (CANCER), tissue (TISSUE), organs (ORGAN), cell (CELL), 3D (GENE_OR_GENE_PRODUCT), human (9606), CL (MULTI_TISSUE_STRUCTURE) |
| Coronavirus Infectious Disease Ontology | CIDO | The Ontology of Coronavirus Infectious Disease (CIDO) is a community-driven open-source biomedical ontology in the area of coronavirus infectious disease. The CIDO is developed to provide standardized human- and computer-interpretable annotation and representation of various coronavirus infectious diseases, including their etiology, transmission, pathogenesis, diagnosis, prevention, and treatment. | 2024-02-16 | production | Coronavirus (694013), CIDO (CANCER), coronavirus (694013) |
| Caroli's Disease and Syndrome Ontology | CAROLIO | We present CaroliO, an ontology for Caroli's disease and syndrome, aimed at enhancing the diagnosis and treatment of these rare liver conditions. Our work addresses the pressing need for structured knowledge resources in the field of rare diseases, which often present significant diagnostic and treatment challenges due to limited information. We developed our ontology in two versions: CaroliO VB, which is the Basic ontology; and CaroliO VA, the Applied Version, which includes more complex axioms i.e. closure axioms and axioms that indicate combinations of scenarios among symptoms, signs, treatments, and outcomes. The Basic Version follows the underlying Open World Assumption and is suitable for ontologists and future researches, while the Applied Version includes more complex axioms to better reflect real world decision making in diagnosis and choosing treatments for Caroli’s disease and syndrome based on our current knowledge. | 2.2 | alpha | Caroli (CANCER), disease (CANCER), liver (ORGAN) |
| Ontology of Information System on Live Births | ONTOSINASC | The SINASC Ontology was developed in the context of DIASUS project. This ontology intent to rescue the data structure presented in Live Birth Certificate form, from Information System on Live Births - SINASC, from Brazilian Public Health System - SUS.  | 1.0.0 | alpha | SINASC (TISSUE) |
| Radiation Oncology Ontology | ROO | The Radiation Oncology Ontology aims to cover the radiation oncology domain with a strong focus on re-using existing ontologies. | 0.5 | alpha | - |
| Digital medicine Outcomes Value Set (DOVeS) Ontology | DOVES | The Digital medicine Outcomes Value Set (DOVeS) ontology aims to provide users with a set of ontologically structured healthcare outcomes (clinical, operational, economic, technical, and others) tailored to digital medicine and digital therapeutic applications. | releases/2023-03-01 | alpha | Set (GENE_OR_GENE_PRODUCT) |
| Pneumonia Diagnosis Ontology | PNADO | Pneumonia Diagnosis Ontology (PNADO) provides the necessary knowledge to diagnose infective pneumonia. It covers symptoms and clinical signs, clinical history, pathogens, laboratory tests and results, imaging, differential diagnosis, complications and pneumonia types. | Version 2.0, 2024/07/10 | beta | PNADO (CANCER) |
| Bilingual Ontology of Alzheimer's Disease and Related Diseases | ONTOAD | OntoAD is a bilingual (English-French) domain ontology for modeling knowledge about Alzheimer's Disease and Related Syndromes. | 1.0 | beta | OntoAD (GENE_OR_GENE_PRODUCT) |
| Disease core ontology applied to Rare Diseases | HRDO | This resource was designed during a PhD in medical informatics (funded by INSERM, 2010-2012). Its components are (i) a core ontology consistent with a metamodel (disorders and groups of disorders, genes, clinical signs and their relations) and (ii) an instantiation of this metamodel with Orphanet Data (available on http://orphadata.org). Research experiments demonstrated (i) efficient classifications generation based on SPARQL Construct, (ii) perspectives in semantic audit of a knowledge base, (iii) semantic comparison with OMIM (www.omim.org) using proximity measurements and (iv) opened perspectives in knowledge sharing (LORD, http://lord.bndmr.fr). Current production services of Orphanet developed ORDO, released in 2014, an ontology synchronized with their production database. This ontology is now available on Bioportal. | V2 | alpha | - |
| Pathogen and Biomarker Shedding | SHEDDING-HUB | The Shedding Hub Vocabulary (SHV) is a SKOS-based controlled vocabulary designed to standardize terminology for studies involving pathogen and biomarker shedding. |  | alpha | SHV (GENE_OR_GENE_PRODUCT) |
| Dengue Fever Ontology | IDODEN | An ontology for dengue fever. | 0.15b | beta | - |
| Dietary lifestyle ontology | DILON | DILON was created to represent dietary lifestyle data.  Concepts were pulled from Korean dietary assessment scales and English assessment scales. Concepts are labeled in English and Korean translations are also provided. | 2 | alpha | - |
| HIV ontology | HIV | The HIV ontology encompasses all knowledge about HIV | 2.6 | production | all (1) |
| ChemoOnto | CHO | ChemoOnto, an ontology to qualify the course of chemotherapies | Version: 1.0.1 | alpha | - |
| Autism Spectrum Disorder Phenotype Ontology | ASDPTO | Autism Spectrum Disorder Phenotype Ontology encapsulates the Autism Spectrum Disorder behavioral phenotype, informed by the standard ASD assessment instruments and the currently known characteristics of this disorder. http://www.ncbi.nlm.nih.gov/pubmed/24163114 | 1 | alpha | - |
| Computer Assisted Brain Injury Rehabilitation Ontology | CABRO | a web ontology for the semantic representation of the computer assisted brain trauma rehabilitation domain. This is a novel and emerging domain, since it employs the use of robotic devices, adaptation software and machine learning to facilitate interactive, adaptive and personalized rehabilitation care, patient monitoring and assisted living.  | 0.5 | alpha | brain (ORGAN) |
| Inherited Retinal Dystrophy | HAMIDEHSGH | Inherited retinal dystrophy (IRD) as one of the rare ocular diseases encompasses different types of hereditary retinal degenerations appearing by involvement of the different layers of the posterior ocular segment. In respect of the importance of the classification systems regarding the recorded history of biologic and natural sciences, there is no classification regarding the IRD diagnoses. Therefore, we aimed to present a novel classification in this regard. |  | alpha | retinal (MULTI_TISSUE_STRUCTURE), posterior ocular (MULTI_TISSUE_STRUCTURE) |
| Sepsis Ontology | SEPON | The Sepsis Ontology (SEPON) is a disease-specific ontology that defines the concepts and relationships between concepts in the field of sepsis and related diseases, contributing to data standardization and providing precise and personalized diagnosis, treatment and prognosis recommandations for sepsis. | V1.0.0 | alpha | SEPON (CANCER) |
| Timebank Ontology | TIMEBANK | The Timebank Ontology is used to describe Timebank systems for Peer-to-Peer Service Exchange. A Timebank allows user to store virtual money, often called a Time Dollar, into a bank account. This virtual currency can be earned by helping fellow Timebank users and spent by requesting help from others. The main goal of the ontology is to facilitate the matching between helpers and requesters of help. | 0.3 | production | - |
| Non Clinical Determinants of Health | N-CDOH | Supporting ontology reuse by integrating CDoH with an existing SDoH ontology. We are presenting an overarching conception for all non-clinical determinants of health (N-CDoH). | v1 | alpha | all (1) |
| Psychometric Ontology of Experiences and Measures | POEM | POEM supports the usage of psychometric instruments, particularly questionnaires, representing their structure, measured constructs, intended contexts, and provenance. | 1.0 | alpha | - |
| Ontology of Drug Neuropathy Adverse Events | ODNAE | ODNAE is an extension of OAE with a focus on drug-associated neuropathy adverse events. | Vision Release; 1.0.13 | production | ODNAE (GENE_OR_GENE_PRODUCT) |
| Pulmonary Embolism Ontology | PE | A controlled vocabulary of PE-relevant information. Currently, only treatment information is available.  | version 1.0 | alpha | Pulmonary (ORGAN) |
| Culture, Race, Ethnicity and Nationality Ontology | CRENO | This work undertakes the possibility of using the Web Ontology Language (OWL2) to describe concepts related to "culture", "race", "ethnicity", and "nationality" - concepts that are sometimes misunderstood/misused due to lack of standardization and biases. The project also attempts to link data relating to these broad concepts to health data, like cancer rates, and shed light on understanding health disparities that impact certain communities. This work is early in development and preliminary.  | 0.6 | alpha | cancer (6754) |
| Ontology of Glucose Metabolism Disorder | OGMD | Including the disease names. phenotypes and their classifications involved in Glucose Metabolism Disorder, Diabetes. | 1 | alpha | disease (CANCER) |
| Social Determinants of Health Ontology | SOHO | SOHO has terminologies covering societal factors, such as where a person was born, grew up, works, lives, etc., along with socio-economic and community factors that affect an individual’s health. | 4 | production | - |
| Pre-eclampsia Ontology | PE-O | The PEO incorporates a wide range of key concepts and terms of PE from clinical and biomedical research in structuring the knowledge base that is specific to PE. | v15 | production | Pre-eclampsia (CANCER) |
| Occupation Ontology | OCCO | This Occupaton Ontology (OccO) is an ontology in the domain of human occupations. Current OccO version is developed based on the US Bureau of Labor Statistics Standard Occupation Classification (SOC) and the related O*Net System. The OccO development follows the principles of the Open Biological and Biomedical (OBO) Foundry. | 2023-08-20 | production | human (9606) |
| International Classification of Diseases for Oncology, 3rd edition | ICD-O-3 | Ontology used for oncology research derived from the European Cancer Information System (ECIS) | OCD-O Revision 3.2 (complete) | alpha | Cancer (CANCER) |
| Ontology for Modeling and Representation of Social Entities | OMRSE | The Ontology for Modeling and Representation of Social Entities (OMRSE) is an OBO Foundry ontology that represents the various entities that arise from human social interactions, such as social acts, social roles, social groups, and organizations. | 2025-03-26 | production | OMRSE (CANCER), human (9606) |
| Chinese Diabetes Mellitus Ontology | CDO | Note: The abbreviation of Chinese Diabetes Mellitus Ontology was changed to CDMO. CDMO is an OWL 2 ontology designed to cover the diabetes mellitus knowledge represented in Chinese language. The ontology is based on BFO and OGMS. In addition, the top hierarchical structure was adapted from SNOMED-CT, and strive for mapping with other medical ontologies in English, such as SNOMED-CT, HPO, DOID, DRON. | 3.0 | production | HPO (CANCER), DRON (GENE_OR_GENE_PRODUCT) |
| Association pour le Développement de l'Informatique en Cytologie et Anatomie Pathologique | ADICAP | Modes de prélèvement, types de technique, organes, lésions, topographies, grading | 10-2024 | beta | - |
| Diabetes Mellitus Treatment Ontology | DMTO | DMTO is an OWL 2 ontology for creating customized treatment plans for diabetic patients. The ontology is based on BFO and OGMS. In addition, it extends the DDO ontology by adding the treatment classes and axioms to the existing diagnosis part.  | 2015-11-20 | alpha | - |
| Brain-Computer Interaction (BCI) Ontology | BCI-O | The BCI ontology specifies a foundational metadata model set for real-world multimodal Brain-Computer Interaction (BCI) data capture activities.  Its structure depicts a conceptual framework that BCI applications can extend and use in their implementations, to define core concepts that capture a relevant and interoperable metadata vocabulary. This ontology is aligned to the Semantic Sensor Network Ontology (SSN): a domain-independent and end-to-end model for sensor/actuator applications.  Hence, its structure has been normalized to assist its use in conjunction with other ontologies or linked data resources to specify any particular definitions (such as units of measurement, time and time series, and location and mobility), that specialized applications in the BCI domain might need. Its spec provides general alignment data modeling guidelines for core concepts, to help BCI applications in their design. | 0.9.6 | beta | - |
| Human Phenotype Ontology | HP | The Human Phenotype Ontology is being developed to provide a structured and controlled vocabulary for the phenotypic features encountered in human hereditary and other disease. Our goal is to provide resource for the computational analysis of the human phenome, with a current focus on monogenic diseases listed in the Online Mendelian Inheritance in Man (OMIM) and Orphanet databases, for which annotations are also provided. Please see documentation and license information at http://www.human-phenotype-ontology.org | hp/releases/2025-05-06 | production | Human (9606), human (9606) |
| Human Ancestry Ontology | HANCESTRO | The Human Ancestry Ontology (HANCESTRO) provides a systematic description of the ancestry concepts used in the NHGRI-EBI Catalog of published genome-wide association studies. | 2025-04-01 | beta | Human (9606) |
| The Ontology of Information System on Mortality | ONTOSIM | The SIM Ontology was developed in the context of DIASUS project. This ontology intent to rescue the data structure presented in Death Certificate form, from Information System on Mortality - SIM, from Brazilian Public Health System - SUS. | 1.0.0 | alpha | SIM (CANCER) |
| Melanoma Ontology | MELO | Ontology of melanoma disease, composed of terms of disease, drug, symptoms, molecular abnormality，anatomical structure, diagnosis, treatment and relationships between them. | 1.0 | alpha | melanoma (CANCER), disease (CANCER) |
| COVID-19 Ontology | COVID-19 | The COVID-19 ontology covers the role of molecular and cellular entities in virus-host-interactions, in the virus life cycle,  as well as a wide spectrum of medical and epidemiological concepts linked to COVID-19. | Version Release: 1.0.0 | alpha | cellular (CELL) |
| Ontology of Risk Stratification in Mental Health | ONTRISCAL | This ontology organizes concepts and properties of risk stratification in mental health. Stratification aims to determine the patient's risk level by scoring signs and symptoms. Psychotherapeutic treatments aim to minimize patient risk by producing qualitative data reported in therapeutic sessions. Thus, the ONTRISCAL is applied in projects that organize the knowledge about quali-quanti data produced by risk stratification and mental health treatments. Quali-quanti data, semantically annotated by ONTRISCAL, can be ingested into knowledge graphs. Domain specialists can use knowledge graphs to navigate the ONTRISCAL concepts, analyze psychotherapeutic treatments, and control risks in mental health. This is a non-profit and open-source project. | 1.0 | alpha | ONTRISCAL (CANCER) |
| DSIP_Fieldlab_7 | DSIPFL7 | The Human Trafficking Vocabulary is an essential resource for researchers and analysts working with raw interview data gathered from refugees affected by human trafficking. It offers a standardized framework for categorizing and understanding various aspects of human trafficking incidents. The vocabulary includes terms for describing the nationality and location of human traffickers, the number of times traffickers are mentioned by independent sources, the type of observation obtained (first-hand or second-hand), and more. This vocabulary was created as part of Data Science in Practice Fieldlab-7, with guidance from Kai Smiths. It is continuously updated to reflect new insights and developments in the field of human trafficking research. | latest_v1 | alpha | Human (9606), human (9606) |
| Nanbyo Disease Ontology | NANDO | Creating a comprehensive hierarchical controlled vocabulary for intractable and rare disease (i.e., nanbyo) representation in Japan. | 2024-11-21 | production | - |
| Ontology for Newborn Screening Follow-up and Translational Research | ONSTR | Ontology for Newborn Screening Follow-up and Translational Research (ONSTR) is an application ontology covering the domain of newborn screening, follow-up and translational research pertaining to patients diagnosed with inheritable and congenital diseases mainly identified through newborn dried blood spot screening. ONSTR is a central component of the project Newborn Screening Follow-up Data Integration Collaborative (NBSDC). ONSTR uses the Basic Formal Ontology v2 (BFO2, v2012-07-20) as top-level ontology and extends the classes imported from OBO Foundry ontologies and candidate ontologies. | 0.4 | alpha | blood (ORGANISM_SUBSTANCE), NBSDC (CANCER), BFO2 (GENE_OR_GENE_PRODUCT), v2012-07 (GENE_OR_GENE_PRODUCT) |
| Prostate Cancer Lifestyle Ontology | PCALION | PCLiON:Prostate Cancer Lifestyle Ontology | 3.0 | production | Prostate Cancer (CANCER) |
| MultiCaRe Taxonomy | MCR_TX | The MultiCaRe Taxonomy contains the class structure used for medical image classification in the MultiCaRe dataset (v2). | 1.0 | production | - |
| Bone Dysplasia Ontology | BDO | The Bone Dysplasia ontology provides a comprehensive and formal representation of the different domain concepts involved in documenting the full complexity of the skeletal dysplasia domain. It captures and combines the genetic features that discriminate the bone dysplasias with the multitude of phenotypic characteristics manifested by patients and required to be taken into account in order to support the diagnosis process. | 1.5 | production | Bone Dysplasia (PATHOLOGICAL_FORMATION), skeletal dysplasia (PATHOLOGICAL_FORMATION), bone (ORGAN) |
| OntoPsychia, social module | ONTOPSYCHIA | Ontology of social and environmental determinants for psychiatry | 1.0 | beta | OntoPsychia (GENE_OR_GENE_PRODUCT) |
| MultiCaRe Taxonomy (ML Labels) | ML_MCR_TX | This is a reduced version of the MultiCaRe Taxonomy (MCR_TX), the class structure used for medical image classification in the MultiCaRe dataset. This reduced version includes only the labels used in Machine Learning (ML) model training. |  | beta | - |
| Radiation Oncology Structures Ontology | ROS | This ontology describes commonly contoured (anatomical and treatment planning) structures for radiation treatment planning. 22000 structures labels (created over a 16 years period in our radiation department) were extracted, classified and categorized to produce this ontology. Lymph nodes delineation international guidelines are provided. This ontology was created to ease and standardize the integration of radiation oncology data into clinical datawarehouses for multicentric studies. The high granularity of this ontology will allow precise dosimetric evaluations. | 1.2.2 | production | Lymph nodes (MULTI_TISSUE_STRUCTURE) |
| Dispedia Core Ontology | DCO | Dispedia Core Ontology is a schema for information brokering and knowledge management in the complex field of rare diseases. On the one hand, the Dispedia schema uses semantic technologies to describe patients affected by rare diseases, and on the other hand, to record expertise concerning these particular diseases in machine-processable form. In addition to the differentiated attribution of facts to the individual need for information, Dispedia enables information brokering which adjusts to the type of recipient, its character as well as its personal attitude. The schema was initially geared to an exemplified disease, Amyotrophe Lateralsclerose (ALS). | 0.7.0.0002 | beta | disease (CANCER) |
| Ontology of Vaccine Adverse Events | OVAE | The Ontology of Vaccine Adverse Events (OVAE) is a biomedical ontology in the area of vaccine adverse events. OVAE is an extension of the Ontology of Adverse Events (OAE) and the Vaccine Ontology (VO). OVAE is aimed to represent and analyze various vaccine-specific adverse events.  | Vision Release: 1.0.34 | production | VO (CANCER) |
| COVID-19 Common Datamodel | GECCO | To ensure the interoperability and compatibility among COVID-19 datasets, we present here a Common Data Model (CDM) which has been built from 11 different COVID-19 datasets from various geographical locations. The current version of the CDM holds 4639 data variables related to COVID-19 such as basic patient information (age, biological sex, and diagnosis) as well as disease-specific data variables, for example, Anosmia and Dispnea. Each of the data variables in the data model is associated with specific data types, variable mappings, value ranges, data units, and data encodings that could be used for standardizing any dataset. | 1.0.0 | production | - |
| Myocardial Infarction Ontology | MIO | The myocardial infarction ontology (MIO) is a disease-specific ontology that characterizes the knowledge field of myocardial infarction. | V1.0.0 | production | myocardial (TISSUE) |
| Homo sapiens Chromosomal Location Ontology for GRCh38 | HSCLO38 | HSCLO38's purpose is to simplify the integration of genomic experimental data at different resolution scales within biomedical knowledge graphs. It defines chromosomal locations within the GRCh38 release for chr 1-22, X,Y and M.  HSCLO38 provides 3,431,153 classes at five distinct resolution levels: whole chromosome, 1 megabase pair (Mbp), 100 kilobase pairs (kbp), 10 kbp, and 1 kbp.  Each node within these class levels is interconnected to its scale parent and to the immediate neighbors on either side to support mapping and association between genomic datasets and features. For example, the 1kbp element HSCLO38:chr1.20517001-20518000 is_a connected to its "scale parent" HSCLO38:chr1.20510001-20520000 as well as to the 5' neighbor HSCLO38:chr1.20516001-20517000 and 3' neighbor HSCLO38:chr1.20518001-20519000.  We provide knowledge-graph-ready files for HSCLO38 on the project home page. |  | alpha | HSCLO38 (CANCER), chromosomal (CELLULAR_COMPONENT), chr 1 (GENE_OR_GENE_PRODUCT), X (CELLULAR_COMPONENT), 3,431,153 (GENE_OR_GENE_PRODUCT), node (MULTI_TISSUE_STRUCTURE), 3' neighbor (CELLULAR_COMPONENT) |
| Exposure Value Set | EVS | A collection of curated and standardized values used in metadata specifications produced by ADAMS (Accelerating DAta and Metadata Standards in the Environmental Health Sciences Study of Emerging Water Contaminants) group. These values are designed to ensure uniformity in the description of the various metadata standards produced by the group.  | 0.0.6 | beta | collection (32644) |
| Presence Ontology | PREO | The Presence Ontology is a systematic vocabulary of terms with defined relationships that models the encounters taking place every day among providers, patients, and family members or friends in environments such as hospitals and clinics. The Presence Ontology provides a conceptual model for the human experience in medicine. This is a preliminary approach, but further use of methods we have developed here may aid in providing clarity and consensus to topics in healthcare. The ontology is also a model for interdisciplinary collaboration, as it was developed in conjunction with experts from bioinformatics, medicine, anthropology, business, and communication sciences through multiple iterative stages. | 1.0.0 | alpha | human (9606) |
| International Classification of Diseases Ontology | ICDO | A biomedical ontology for logical representation of the terms and relations related to the International Classification of Diseases (ICD) | 1.0.83 | beta | - |
| Combined Ontology for Inflammatory Diseases | COID | A domain ontology for inflammatory diseases, with ICD-10 used as a foundation. Ontology is expanded to include medical therapies, anatomy, and symptoms. With the addition of synonyms and lay terms derived from embedding models. | 1.0.0 | alpha | - |
| Catalog Themes | CATALOG-THEMES | Controlled Vocabulary for Health Data Themes. A custom vocabulary for classifying health-related datasets in our organization. |  | production | - |
| Imaging Biomarker Ontology | IBO | The Imaging Biomarker Ontology describes radiological measurements that characterize biological processes (such as anisotropic water diffusion in a particular tissue region) of imaged subjects (such as neuro-imaging observations) and help clinicians, particularly in the assessment of therapeutic responses and the early prediction of pathologies. IBO reuses several OBO ontologies namely BFO, IAO, PATO, GO, FMA, and OGMS.  |  | beta | tissue (TISSUE), PATO (GENE_OR_GENE_PRODUCT) |
| postpartum depression ontology | PARTUMDO | This ontology is for use in extracting postpartum depression condition codes, and also comorbidities and diseases often mistaken for postpartum depression. The ontology describes not only the codes but also the relationships between these various diseases/risk factors for use in automatically extracting relevant patient populations from Electronic Health Records |  | alpha | - |
| Mental Stress Ontology | STRONG | This ontology is a module of the human stress ontology, developed in the context of the PGHD(person-generated health data) project, supported by the National Research Foundation of Korea. This ontology included such extension for 11 relatively standard instruments.  |  | production | human (9606) |
| Ontology of Pneumology | ONTOPNEUMO | Ontology of pneumology (french version). | 0.1 | beta | - |
| Children's Health Exposure Analysis Resource | CHEAR | Children's health and wellbeing are influenced by interactions between environmental and genetic factors. NIEHS is establishing an infrastructure, the Children's Health Exposure Analysis Resource (CHEAR), to provide the extramural research community access to laboratory and data analyses that add or expand the inclusion of environmental exposures in children's health research. The goal of CHEAR is to provide tools so researchers can assess the full range of environmental exposures which may affect children's health. We anticipate that CHEAR will be used by children's health researchers conducting epidemiological or clinical studies that currently have limited consideration of environmental exposures, or those who have collected exposure data but seek more extensive analyses. | 2.8 | production | CHEAR (CANCER), extramural (ANATOMICAL_SYSTEM) |
| Orphanet Rare Disease Ontology | ORDO | The Orphanet Rare Disease ontology (ORDO) is jointly developed by Orphanet and the EBI to provide a structured vocabulary for rare diseases capturing relationships between diseases, genes and other relevant features which will form a useful resource for the computational analysis of rare diseases. It derived from the Orphanet database (www.orpha.net) , a multilingual database dedicated to rare diseases populated from literature and validated by international experts. It integrates a nosology (classification of rare diseases), relationships (gene-disease relations, epiemological data) and connections with other terminologies (MeSH, SNOMED CT, UMLS, MedDRA),databases (OMIM, UniProtKB, HGNC, ensembl, Reactome, IUPHAR, Geantlas) or classifications (ICD10). The ontology will be maintained by Orphanet and further populated with new data. Orphanet classifications can be browsed in the OLS view. The Orphanet Rare Disease Ontology is updated monthly and follows the OBO guidelines on deprecation of terms. It constitutes the official ontology of rare diseases produced and maintained by Orphanet (INSERM, US14).  | 4.7 | production | ORDO (CANCER), gene-disease (GENE_OR_GENE_PRODUCT), UMLS (CANCER) |
| COntextualised and Personalised Physical activity and Exercise Recommendations | COPPER | The Contextualised and Personalised Physical activity and Exercise Recommendations (COPPER) ontology was created in order to support action and coping planning in the context of physical activity promotion by providing personalised recommendations for activities, activity context, barriers and coping strategies. | 2025-10-07 | production | - |
| Physical Activity Ontology | PACO | Physical Activity Concept Ontology (PACO) contains various concepts that are required to describe one's physical activity.  | 0.2 | alpha | - |
| Anthology of Biosurveillance Diseases | ABD | Our disease ontology provides information on infectious diseases, disease synonyms, transmission pathways, disease agents, affected populations and disease properties. Diseases have been grouped into syndromic disease categories, such that programmers can look through relevant categories, as well as at specific diseases. Organisms, linked to both agents and populations, are structured hierarchically, to provide multiple levels of organism resolution. In addition, both disease transmission and relevant disease properties are available to search. Disease properties include tags like 'notifiable diseases' and 'economic importance' to flag particular disease characteristics that may be of interest, but are not captured elsewhere. |  | alpha | disease (CANCER), Organisms (CANCER), flag (34205) |
| Thyroid Cancer Ontology | TCO | Thyroid cancer ontology (TCO) contains 578 concepts organized under 6 biological viewpoints, i.e., Anatomy and Histoembryology, Basic Information of Patient, Cellular or Molecular Interactions, Clinical Aspects, Etiology, Pathology | 1.0 | beta | cancer (6754), TCO (CANCER), Cellular (CELL) |
| Vertebrate Homologous Organ Group Ontology | VHOG | vHOG is a multi-species anatomical ontology for the vertebrate lineage, developed as part of the Bgee project. This mapping represents homology-strict relationships, in the sense of HOM:0000007 "historical homology". Each mapping has been manually reviewed, and we provide confidence codes and references when available. | unknown | production | - |
| COVID-19 Surveillance Ontology | COVID19 | The COVID-19 Surveillance Ontology is an application ontology used to support SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2)  surveillance in primary care. The ontology facilitates monitoring of COVID-19 cases and related respiratory conditions using data from multiple brands of computerised medical record systems. | V1.0 | beta | - |
| Medical Imaging and Diagnostic Ontology | MIDO | MIDO is an OWL 2 ontology for creating a standardized format for modeling data in medical imaging tasks such as Disease Classification/ Diagnosis as well as Disease localization/segmentation | 2015-11-20 | alpha | Disease (CANCER) |
| CODO: an Ontology for collection and analysis of COviD-19 data | CODO | The initial version, CODO ontology v1.0 to v1.3, encompasses pivotal features such as classes like patients, clinical findings, symptoms, and properties like relationships between patients, travel history, and test results. It facilitates tracking specific pandemic cases, detailing how patients may have been infected, and identifying potential contacts at risk due to their connection with the infected individual. CODO also enables monitoring of clinical tests, travel history, available resources, and actual needs like ICU beds and invasive ventilators. With capabilities for advanced analytics, contact tracing, trend studies, and growth projections based on daily COVID-19 data, CODO supports the organization and representation of COVID-19 data on a daily basis. It allows semantic querying and data retrieval and aids in behavior analysis of the disease and transmission routes. | 1.5 | beta | CODO (CANCER), disease (CANCER) |
| Obstetric and Neonatal Ontology | ONTONEO | The Obstetric and Neonatal Ontology is a structured controlled vocabulary to provide a representation of the data from electronic health records (EHRs) involved in the care of the pregnant woman, and of her baby. | 2025-03-14 | alpha | EHRs (CANCER) |
| Cognitive Reserves Assessment Tasks | ONL-TASKS | This ontology is an extension of the OntoNeuroLOG ontology, developed in the context of a project supported by Russian Foundation for Basic Research (19-29-01017 grant). The ontology organizes tests that are used for assessing cognitive reserves with the focus on keywords that can be employed in mining of PubMed publications. |  | alpha | - |
| Food Interactions with Drugs Evidence Ontology | FIDEO | The Food Interactions with Drugs Evidence Ontology (FIDEO) is an ontology used for annotation and retrieval of scientific articles about food-drug interactions.  | 2023-12-18 | alpha | FIDEO (CANCER) |
| Bionutrition Ontology | BNO | The Bionutrition Ontology relates concepts and terminologies used for human nutrition in a clinical and biomedical setting. | 0.2 | alpha | human (9606) |
| Breast Cancer Ontology | BCON | The Breast Cancer Ontology (BCON) aims to capture the diverse aspects of breast cancer, including its molecular biology, pathology, epidemiology, clinical data, and lifestyle factors. By representing this knowledge in a standardized and machine-readable format, the ontology enables researchers, clinicians, and other stakeholders to access, share, and analyze breast cancer-related information more effectively. It serves as a valuable resource for organizing existing knowledge, discovering new insights, and supporting decision-making processes in the field of breast cancer research and healthcare. | 1.0 | alpha | Breast Cancer (CANCER), cancer (6754), breast (CANCER) |
| Epilepsy Ontology | EPILONT | Ontology about the epilepsy domain and epileptic seizures. Based on the diagnosis proposed by the International League Against Epilepsy (ILAE). | V1 | beta | - |
| Health Ontology for Minority Equity | HOME | An ontology to record the physical, emotional and psychological effects resulting from differences in treatment that citizens receive based on their identity when entering healthcare institutions. We present the first version of such a Health Ontology for Minority Equity (HOME) | 1 | alpha | - |
| Breast Cancer-Microbiome Ontology  | BCMO | The aim of BCMO is to construct a comprehensive and standardized data collection method to study the interaction between the microbiome and breast cancer, including clinical data, pathology, epidemiology, demographic characteristics and lifestyle of breast cancer. Through standardized methods, research results in related research fields can be structured, shareable and comparable. It will contribute to the study of how the microbiome interacts with breast cancer. |  | alpha | BCMO (GENE_OR_GENE_PRODUCT), collection (32644), cancer (6754) |
| Cell Cycle Ontology | CCO | An application ontology integrating knowledge about the eukaryotic cell cycle. | 2.10 | beta | cell (CELL) |
| Neural ElectroMagnetic Ontology | NEMO | Neural ElectroMagnetic Ontology (NEMO) describes classes of event-related brain potentials (ERP) and their properties, including spatial, temporal, and functional (cognitive/behavioral) attributes, and data-level attributes (acquisition and analysis parameters). Working (unstable), Release (stable), and Old (archival) versions of NEMO can be found on sourceforge: http://sourceforge.net/projects/nemoontologies/. | v3.10 | production | Neural ElectroMagnetic (MULTI_TISSUE_STRUCTURE), brain (ORGAN), ERP (CANCER) |
| Mental State Assessment | ONL-MSA | This ontology is a module of the OntoNeuroLOG ontology, developed in the context of the NeuroLOG project, a french project supported by ANR (ANR-06-TLOG-024 grant) aiming at integrating distributed heterogeous resources in neuroimaging. This MSA ontology module covers the field of mental state assessments, i.e. instruments, instrument variables, assessments, and resulting scores. It includes a generic domain core ontology, that provides a general model of such entities and a general taxonomy of behavioural, neurosychological and neuroclinical instruments, that can be easily extended to model any particular kind of instrument. It also includes such extensions for 8 relatively standard instruments. | v1 | production | MSA (CANCER) |
| EmpowerBP | EBP | Clinical practice guidelines, behaviour change theories, and associated behaviour change strategies for the management of hypertension. | 2.5 | alpha | - |
| SLEEP ontology | SLEEP | The SLEEP ontology aims to facilitate the collection of PGHD related to sleep patterns and behaviors. It was created by organizing sleep-related terms gathered from sleep questionnaires and specialized literature. | 1.0 | alpha | collection (32644) |
| COPD Ontology | COPDO | The COPD Ontology  is a biomedical ontology used for modelling concepts associated to chronic obstructive pulmonary disease in routine clinical databases. | 1 | production | COPD (ORGAN), pulmonary (ORGAN) |
| Cognitive Atlas Ontology | COGAT | The Cognitive Atlas is a collaborative knowledge building project that aims to develop an ontology that characterizes the state of current thought in cognitive science.  It defines a set of mental concepts along with a set of mental tasks, and the measurement relations between those classes. | 0.4 | alpha | - |
| FAIR Genomes | FG | The FAIR Genomes semantic metadata schema to power reuse of NGS data in research and healthcare. | 1.2-Minor | production | - |
| Alzheimer Disease Map Ontology | ADMO | Alzheimer’s disease (AD) pathophysiology is still imperfectly understood and current paradigms have not led to curative outcome. Omics technologies offer great promises for improving our understanding and generating new hypotheses. However, integration and interpretation of such data pose major challenges, calling for adequate knowledge models. AlzPathway is a disease map that gives a detailed and broad account of AD pathophysiology. However, AlzPathway lacks formalism, which can lead to ambiguity and misinterpretation. Ontologies are an adequate framework to overcome this limitation, through their axiomatic definitions and logical reasoning prop-erties. We introduce the AD Map Ontology (ADMO), an ontological upper model based on systems biology terms. We then propose to convert AlzPathway into an ontology and to integrate it into ADMO. | beta | beta | AD (GENE_OR_GENE_PRODUCT), disease (CANCER) |
| Resource of Asian Primary Immunodeficiency Diseases (RAPID) Phenotype Ontology | RPO | RAPID phenotype ontology presents controlled vocabulary of ontology class structures and entities of observed phenotypic terms for primary immunodeficiency diseases (PIDs) that facilitate global sharing and free exchange of PID data with users’ communities | Version 1.0 | production | PIDs (CANCER), PID (CANCER) |
| Exercise Medicine Ontology | EXMO | A core reference ontology built upon BFO about exercise medicine. This ontology contains the related terms for healthy people, people with chronic conditions and people living with diability to exercise. | 2025-06-05 | beta | - |
| Dependency Layered Ontology for Radiation Oncology | DLORO | Dependency Layered Ontology for Radiation Oncology | 1.2 | alpha | - |
| HNS_Ontolgoy | HNS | The purpose of this research is to develop an ontology with subsequent testing and evaluation, for identifying utility and value. The domain that has been chosen is human nervous system (HNS) disorders. It is hypothesized here that an ontology-based patient records management system is more effective in meeting and addressing complex information needs of health-care personnel. Therefore, this study has been based on the premise that developing an ontology and using it as a component of the search interface in hospital records management systems will lead to more efficient and effective management of health-care. It is proposed here to develop an ontology of the domain of HNS disorders using a standard vocabulary such as MeSH or SNOMED CT. The principal classes of an ontology include facet analysis for arranging concepts based on their common characteristics to build mutually exclusive classes. We combine faceted theory with description logic, which helps us to better query and retrieve data by implementing an ontological model. Protégé 5.2.0 was used as ontology editor. The use of ontologies for domain modelling will be of acute help to doctors for searching patient records. In this paper we show how the faceted approach helps us to build a flexible model and retrieve better information. We use the medical domain as a case study to show examples and implementation. | 4.0 | beta | human (9606), HNS (CANCER) |
| Living Environment Ontology | LEO | The Living Environment Ontology (LEO) is a structured framework designed to classify and standardize concepts related to human living conditions. It integrates keywords and concepts from diverse sources, such as questionnaires, references, and abstracts, to create a comprehensive ontology for assessing human living environments. | 1.0 | alpha | human (9606) |
| Mapping of Epilepsy Ontologies | MEPO | The Mapping of Epilepsy Ontologies (MEPO) provides owl:sameAs relationships of similar concepts for the ontologies EpSO, ESSO, EPILONT, EPISEM, and FENICS. Two concepts are considered similar if they share the same concept name or the same synonym. Additional synonyms for each of the concepts are generated using the Snowball stemming algorithm. The software implementation is available on github within the Maven project snoke in the module snoke.ontology: https://github.com/bernd-mueller/snoke/tree/master/snoke.ontology | 0.6 | alpha | MEPO (CANCER) |
| Ontology for Nutritional Studies | ONS | Ontology For Nutritional Studies (ONS) is the first systematic effort to provide a formal ontology framework for the description of nutritional studies. | 2024-07-03 | production | ONS (CANCER) |
| Ontology for Nutritional Epidemiology | ONE | Nutritional epidemiology is a specific research area. The generic ontologies for food science, nutrition science or medical science failed to cover the specific characteristics of nutritional epidemiologic studies. As a result, we developed the ontology for nutritional epidemiology (ONE) in order to describe nutritional epidemiologic studies accurately. | V2.2 | alpha | - |
| Prostate Cancer Ontology | PCAO | Here, we introduce the Prostate cancer ontology (PCaO) as a comprehensive semantic framework covering the whole breadth of the PCa’s knowledge domain. This ontology with a subclass-based taxonomic hierarchy not only covers the broad spectrum of major biomedical concepts from molecular to clinical features of the disease, but also the different views on disease features held by molecular biologists, clinicians and drug developers.  PCaO organized under three biological viewpoints, i.e., epidemiology, diagnosis, and treatment. | 2.0 | beta | cancer (6754), PCa (CANCER), disease (CANCER) |
| Breast Cancer Lifestyle Ontology | BCLION | The Breast Cancer Lifestyle Ontology (BCLION) captures various factors related to lifestyle choices and behaviors that may influence the risk of developing breast cancer. It provides a structured representation of these factors in a hierarchical manner, allowing for a comprehensive understanding of the different aspects of breast cancer lifestyle. | 1.0 | alpha | Breast Cancer (CANCER), cancer (6754) |
| metaBUS controlled vocabulary for organizational and applied psychology research | METABUS | The metaBUS hierarchical taxonomy of ~5,000 constructs studied in applied psychology, organizational behavior, and human resource management. | 0.1 | alpha | metaBUS (473143), human (9606) |
| Ontology of Chinese Medicine for Rheumatism | OCMR | OCMR is a biomedical ontology that represents anti-rheumatism traditional Chinese medicines and related information. | Vision Release; 1.0.11 | production | - |
| ICHOM Set Pregnancy and Childbirth | ICHOM-PROMS-PCB | The ICHOM Set of Patient-Centered Outcome Measures for Pregnancy And Childbirth is the result of hard work by a group of leading physicians, measurement experts and patients. It is a recommendation of the outcomes that matter most to persons experiencing Pregnancy And Childbirth.  |  | alpha | - |
| Base de Données Publique des Médicaments | BDPM | LA BDPM est un des référentiels du médicament gérée et distribuée par l’ANSM. Elle recense l’ensemble des spécialités et des présentations effectivement commercialisées durant les 5 dernières années. De ce fait, elle ne propose pas un historique exhaustif des codes de présentation et spécialités, celui-ci peut être obtenu grâce à la base RCP disponible sur le site de l’ANSM. | 2025-03-04 | production | - |
| Glioblastoma | GBM | Ontology of Glioblastoma |  | alpha | Glioblastoma (CANCER) |
| Illness and Injury | ILLNESSINJURY | This list of Illness and Injury (Morbidity and Mortality) classifications is an abbreviated version of the World Health Organization's (WHO) Startup Mortality List (ICD-10-SMoL).  ICD-10-SMoL is the WHO's application of ICD-10 for low-resource settings initial cause of death collection. | 1.0.2 | beta | collection (32644) |
| Genomic Clinical Decision Support Ontology | GENE-CDS | The Genomic CDS ontology aims to unify several functionalities in a single resource, being A knowledge base for clinical pharmacogenomics/pharmacogenetics that can be used for question-answering (e.g., which SNPs are associated with this drug) A rule base for clinical decision support (e.g., inferring that a patient with a specific set of SNPs requires a lowered dose of warfarin and generating a CDS message that can be viewed by clinicians) A tool for checking data consistency (e.g., highlighting which allele definitions in PharmGKB are overlapping, or which clinical decision support rules are matching the same group of patients) | 0.8.0 | alpha | - |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |
| Ontology for MicroRNA Target | OMIT | The purpose of the OMIT ontology is to establish data exchange standards and common data elements in the microRNA (miR) domain. Biologists (cell biologists in particular) and bioinformaticians can make use of OMIT to leverage emerging semantic technologies in knowledge acquisition and discovery for more effective identification of important roles performed by miRs in humans' various diseases and biological processes (usually through miRs' respective target genes). OMIT has reused and extended a set of well-established concepts from existing bio-ontologies, e.g., Gene Ontology, Sequence Ontology, Protein Ontology, NCBI Organism Taxonomy, Human Disease Ontology, Foundational Model of Anatomy, and so forth. | OBO 4.0-12122013-2338 | alpha | cell (CELL), miRs (GENE_OR_GENE_PRODUCT), humans (9605), Human (9606) |
| Human Disease Ontology | DOID | Creating a comprehensive classification of human diseases organized by etiology. | 2025-05-30 | production | human (9606) |
| Sleep Domain Ontology | SDO | An application ontology for the domain of Sleep Medicine. | Last modification: Feb. 11, 2013 Sivaram Arabandi | beta | - |
| Ontology of Physical Exercises | OPE | The Ontology of Physical Exercises (OPE) provides a reference for describing an exercise in terms of functional movements, engaged musculoskeletal system parts, related equipment or monitoring devices, intended health outcomes, as well as target ailments for which the exercise might be employed as a treatment or preventative measure.  | 0.0.1 | alpha | - |
| Ontology of Genes and Genomes | OGG | OGG is a biological ontology in the area of genes and genomes. OGG uses the Basic Formal Ontology (BFO) as its upper level ontology. This OGG document contains the genes and genomes of a list of selected organisms, including human, two viruses (HIV and influenza virus), and bacteria (B. melitensis strain 16M, E. coli strain K-12 substrain MG1655, M. tuberculosis strain H37Rv, and P. aeruginosa strain PAO1). More OGG information for other organisms (e.g., mouse, zebrafish, fruit fly, yeast, etc.) may be found in other OGG subsets. | Vision Release: 1.0.59 | production | OGG (GENE_OR_GENE_PRODUCT), human (9606), viruses (10239), HIV (57667), influenza virus (11309), bacteria (629395), E. coli (562), mouse (10090), zebrafish (7955), fruit fly (103775) |
| The Drug Ontology | DRON | We built this ontology primarily to support comparative effectiveness researchers studying claims data. They need to be able to query U.S. National Drug Codes (NDCs) by ingredient, mechanism of action (beta-adrenergic blockade), physiological effect (diuresis), and therapeutic intent (anti-hypertensive). | 2025-04-18 | production | - |
| HPO - ORDO Ontological Module | HOOM | Orphanet provides phenotypic annotations of the rare diseases in the Orphanet nomenclature using the Human Phenotype Ontology (HPO). HOOM is a module that qualifies the annotation between a clinical entity and phenotypic abnormalities according to a frequency and by integrating the notion of diagnostic criterion. In ORDO a clinical entity is either a group of rare disorders, a rare disorder or a subtype of disorder. The phenomes branch of ORDO has been refactored as a logical import of HPO, and the HPO-ORDO phenotype disease-annotations have been provided in a series of triples in OBAN format in which associations, frequency and provenance are modeled. HOOM is provided as an OWL (Ontologies Web Languages) file, using OBAN, the Orphanet Rare Disease Ontology (ORDO), and HPO ontological models. HOOM provides extra possibilities for researchers, pharmaceutical companies and others wishing to co-analyse rare and common disease phenotype associations, or re-use the integrated ontologies in genomic variants repositories or match-making tools.  | 2.3 | beta | Human (9606), HPO (CANCER), ORDO (CANCER) |
| Environmental Determinants of Health | ENDOH | Clean air, stable climate, adequate water, sanitation and hygiene, safe use of chemicals, protection from radiation, healthy and safe workplaces, sound agricultural practices, health-supportive cities and built environments, and a preserved nature are all prerequisites for good health. | BETA | beta | all (1) |

<a name="bioportal-category-human-developmental-anatomy"></a>
## BioPortal Category "Human Developmental Anatomy"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Human Developmental Anatomy Ontology, abstract version 2 | EHDAA2 | A structured controlled vocabulary of stage-specific anatomical structures of the human. It has been designed to mesh with the mouse anatomy and incorporates each Carnegie stage of development (CS1-20). The abstract version of the human developmental anatomy ontology compresses all the tissues present over Carnegie stages 1-20 into a single hierarchy. The heart, for example, is present from Carnegie Stage 9 onwards and is thus represented by 12 EHDA IDs (one for each stage). In the abstract mouse, it has a single ID so that the abstract term given as just heart really means heart (CS 9-20). Timing details will be added to the abstract version of the ontology in a future release. | releases/2024-01-11 | beta | stage-specific (GENE_OR_GENE_PRODUCT), human (9606), mouse (10090), all (1), tissues (TISSUE), heart (ORGAN), CS 9 (GENE_OR_GENE_PRODUCT) |
| Association pour le Développement de l'Informatique en Cytologie et Anatomie Pathologique | ADICAP | Modes de prélèvement, types de technique, organes, lésions, topographies, grading | 10-2024 | beta | - |
| Vertebrate Homologous Organ Group Ontology | VHOG | vHOG is a multi-species anatomical ontology for the vertebrate lineage, developed as part of the Bgee project. This mapping represents homology-strict relationships, in the sense of HOM:0000007 "historical homology". Each mapping has been manually reviewed, and we provide confidence codes and references when available. | unknown | production | - |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |

<a name="bioportal-category-imaging"></a>
## BioPortal Category "Imaging"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Cognitive Task Ontology | COGITO | CogiTO relates Tasks from Cognitive Atlas to Hierarchical Event Descriptor (HED) tags via logical relationships (as equivalent class axioms) between concepts. In other words, terms for cognitive tasks are defined by their relation to HED tags. The purpose of this ontology is to enable querying partially annotated datasets either by a Cognitive Task or by HED tags.  | 1.0.1 | beta | - |
| Biomedical Image Ontology | BIM | Biomedical images published within the scientific literature play a central role in reporting and facilitating life science discoveries. Existing ontologies and vocabularies describing biomedical images, particularly sequence images, do not provide sufficient semantic representation for image annotations generated automatically and/or semi-automatically. We present an open ontology for the annotation of biomedical images (BIM) scripted in OWL/RDF. The BIM ontology provides semantic vocabularies to describe the manually curated image annotations as well as annotations generated by online bioinformatics services using content extracted from images by the Semantic Enrichment of Biomedical Images (SEBI) system. The BIM ontology is represented in three parts; (i) image vocabularies - which holds vocabularies for the annotation of an image and/or region of interests (ROI) inside an image, as well as vocabularies to represent the pre and post processing states of an image, (ii) text entities - covers annotations from the text that are associated with an image (e.g. image captions) and provides semantic representation for NLP algorithm outputs, (iii) a provenance model  - that contributes towards the maintenance of annotation versioning. To illustrate the BIM ontology’s utility, we provide three annotation cases generated by BIM in conjunction with the SEBI image annotation engine. | 1.3 | alpha | BIM (GENE_OR_GENE_PRODUCT) |
| Radiology Gamuts Ontology | GAMUTS | In radiology, a "gamut" is the set of conditions that can cause a specified imaging finding.  The Radiology Gamuts Ontology (RGO) links more than 16,000 diseases and imaging findings.  The causal relation (may_cause; inverse, may_be_caused_by) connects findings and possible causes. This work is made available under provisions of the Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0) license. | 2.0 | production | - |
| Biomedical Informatics Research Network Project Lexicon | BIRNLEX | The BIRN Project lexicon will provide entities for data and database annotation for the BIRN project, covering anatomy, disease, data collection, project management and experimental design. | 1.3.1 | production | BIRN (CANCER), collection (32644) |
| Cardiovascular Magnetic Resonance Quality Assessment Ontology | CMR-QA | CMR-QA ontology aims at defining the semantics of free-text quality assessment annotations associated to Cardiac Magnetic Resonance (CMR) Imaging. | 0.4.5 | beta | Cardiac (ORGAN) |
| Radiation Oncology Ontology | ROO | The Radiation Oncology Ontology aims to cover the radiation oncology domain with a strong focus on re-using existing ontologies. | 0.5 | alpha | - |
| Magnetic Resonance Dataset Acquisition Ontology | ONL-MR-DA | This ontology is a module of the OntoNeuroLOG ontology, developed in the context of the NeuroLOG project, a french project supported by ANR (ANR-06-TLOG-024 grant) aiming at integrating distributed heterogeneous resources in neuroimaging. If covers the domain of Magnetic Resonance Imaging (MRI) dataset acquisition, i.e. MRI protocols, and MRI sequence parameters. In particular, it includes a multi-axial classification of MR sequences. | V1 | production | - |
| Multiplex Classification Ontology for the MultiCaRe Experiment | MCO4MCRE | Ontology used for one of the experiments from the Multiplex Classification Framework (paper under review). This ontology is based on the set of classes from the MultiCaRe dataset. The MultiCaRe dataset is a multimodal case report dataset containing medical images that were programmatically labeled based on the content of their captions (Nievas Offidani & Delrieux, 2024). Images may have more than one label, making this a multi-label classification task. For this experiment, we selected a subset of 31 classes, including “endoscopy”, “colonoscopy”, and “mri”. The original ontology is flat, and it does not contain any logical constraints among the classes. Data Article: Nievas Offidani, M. A., & Delrieux, C. A. (2024). Dataset of clinical cases, images, image labels and captions from open access case reports from PubMed Central (1990–2023). Data in Brief, 52, 110008. https://doi.org/10.1016/j.dib.2023.110008 |  | alpha | - |
| Radiomics Ontology | RO | The Radiomics Ontology aims to cover the radiomics feature domain with a strong focus on first order, shape, textural radiomics features. In addition, in the original version. it includes classes about segmentation algorithms and imaging filters. Due to a recent collaboration with the IBSI (International Biomarkers Standardization Initiative), the ontology has been expanded (v 1.6) and it includes all the entities presented in the IBSI document. Therefore, a broad coverage of not only radiomics features, but also every entity (e.g. software properties, filter properties, features extraction parameters) involved into radiomics computation has been added. In the latest version (v2.0), the ontology URIs have been updated to reflect the codes avaialble in the IBSI latest manual | 2.1 | alpha | all (1) |
| SSBD Ontology | SSBD | SSBD Ontology is a two-tier model for FAIR metadata of bioimaging. The repository layer captures the minimum information required for rapid, DOI-based release of image datasets, while the added-value layer enriches the same instances with detailed biological context (organism, strain, cell/cell-line, anatomy, GO terms) and imaging provenance (imaging method, instrument, OME-Zarr links). SSBD Ontology re-uses and aligns with OBO Foundry ontologies— ChEBI, CL, CLO, UBERON, GO, FBbi, OBI, IAO—so that multi scale data information can be integrated uniformly. The ontology is distributed under CC-BY 4.0., The SSBD Ontology is an ontology of SSBD (https://ssbd.riken.jp). SSBD is a platform for sharing and reusing bioimaging data. SSBD:database is an added-value database for bioimages and biological dynamics data. It provides a rich set of open resources for analyzing microscopy images and qunatitative data of biological objects, such as single-molecule, cell, tissue, individual, etc., and software tools for analysis. Microscopy images and quantitative biological data are collected from a variety of species, sources, and methods. These include data obtained from both experiments and computational simulations. | 2025-05-12 | alpha | CL (MULTI_TISSUE_STRUCTURE), FBbi (GENE_OR_GENE_PRODUCT), SSBD (CANCER), SSBD (https://ssbd.riken.jp). (GENE_OR_GENE_PRODUCT), cell (CELL), tissue (TISSUE) |
| Breast cancer screening recommendation ontology | BCSR-ONTO | BCSR-Onto represents knowledge about risk assessment and recommendations by diverse guidelines for breast cancer screening both for women at high and average risk of developing breast cancer. | 2.1 | alpha | BCSR-Onto (GENE_OR_GENE_PRODUCT), cancer (6754) |
| Mammography Screening Ontology | MAMO-SCR-ONTO | The Mammography Screening Ontology (MAMO-SCR-Onto) is designed to provide mammography application recommendations for the early detection of breast cancer in patients at average risk of the disease. | 1.0 | alpha | cancer (6754) |
| Brain-Computer Interaction (BCI) Ontology | BCI-O | The BCI ontology specifies a foundational metadata model set for real-world multimodal Brain-Computer Interaction (BCI) data capture activities.  Its structure depicts a conceptual framework that BCI applications can extend and use in their implementations, to define core concepts that capture a relevant and interoperable metadata vocabulary. This ontology is aligned to the Semantic Sensor Network Ontology (SSN): a domain-independent and end-to-end model for sensor/actuator applications.  Hence, its structure has been normalized to assist its use in conjunction with other ontologies or linked data resources to specify any particular definitions (such as units of measurement, time and time series, and location and mobility), that specialized applications in the BCI domain might need. Its spec provides general alignment data modeling guidelines for core concepts, to help BCI applications in their design. | 0.9.6 | beta | - |
| Cryo Electron Microscopy | CRYOEM | Ontology that describes data types and image processing operations in Cryo Electron Microscopy of Single Particles | 2021-03-09 | alpha | - |
| MultiCaRe Taxonomy | MCR_TX | The MultiCaRe Taxonomy contains the class structure used for medical image classification in the MultiCaRe dataset (v2). | 1.0 | production | - |
| MultiCaRe Taxonomy (ML Labels) | ML_MCR_TX | This is a reduced version of the MultiCaRe Taxonomy (MCR_TX), the class structure used for medical image classification in the MultiCaRe dataset. This reduced version includes only the labels used in Machine Learning (ML) model training. |  | beta | - |
| Radiation Oncology Structures Ontology | ROS | This ontology describes commonly contoured (anatomical and treatment planning) structures for radiation treatment planning. 22000 structures labels (created over a 16 years period in our radiation department) were extracted, classified and categorized to produce this ontology. Lymph nodes delineation international guidelines are provided. This ontology was created to ease and standardize the integration of radiation oncology data into clinical datawarehouses for multicentric studies. The high granularity of this ontology will allow precise dosimetric evaluations. | 1.2.2 | production | Lymph nodes (MULTI_TISSUE_STRUCTURE) |
| Imaging Biomarker Ontology | IBO | The Imaging Biomarker Ontology describes radiological measurements that characterize biological processes (such as anisotropic water diffusion in a particular tissue region) of imaged subjects (such as neuro-imaging observations) and help clinicians, particularly in the assessment of therapeutic responses and the early prediction of pathologies. IBO reuses several OBO ontologies namely BFO, IAO, PATO, GO, FMA, and OGMS.  |  | beta | tissue (TISSUE), PATO (GENE_OR_GENE_PRODUCT) |
| DICOM Controlled Terminology | DCM | DICOM Controlled Terminology (PS3.16 2025c Annex D) | 2025c_20250714 | alpha | - |
| Biological Imaging Methods Ontology | FBbi | A structured controlled vocabulary of sample preparation, visualization and imaging methods used in biomedical research. |  | production | - |
| Image and Data Quality Assessment Ontology | IDQA | Image and Data Quality Assessment for scientific data management | v50 | beta | - |
| NeuroBridge Ontology | NEUROBRG | NeuroBridge ontology is being developed as part of the NeuroBridge project to implement FAIR principles for neuroimaging and neurophysiology data using computable provenance metadata. The NeuroBridge ontology extends the W3C PROV ontology and the ProvCaRe ontology with terms from the SchizConnect and Data Bridge ontology aligned and merged into the original class structure of the ProvCaRe ontology.  | 1.0 (Release 2023) | production | SchizConnect (GENE_OR_GENE_PRODUCT) |
| Medical Imaging and Diagnostic Ontology | MIDO | MIDO is an OWL 2 ontology for creating a standardized format for modeling data in medical imaging tasks such as Disease Classification/ Diagnosis as well as Disease localization/segmentation | 2015-11-20 | alpha | Disease (CANCER) |
| Medical image simulation | OntoVIP | The OntoVIP ontology was developed in the context of the Virtual Imaging Platform project (VIP), a french project supported by ANR (ANR-09-COSI-03 grant) aiming at sharing medical image simulation resources. This ontology describes the content of the models used in medical image simulation. This ontology can be used to annotate such models in order to highlight the different entities that are present in the 3D scene to be imaged, i.e. anatomical structures, pathological structures, foreign bodies, contrast agents etc. The model allows also to associate to these entities information about their physical qualities, which are used in the medical image simulation process (to mimick physical phenomena involved in CT, MR, US and PET imaging). This ontology partly relies on the OntoNeuroLOG ontology (ONL-DP ONL-MR-DA), as well as PATO, RadLex, FMA and ChEBI. | v1 | production | PATO (CANCER) |
| Neural ElectroMagnetic Ontology | NEMO | Neural ElectroMagnetic Ontology (NEMO) describes classes of event-related brain potentials (ERP) and their properties, including spatial, temporal, and functional (cognitive/behavioral) attributes, and data-level attributes (acquisition and analysis parameters). Working (unstable), Release (stable), and Old (archival) versions of NEMO can be found on sourceforge: http://sourceforge.net/projects/nemoontologies/. | v3.10 | production | Neural ElectroMagnetic (MULTI_TISSUE_STRUCTURE), brain (ORGAN), ERP (CANCER) |
| Information Artifact Ontology | IAO | The Information Artifact Ontology (IAO) is an  ontology of information entities, originally driven by work by the OBI consortium. | 2022-11-07 | beta | - |
| Early Pregnancy Ontology | EPO | The early pregnancy ontology contains the concepts to describe ectopic pregnancy ultrasound images, organized into several subsumption hierarchies for types of ectopic pregnancies and the signs, anatomical structures and technical elements of imaging associated with ectopic pregnancy. | 1.1 | alpha | - |
| Multiplex Classification Ontology for the HyperKvasir Experiment | MCO4HKE | Ontology used for one of the experiments from the Multiplex Classification Framework (paper under review). This ontology is based on the set of classes from the HyperKvasir dataset. The HyperKvasir dataset is a manually annotated, multi-class image and video dataset for gastrointestinal endoscopy (Borgli et al., 2020). For the Multiplex experiment, only pathological finding labels were used (such as “Esophagitis grade A,” “Esophagitis grade B-D,” and “Hemorrhoids"). |  | beta | gastrointestinal endoscopy (CANCER) |
| NIDM-Results | NIDM-RESULTS | NIDM-Results is a data model that enable sharing neuroimaging results using a common descriptive standard across neuroimaging software. NIDM-Results focuses on mass-univariate studies and is mostly targeted at fMRI but is also suitable for anatomical MRI (with Voxel-Based Morphometry), and Positron Emission Tomography (PET).  NIDM-Results packs provide a lightweight solution to share maps generated by neuroimaging studies along with their metadata. Export to NIDM-Results is natively implemented in SPM, NeuroVault and CBRAIN and a Python library is ready for  integration in FSL. NeuroVault additionally can import NIDM-Results packs. | Recommendation version 2013-04-30 | production | - |
| Dataset processing | ONL-DP | This ontology is a module of the OntoNeuroLOG ontology, developed in the context of the NeuroLOG project, a french project supported by ANR (ANR-06-TLOG-024 grant) aiming at integrating distributed heterogeous resources in neuroimaging. If covers the domain of Datasets and the processing of datasets. It includes a detailed taxonomy of datasets in the area of neuroimaging (and especially MR imaging) as well as a taxonomy of medical image processing. | V1 | production | - |
| EDAM Bioimaging Ontology | EDAM-BIOIMAGING | Bioimage informatics operations, types of data, data formats, and bioimaging topics. EDAM Bioimaging is an extension of the EDAM ontology (edamontology.org) for image analysis, bioimage informatics, bioimaging, and imaging in general. It is being developed in collaboration between the ELIXIR Research Infrastructure and the NEUBIAS and COMULIS COST Actions (neubias.org CA15124, comulis.eu CA17121, funded by the Horizon 2020 Framework Programme of the European Union), in coordination with the Euro-BioImaging Research Infrastructure (PP2) and the Global BioImaging network. | alpha06 | alpha | EDAM (GENE_OR_GENE_PRODUCT), comulis.eu CA17121 (GENE_OR_GENE_PRODUCT) |

<a name="bioportal-category-immunology"></a>
## BioPortal Category "Immunology"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Allergy Ontology | ALLERGIDOC | L'ontologie ALLERGIDOC représente la terminologie et les connaissances scientifiques, cliniques et généralistes du domaine de l'allergologie. Elle a été élaborée par Marcin Trzmielewski pour l'Unité d'Exploration des Allergies du CHU de Montpellier (référents : Docteur Davide Caimmi et Prof. Pascal Demoly), dans le cadre d'une thèse en sciences de l'information, de la communication et de la documentation à l'Université Paul Valéry Montpellier 3, financée par la Région Occitanie (France) entre 2019 et 2023 et encadrée par Prof. Céline Paganelli. L'ontologie a été élaborée dans une approche constructiviste, à partir d'une analyse des pratiques informationnelles des employés de l'unité d'allergologie et d'une analyse des documents que ces employés utilisent dans leur quotidien. La version 3.0, élaborée après la thèse, comprend des modifications des versions 1.1 et 2.0 effectuées en février-mars 2025 par Marcin Trzmielewski (Laboratoire d'Etudes et de Recherches Appliquées en Sciences Sociales - LERASS), avec la contribution des étudiantes de la licence professionnelle en documentation : Anne Guerre, Nada Bouhsina, Sabrina Duc et Claire Casasso (Université de Montpellier Paul-Valéry). La version 3.0 contient également une nouvelle structuration au niveau des classes principales (facettes) et d'autres niveaux de profondeur classificatoire. L'analyse à facettes a été basée sur les catégories représentant les cibles informationnelles des professionnels d'allergologie et celles de la catégorie Descriptive Ontology for Linguistic and Cognitive Engineering (DOLCE). Les concepts, les termes et les relations qui composent l'ontologie sont issus des versions précédentes 1.1 et 2.0, des extractions terminologiques effectuées lors de l'élaboration de la version 1.0, comme des réunions de validation des versions 1.0 et 2.0. Ils proviennent également des systèmes d'organisation des connaissances existants : Ontology for General Medical Science, Information Artifact Ontology, MeSH, SNOMED, LOINC, Allergen Nomenclature, catalogues des allergènes ThermoFisher et Eurofins, site IMAIOS, taxonomies biologiques consultées sur Wikipédia, VIDAL, Gender, Sex, and Sexual Orientation Ontology, et classification d'Eckman. La version 3.0 est actuellement en cours de validation par les professionnels d'allergologie. | 3.0 | alpha | encadrée (PATHOLOGICAL_FORMATION), facettes (TISSUE), Eurofins (CELLULAR_COMPONENT), IMAIOS (CANCER), Sex (GENE_OR_GENE_PRODUCT) |
| Ontology of Host-Pathogen Interactions | OHPI | The Ontology of Host-Pathogen Interactions (OHPI) is a biomedical ontology of the host-pathogen interactions. Its development follows the OBO Foundry Principles. | 1.0.26 | production | OHPI (CANCER) |
| Therapeutic Antibody Ontology | TAON | TAON is a comprehensive ontology of the therapeutic antibody, which  included antibody terms and drug terms. | 1.1 | alpha | - |
| Biological Pathway Taxonomy | BPT | Biological Pathway Taxonomy (BPT) - hierarchical taxonomy of cellular and molecular mechanisms, metabolic reactions, and diseases that can be shown as signaling pathways. BPT was created manually based on the Elsevier Pathway Collection available for public usage. There are 3 types of entities in BPT – pathway, group, and folder of pathways and groups. Group is the list of genes and proteins associated with biological function and diseases. Pathway is an interactive model of molecular interactions in human and mammalian cells described in the literature. Each model was reconstructed based on facts published in peer-reviewed journals, with manually quality control. BPT pathway entity includes several properties such as link to the model in Pathway Studio software which allows reading supported sentences from published articles and additional information about members. A list of members for each model can be provided by request. Property “network” describes maps without logical causal relationships between members to distinguish them from actual models of molecular mechanisms (“signaling”). The index folder contains all pathways and groups sorted alphabetically. | synonymes2022 | alpha | cellular (CELL), human (9606), mammalian cells (CELL), all (1) |
| COVID-19 Ontology | COVID-19 | The COVID-19 ontology covers the role of molecular and cellular entities in virus-host-interactions, in the virus life cycle,  as well as a wide spectrum of medical and epidemiological concepts linked to COVID-19. | Version Release: 1.0.0 | alpha | cellular (CELL) |
| Ontology of Vaccine Adverse Events | OVAE | The Ontology of Vaccine Adverse Events (OVAE) is a biomedical ontology in the area of vaccine adverse events. OVAE is an extension of the Ontology of Adverse Events (OAE) and the Vaccine Ontology (VO). OVAE is aimed to represent and analyze various vaccine-specific adverse events.  | Vision Release: 1.0.34 | production | VO (CANCER) |
| Ontology of Host-Microbe Interactions | OHMI | OHMI is a biomedical ontology that represents the entities and relations in the domain of host-microbiome interactions. | 2019-09-17 | production | - |
| Immunogenetics Ontology | IMGT-ONTOLOGY | IMGT-ONTOLOGY is the first ontology for immunogenetics and immunoinformatics. It provides a semantic specification of the terms to be used in immunogenetics and immunoinformatics and manages the related knowledge, thus allowing the standardization for immunogenetics data from genome, proteome, genetics, two-dimensional (2D) and three-dimensional (3D) structures. | 1.0.3 | beta | - |
| Viral Disease Ontology Trunk | VDOT | VDOT is a middle-layer ontology for the health care domain with special emphasis on viral infections and transplantation medicine. | Initial Version release date 28/02/2017 | production | - |
| Vaccination Informed Consent Ontology | VICO | The Vaccination Informed Consent Ontology (VICO) is a community-driven ontology in the domain of vaccination/immunization informed consent. | Arbor version; 1.0.52 | production | VICO (CANCER) |
| Vaccine Ontology | VO | A biomedical ontology in the vaccine domain | 2025-05-24 | production | - |
| HIVOntology | HIVO004 | A Project of Biomedical Ontology for Systems Biomedicine. The goals of the Project include: (1). To pilot the novel researches on AIDS and HIV or other diseases by viruses, by means of the ontology and other biomedical informatics technologies, connecting the insights of the Biomedical Knowledge Integration (BMKI), systems biomedicine, complexity sciences etc. (2). To Carry out the methodology explorations for the goals above. | 020 | alpha | HIV (57667), viruses (10239) |
| Resource of Asian Primary Immunodeficiency Diseases (RAPID) Phenotype Ontology | RPO | RAPID phenotype ontology presents controlled vocabulary of ontology class structures and entities of observed phenotypic terms for primary immunodeficiency diseases (PIDs) that facilitate global sharing and free exchange of PID data with users’ communities | Version 1.0 | production | PIDs (CANCER), PID (CANCER) |
| Base de Données Publique des Médicaments | BDPM | LA BDPM est un des référentiels du médicament gérée et distribuée par l’ANSM. Elle recense l’ensemble des spécialités et des présentations effectivement commercialisées durant les 5 dernières années. De ce fait, elle ne propose pas un historique exhaustif des codes de présentation et spécialités, celui-ci peut être obtenu grâce à la base RCP disponible sur le site de l’ANSM. | 2025-03-04 | production | - |

<a name="bioportal-category-microbial-anatomy"></a>
## BioPortal Category "Microbial Anatomy"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Dictyostelium Discoideum Anatomy Ontology | DDANAT | A structured controlled vocabulary of the anatomy of the slime-mould Dictyostelium discoideum. | 1.19 | production | slime-mould Dictyostelium discoideum (CELLULAR_COMPONENT) |
| HIV ontology | HIV | The HIV ontology encompasses all knowledge about HIV | 2.6 | production | all (1) |
| Ontology of Host-Microbe Interactions | OHMI | OHMI is a biomedical ontology that represents the entities and relations in the domain of host-microbiome interactions. | 2019-09-17 | production | - |
| Fungal Gross Anatomy Ontology | FAO | A structured controlled vocabulary for the anatomy of fungi. |  | production | fungi (4751) |
| Microbial Phenotype Ontology | MPO | An ontology for describing microbial phenotypes | Version 0.74 alpha | alpha | - |
| Salmonella Serotype vocabulary | SALMO | N/A | Unknown | Unknown | - |
| DSMZ Digital Diversity Ontology | D3O | The DSMZ Digital Diversity Ontology (D3O) is an ontology designed to standardize and integrate data related to microbial diversity and associated resources within the context of the DSMZ’s Digital Diversity Knowledge Graph. This ontology provides a structured framework to represent various aspects of microbial data, including taxonomic information, strain properties, cultivation conditions, and media links. D3O facilitates cross-referencing between internal DSMZ datasets and external biological resources, supporting enhanced data discovery, interoperability, and research in microbiology, genomics, and metabolomics. The ontology includes both object properties, which link entities such as bacterial strains to external resources (e.g., literature references or media descriptions), and data properties, which associate entities with literal values (e.g., taxonomic identifiers, growth conditions). Designed for use in RDF-based knowledge graphs, D3O supports flexible integration with other life science ontologies, promoting consistency and reuse in the broader scientific community. | 1.1 | alpha | DSMZ (CELL), D3O (GENE_OR_GENE_PRODUCT) |

<a name="bioportal-category-molecule"></a>
## BioPortal Category "Molecule"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| In Vitro Methodologies Ontology for MAATrica metric | VIMM | MAATrica is a novel metric for assessing the coherence of methodologies in research papers within the fields of medicinal and nutraceutical chemistry. MAATrica employs user-controlled and customizable ontologies (in silico, in vitro and in vivo), enabling personalized analysis of research papers in the fields of medicinal and nutraceutical chemistry. Here we present the in vitro ontology we developed for the scope. |  | alpha | - |
| Whole-cell modeling ontology | WC | Ontology for whole-cell (WC) modeling. See https://wholecell.org for more information about WC modeling | 0.0.1 | alpha | - |
| Drug Target Ontology | DTO | Drug Target Ontology (DTO) is being developed at the University of Miami in the research group of Stephan Schürer. DTO is supported by grant U54CA189205 (Illuminating the Druggable Genome Knowledge Management Center, Tudor Oprea, PI) awarded by the NCI through the NIH Common Fund. It is a component of the Illuminating the Druggable Genome (IDG) project (https://commonfund.nih.gov/idg). The DTO project develops a novel semantic framework to formalize knowledge about drug targets with a focus on the current IDG protein families. The DTO is developed as a reference for drug targets with the longer-term goal to create a community standard that will facilitate the integration of diverse drug discovery information from numerous heterogeneous resources. The first version of the DTO consists of asserted class hierarchies of the four IDG protein families, GPCRs, kinases, ion channels, and nuclear hormone receptors. Protein classes are linked to tissue and disease via different levels of confidence. DTO also contains drug target development level classifications developed in the IDG project (http://targetcentral.ws/), and functional and qualitative annotations and classifications for kinase proteins, GPCR ligands and ion channels. DTO is modeled in OWL2-DL to enable further classification by inference reasoning and SPARQL queries. DTO is implemented following a modularization approach. DTO is used as the organizational framework for drug targets in the IDG PHAROS User Interface Portal (https://pharos.nih.gov) and also the Tin-X Target Importance and Novelty Explorer (http://newdrugtargets.org). | 1.1.1 | beta | DTO (TISSUE), U54CA189205 (GENE_OR_GENE_PRODUCT), NCI (CELL), GPCRs (GENE_OR_GENE_PRODUCT), nuclear hormone receptors (GENE_OR_GENE_PRODUCT), tissue (TISSUE), disease (CANCER) |
| Glycomics Ontology | GLYCO | The GlycO ontology provides an up-to-date knowledge base of experimentally verified glycan structures. Glycan (oligosaccharide or polysaccharide) structures are represented as trees of monosaccharide residues. Linkage to proteins and lipids is supported as well. Insertion of a new glycan is controlled by curation process that includes matching the new glycan against a canonical glyco-tree (a highly branched representation for a family of glycans). | 4.6.0.0 | production | - |
| Ontology of Host-Pathogen Interactions | OHPI | The Ontology of Host-Pathogen Interactions (OHPI) is a biomedical ontology of the host-pathogen interactions. Its development follows the OBO Foundry Principles. | 1.0.26 | production | OHPI (CANCER) |
| Ontology of chemically concrete, composable DNA, RNA, and protein residue and crosslinks | BPFORMS | BpForms is an ontology of molecularly-concrete DNA, RNA, and protein residues and crosslinks which can be composed into molecularly-concrete forms of DNA, RNA, proteins, and complexes. The ontology can be used with the BpForms grammar and software tools for polymers and the BcForms grammar and software tools for complexes. See https://www.bpforms.org and https://www.bcforms.org for more information. | 0.0.9 | beta | DNA (CELLULAR_COMPONENT), https://www.bcforms.org (GENE_OR_GENE_PRODUCT) |
| HIV ontology | HIV | The HIV ontology encompasses all knowledge about HIV | 2.6 | production | all (1) |
| BioAssay Ontology | BAO | The BioAssay Ontology (BAO) describes chemical biology screening assays and their results including high-throughput screening (HTS) data for the purpose of categorizing assays and data analysis.  BAO is an extensible, knowledge-based, highly expressive (currently SHOIQ(D)) description of biological assays making use of descriptive logic based features of the Web Ontology Language (OWL).  BAO currently has over 1000 classes and also makes use of several other ontologies.  It describes several concepts related to biological screening, including Perturbagen, Format, Meta Target, Design,  Detection Technology, and Endpoint.  Perturbagens are perturbing agents that are screened in an assay; they are mostly small molecules. Assay Meta Target describes what is known about the biological system and / or its components interrogated in the assay (and influenced by the Perturbagen).  Meta target can be directly described as a molecular entity (e.g. a purified protein or a protein complex), or indirectly by a biological process or event (e.g. phosphorylation).  Format describes the biological or chemical features common to each test condition in the assay and includes biochemical, cell-based, organism-based, and variations thereof.  The assay Design describes the assay methodology and implementation of how the perturbation of the biological system is translated into a detectable signal.  Detection Technology relates to the physical method and technical details to detect and record a signal.  Endpoints are the final HTS results as they are usually published (such as IC50, percent inhibition, etc.).  BAO has been designed to accommodate multiplexed assays.  All main BAO components include multiple levels of sub-categories and specification classes, which are linked via object property relationships forming an expressive knowledge-based representation. | 2.8.16 | production | SHOIQ(D (GENE_OR_GENE_PRODUCT) |
| NIST Genome Editing Lexicon | NIST_GEL | Genome editing technology is a fast-growing and rapidly advancing global bioscience field with applications in many biotechnology sectors. Genome editing is used to modify the nucleic acids of a genetic code, which can be composed of DNA or RNA, in a site-specific manner. Modifications can include insertion, deletion or alteration of nucleic acids. The technology operates by biochemical principles generally applicable to every kind of cell. Examples of genome editing technology applications with global significance include human cell-based therapeutics, agriculture, microbial based therapeutics, synthetic biology and biomanufacturing. While genome editing technology is being actively utilized, there is a need for international standardization in terms and definitions for this field, so as to enhance interpretation and communication of concepts, data and results. This document has been developed to provide a unified standard set of terms and definitions that serve the needs of biotechnology stakeholders and act as a reference for genome editing technology. | 2.2 | production | DNA (CELLULAR_COMPONENT), insertion (2673), cell (CELL), genome (CELLULAR_COMPONENT), human (9606), synthetic (32630) |
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Neuroscience Information Framework (NIF) Standard Ontology | NIFSTD | NIF Standard ontology (NIFSTD) is a core component of Neuroscience Information Framework (NIF) project (http://neuinfo.org), a semantically enhanced portal for accessing and integrating neuroscience data, tools and information. NIFSTD includes a set of modular ontologies that provide a comprehensive collection of terminologies to describe neuroscience data and resources. | 3.1 - January 15, 2018 | production | portal (TISSUE), collection (32644) |
| Cancer Chemoprevention Ontology | CANCO | The Cancer Chemoprevention Ontology constitutes a vocabulary that is able to describe and semantically interconnect the different paradigms of the cancer chemoprevention domain. | 0.3 | production | Cancer (CANCER), cancer (6754) |
| G Protein-Coupled Receptor BioAssays Ontology | BAO-GPCR | The G protein-coupled receptors (GPCRs) ontology (http://www.bioassayontology.org/bao_gpcr) describes pharmacology, biochemistry and physiology of these important and therapeutically promising class of academic and pharmaceutical research targets. Incorporation and comparison of various small molecule screening data sets, such as those deposited in PubChem, ChEMBL, KEGG, PDSP, and/or IUPHAR databases, requires a formalized electronic organization system. In order to bridge the gap between the overflow of HTS data and the bottleneck of integrated analysis tools, herein, we provide the first comprehensive GPCR ontology. The development and utility of GPCR ontology was based on previously developed BioAssay Ontology (BAO). The GPCR ontology contains information about biochemical, pharmacological, and functional properties of individual GPCRs as well as GPCR-selective ligands inclusive of their HTS screening results and other records. This provides the first all-inclusive GPCR ontology with all available data to model the relationship between the GPCR binding sites and their physiologic and pharmacologic role in physiology via small molecule chemical structures. We developed this system using emerging semantic technologies, by leveraging existing and descriptive domain level ontologies. | 1.1 | beta | GPCRs (GENE_OR_GENE_PRODUCT), all (1) |
| LipidOntology | LION | Lipid ontology (LION) associates >50,000 lipid species to biophysical, chemical, and cell biological features.  | 1.1 | production | cell (CELL) |
| NMR-Controlled Vocabulary | NMR | This is the new msi-sanctioned nmr CV, created within the COSMOS EU project, to support the nmrML data standard for nuclear magnetic resonance data in metabolomics with meaningful raw data descriptors. | 1.1.0 | beta | - |
| Ontology of Homology and Related Concepts in Biology | HOM | This ontology represents concepts related to homology, as well as other concepts used to describe similarity and non-homology. | releases/2015-01-07 | production | - |
| HIVOntology | HIVO004 | A Project of Biomedical Ontology for Systems Biomedicine. The goals of the Project include: (1). To pilot the novel researches on AIDS and HIV or other diseases by viruses, by means of the ontology and other biomedical informatics technologies, connecting the insights of the Biomedical Knowledge Integration (BMKI), systems biomedicine, complexity sciences etc. (2). To Carry out the methodology explorations for the goals above. | 020 | alpha | HIV (57667), viruses (10239) |
| Venom Ontology | CU-VO | An ontology of venoms and venom-related compounds, designed to facilitate the discovery of novel venom-derived therapeutics. | 1.0 | production | venom-derived (MULTI_TISSUE_STRUCTURE) |
| Minimal Information about any Sequence Ontology | MIXS | An OWL representation of the Minimum Information for any (x) Standard (MIxS), managed by the Genomic Standards Consortium.  | 6.2.0 | beta | OWL (GENE_OR_GENE_PRODUCT) |
| Ontology of Genes and Genomes - Mouse | OGG-MM | OGG-Mm is the OGG Mus musculus (mouse) subset. The OGG (Ontology of Genes and Genomes) is a formal ontology of genes and genomes of biological organisms. OGG is developed by following OBO Foundry principles and aligning with the BFO top ontology.  | Vision Release: 1.0.66 | production | OGG-Mm (GENE_OR_GENE_PRODUCT), Mus musculus (10090), mouse (10090), OGG (GENE_OR_GENE_PRODUCT) |
| In Silico Methodologies Ontology for MAATrica metric | SIMM | MAATrica is a novel metric for assessing the coherence of methodologies in research papers within the fields of medicinal and nutraceutical chemistry. MAATrica employs user-controlled and customizable ontologies (in silico, in vitro and in vivo), enabling personalized analysis of research papers in the fields of medicinal and nutraceutical chemistry. |  | alpha | - |
| Epigenome Ontology | EGO | The Epigenome Ontology (EGO) a biomedical ontology for integrative epigenome knowledge representation and data analysis. | Arbor version; 1.0.1 | alpha | - |
| Base de Données Publique des Médicaments | BDPM | LA BDPM est un des référentiels du médicament gérée et distribuée par l’ANSM. Elle recense l’ensemble des spécialités et des présentations effectivement commercialisées durant les 5 dernières années. De ce fait, elle ne propose pas un historique exhaustif des codes de présentation et spécialités, celui-ci peut être obtenu grâce à la base RCP disponible sur le site de l’ANSM. | 2025-03-04 | production | - |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |
| Ontology for MicroRNA Target | OMIT | The purpose of the OMIT ontology is to establish data exchange standards and common data elements in the microRNA (miR) domain. Biologists (cell biologists in particular) and bioinformaticians can make use of OMIT to leverage emerging semantic technologies in knowledge acquisition and discovery for more effective identification of important roles performed by miRs in humans' various diseases and biological processes (usually through miRs' respective target genes). OMIT has reused and extended a set of well-established concepts from existing bio-ontologies, e.g., Gene Ontology, Sequence Ontology, Protein Ontology, NCBI Organism Taxonomy, Human Disease Ontology, Foundational Model of Anatomy, and so forth. | OBO 4.0-12122013-2338 | alpha | cell (CELL), miRs (GENE_OR_GENE_PRODUCT), humans (9605), Human (9606) |

<a name="bioportal-category-mousse-anatomy"></a>
## BioPortal Category "Mouse Anatomy"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Allen Brain Atlas (ABA) Adult Mouse Brain Ontology | ABA-AMB | Allen Brain Atlas P56 Mouse Ontology | 1 | production | - |
| Vertebrate Homologous Organ Group Ontology | VHOG | vHOG is a multi-species anatomical ontology for the vertebrate lineage, developed as part of the Bgee project. This mapping represents homology-strict relationships, in the sense of HOM:0000007 "historical homology". Each mapping has been manually reviewed, and we provide confidence codes and references when available. | unknown | production | - |
| Mouse Adult Gross Anatomy Ontology | MA | A structured controlled vocabulary of the adult anatomy of the mouse (Mus). | releases/2017-02-07 | alpha | mouse (10090), Mus (862507) |
| NeuroMorpho.Org brain region ontologies | NMOBR | The species, brain regions and cell types are integrated into a single ontology for enabling OntoSearch functionality to mine the latest v7.0 release of NeuroMorpho.Org. | 2.0 | beta | brain (ORGAN), cell (CELL) |

<a name="bioportal-category-neurologic-disease"></a>
## BioPortal Category "Neurologic Disease"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Alzheimer's Disease Ontology | ADO | Alzheimer's Disease Ontology is a knowledge-based ontology that encompasses varieties of concepts related to Alzheimer's Disease, structured by upper level Basic Formal Ontology (BFO). This ontology is enriched by the interrelated entities that demonstrate the network of the understanding on Alzheimer's disease and can be readily applied for text mining. | 2.0.1 | production | - |
| Multiple sclerosis ontology | MSO | Multiple sclerosis ontology integrated with Basic formal ontology. |  | alpha | - |
| Epilepsy and Seizure Ontology | EPSO | The Epilepsy Ontology models seizure types, syndromes, etiology, electrophysiology, medication, neuropathology, including  immunohistochemistry, microscopy, and anatomical description, together with terms describing preclinical model systems. The Epilepsy Ontology has been integrated with generative artificial intelligence (AI)/LLMs and machine learning models for natural language processing (NLP), drug efficacy analysis, and translational research. | 3.1 | production | artificial (32630) |
| Parkinson and Movement Disorder Ontology | PMDO | The Parkinson Movement Disorder Ontology (PMDO) uses a systematic analysis of movement disorder instruments, extensive review of literature led by movement disorder specialists, and the AMP PD knowledge portal with an iterative ontology engineering process. The current version of PMDO is focused on parkinsonian disorders with classes representing three broad categories of: (1) neurological findings, (2) treatment plans, and (3) instruments used to evaluate various traits of PD. | 1.1 | beta | Parkinson Movement (PATHOLOGICAL_FORMATION), PMDO (GENE_OR_GENE_PRODUCT), portal (TISSUE) |
| Computational Ontology Framework for Brain Profiles | BRAIN-COF | BRAIN-COF is an ontology for representing neuroanatomical concepts aligned with the Desikan–Killiany–Tourville (DKT) brain atlas. Using hierarchical relationships, our ontology explicitly encodes a priori neuroanatomical relationships amongst brain regions. Brain-COF implements computationally efficient calculation of volumetric deviation indicative of disease pathology. | 3 | production | DKT (CANCER), brain (ORGAN), disease (CANCER) |
| Bilingual Ontology of Alzheimer's Disease and Related Diseases | ONTOAD | OntoAD is a bilingual (English-French) domain ontology for modeling knowledge about Alzheimer's Disease and Related Syndromes. | 1.0 | beta | OntoAD (GENE_OR_GENE_PRODUCT) |
| Computer Assisted Brain Injury Rehabilitation Ontology | CABRO | a web ontology for the semantic representation of the computer assisted brain trauma rehabilitation domain. This is a novel and emerging domain, since it employs the use of robotic devices, adaptation software and machine learning to facilitate interactive, adaptive and personalized rehabilitation care, patient monitoring and assisted living.  | 0.5 | alpha | brain (ORGAN) |
| Neuropsychological Integrative Ontology | NIO | An integrative ontology of the four main domains of neuropsychological diseases, brain areas, cognitive functions and neuropsychological tests | v15 | alpha | brain (ORGAN) |
| Brain-Computer Interaction (BCI) Ontology | BCI-O | The BCI ontology specifies a foundational metadata model set for real-world multimodal Brain-Computer Interaction (BCI) data capture activities.  Its structure depicts a conceptual framework that BCI applications can extend and use in their implementations, to define core concepts that capture a relevant and interoperable metadata vocabulary. This ontology is aligned to the Semantic Sensor Network Ontology (SSN): a domain-independent and end-to-end model for sensor/actuator applications.  Hence, its structure has been normalized to assist its use in conjunction with other ontologies or linked data resources to specify any particular definitions (such as units of measurement, time and time series, and location and mobility), that specialized applications in the BCI domain might need. Its spec provides general alignment data modeling guidelines for core concepts, to help BCI applications in their design. | 0.9.6 | beta | - |
| Neuroscience Information Framework (NIF) Standard Ontology | NIFSTD | NIF Standard ontology (NIFSTD) is a core component of Neuroscience Information Framework (NIF) project (http://neuinfo.org), a semantically enhanced portal for accessing and integrating neuroscience data, tools and information. NIFSTD includes a set of modular ontologies that provide a comprehensive collection of terminologies to describe neuroscience data and resources. | 3.1 - January 15, 2018 | production | portal (TISSUE), collection (32644) |
| Neurodegenerative Disease Data Ontology | NDDO | Neurodegenerative Disease Data Ontology (NDDO) is an ontology for describing data on neurodegenerative disease patients. | 0.2 | alpha | - |
| National Institutes of Health Stroke Scale Ontology | NIHSS | Domain ontology for National Institutes of Health Stroke Scale (NIHSS). Scores range from 0-42. Patients are given more points for greater deficiencies. A score of 0 indicates that the test is normal. | 2024/06 | alpha | - |
| postpartum depression ontology | PARTUMDO | This ontology is for use in extracting postpartum depression condition codes, and also comorbidities and diseases often mistaken for postpartum depression. The ontology describes not only the codes but also the relationships between these various diseases/risk factors for use in automatically extracting relevant patient populations from Electronic Health Records |  | alpha | - |
| Ontology of Amyotrophic Lateral Sclerosis, all modules | ONTOPARON | Ontology of ALS (amyotrophic lateral sclerosis), social module + coordination module + medical module developped by neurologists (ICM) and knowledge engineers (LIMICS). | 1.1 | beta | ICM (CANCER) |
| NeuroBridge Ontology | NEUROBRG | NeuroBridge ontology is being developed as part of the NeuroBridge project to implement FAIR principles for neuroimaging and neurophysiology data using computable provenance metadata. The NeuroBridge ontology extends the W3C PROV ontology and the ProvCaRe ontology with terms from the SchizConnect and Data Bridge ontology aligned and merged into the original class structure of the ProvCaRe ontology.  | 1.0 (Release 2023) | production | SchizConnect (GENE_OR_GENE_PRODUCT) |
| Epilepsy Syndrome Seizure Ontology | ESSO | This ontology contains epilepsy syndromes, seizure types, and data elements associated with them. | 1.03 | beta | - |
| BrainTeaser Ontology (BTO) | BT-ONTOLOGY | BrainTeaser Ontology (BTO) whose purpose is to jointly model both Amyotrophic Lateral Sclerosis (ALS) and Multiple Sclerosis (MS). The BTO serves multiple purposes: to provide a unified and comprehensive conceptual view about ALS and MS, which are typically dealt with separately, allowing us to coherently integrate the data coming from the different medical partners in the project; to seamlessly represent both retrospective data and prospective data, produced during the lifetime of BRAINTEASER; to allow for sharing and re-using the BRAINTEASER datasets according to Open Science and FAIR principles. | 2.0 | alpha | - |
| Epilepsy Ontology | EPILONT | Ontology about the epilepsy domain and epileptic seizures. Based on the diagnosis proposed by the International League Against Epilepsy (ILAE). | V1 | beta | - |
| Spinal Cord Injury Ontology | SCIO | Representation of pre-clinical studies in the domain of spinal cord injury therapies | 1 | alpha | spinal cord (ORGAN) |
| NeuralReprogrammingOntology | NRO | The neural reprogramming ontology encompasses the domain of neural regenerative cell reprogramming knowledge. As research progresses, researchers and specialists in the field of reprogramming could continue to expand the bounds of this foundation, so adding to its depth. | 1.0.0 | beta | neural (TISSUE), cell (CELL) |
| Epilepsy Semiology Ontology | ESO_C | Semiology of Epilepsy  | 1.0 | alpha | - |
| HNS_Ontolgoy | HNS | The purpose of this research is to develop an ontology with subsequent testing and evaluation, for identifying utility and value. The domain that has been chosen is human nervous system (HNS) disorders. It is hypothesized here that an ontology-based patient records management system is more effective in meeting and addressing complex information needs of health-care personnel. Therefore, this study has been based on the premise that developing an ontology and using it as a component of the search interface in hospital records management systems will lead to more efficient and effective management of health-care. It is proposed here to develop an ontology of the domain of HNS disorders using a standard vocabulary such as MeSH or SNOMED CT. The principal classes of an ontology include facet analysis for arranging concepts based on their common characteristics to build mutually exclusive classes. We combine faceted theory with description logic, which helps us to better query and retrieve data by implementing an ontological model. Protégé 5.2.0 was used as ontology editor. The use of ontologies for domain modelling will be of acute help to doctors for searching patient records. In this paper we show how the faceted approach helps us to build a flexible model and retrieve better information. We use the medical domain as a case study to show examples and implementation. | 4.0 | beta | human (9606), HNS (CANCER) |
| Mapping of Epilepsy Ontologies | MEPO | The Mapping of Epilepsy Ontologies (MEPO) provides owl:sameAs relationships of similar concepts for the ontologies EpSO, ESSO, EPILONT, EPISEM, and FENICS. Two concepts are considered similar if they share the same concept name or the same synonym. Additional synonyms for each of the concepts are generated using the Snowball stemming algorithm. The software implementation is available on github within the Maven project snoke in the module snoke.ontology: https://github.com/bernd-mueller/snoke/tree/master/snoke.ontology | 0.6 | alpha | MEPO (CANCER) |
| EMIF-AD ontology | EMIF-AD | Ontology developed as part of European Medical Information Framework' s (EMIF) Alzheimer’s disease (AD) project.  | 0.0.2 | alpha | AD (CANCER) |
| Neural Electronic Interface Ontology | NEIO | The Neural Electronic Interface Ontology represents the types of devices used to interface with the central nervous system to enhance (or augment) stimuli from a subject's external environment, the conditions under which these devices should be employed, the assessment of a subject's sensory ability, and the metrics for evaluating the performance. | 2024-06-25 | alpha | central nervous system (ANATOMICAL_SYSTEM) |
| Human Disease Ontology | DOID | Creating a comprehensive classification of human diseases organized by etiology. | 2025-05-30 | production | human (9606) |
| Sleep Domain Ontology | SDO | An application ontology for the domain of Sleep Medicine. | Last modification: Feb. 11, 2013 Sivaram Arabandi | beta | - |
| Neurologic Examination Ontology | NEO | An ontology to capture findings from the neurological examination.  The ontology is based on concepts from UMLS Metathesaurus and SNOMED CT.  Only abnormal findings are included.  The purpose of the ontology is to act as a limited vocabulary for coding the neurological examination for big data projects or machine learning. | 49.0 | production | - |
| Women with Epilepsy of Child-bearing Age Ontology | WWECA | This is an ontology for the management of women of child-bearing age with epilepsy. | 3.0 | alpha | - |

<a name="bioportal-category-neurological-disorder"></a>
## BioPortal Category "Neurological Disorder"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| NND_Neuropathological_Diagnosis | NND_ND | Netherlands Neurogenetics Database Neuropathological Diagnosis ontology. This ontology describes the most common neurodegenerative and psychiatric disorders in the Netherlands Brain Bank and can be used for cross-disorder brain research. | 2023-03-31 | beta | Brain Bank (CANCER), brain (ORGAN) |
| Alzheimer's Disease Ontology | ADO | Alzheimer's Disease Ontology is a knowledge-based ontology that encompasses varieties of concepts related to Alzheimer's Disease, structured by upper level Basic Formal Ontology (BFO). This ontology is enriched by the interrelated entities that demonstrate the network of the understanding on Alzheimer's disease and can be readily applied for text mining. | 2.0.1 | production | - |
| Epilepsy and Seizure Ontology | EPSO | The Epilepsy Ontology models seizure types, syndromes, etiology, electrophysiology, medication, neuropathology, including  immunohistochemistry, microscopy, and anatomical description, together with terms describing preclinical model systems. The Epilepsy Ontology has been integrated with generative artificial intelligence (AI)/LLMs and machine learning models for natural language processing (NLP), drug efficacy analysis, and translational research. | 3.1 | production | artificial (32630) |
| EpilepsyOntology | EPIO | The Epilepsy Ontology (EPO) is an assembly of structured knowledge on various aspects of epilepsy, developed according to basic formal ontology (BFO) and Open Biological and Biomedical Ontology (OBO) Foundry principles. Entities and definitions are collected from the latest International League against Epilepsy (ILAE) classification, as well as from domain-specific ontologies such as Epilepsy Ontology (EPILONT), Epilepsy Syndrome Seizure Ontology (ESSO), Epilepsy Semiology(EPISEM) and Epilepsy and Seizure Ontology (EPSO) and scientific literature. This ontology is intended to be used for data management and for text mining purposes. The current release of the ontology is publicly available and is a community based effort to assemble various facets of the complex disease Epilepsy. | Version Release: 1.0.0 | alpha | Semiology(EPISEM (GENE_OR_GENE_PRODUCT) |
| NND_Clinical_history | NND_CH | Netherlands Neurogenetics Database Clinical History ontology. This ontology describes clinical signs, symptoms and findings from the most common neurodegenerative and psychiatric disorders in the Netherlands Brain Bank and can be used for cross-disorder brain research. These attributes encompass 19 groupings, including ‘Disturbances in mood and behavior’, ‘Extrapyramidal signs’, ‘Cognitive and memory impairment’ in 5 broad domains: Psychiatric, Cognitive, Motor, Sensory/autonomic, and General. | 1 | beta | Brain Bank (CANCER), brain (ORGAN) |
| Parkinson and Movement Disorder Ontology | PMDO | The Parkinson Movement Disorder Ontology (PMDO) uses a systematic analysis of movement disorder instruments, extensive review of literature led by movement disorder specialists, and the AMP PD knowledge portal with an iterative ontology engineering process. The current version of PMDO is focused on parkinsonian disorders with classes representing three broad categories of: (1) neurological findings, (2) treatment plans, and (3) instruments used to evaluate various traits of PD. | 1.1 | beta | Parkinson Movement (PATHOLOGICAL_FORMATION), PMDO (GENE_OR_GENE_PRODUCT), portal (TISSUE) |
| Bilingual Ontology of Alzheimer's Disease and Related Diseases | ONTOAD | OntoAD is a bilingual (English-French) domain ontology for modeling knowledge about Alzheimer's Disease and Related Syndromes. | 1.0 | beta | OntoAD (GENE_OR_GENE_PRODUCT) |
| Autism Spectrum Disorder Phenotype Ontology | ASDPTO | Autism Spectrum Disorder Phenotype Ontology encapsulates the Autism Spectrum Disorder behavioral phenotype, informed by the standard ASD assessment instruments and the currently known characteristics of this disorder. http://www.ncbi.nlm.nih.gov/pubmed/24163114 | 1 | alpha | - |
| Computer Assisted Brain Injury Rehabilitation Ontology | CABRO | a web ontology for the semantic representation of the computer assisted brain trauma rehabilitation domain. This is a novel and emerging domain, since it employs the use of robotic devices, adaptation software and machine learning to facilitate interactive, adaptive and personalized rehabilitation care, patient monitoring and assisted living.  | 0.5 | alpha | brain (ORGAN) |
| Clinical Trials Ontology - Neurodegenerative Diseases | CTO-NDD | The CTO-NDD stands for Clinical Trial Ontology-Neurodegenerative Diseases, which was built with the purpose of organizing the knowledge domain of clinical trials in neurodegeneration field. This resource has been created for use in the IMI-funded AETIONOMY project (www.aetionomy.org).  | 0.6 | alpha | - |
| Brain-Computer Interaction (BCI) Ontology | BCI-O | The BCI ontology specifies a foundational metadata model set for real-world multimodal Brain-Computer Interaction (BCI) data capture activities.  Its structure depicts a conceptual framework that BCI applications can extend and use in their implementations, to define core concepts that capture a relevant and interoperable metadata vocabulary. This ontology is aligned to the Semantic Sensor Network Ontology (SSN): a domain-independent and end-to-end model for sensor/actuator applications.  Hence, its structure has been normalized to assist its use in conjunction with other ontologies or linked data resources to specify any particular definitions (such as units of measurement, time and time series, and location and mobility), that specialized applications in the BCI domain might need. Its spec provides general alignment data modeling guidelines for core concepts, to help BCI applications in their design. | 0.9.6 | beta | - |
| Neuroscience Information Framework (NIF) Standard Ontology | NIFSTD | NIF Standard ontology (NIFSTD) is a core component of Neuroscience Information Framework (NIF) project (http://neuinfo.org), a semantically enhanced portal for accessing and integrating neuroscience data, tools and information. NIFSTD includes a set of modular ontologies that provide a comprehensive collection of terminologies to describe neuroscience data and resources. | 3.1 - January 15, 2018 | production | portal (TISSUE), collection (32644) |
| Ontology of Risk Stratification in Mental Health | ONTRISCAL | This ontology organizes concepts and properties of risk stratification in mental health. Stratification aims to determine the patient's risk level by scoring signs and symptoms. Psychotherapeutic treatments aim to minimize patient risk by producing qualitative data reported in therapeutic sessions. Thus, the ONTRISCAL is applied in projects that organize the knowledge about quali-quanti data produced by risk stratification and mental health treatments. Quali-quanti data, semantically annotated by ONTRISCAL, can be ingested into knowledge graphs. Domain specialists can use knowledge graphs to navigate the ONTRISCAL concepts, analyze psychotherapeutic treatments, and control risks in mental health. This is a non-profit and open-source project. | 1.0 | alpha | ONTRISCAL (CANCER) |
| Ontology of amyotrophic  lateral sclerosis, social module | ONTOPARON_SOCIAL | Ontology of ALS (amyotrophic lateral sclerosis), social module developped by neurologist (ICM) and knowledge engineers (LIMICS). | 1.0 | alpha | ICM (CANCER) |
| Neurodegenerative Disease Data Ontology | NDDO | Neurodegenerative Disease Data Ontology (NDDO) is an ontology for describing data on neurodegenerative disease patients. | 0.2 | alpha | - |
| postpartum depression ontology | PARTUMDO | This ontology is for use in extracting postpartum depression condition codes, and also comorbidities and diseases often mistaken for postpartum depression. The ontology describes not only the codes but also the relationships between these various diseases/risk factors for use in automatically extracting relevant patient populations from Electronic Health Records |  | alpha | - |
| Cognitive Reserves Assessment Tasks | ONL-TASKS | This ontology is an extension of the OntoNeuroLOG ontology, developed in the context of a project supported by Russian Foundation for Basic Research (19-29-01017 grant). The ontology organizes tests that are used for assessing cognitive reserves with the focus on keywords that can be employed in mining of PubMed publications. |  | alpha | - |
| Epilepsy Ontology | EPILONT | Ontology about the epilepsy domain and epileptic seizures. Based on the diagnosis proposed by the International League Against Epilepsy (ILAE). | V1 | beta | - |
| Epilepsy Semiology | EPISEM | An ontology designed to capture the semiology of epilepsy including ictal, post-ictal, inter-ictal, and aura signs. For the branches entitled "disease", "etiologic factor", "epidemiologic factor"--see ESSO ontology | 5.0 | alpha | disease (CANCER) |
| Mental State Assessment | ONL-MSA | This ontology is a module of the OntoNeuroLOG ontology, developed in the context of the NeuroLOG project, a french project supported by ANR (ANR-06-TLOG-024 grant) aiming at integrating distributed heterogeous resources in neuroimaging. This MSA ontology module covers the field of mental state assessments, i.e. instruments, instrument variables, assessments, and resulting scores. It includes a generic domain core ontology, that provides a general model of such entities and a general taxonomy of behavioural, neurosychological and neuroclinical instruments, that can be easily extended to model any particular kind of instrument. It also includes such extensions for 8 relatively standard instruments. | v1 | production | MSA (CANCER) |
| Functional Electrophysiology Nomenclature for Ion Channels | FENICS | The Functional Epilepsy Nomenclature for Ion Channels (FENICS) is allowing the consistent evaluation of electrophysiology experiments on Ion Channels in the context of neurological disorders, in particular epilepsy.  | 0.0.5 | alpha | Ion (TISSUE) |
| Spinal Cord Injury Ontology | SCIO | Representation of pre-clinical studies in the domain of spinal cord injury therapies | 1 | alpha | spinal cord (ORGAN) |
| Mapping of Epilepsy Ontologies | MEPO | The Mapping of Epilepsy Ontologies (MEPO) provides owl:sameAs relationships of similar concepts for the ontologies EpSO, ESSO, EPILONT, EPISEM, and FENICS. Two concepts are considered similar if they share the same concept name or the same synonym. Additional synonyms for each of the concepts are generated using the Snowball stemming algorithm. The software implementation is available on github within the Maven project snoke in the module snoke.ontology: https://github.com/bernd-mueller/snoke/tree/master/snoke.ontology | 0.6 | alpha | MEPO (CANCER) |
| Neural Electronic Interface Ontology | NEIO | The Neural Electronic Interface Ontology represents the types of devices used to interface with the central nervous system to enhance (or augment) stimuli from a subject's external environment, the conditions under which these devices should be employed, the assessment of a subject's sensory ability, and the metrics for evaluating the performance. | 2024-06-25 | alpha | central nervous system (ANATOMICAL_SYSTEM) |
| Human Disease Ontology | DOID | Creating a comprehensive classification of human diseases organized by etiology. | 2025-05-30 | production | human (9606) |
| Sleep Domain Ontology | SDO | An application ontology for the domain of Sleep Medicine. | Last modification: Feb. 11, 2013 Sivaram Arabandi | beta | - |
| Neurologic Examination Ontology | NEO | An ontology to capture findings from the neurological examination.  The ontology is based on concepts from UMLS Metathesaurus and SNOMED CT.  Only abnormal findings are included.  The purpose of the ontology is to act as a limited vocabulary for coding the neurological examination for big data projects or machine learning. | 49.0 | production | - |
| Atherosclerotic Cerebrovascular Disease Ontology | ACVD_ONTOLOGY | The ontology describes the related aspects of atherosclerotic cerebrovascular disease, and provides a clear set of cross-lingual concepts and terms with an explicit hierarchical structure, helping scientific researchers to quickly retrieve relevant medical literature, assisting data scientists to efficiently identify relevant contents in electronic health records, and providing a clear domain framework for academic reference. The current ontology included 10 top-level classes, respectively clinical manifestation, comorbidity, complication, diagnosis, model of atherosclerotic cerebrovascular disease, pathogenesis, prevention, rehabilitation, risk factor, and treatment. Totally, there are 1715 concepts in the 11-level ontology, covering 4588 Chinese terms, 6617 English terms, and 972 definitions. | 2.0 | alpha | atherosclerotic cerebrovascular (PATHOLOGICAL_FORMATION) |
| Women with Epilepsy of Child-bearing Age Ontology | WWECA | This is an ontology for the management of women of child-bearing age with epilepsy. | 3.0 | alpha | - |

<a name="bioportal-category-phenotype"></a>
## BioPortal Category "Phenotype"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| New Approach Methodology Ontology | NAMO | The New Approach Methodology Ontology and Schema (NAMO) is a comprehensive data framework designed to standardize and integrate information about alternative methods to animal testing in biomedical research. Built on LinkML, NAMO provides a common vocabulary and data structure for describing diverse model systems, from cellular cultures and organ-on-chip devices to computational models and digital twins. |  | production | Schema (GENE_OR_GENE_PRODUCT), cellular cultures (CELL), twins (CANCER) |
| INRAE Thesaurus | INRAETHES | INRAE Thesaurus contains more than 16,000 concepts relevant to domains covered by the National Research Institute for Agriculture, Food and the Environment. Concepts are described with terms in French (100%) and English (81%) terms, some having textual definitions and mappings to other semantic artifacts. The concepts are organized in 12 domains subdivided in thematic sections called micro-thesauri., Thésaurus ouvert et partagé couvrant les domaines de recherche d’INRAE. Il sert de référentiel au sein de l’institut pour indexer et annoter des documents, pages web, descriptions d’activités, jeux de données, à des fins de recherche ou d’analyse de l’information. | 2.4 | production | INRAE (CANCER), fins (ORGAN) |
| Heart Failure Ontology | HFO | Heart failure ontology is an attempt to define heart-failure-relevant information including the causes and risk factors, signs and symptoms, diagnostic tests and results, and treatment.  | version 1.0 | alpha | Heart (ORGAN) |
| Clinical smell and taste disorders | CSTD | This is an knowledge domain about smell and taste disoders. The aim of this ontology domain is the facilitate high throughput and deep-phenotyping assessment in clinical smell and taste research. | 1 | alpha | - |
| Coleoptera Anatomy Ontology | COLAO | The Coleoptera Anatomy Ontology contains terms used for describing the anatomy and phenotype of beetles in biodiversity research. It has been built using the Ontology Develoment Kit with the Ontology for the Anatomy of the Insect SkeletoMuscular system (AISM) as a backbone. The project is open for collaboration. | 2024-06-21 | alpha | beetles (7041) |
| Plant Gall Ontology | GALLONT | Ontology of plant gall phenotypes. Plant galls are novel plant structures, generated by plants in response to biotic stressors. This ontology is used to annotate gall phenotypes (e.g., their colors, textures, sizes, locations on the plant) in a semantic way, in order to facilitate discoveries about the genetic and physiologic mechanisms responsible for such phenotypes. The ontology can also be used as a controlled vocabulary for natural language descriptions of plant galls. | 2024-04-19 | production | gall (ORGAN), galls (CELL), plants (3193) |
| Cell Line Ontology | CLO | The Cell Line Ontology (CLO) is a community-based ontology in domain of biological cell lines with a focus on permanent cell lines from culture collections. Upper ontology structures that frame the skeleton of CLO include Basic Formal Ontology and Relation Ontology. Cell lines contained in CLO are associated with terms from other ontologies such as Cell Type Ontology, NCBI Taxonomy, and Ontology for Biomedical Investigation. A common design pattern for the cell line is used to model cell lines and their attributes, the Jurkat cell line provides ane xample.  Currently CLO contains over 36,000 cell line entries obtained from ATCC, HyperCLDB, Coriell, and bymanual curation. The cell lines are derived from 194 cell types, 656 anatomical entries, and 217 organisms. The OWL-based CLO is machine-readable and can be used in various applications. | 2.1.188 | production | Cell Line (CELL), cell lines (CELL), Cell lines (CELL), Cell Type (CELL), cell line (CELL), Jurkat cell line (CELL), ATCC (CELL), Coriell (CELL), cell (CELL) |
| Pharmacogenomic Relationships Ontology | PHARE | The PHArmacogenomic RElationships Ontology (or PHARE) proposes concepts and roles to represent relationships of pharmacogenomics interest. | version id: 110114 | alpha | - |
| Ontology of Microbial Phenotypes | OMP | OMP is a formal ontology of phenotypes observed in all microbes: bacteria, archaea, protists, fungi, and viruses. OMP is being developed to be compatible with other microbial phenotype ontologies, such as the Fission Yeast Phenotype Ontology (FYPO) and the Ascomycete Phenotype Ontology (APO), and aims to standardize the capture of phenotypic information to allow improved cross-species data mining and analysis. | releases/2019-06-07 | production | OMP (GENE_OR_GENE_PRODUCT), all (1), bacteria (629395), archaea (2157), fungi (4751), viruses (10239), APO (GENE_OR_GENE_PRODUCT), cross-species (GENE_OR_GENE_PRODUCT) |
| Planarian Phenotype Ontology | PLANP | Planarian Phenotype Ontology is an ontology of phenotypes observed in the planarian Schmidtea mediterranea. Many of the phenotypes were seen durning RNAi experiments. |  | production | planarian Schmidtea mediterranea (CANCER) |
| Parkinson and Movement Disorder Ontology | PMDO | The Parkinson Movement Disorder Ontology (PMDO) uses a systematic analysis of movement disorder instruments, extensive review of literature led by movement disorder specialists, and the AMP PD knowledge portal with an iterative ontology engineering process. The current version of PMDO is focused on parkinsonian disorders with classes representing three broad categories of: (1) neurological findings, (2) treatment plans, and (3) instruments used to evaluate various traits of PD. | 1.1 | beta | Parkinson Movement (PATHOLOGICAL_FORMATION), PMDO (GENE_OR_GENE_PRODUCT), portal (TISSUE) |
| Ontological Knowledge Base Model for Cystic Fibrosis | ONTOKBCF | OntoKBCF is an ontological knowledge base model for cystic fibrosis. There are molecular genetic information (i.e. gene mutations) and health information included in OntoKBCF. The purposes of OntoKBCF include management of molecular genetic information and health information and embedding OntoKBCF into EHR settings. | 1301 | production | cystic (PATHOLOGICAL_FORMATION) |
| Neuron Phenotype Ontology | NPOKB | An ontology of neuron types based on the phenotypic dimensions of cells that can be measure experimentally. | 2025-05-02 | beta | neuron (CELL), cells (CELL) |
| C. elegans Phenotype Vocabulary | WB-PHENOTYPE | A structured controlled vocabulary of Caenorhabditis elegans phenotypes | releases/2025-04-01 | beta | - |
| Plant Trait Ontology | PTO | A controlled vocabulary to describe phenotypic traits in plants. Each trait is a distinguishable feature, characteristic, quality or phenotypic feature of a developing or mature plant, or a plant part. | May 2025 | production | plants (3193) |
| Cerebrotendinous Xanthomatosis Ontology | CTX | The ontology represents CTX phenotypes, genetic variants, and bidirectional relationships between them though a patient model. The CTX ontology was build reusing the Human Phenotype Ontology (HPO) and the Snomed ct ontologies. A set of temporal clinical manifestations are semantically annotated with a domain phenotype ontology and registered with a time-stamped value. | vtemp2 | alpha | CTX (ORGAN), Human (9606), HPO (CANCER) |
| Flora Phenotype Ontology | FLOPO | The Flora Phenotype Ontology is an ontology of phenotypes reported in Floras. The original version was developed at the pro-iBiosphere Hackathon in Leiden. This is the pre-classified version of the ontology; the original OWL file is at https://github.com/flora-phenotype-ontology/flopoontology/blob/master/ontology/flopo.owl The Flora Phenotype Ontology is generated from the Flora Malesiana, Flora Gabon, Flora of Central Africa, and a collection of Kew's African Floras. Every class in the ontology has at least one taxon annotation. The (draft) taxon annotation are available at http://jagannath.pdn.cam.ac.uk/plant/flora/clean-rerun/ | 0.9 | beta | Leiden (CANCER), collection (32644) |
| Measurement Method Ontology | MMO | The Measurement Method Ontology is designed to represent the variety of methods used to make qualitative and quantitative clinical and phenotype measurements both in the clinic and with model organisms. | 2.109 | production | - |
| Porifera Ontology | PORO | An ontology covering the anatomy of Porifera (sponges) | unknown | beta | sponges (6040) |
| Autism Spectrum Disorder Phenotype Ontology | ASDPTO | Autism Spectrum Disorder Phenotype Ontology encapsulates the Autism Spectrum Disorder behavioral phenotype, informed by the standard ASD assessment instruments and the currently known characteristics of this disorder. http://www.ncbi.nlm.nih.gov/pubmed/24163114 | 1 | alpha | - |
| Solanaceae Phenotype Ontology | SPTO | Solanaceae crop phenotypes and traits, developed in collaboration with the research community, especially for breeder traits of agronomic importance. | unknown | alpha | Solanaceae Phenotype Ontology (GENE_OR_GENE_PRODUCT) |
| Cellular microscopy phenotype ontology | CMPO | The Cellular Microscopy Phenotype Ontology (CMPO) is a species neutral ontology for describing general phenotypic observations relating to the whole cell, cellular components, cellular processes and cell populations. CMPO is an application ontology developed in OWL that contains precomposed phenotypes descriptions that are defined using the Gene Ontology (GO) and the Phenotype Trait Ontology (PATO). | 1.3 | production | Cellular Microscopy (CELL), cell (CELL), cellular (CELL) |
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Units of Measurement Ontology | UO | Metrical units for use in conjunction with PATO | 2023-05-25 | production | - |
| Bleeding History Phenotype Ontology | BHO | An application ontology devoted to the standardized recording of phenotypic data related to hemorrhagic disorders. | 0.4 | production | - |
| Human Phenotype Ontology | HP | The Human Phenotype Ontology is being developed to provide a structured and controlled vocabulary for the phenotypic features encountered in human hereditary and other disease. Our goal is to provide resource for the computational analysis of the human phenome, with a current focus on monogenic diseases listed in the Online Mendelian Inheritance in Man (OMIM) and Orphanet databases, for which annotations are also provided. Please see documentation and license information at http://www.human-phenotype-ontology.org | hp/releases/2025-05-06 | production | Human (9606), human (9606) |
| Home Cage Monitoring definition | HCM-D | From COST-TEATIME (https://www.cost-teatime.org/) | v0.1 | alpha | - |
| Ontology for Newborn Screening Follow-up and Translational Research | ONSTR | Ontology for Newborn Screening Follow-up and Translational Research (ONSTR) is an application ontology covering the domain of newborn screening, follow-up and translational research pertaining to patients diagnosed with inheritable and congenital diseases mainly identified through newborn dried blood spot screening. ONSTR is a central component of the project Newborn Screening Follow-up Data Integration Collaborative (NBSDC). ONSTR uses the Basic Formal Ontology v2 (BFO2, v2012-07-20) as top-level ontology and extends the classes imported from OBO Foundry ontologies and candidate ontologies. | 0.4 | alpha | blood (ORGANISM_SUBSTANCE), NBSDC (CANCER), BFO2 (GENE_OR_GENE_PRODUCT), v2012-07 (GENE_OR_GENE_PRODUCT) |
| Pathogen Host Interaction Phenotype Ontology | PHIPO | PHIPO is a formal ontology of species-neutral phenotypes observed in pathogen-host interactions. PHIPO is being developed to support the comprehensive and detailed representation of phenotypes in PHI-base, the multi-species Pathogen-Host Interactions database available online at http://www.phi-base.org. PHIPO is pre-composed and logically defined. | 2024-08-29 | production | PHI-base (GENE_OR_GENE_PRODUCT) |
| Ascomycete Phenotype Ontology | APO | A structured controlled vocabulary for the phenotypes of Ascomycete fungi, A structured controlled vocabulary for the phenotypes of Ascomycete fungi. | releases/2025-08-01 | production | fungi (4751) |
| Mouse Pathology Ontology | MPATH | A structured controlled vocabulary of mutant and transgenic mouse pathological lesions and processes | 2020-05-19 | production | mouse (10090), lesions (PATHOLOGICAL_FORMATION) |
| Ontology of Vaccine Adverse Events | OVAE | The Ontology of Vaccine Adverse Events (OVAE) is a biomedical ontology in the area of vaccine adverse events. OVAE is an extension of the Ontology of Adverse Events (OAE) and the Vaccine Ontology (VO). OVAE is aimed to represent and analyze various vaccine-specific adverse events.  | Vision Release: 1.0.34 | production | VO (CANCER) |
| Myocardial Infarction Ontology | MIO | The myocardial infarction ontology (MIO) is a disease-specific ontology that characterizes the knowledge field of myocardial infarction. | V1.0.0 | production | myocardial (TISSUE) |
| Combined Ontology for Inflammatory Diseases | COID | A domain ontology for inflammatory diseases, with ICD-10 used as a foundation. Ontology is expanded to include medical therapies, anatomy, and symptoms. With the addition of synonyms and lay terms derived from embedding models. | 1.0.0 | alpha | - |
| Imaging Biomarker Ontology | IBO | The Imaging Biomarker Ontology describes radiological measurements that characterize biological processes (such as anisotropic water diffusion in a particular tissue region) of imaged subjects (such as neuro-imaging observations) and help clinicians, particularly in the assessment of therapeutic responses and the early prediction of pathologies. IBO reuses several OBO ontologies namely BFO, IAO, PATO, GO, FMA, and OGMS.  |  | beta | tissue (TISSUE), PATO (GENE_OR_GENE_PRODUCT) |
| PhenX Phenotypic Terms | PHENX | Standard measures related to complex diseases, phenotypic traits and environmental exposures | 1.2 | production | - |
| postpartum depression ontology | PARTUMDO | This ontology is for use in extracting postpartum depression condition codes, and also comorbidities and diseases often mistaken for postpartum depression. The ontology describes not only the codes but also the relationships between these various diseases/risk factors for use in automatically extracting relevant patient populations from Electronic Health Records |  | alpha | - |
| Orphanet Rare Disease Ontology | ORDO | The Orphanet Rare Disease ontology (ORDO) is jointly developed by Orphanet and the EBI to provide a structured vocabulary for rare diseases capturing relationships between diseases, genes and other relevant features which will form a useful resource for the computational analysis of rare diseases. It derived from the Orphanet database (www.orpha.net) , a multilingual database dedicated to rare diseases populated from literature and validated by international experts. It integrates a nosology (classification of rare diseases), relationships (gene-disease relations, epiemological data) and connections with other terminologies (MeSH, SNOMED CT, UMLS, MedDRA),databases (OMIM, UniProtKB, HGNC, ensembl, Reactome, IUPHAR, Geantlas) or classifications (ICD10). The ontology will be maintained by Orphanet and further populated with new data. Orphanet classifications can be browsed in the OLS view. The Orphanet Rare Disease Ontology is updated monthly and follows the OBO guidelines on deprecation of terms. It constitutes the official ontology of rare diseases produced and maintained by Orphanet (INSERM, US14).  | 4.7 | production | ORDO (CANCER), gene-disease (GENE_OR_GENE_PRODUCT), UMLS (CANCER) |
| Plant Experimental Conditions Ontology | PECO | The Plant Experimental Conditions Ontology describes the treatments, growing conditions, and/or study types used in plant biology experiments. The subjects of the studies may also include in vitro plant structures (PO:0000004). A plant treatment is the most specific,and may be a component of a plant growing condition, and the plant study type. The broadest classes are the plant study types where the terms can be used to identify the growth study facility. Each growth facility such as field study, growth chamber, green house etc is a environment on its own and it may also involve instances of biotic and abiotic treatments and different growing conditions. | releases/2023-11-10 | production | - |
| Phenotypic Quality Ontology | PATO | Phenotypic qualities (properties). This ontology can be used in conjunction with other ontologies such as GO or anatomical ontologies to refer to phenotypes. Examples of qualities are red, ectopic, high temperature, fused, small, edematous and arrested. | 2025-05-14 | production | - |
| Ontology of Host-Microbe Interactions | OHMI | OHMI is a biomedical ontology that represents the entities and relations in the domain of host-microbiome interactions. | 2019-09-17 | production | - |
| Ontology of Biological Attributes | OBA | A collection of biological attributes (traits) covering all kingdoms of life. Interoperable with VT (vertebrate trait ontology) and TO (plant trait ontology). Extends PATO. | 2025-03-28 | production | collection (32644), all (1) |
| Long Covid Phenotype Ontology | LONGCOVID | Oxford-RCGP RSC Long COVID-19 (LC) Phenotype Ontology | 20220106 | alpha | Oxford-RCGP RSC (GENE_OR_GENE_PRODUCT) |
| Zebrafish Phenotype Ontology | ZP | The Zebrafish Phenotype Ontology formally defines all phenotypes of the Zebrafish model organism. | 2024-04-18 | production | all (1), Zebrafish (7955) |
| Fission Yeast Phenotype Ontology | FYPO | Fission Yeast Phenotype Ontology | 2025-05-09 | alpha | - |
| Experimental Conditions Ontology | XCO | The Experimental Conditions Ontology is designed to represent the conditions under which physiological and morphological measurements are made both in the clinic and in studies involving humans or model organisms. | 4.205 | production | humans (9605) |
| Cognitive Atlas Ontology | COGAT | The Cognitive Atlas is a collaborative knowledge building project that aims to develop an ontology that characterizes the state of current thought in cognitive science.  It defines a set of mental concepts along with a set of mental tasks, and the measurement relations between those classes. | 0.4 | alpha | - |
| Plant Breeding Ontology | PBO | Plant Breeding Ontology (PBO): an ontology for the plant breeding community which captures more than 2200 entries where 80 represent the core terms. PBO has translations in 8 different languages: English (main language), Spanish, French, Dutch, German, Japanese, Catalan and Thai, as well as their definitions, synonyms, derived terms and samples of their usage. PBO has been built partially manually and semi automatically. |  | beta | - |
| Crop Ontology | CO | N/A | Unknown | Unknown | Crop (CELL) |
| Resource of Asian Primary Immunodeficiency Diseases (RAPID) Phenotype Ontology | RPO | RAPID phenotype ontology presents controlled vocabulary of ontology class structures and entities of observed phenotypic terms for primary immunodeficiency diseases (PIDs) that facilitate global sharing and free exchange of PID data with users’ communities | Version 1.0 | production | PIDs (CANCER), PID (CANCER) |
| Ocular Immune-Mediated Inflammatory Diseases Ontology | OCIMIDO | Application ontology for Ocular Immune-mediated Inflammatory Diseases, built from domain experts in ophthalmology, clinical guidelines, and enhanced with patient-preferred terms. | 1.2 | production | - |
| NeuralReprogrammingOntology | NRO | The neural reprogramming ontology encompasses the domain of neural regenerative cell reprogramming knowledge. As research progresses, researchers and specialists in the field of reprogramming could continue to expand the bounds of this foundation, so adding to its depth. | 1.0.0 | beta | neural (TISSUE), cell (CELL) |
| Xenopus Phenotype Ontology | XPO | An ontology of anatomical, cellular, and gene function phenotypes in Xenopus, the African clawed frogs. The Xenopus Phenotype Ontology represents and standardizes anatomical, cellular, and gene function phenotypes in Xenopus, the African clawed frogs. The XPO is being designed primarily to support phenotype curation in Xenbase, the model organism database for Xenopus, and to facilitate mappings between frog phenotypes and human disease. | 2024-04-18 | production | cellular (CELL), Xenopus (262014), frogs (8342), XPO (GENE_OR_GENE_PRODUCT), human (9606) |
| Microbial Phenotype Ontology | MPO | An ontology for describing microbial phenotypes | Version 0.74 alpha | alpha | - |
| Mammalian Phenotype Ontology | MP | Standard terms for annotating mammalian phenotypic data. | 2025-06-18 | production | - |
| Biolink Model | BIOLINK | A high level datamodel of biological entities (genes, diseases, phenotypes, pathways, individuals, substances, etc) and their associations | 2.2.13 | production | - |
| Base de Données Publique des Médicaments | BDPM | LA BDPM est un des référentiels du médicament gérée et distribuée par l’ANSM. Elle recense l’ensemble des spécialités et des présentations effectivement commercialisées durant les 5 dernières années. De ce fait, elle ne propose pas un historique exhaustif des codes de présentation et spécialités, celui-ci peut être obtenu grâce à la base RCP disponible sur le site de l’ANSM. | 2025-03-04 | production | - |
| Ontology of Physics for Biology | OPB | The OPB is a reference ontology of classical physics and thermodynamics as applied to the dynamics of biological systems. It is designed to encompass the multiple structural scales (atoms to organisms) and multiple physical domains (fluid dynamics, chemical kinetics, particle diffusion, etc.) that are encountered in the study and analysis of biological organisms. | 1.2 | beta | OPB (GENE_OR_GENE_PRODUCT) |
| Drosophila Phenotype Ontology | DPO | An ontology of commonly encountered and/or high level Drosophila phenotypes. It has significant formalisation - utilising terms from GO, CL, PATO and the Drosophila anatomy ontology. It has been used by FlyBase for > 159000 annotations of phenotype. | 2025-05-30 | production | Drosophila (2081351), CL (MULTI_TISSUE_STRUCTURE), FlyBase (GENE_OR_GENE_PRODUCT) |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |
| Sleep Domain Ontology | SDO | An application ontology for the domain of Sleep Medicine. | Last modification: Feb. 11, 2013 Sivaram Arabandi | beta | - |
| NeuroMorpho.Org brain region ontologies | NMOBR | The species, brain regions and cell types are integrated into a single ontology for enabling OntoSearch functionality to mine the latest v7.0 release of NeuroMorpho.Org. | 2.0 | beta | brain (ORGAN), cell (CELL) |
| HPO - ORDO Ontological Module | HOOM | Orphanet provides phenotypic annotations of the rare diseases in the Orphanet nomenclature using the Human Phenotype Ontology (HPO). HOOM is a module that qualifies the annotation between a clinical entity and phenotypic abnormalities according to a frequency and by integrating the notion of diagnostic criterion. In ORDO a clinical entity is either a group of rare disorders, a rare disorder or a subtype of disorder. The phenomes branch of ORDO has been refactored as a logical import of HPO, and the HPO-ORDO phenotype disease-annotations have been provided in a series of triples in OBAN format in which associations, frequency and provenance are modeled. HOOM is provided as an OWL (Ontologies Web Languages) file, using OBAN, the Orphanet Rare Disease Ontology (ORDO), and HPO ontological models. HOOM provides extra possibilities for researchers, pharmaceutical companies and others wishing to co-analyse rare and common disease phenotype associations, or re-use the integrated ontologies in genomic variants repositories or match-making tools.  | 2.3 | beta | Human (9606), HPO (CANCER), ORDO (CANCER) |
| DSMZ Digital Diversity Ontology | D3O | The DSMZ Digital Diversity Ontology (D3O) is an ontology designed to standardize and integrate data related to microbial diversity and associated resources within the context of the DSMZ’s Digital Diversity Knowledge Graph. This ontology provides a structured framework to represent various aspects of microbial data, including taxonomic information, strain properties, cultivation conditions, and media links. D3O facilitates cross-referencing between internal DSMZ datasets and external biological resources, supporting enhanced data discovery, interoperability, and research in microbiology, genomics, and metabolomics. The ontology includes both object properties, which link entities such as bacterial strains to external resources (e.g., literature references or media descriptions), and data properties, which associate entities with literal values (e.g., taxonomic identifiers, growth conditions). Designed for use in RDF-based knowledge graphs, D3O supports flexible integration with other life science ontologies, promoting consistency and reuse in the broader scientific community. | 1.1 | alpha | DSMZ (CELL), D3O (GENE_OR_GENE_PRODUCT) |

<a name="bioportal-category-physicochemical"></a>
## BioPortal Category "Physicochemical"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| New Approach Methodology Ontology | NAMO | The New Approach Methodology Ontology and Schema (NAMO) is a comprehensive data framework designed to standardize and integrate information about alternative methods to animal testing in biomedical research. Built on LinkML, NAMO provides a common vocabulary and data structure for describing diverse model systems, from cellular cultures and organ-on-chip devices to computational models and digital twins. |  | production | Schema (GENE_OR_GENE_PRODUCT), cellular cultures (CELL), twins (CANCER) |
| The Biological and Environmental Research Variable Ontology | BERVO | The Biological and Environmental Research Variable Ontology (BERVO) is a model of the experimental variables, conditions, and concepts used in the study of environmental research, including studies in earth science, plant science, and geochemistry. | 2025-08-05 | alpha | - |
| Ontology of chemically concrete, composable DNA, RNA, and protein residue and crosslinks | BPFORMS | BpForms is an ontology of molecularly-concrete DNA, RNA, and protein residues and crosslinks which can be composed into molecularly-concrete forms of DNA, RNA, proteins, and complexes. The ontology can be used with the BpForms grammar and software tools for polymers and the BcForms grammar and software tools for complexes. See https://www.bpforms.org and https://www.bcforms.org for more information. | 0.0.9 | beta | DNA (CELLULAR_COMPONENT), https://www.bcforms.org (GENE_OR_GENE_PRODUCT) |
| BioAssay Ontology | BAO | The BioAssay Ontology (BAO) describes chemical biology screening assays and their results including high-throughput screening (HTS) data for the purpose of categorizing assays and data analysis.  BAO is an extensible, knowledge-based, highly expressive (currently SHOIQ(D)) description of biological assays making use of descriptive logic based features of the Web Ontology Language (OWL).  BAO currently has over 1000 classes and also makes use of several other ontologies.  It describes several concepts related to biological screening, including Perturbagen, Format, Meta Target, Design,  Detection Technology, and Endpoint.  Perturbagens are perturbing agents that are screened in an assay; they are mostly small molecules. Assay Meta Target describes what is known about the biological system and / or its components interrogated in the assay (and influenced by the Perturbagen).  Meta target can be directly described as a molecular entity (e.g. a purified protein or a protein complex), or indirectly by a biological process or event (e.g. phosphorylation).  Format describes the biological or chemical features common to each test condition in the assay and includes biochemical, cell-based, organism-based, and variations thereof.  The assay Design describes the assay methodology and implementation of how the perturbation of the biological system is translated into a detectable signal.  Detection Technology relates to the physical method and technical details to detect and record a signal.  Endpoints are the final HTS results as they are usually published (such as IC50, percent inhibition, etc.).  BAO has been designed to accommodate multiplexed assays.  All main BAO components include multiple levels of sub-categories and specification classes, which are linked via object property relationships forming an expressive knowledge-based representation. | 2.8.16 | production | SHOIQ(D (GENE_OR_GENE_PRODUCT) |
| Microbial Isolation Source Ontology | MISO | This ontology aims to classify and tag microbial isolation sources, which are often only provided as free-text commentaries, to improve their accessibility and usability. |  | production | - |
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Ontology of Chemical Elements | OCE | The Ontology of Chemical Elements (OCE) is a community-based ontology in the domain of chemical elements. OCE systematically represents and annotates the known 118 chemical elements and their charactistics and relations with other entities. OCE imports the basic chemical element information from ChEBI and adds more classes, annotations, and axioms. OCE specifically focuses on adding all the related information from the Periodic Table of Chemical Elements.  | Vision Release: 1.0.29 | production | all (1) |
| G Protein-Coupled Receptor BioAssays Ontology | BAO-GPCR | The G protein-coupled receptors (GPCRs) ontology (http://www.bioassayontology.org/bao_gpcr) describes pharmacology, biochemistry and physiology of these important and therapeutically promising class of academic and pharmaceutical research targets. Incorporation and comparison of various small molecule screening data sets, such as those deposited in PubChem, ChEMBL, KEGG, PDSP, and/or IUPHAR databases, requires a formalized electronic organization system. In order to bridge the gap between the overflow of HTS data and the bottleneck of integrated analysis tools, herein, we provide the first comprehensive GPCR ontology. The development and utility of GPCR ontology was based on previously developed BioAssay Ontology (BAO). The GPCR ontology contains information about biochemical, pharmacological, and functional properties of individual GPCRs as well as GPCR-selective ligands inclusive of their HTS screening results and other records. This provides the first all-inclusive GPCR ontology with all available data to model the relationship between the GPCR binding sites and their physiologic and pharmacologic role in physiology via small molecule chemical structures. We developed this system using emerging semantic technologies, by leveraging existing and descriptive domain level ontologies. | 1.1 | beta | GPCRs (GENE_OR_GENE_PRODUCT), all (1) |
| LipidOntology | LION | Lipid ontology (LION) associates >50,000 lipid species to biophysical, chemical, and cell biological features.  | 1.1 | production | cell (CELL) |
| NMR-Controlled Vocabulary | NMR | This is the new msi-sanctioned nmr CV, created within the COSMOS EU project, to support the nmrML data standard for nuclear magnetic resonance data in metabolomics with meaningful raw data descriptors. | 1.1.0 | beta | - |
| Environment Ontology | ENVO | EnvO is an OBO Foundry and Library ontology for the concise, controlled description of environmental entities such as ecosystems, environmental processes, and environmental qualities. It closely interoperates with a broad collection of other OBO ontologies and is used in a diverse range of projects.  | 2024-07-01 | production | EnvO (GENE_OR_GENE_PRODUCT), collection (32644) |
| Base de Données Publique des Médicaments | BDPM | LA BDPM est un des référentiels du médicament gérée et distribuée par l’ANSM. Elle recense l’ensemble des spécialités et des présentations effectivement commercialisées durant les 5 dernières années. De ce fait, elle ne propose pas un historique exhaustif des codes de présentation et spécialités, celui-ci peut être obtenu grâce à la base RCP disponible sur le site de l’ANSM. | 2025-03-04 | production | - |
| Ontology of Physics for Biology | OPB | The OPB is a reference ontology of classical physics and thermodynamics as applied to the dynamics of biological systems. It is designed to encompass the multiple structural scales (atoms to organisms) and multiple physical domains (fluid dynamics, chemical kinetics, particle diffusion, etc.) that are encountered in the study and analysis of biological organisms. | 1.2 | beta | OPB (GENE_OR_GENE_PRODUCT) |

<a name="bioportal-category-plant"></a>
## BioPortal Category "Plant"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| INRAE Thesaurus | INRAETHES | INRAE Thesaurus contains more than 16,000 concepts relevant to domains covered by the National Research Institute for Agriculture, Food and the Environment. Concepts are described with terms in French (100%) and English (81%) terms, some having textual definitions and mappings to other semantic artifacts. The concepts are organized in 12 domains subdivided in thematic sections called micro-thesauri., Thésaurus ouvert et partagé couvrant les domaines de recherche d’INRAE. Il sert de référentiel au sein de l’institut pour indexer et annoter des documents, pages web, descriptions d’activités, jeux de données, à des fins de recherche ou d’analyse de l’information. | 2.4 | production | INRAE (CANCER), fins (ORGAN) |
| AGROVOC | AGROVOC | AGROVOC is a multilingual and controlled vocabulary designed to cover concepts and terminology under FAO's areas of interest. It is a large Linked Open Data set about agriculture, available for public use, and its highest impact is through facilitating the access and visibility of data across domains and languages. |  | production | - |
| Plant Stress Ontology | PLANTSO | This ontology describes biotic and abiotic stresses that a plant may encounter. | 2023-11-14 | production | - |
| Ontology of units of Measure | OM | The Ontology of units of Measure (OM) 2.0 models concepts and relations important to scientific research. It has a strong focus on units, quantities, measurements, and dimensions. OM is modelled in OWL 2 - Web Ontology Language. The OM ontology provides classes, instances, and properties that represent the different concepts used for defining and using measures and units. It includes, for instance, common units such as the SI units metre and kilogram, but also units from other systems of units such as the mile or nautical mile. For many application areas it includes more specific units and quantities, such as the unit of the Hubble constant or the quantity vaselife. The following application areas are supported by OM: Geometry; Mechanics; Thermodynamics; Electromagnetism; Fluid mechanics; Chemical physics; Photometry; Radiometry and Radiobiology; Nuclear physics; Astronomy and Astrophysics; Cosmology; Earth science; Meteorology; Material science; Microbiology; Economics; Information technology and Typography. | 2.0.59 | production | OM (MULTI_TISSUE_STRUCTURE), Fluid (GENE_OR_GENE_PRODUCT), Nuclear (CELLULAR_COMPONENT), Material (CELL) |
| NMDC Ontology | NMDCO | The NMDC Ontology (NMDCO) is used by the National Microbiome Data Collaborative (NMDC, https://microbiomedata.org) to annotate multi-omic microbiome metadata. It merges a number of ontologies relevant to the NMDC (such as the plant and environment ontologies) into a single ontology. The ontology can be download from Github at https://github.com/microbiomedata/nmdc-ontology. | 2024-03-15 | beta | - |
| Plant Trait Ontology | PTO | A controlled vocabulary to describe phenotypic traits in plants. Each trait is a distinguishable feature, characteristic, quality or phenotypic feature of a developing or mature plant, or a plant part. | May 2025 | production | plants (3193) |
| Flora Phenotype Ontology | FLOPO | The Flora Phenotype Ontology is an ontology of phenotypes reported in Floras. The original version was developed at the pro-iBiosphere Hackathon in Leiden. This is the pre-classified version of the ontology; the original OWL file is at https://github.com/flora-phenotype-ontology/flopoontology/blob/master/ontology/flopo.owl The Flora Phenotype Ontology is generated from the Flora Malesiana, Flora Gabon, Flora of Central Africa, and a collection of Kew's African Floras. Every class in the ontology has at least one taxon annotation. The (draft) taxon annotation are available at http://jagannath.pdn.cam.ac.uk/plant/flora/clean-rerun/ | 0.9 | beta | Leiden (CANCER), collection (32644) |
| Soy Ontology | SOY | Growth, trait and development ontology for soybean | unknown | beta | soybean (3847) |
| Plant Ontology | PO | The Plant Ontology is a structured vocabulary and database resource that links plant anatomy, morphology and growth and development to plant genomics data. The PO is under active development to expand to encompass terms and annotations from all plants. | releases/2024-04-17 | production | all (1), plants (3193) |
| The FoodOn Food Ontology | FOODON | FoodOn is an ontology built to represent entities which bear a “food role” and is initially focused on categorizing and processing of food for humans. We aim to develop semantics for food safety, food security, the agricultural and animal husbandry practices linked to food production, culinary, nutritional and chemical ingredients and processes.  FoodOn belongs to the OBOFoundry.org family of ontologies. | 2025-06-07 | production | bear (9641), humans (9605) |
| GoMapMan | GMM | GoMapMan is an open web-accessible resource for gene functional annotations in the plant sciences. It was developed to facilitate improvement, consolidation and visualization of gene annotations across several plant species. | 1.2 | production | - |
| Control y Monitoreo de afecciones en los cultivos | AGROCYMAC | Agrocymac es una ontología creada para estructurar información sobre el control y monitoreo de plagas, malezas y enfermedades que afectan específicamente a los cultivos de Arroz, Café y Cacao. |  | alpha | ontología (ORGANISM_SUBDIVISION) |
| Anthology of Biosurveillance Diseases | ABD | Our disease ontology provides information on infectious diseases, disease synonyms, transmission pathways, disease agents, affected populations and disease properties. Diseases have been grouped into syndromic disease categories, such that programmers can look through relevant categories, as well as at specific diseases. Organisms, linked to both agents and populations, are structured hierarchically, to provide multiple levels of organism resolution. In addition, both disease transmission and relevant disease properties are available to search. Disease properties include tags like 'notifiable diseases' and 'economic importance' to flag particular disease characteristics that may be of interest, but are not captured elsewhere. |  | alpha | disease (CANCER), Organisms (CANCER), flag (34205) |
| Plant Experimental Conditions Ontology | PECO | The Plant Experimental Conditions Ontology describes the treatments, growing conditions, and/or study types used in plant biology experiments. The subjects of the studies may also include in vitro plant structures (PO:0000004). A plant treatment is the most specific,and may be a component of a plant growing condition, and the plant study type. The broadest classes are the plant study types where the terms can be used to identify the growth study facility. Each growth facility such as field study, growth chamber, green house etc is a environment on its own and it may also involve instances of biotic and abiotic treatments and different growing conditions. | releases/2023-11-10 | production | - |
| Plant Breeding Ontology | PBO | Plant Breeding Ontology (PBO): an ontology for the plant breeding community which captures more than 2200 entries where 80 represent the core terms. PBO has translations in 8 different languages: English (main language), Spanish, French, Dutch, German, Japanese, Catalan and Thai, as well as their definitions, synonyms, derived terms and samples of their usage. PBO has been built partially manually and semi automatically. |  | beta | - |
| Crop Ontology | CO | N/A | Unknown | Unknown | Crop (CELL) |
| Plant Diversity Ontology | PDO_CAS | Plant Diversity Ontology (PDO) is the outcome of the construction tasks of domain ontology of science in the project of Scientific & Technological Knowledge Organization Systems (STKOS). The structure of PDO centers on species, including micro level of molecular, cell and tissue, medium level of plant organs, species and population, and macro level of ecosystem. The classes of PDO also capture the habitat, environmental conditions and expand the general characteristics of plants, completing the instances richness for plant species, geographical, classification and morphology. Furthermore, PDO builds the relationships among species, region, morphology, plant organs, developmental stage, revealing the list of species in an area, the morphology and life form of plants, and development process of species. In sum, PDO shows a growth process or development stage of a species and its structure characteristics in particular space and time scale, realizing the description of species diversity, ecosystem diversity and genetic diversity. At present, PDO has been applied to the platform of Go to Selected Open Access Journals (GoOA) developed by National Science Library of Chinese Academy of Sciences, supporting concept retrieval and concept extension for GoOA. |  | alpha | STKOS (GENE_OR_GENE_PRODUCT), cell (CELL), tissue (TISSUE), organs (ORGAN), plants (3193), PDO (TISSUE), GoOA (CANCER) |

<a name="bioportal-category-plant-anatomy"></a>
## BioPortal Category "Plant Anatomy"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| The Biological and Environmental Research Variable Ontology | BERVO | The Biological and Environmental Research Variable Ontology (BERVO) is a model of the experimental variables, conditions, and concepts used in the study of environmental research, including studies in earth science, plant science, and geochemistry. | 2025-08-05 | alpha | - |
| Plant Ontology | PO | The Plant Ontology is a structured vocabulary and database resource that links plant anatomy, morphology and growth and development to plant genomics data. The PO is under active development to expand to encompass terms and annotations from all plants. | releases/2024-04-17 | production | all (1), plants (3193) |
| Fern Ontology | FO | Fern Ontology (FO) is a controlled and structured vocabulary of fern specific anatomy and lifecycle.  | 0.05 | beta | - |

<a name="bioportal-category-plant-development"></a>
## BioPortal Category "Plant Development"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| The Biological and Environmental Research Variable Ontology | BERVO | The Biological and Environmental Research Variable Ontology (BERVO) is a model of the experimental variables, conditions, and concepts used in the study of environmental research, including studies in earth science, plant science, and geochemistry. | 2025-08-05 | alpha | - |
| Plant Ontology | PO | The Plant Ontology is a structured vocabulary and database resource that links plant anatomy, morphology and growth and development to plant genomics data. The PO is under active development to expand to encompass terms and annotations from all plants. | releases/2024-04-17 | production | all (1), plants (3193) |
| Control y Monitoreo de afecciones en los cultivos | AGROCYMAC | Agrocymac es una ontología creada para estructurar información sobre el control y monitoreo de plagas, malezas y enfermedades que afectan específicamente a los cultivos de Arroz, Café y Cacao. |  | alpha | ontología (ORGANISM_SUBDIVISION) |

<a name="bioportal-category-protein"></a>
## BioPortal Category "Protein"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| FamPlex | FPLX | FamPlex is a collection of resources for grounding biological entities from text and describing their hierarchical relationships. | 1.0 | alpha | FamPlex (GENE_OR_GENE_PRODUCT), collection (32644) |
| Drug Target Ontology | DTO | Drug Target Ontology (DTO) is being developed at the University of Miami in the research group of Stephan Schürer. DTO is supported by grant U54CA189205 (Illuminating the Druggable Genome Knowledge Management Center, Tudor Oprea, PI) awarded by the NCI through the NIH Common Fund. It is a component of the Illuminating the Druggable Genome (IDG) project (https://commonfund.nih.gov/idg). The DTO project develops a novel semantic framework to formalize knowledge about drug targets with a focus on the current IDG protein families. The DTO is developed as a reference for drug targets with the longer-term goal to create a community standard that will facilitate the integration of diverse drug discovery information from numerous heterogeneous resources. The first version of the DTO consists of asserted class hierarchies of the four IDG protein families, GPCRs, kinases, ion channels, and nuclear hormone receptors. Protein classes are linked to tissue and disease via different levels of confidence. DTO also contains drug target development level classifications developed in the IDG project (http://targetcentral.ws/), and functional and qualitative annotations and classifications for kinase proteins, GPCR ligands and ion channels. DTO is modeled in OWL2-DL to enable further classification by inference reasoning and SPARQL queries. DTO is implemented following a modularization approach. DTO is used as the organizational framework for drug targets in the IDG PHAROS User Interface Portal (https://pharos.nih.gov) and also the Tin-X Target Importance and Novelty Explorer (http://newdrugtargets.org). | 1.1.1 | beta | DTO (TISSUE), U54CA189205 (GENE_OR_GENE_PRODUCT), NCI (CELL), GPCRs (GENE_OR_GENE_PRODUCT), nuclear hormone receptors (GENE_OR_GENE_PRODUCT), tissue (TISSUE), disease (CANCER) |
| Glycomics Ontology | GLYCO | The GlycO ontology provides an up-to-date knowledge base of experimentally verified glycan structures. Glycan (oligosaccharide or polysaccharide) structures are represented as trees of monosaccharide residues. Linkage to proteins and lipids is supported as well. Insertion of a new glycan is controlled by curation process that includes matching the new glycan against a canonical glyco-tree (a highly branched representation for a family of glycans). | 4.6.0.0 | production | - |
| Enzyme Structure Function Ontology | ESFO | The ESFO provides a new paradigm for organizing enzyme sequence, structure, and function information, whereby specific elements of enzyme sequence and structure are mapped to specific conserved aspects of function, thus facilitating the functional annotation of uncharacterized enzymes. The ESFO helps prevent misannotation by emphasizing annotation of enzymes only at the level of functional granularity warranted by available information. | 1 | alpha | ESFO (GENE_OR_GENE_PRODUCT) |
| Therapeutic Antibody Ontology | TAON | TAON is a comprehensive ontology of the therapeutic antibody, which  included antibody terms and drug terms. | 1.1 | alpha | - |
| Ontology of chemically concrete, composable DNA, RNA, and protein residue and crosslinks | BPFORMS | BpForms is an ontology of molecularly-concrete DNA, RNA, and protein residues and crosslinks which can be composed into molecularly-concrete forms of DNA, RNA, proteins, and complexes. The ontology can be used with the BpForms grammar and software tools for polymers and the BcForms grammar and software tools for complexes. See https://www.bpforms.org and https://www.bcforms.org for more information. | 0.0.9 | beta | DNA (CELLULAR_COMPONENT), https://www.bcforms.org (GENE_OR_GENE_PRODUCT) |
| HIV ontology | HIV | The HIV ontology encompasses all knowledge about HIV | 2.6 | production | all (1) |
| BioAssay Ontology | BAO | The BioAssay Ontology (BAO) describes chemical biology screening assays and their results including high-throughput screening (HTS) data for the purpose of categorizing assays and data analysis.  BAO is an extensible, knowledge-based, highly expressive (currently SHOIQ(D)) description of biological assays making use of descriptive logic based features of the Web Ontology Language (OWL).  BAO currently has over 1000 classes and also makes use of several other ontologies.  It describes several concepts related to biological screening, including Perturbagen, Format, Meta Target, Design,  Detection Technology, and Endpoint.  Perturbagens are perturbing agents that are screened in an assay; they are mostly small molecules. Assay Meta Target describes what is known about the biological system and / or its components interrogated in the assay (and influenced by the Perturbagen).  Meta target can be directly described as a molecular entity (e.g. a purified protein or a protein complex), or indirectly by a biological process or event (e.g. phosphorylation).  Format describes the biological or chemical features common to each test condition in the assay and includes biochemical, cell-based, organism-based, and variations thereof.  The assay Design describes the assay methodology and implementation of how the perturbation of the biological system is translated into a detectable signal.  Detection Technology relates to the physical method and technical details to detect and record a signal.  Endpoints are the final HTS results as they are usually published (such as IC50, percent inhibition, etc.).  BAO has been designed to accommodate multiplexed assays.  All main BAO components include multiple levels of sub-categories and specification classes, which are linked via object property relationships forming an expressive knowledge-based representation. | 2.8.16 | production | SHOIQ(D (GENE_OR_GENE_PRODUCT) |
| Biological Pathway Taxonomy | BPT | Biological Pathway Taxonomy (BPT) - hierarchical taxonomy of cellular and molecular mechanisms, metabolic reactions, and diseases that can be shown as signaling pathways. BPT was created manually based on the Elsevier Pathway Collection available for public usage. There are 3 types of entities in BPT – pathway, group, and folder of pathways and groups. Group is the list of genes and proteins associated with biological function and diseases. Pathway is an interactive model of molecular interactions in human and mammalian cells described in the literature. Each model was reconstructed based on facts published in peer-reviewed journals, with manually quality control. BPT pathway entity includes several properties such as link to the model in Pathway Studio software which allows reading supported sentences from published articles and additional information about members. A list of members for each model can be provided by request. Property “network” describes maps without logical causal relationships between members to distinguish them from actual models of molecular mechanisms (“signaling”). The index folder contains all pathways and groups sorted alphabetically. | synonymes2022 | alpha | cellular (CELL), human (9606), mammalian cells (CELL), all (1) |
| Protein Ontology | PR | PRO provides an ontological representation of protein-related entities by explicitly defining them and showing the relationships between them. Each PRO term represents a distinct class of entities (including specific modified forms, orthologous isoforms, and protein complexes) ranging from the taxon-neutral to the taxon-specific. The ontology has a meta-structure encompassing three areas: proteins based on evolutionary relatedness (ProEvo); protein forms produced from a given gene locus (ProForm); and protein-containing complexes (ProComp). | 71.0 | production | - |
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Feature Annotation Location Description Ontology  | FALDO | FALDO is the Feature Annotation Location Description Ontology. It is a simple ontology to describe sequence feature positions and regions as found in GFF3, DBBJ, EMBL, GenBank files, UniProt, and many other bioinformatics resources. The aim of this ontology is to describe the position of a sequence region or a feature. It does not aim to describe features or regions itself, but instead depends on resources such as the Sequence Ontology or the UniProt core ontolgy. | Created at the Biohackathon 2012 and 2013 | production | EMBL (GENE_OR_GENE_PRODUCT) |
| G Protein-Coupled Receptor BioAssays Ontology | BAO-GPCR | The G protein-coupled receptors (GPCRs) ontology (http://www.bioassayontology.org/bao_gpcr) describes pharmacology, biochemistry and physiology of these important and therapeutically promising class of academic and pharmaceutical research targets. Incorporation and comparison of various small molecule screening data sets, such as those deposited in PubChem, ChEMBL, KEGG, PDSP, and/or IUPHAR databases, requires a formalized electronic organization system. In order to bridge the gap between the overflow of HTS data and the bottleneck of integrated analysis tools, herein, we provide the first comprehensive GPCR ontology. The development and utility of GPCR ontology was based on previously developed BioAssay Ontology (BAO). The GPCR ontology contains information about biochemical, pharmacological, and functional properties of individual GPCRs as well as GPCR-selective ligands inclusive of their HTS screening results and other records. This provides the first all-inclusive GPCR ontology with all available data to model the relationship between the GPCR binding sites and their physiologic and pharmacologic role in physiology via small molecule chemical structures. We developed this system using emerging semantic technologies, by leveraging existing and descriptive domain level ontologies. | 1.1 | beta | GPCRs (GENE_OR_GENE_PRODUCT), all (1) |
| CYTOKINE | CYTO | It is flat ontology showing cytokine synonyms. It has been produced by merging cytokine branches, using SOMA (Synonym Ontology Mapping Approach) method, of 10 Bioportal ontologies. | Cytokines | alpha | SOMA (GENE_OR_GENE_PRODUCT) |
| Viral Disease Ontology Trunk | VDOT | VDOT is a middle-layer ontology for the health care domain with special emphasis on viral infections and transplantation medicine. | Initial Version release date 28/02/2017 | production | - |
| NMR-Controlled Vocabulary | NMR | This is the new msi-sanctioned nmr CV, created within the COSMOS EU project, to support the nmrML data standard for nuclear magnetic resonance data in metabolomics with meaningful raw data descriptors. | 1.1.0 | beta | - |
| Cell Cycle Ontology | CCO | An application ontology integrating knowledge about the eukaryotic cell cycle. | 2.10 | beta | cell (CELL) |
| Functional Electrophysiology Nomenclature for Ion Channels | FENICS | The Functional Epilepsy Nomenclature for Ion Channels (FENICS) is allowing the consistent evaluation of electrophysiology experiments on Ion Channels in the context of neurological disorders, in particular epilepsy.  | 0.0.5 | alpha | Ion (TISSUE) |
| Chromatin Regulator Ontology | CHRO | Functional and biochemical description of covalent histone modifiers and chromatin remodeler proteins. | 2023-08-18 | alpha | histone (GENE_OR_GENE_PRODUCT), chromatin remodeler (GENE_OR_GENE_PRODUCT) |
| HIVOntology | HIVO004 | A Project of Biomedical Ontology for Systems Biomedicine. The Project includes, so far, a main ontology which is imported by several other sub ontologies AIDSClinic*.owl, etc. The goals of the Project include: (1). To pilot the novel researches on AIDS and HIV or other diseases by viruses, by means of the ontology and other biomedical informatics technologies, connecting the insights of the Biomedical Knowledge Integration (BMKI), systems biomedicine, complexity sciences etc. (2). To Carry out the methodology explorations for the goals above. | 020 | alpha | HIV (57667), viruses (10239) |
| Cell Line Ontology [by Mahadevan] | MCCL | Ontologies provide a common platform of controlled vocabulary for researchers who need to share information across a particular domain, inclusive of machine-interpretable definitions of basic concepts in the domain and relations among them. The need for ontologies that provide a systematic arrangement of available data and allow cross talk across various related domains is gaining momentum. In this backdrop, we have developed a comprehensive ontology on primary and established cell lines-both normal and pathologic. It covers around 400 cell lines. This ontology has been built to include the major domains in the field of biology like anatomy, bio-molecules, chemicals and drugs, pathological conditions and genetic variations around the cell lines. An extensive network of relations has been built across these concepts to enable different combinations of queries. The ontology covers all cell lines from major sources like ATCC, DSMZ, ECACC, ICLC etc. and is built in OWL format. | 2 | beta | cell lines-both (CANCER), cell lines (CELL), bio-molecules (CANCER), all (1), DSMZ (CELL), ECACC (CELL) |
| Venom Ontology | CU-VO | An ontology of venoms and venom-related compounds, designed to facilitate the discovery of novel venom-derived therapeutics. | 1.0 | production | venom-derived (MULTI_TISSUE_STRUCTURE) |
| Enzyme Mechanism Ontology | EMO | This ontology describes the components of an enzyme and its reaction mechanism including the roles that the components play in the reaction mechanism. |  | alpha | - |
| Genetic Feature Format Ontology | GFFO | An ontology to represent genome feature annotations in gff3 format. | 0.0.1 | alpha | - |
| Mapping of Epilepsy Ontologies | MEPO | The Mapping of Epilepsy Ontologies (MEPO) provides owl:sameAs relationships of similar concepts for the ontologies EpSO, ESSO, EPILONT, EPISEM, and FENICS. Two concepts are considered similar if they share the same concept name or the same synonym. Additional synonyms for each of the concepts are generated using the Snowball stemming algorithm. The software implementation is available on github within the Maven project snoke in the module snoke.ontology: https://github.com/bernd-mueller/snoke/tree/master/snoke.ontology | 0.6 | alpha | MEPO (CANCER) |
| Mass Spectrometry Ontology | MS | A structured controlled vocabulary for the annotation of experiments concerned with proteomics mass spectrometry. Developed by the HUPO Proteomics Standards Initiative (PSI). | 4.1.195 | alpha | - |
| An Ontology for integration and interoperability of multi-aspect data of Protein food production chains | ALTPROTON | The ALTPROTON ontology is the foundation of a cloud-based interoperable data platform specifically tailored to address the complexities of the protein food production chain. The ALTPROTON consists of three main domains food quality, sustainability, and food processing. The multi-aspect data generated within each domain are connected via the most representative concepts to effectively integrate and interoperable for future advancement in the food industry and protein food research sector. The paper describes the concepts, classes, and relationships used in ALTPROTON to fill the gaps in existing food ontologies and databases. | 1.2 | production | - |
| In Silico Methodologies Ontology for MAATrica metric | SIMM | MAATrica is a novel metric for assessing the coherence of methodologies in research papers within the fields of medicinal and nutraceutical chemistry. MAATrica employs user-controlled and customizable ontologies (in silico, in vitro and in vivo), enabling personalized analysis of research papers in the fields of medicinal and nutraceutical chemistry. |  | alpha | - |
| Ontology for MicroRNA Target | OMIT | The purpose of the OMIT ontology is to establish data exchange standards and common data elements in the microRNA (miR) domain. Biologists (cell biologists in particular) and bioinformaticians can make use of OMIT to leverage emerging semantic technologies in knowledge acquisition and discovery for more effective identification of important roles performed by miRs in humans' various diseases and biological processes (usually through miRs' respective target genes). OMIT has reused and extended a set of well-established concepts from existing bio-ontologies, e.g., Gene Ontology, Sequence Ontology, Protein Ontology, NCBI Organism Taxonomy, Human Disease Ontology, Foundational Model of Anatomy, and so forth. | OBO 4.0-12122013-2338 | alpha | cell (CELL), miRs (GENE_OR_GENE_PRODUCT), humans (9605), Human (9606) |

<a name="bioportal-category-subcellular"></a>
## BioPortal Category "Subcellular"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| In Vitro Methodologies Ontology for MAATrica metric | VIMM | MAATrica is a novel metric for assessing the coherence of methodologies in research papers within the fields of medicinal and nutraceutical chemistry. MAATrica employs user-controlled and customizable ontologies (in silico, in vitro and in vivo), enabling personalized analysis of research papers in the fields of medicinal and nutraceutical chemistry. Here we present the in vitro ontology we developed for the scope. |  | alpha | - |
| HIV ontology | HIV | The HIV ontology encompasses all knowledge about HIV | 2.6 | production | all (1) |
| BioAssay Ontology | BAO | The BioAssay Ontology (BAO) describes chemical biology screening assays and their results including high-throughput screening (HTS) data for the purpose of categorizing assays and data analysis.  BAO is an extensible, knowledge-based, highly expressive (currently SHOIQ(D)) description of biological assays making use of descriptive logic based features of the Web Ontology Language (OWL).  BAO currently has over 1000 classes and also makes use of several other ontologies.  It describes several concepts related to biological screening, including Perturbagen, Format, Meta Target, Design,  Detection Technology, and Endpoint.  Perturbagens are perturbing agents that are screened in an assay; they are mostly small molecules. Assay Meta Target describes what is known about the biological system and / or its components interrogated in the assay (and influenced by the Perturbagen).  Meta target can be directly described as a molecular entity (e.g. a purified protein or a protein complex), or indirectly by a biological process or event (e.g. phosphorylation).  Format describes the biological or chemical features common to each test condition in the assay and includes biochemical, cell-based, organism-based, and variations thereof.  The assay Design describes the assay methodology and implementation of how the perturbation of the biological system is translated into a detectable signal.  Detection Technology relates to the physical method and technical details to detect and record a signal.  Endpoints are the final HTS results as they are usually published (such as IC50, percent inhibition, etc.).  BAO has been designed to accommodate multiplexed assays.  All main BAO components include multiple levels of sub-categories and specification classes, which are linked via object property relationships forming an expressive knowledge-based representation. | 2.8.16 | production | SHOIQ(D (GENE_OR_GENE_PRODUCT) |
| Biological Pathway Taxonomy | BPT | Biological Pathway Taxonomy (BPT) - hierarchical taxonomy of cellular and molecular mechanisms, metabolic reactions, and diseases that can be shown as signaling pathways. BPT was created manually based on the Elsevier Pathway Collection available for public usage. There are 3 types of entities in BPT – pathway, group, and folder of pathways and groups. Group is the list of genes and proteins associated with biological function and diseases. Pathway is an interactive model of molecular interactions in human and mammalian cells described in the literature. Each model was reconstructed based on facts published in peer-reviewed journals, with manually quality control. BPT pathway entity includes several properties such as link to the model in Pathway Studio software which allows reading supported sentences from published articles and additional information about members. A list of members for each model can be provided by request. Property “network” describes maps without logical causal relationships between members to distinguish them from actual models of molecular mechanisms (“signaling”). The index folder contains all pathways and groups sorted alphabetically. | synonymes2022 | alpha | cellular (CELL), human (9606), mammalian cells (CELL), all (1) |
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Neuroscience Information Framework (NIF) Standard Ontology | NIFSTD | NIF Standard ontology (NIFSTD) is a core component of Neuroscience Information Framework (NIF) project (http://neuinfo.org), a semantically enhanced portal for accessing and integrating neuroscience data, tools and information. NIFSTD includes a set of modular ontologies that provide a comprehensive collection of terminologies to describe neuroscience data and resources. | 3.1 - January 15, 2018 | production | portal (TISSUE), collection (32644) |
| G Protein-Coupled Receptor BioAssays Ontology | BAO-GPCR | The G protein-coupled receptors (GPCRs) ontology (http://www.bioassayontology.org/bao_gpcr) describes pharmacology, biochemistry and physiology of these important and therapeutically promising class of academic and pharmaceutical research targets. Incorporation and comparison of various small molecule screening data sets, such as those deposited in PubChem, ChEMBL, KEGG, PDSP, and/or IUPHAR databases, requires a formalized electronic organization system. In order to bridge the gap between the overflow of HTS data and the bottleneck of integrated analysis tools, herein, we provide the first comprehensive GPCR ontology. The development and utility of GPCR ontology was based on previously developed BioAssay Ontology (BAO). The GPCR ontology contains information about biochemical, pharmacological, and functional properties of individual GPCRs as well as GPCR-selective ligands inclusive of their HTS screening results and other records. This provides the first all-inclusive GPCR ontology with all available data to model the relationship between the GPCR binding sites and their physiologic and pharmacologic role in physiology via small molecule chemical structures. We developed this system using emerging semantic technologies, by leveraging existing and descriptive domain level ontologies. | 1.1 | beta | GPCRs (GENE_OR_GENE_PRODUCT), all (1) |
| NMR-Controlled Vocabulary | NMR | This is the new msi-sanctioned nmr CV, created within the COSMOS EU project, to support the nmrML data standard for nuclear magnetic resonance data in metabolomics with meaningful raw data descriptors. | 1.1.0 | beta | - |
| HIVOntology | HIVO004 | A Project of Biomedical Ontology for Systems Biomedicine. The goals of the Project include: (1). To pilot the novel researches on AIDS and HIV or other diseases by viruses, by means of the ontology and other biomedical informatics technologies, connecting the insights of the Biomedical Knowledge Integration (BMKI), systems biomedicine, complexity sciences etc. (2). To Carry out the methodology explorations for the goals above. | 020 | alpha | HIV (57667), viruses (10239) |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |

<a name="bioportal-category-subcellular-anatomy"></a>
## BioPortal Category "Subcellular anatomy"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Neuroscience Information Framework (NIF) Standard Ontology | NIFSTD | NIF Standard ontology (NIFSTD) is a core component of Neuroscience Information Framework (NIF) project (http://neuinfo.org), a semantically enhanced portal for accessing and integrating neuroscience data, tools and information. NIFSTD includes a set of modular ontologies that provide a comprehensive collection of terminologies to describe neuroscience data and resources. | 3.1 - January 15, 2018 | production | portal (TISSUE), collection (32644) |
| Kidney Tissue Atlas Ontology | KTAO | The Kidney Tissue Atlas Ontology (KTAO) is an ontology that integrates kidney-related cell types, cell states, locations, gene markers, disease, etc., and their relations. KTAO is developed to support the NIH-NIDDK-funed Kidney Precision Medicine Project (KPMP).  | Vision Release: 1.0.121 | beta | Kidney Tissue Atlas Ontology (CANCER), cell (CELL), disease (CANCER), Kidney (ORGAN) |

<a name="bioportal-category-taxonomic-classification"></a>
## BioPortal Category "Taxonomic Classification"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Cell Line Ontology | CLO | The Cell Line Ontology (CLO) is a community-based ontology in domain of biological cell lines with a focus on permanent cell lines from culture collections. Upper ontology structures that frame the skeleton of CLO include Basic Formal Ontology and Relation Ontology. Cell lines contained in CLO are associated with terms from other ontologies such as Cell Type Ontology, NCBI Taxonomy, and Ontology for Biomedical Investigation. A common design pattern for the cell line is used to model cell lines and their attributes, the Jurkat cell line provides ane xample.  Currently CLO contains over 36,000 cell line entries obtained from ATCC, HyperCLDB, Coriell, and bymanual curation. The cell lines are derived from 194 cell types, 656 anatomical entries, and 217 organisms. The OWL-based CLO is machine-readable and can be used in various applications. | 2.1.188 | production | Cell Line (CELL), cell lines (CELL), Cell lines (CELL), Cell Type (CELL), cell line (CELL), Jurkat cell line (CELL), ATCC (CELL), Coriell (CELL), cell (CELL) |
| National Center for Biotechnology Information (NCBI) Organismal Classification | NCBITAXON | The NCBI Taxonomy Database is a curated classification and nomenclature for all of the organisms in the public sequence databases. | NCBI2024_04_02 | production | all (1) |
| Epilepsy and Seizure Ontology | EPSO | The Epilepsy Ontology models seizure types, syndromes, etiology, electrophysiology, medication, neuropathology, including  immunohistochemistry, microscopy, and anatomical description, together with terms describing preclinical model systems. The Epilepsy Ontology has been integrated with generative artificial intelligence (AI)/LLMs and machine learning models for natural language processing (NLP), drug efficacy analysis, and translational research. | 3.1 | production | artificial (32630) |
| Common Diabetes Medications Ontology | CDMONTO | The Common Diabetes Medications Ontology (CDMOnto) is an ontology-based decision support system that has been developed with the objective of providing appropriate treatment suggestions for diabetic patients. The CDMOnto Ontology serves as a foundation for shared semantics, domain-specific, standard, and interoperable knowledge pertinent to common diabetes medications. It encompasses the knowledge of diabetes medications and offers a unified terminology that can be disseminated and processed by medical professionals, pharmacists, and diabetic patients. |  | beta | - |
| International Classification for Nursing Practice | ICNP | International Classification for Nursing Practice | classified | production | - |
| HIV ontology | HIV | The HIV ontology encompasses all knowledge about HIV | 2.6 | production | all (1) |
| OCD ontology | OCD | The Obsessive Compulsive Disorder Ontology (OCD) is a domain-specific ontology for representing the knowledge of OCD. |  | alpha | - |
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Teleost Taxonomy Ontology | TTO | The Teleost taxonomy ontology is being used to facilitate annotation of phenotypes, particularly for taxa that are not covered by NCBI because no submissions of molecular data have been made. Taxonomy ontologies can also be valuable in annotating legacy data, where authors make phenotype or ecological assertions (e.g., host-parasite associations) that refer to groups that are reorganized or no longer recognized. The taxonomy ontology serves as the source of taxa for our project's use for identifying evolutionary changes that match the phenotype of a zebrafish mutant. | unknown | production | zebrafish (7955) |
| Dementia-Related Emotional And Mood Disturbance Non-Pharmacological Treatment Ontology | DREAMDNPTO | DREAMDNPTO represents the domain knowledge specific to the non-pharmacological treatment of emotional and mood disturbances (e.g., depression and anxiety in dementia) in people with dementia living in the long-term care setting. | v1.0 | alpha | - |
| Electronic Care Plan | ECP | Our focus in this work is to define in a formal way (ontology-based) and platform independent representative metadata and relationships of care plans. This approach has as goal to work as an umbrella over all these systems and provides some advantages regarding the management, the organization and the searching of formal care plans. | 0.6 | beta | all (1) |
| OPTImAL: An ontology for patient adherence modeling in physical activity domain | OPTIMAL | OPTImAL is a reusable formal model of factors affecting CVD patient adherence to physical activity and exercise. The basis for this model relies on the analysis of data/evidence published in the scientific literature and enables identification of adherence based on the patient profile. In the model, the patient profile is seen from the perspective of 60 multidimensional aspects (320 factors). Each factor is associated with defined adherence to a particular patient activity behavior. | 1 | alpha | - |
| HierarchicalDMProcessOnto | HDMPONTO | A Hierarchical Data Mining Process Ontology, by combined OntoKDD and DMWF. |  | alpha | - |
| Vertebrate Taxonomy Ontology | VTO | The Vertebrate Taxonomy Ontology includes both extinct and extant vertebrates, aiming to provide one comprehensive hierarchy. The hierarchy backbone for extant taxa is based on the NCBI taxonomy. Since the NCBI taxonomy only includes species associated with archived genetic data, to complement this, we also incorporate taxonomic information across the vertebrates from the Paleobiology Database (PaleoDB). The Teleost Taxonomy Ontology (TTO) and AmphibiaWeb (AWeb) are incorporated to provide a more authoritative hierarchy and a richer set of names for specific taxonomic groups. | unknown | production | vertebrates (7742), AWeb (CELL) |
| Adherence and Integrated Care | ADHER_INTCARE_EN | This ontology includes concepts in English related to adherence to therapeutic plans in the context of integrated care. Includes maps with Snomed CT, bibliographic references and definitions from the Dictionary of the Royal Academy of Medicine of Spain. Ontology developed with the support of the project StandICT.eu. The equivalent ontology in Spanish is ADHER_INTCARE_SP. | English 051319 | alpha | ADHER_INTCARE_SP (CANCER) |
| Ontology of Nuclear Toxicity | ONTOTOXNUC | Ontology of nuclear toxicity developed during ToxNuc project. | 1.0 | alpha | nuclear (CELLULAR_COMPONENT) |
| Anatomic Pathology Lexicon | PATHLEX | PathLex, a comprehensive lexicon—a unified language of anatomic pathology terms—for standardized indexing and retrieval of anatomic pathology information resources.  | 1.2 | beta | Pathology (CANCER) |
| HIVOntology | HIVO004 | A Project of Biomedical Ontology for Systems Biomedicine. The goals of the Project include: (1). To pilot the novel researches on AIDS and HIV or other diseases by viruses, by means of the ontology and other biomedical informatics technologies, connecting the insights of the Biomedical Knowledge Integration (BMKI), systems biomedicine, complexity sciences etc. (2). To Carry out the methodology explorations for the goals above. | 020 | alpha | HIV (57667), viruses (10239) |
| Dementia-Related Psychotic Symptoms Non-Pharmacological Treatment Ontology | DRPSNPTO | The DRPSNPTO ontology represents the domain knowledge specific to non-pharmacological treatment of psychotic symptoms in dementia in the long-term care setting. | v1.0 | alpha | - |
| eDEM-Connect: Ontology of Dementia-related Agitation and Relationship between Informal Caregivers and Persons with Dementia | EDEM-CONNECTONTO | The EDEM-CONNECTONTO has been created to incorporate domain knowledge about the various types of agitation observed in people with dementia (PwD), as well as the dyadic relationship existing between PwD and their informal caregivers. Furthermore, it offers a structured framework for non-pharmacological interventions, enabling caregivers to effectively manage and mitigate the bidirectional effects of agitation in PwD. | Version 1.0 | production | EDEM-CONNECTONTO (CELLULAR_COMPONENT), PwD (CANCER) |
| International Classification of Wellness | ICW | The International Classification of Wellness (ICW) is Wellness taxonomy. ICW provides a scheme for classifying Wellness events (activities) in which persons (users) can participate. ICW furthers the goals of preventative medicine and population health by supporting goal setting, quantifying user engagement in wellness activities, and correlating user engagement in wellness activities with health outcomes. | 1.2 | production | - |
| Adherence and Integrated Care in Spanish | ADHER_INTCARE_SP | This ontology includes concepts in Spanish related to adherence to therapeutic plans in the context of integrated care. Includes maps with Snomed CT, bibliographic references and definitions from the Dictionary of the Royal Academy of Medicine of Spain. Ontology developed with the support of the project StandICT.eu. | Spanish 052319 | alpha | - |
| An Ontology for integration and interoperability of multi-aspect data of Protein food production chains | ALTPROTON | The ALTPROTON ontology is the foundation of a cloud-based interoperable data platform specifically tailored to address the complexities of the protein food production chain. The ALTPROTON consists of three main domains food quality, sustainability, and food processing. The multi-aspect data generated within each domain are connected via the most representative concepts to effectively integrate and interoperable for future advancement in the food industry and protein food research sector. The paper describes the concepts, classes, and relationships used in ALTPROTON to fill the gaps in existing food ontologies and databases. | 1.2 | production | - |
| TDWG Specimen LSID Ontology | TDWGSPEC | Biodiversity Information Standards (TDWG) is an international not-for-profit group that develops standards and protocols for sharing biodiversity data. They have a detailed collection of vocabularies and ontologies for studies of biological organisms. | 0.3 | alpha | collection (32644) |
| Dementia-Related Agitation Non-Pharmacological Treatment Ontology | DRANPTO | DRANPTO is developed for representing the domain knowledge specific to non-pharmacological intervention for agitation in dementia, particularly for the long-term care setting. | Version 1.0 | alpha | - |
| metaBUS controlled vocabulary for organizational and applied psychology research | METABUS | The metaBUS hierarchical taxonomy of ~5,000 constructs studied in applied psychology, organizational behavior, and human resource management. | 0.1 | alpha | metaBUS (473143), human (9606) |
| Salmon Ontology | SALMON | An ontology representing knowledge about salmon, features of their habitats, salmon stakeholders, and related entities. This ontology is considered to be in the early stages of development, though it is based upon considerable previous work. Contributions of all kinds are welcome and encouraged, including alignments, updates to existing terms, and new terms. | 0.4.0 | beta | all (1) |
| Illness and Injury | ILLNESSINJURY | This list of Illness and Injury (Morbidity and Mortality) classifications is an abbreviated version of the World Health Organization's (WHO) Startup Mortality List (ICD-10-SMoL).  ICD-10-SMoL is the WHO's application of ICD-10 for low-resource settings initial cause of death collection. | 1.0.2 | beta | collection (32644) |
| Neurologic Examination Ontology | NEO | An ontology to capture findings from the neurological examination.  The ontology is based on concepts from UMLS Metathesaurus and SNOMED CT.  Only abnormal findings are included.  The purpose of the ontology is to act as a limited vocabulary for coding the neurological examination for big data projects or machine learning. | 49.0 | production | - |
| Body in Numbers project terminology | BIN | This ontology was created as a part of Vítězslav Vacek's masters thesis and is describing the terminology used within the Body in Numbers (BiN) project that is currently underway (as of 2019-05-01) at the University of West Bohemia, Czech Republic and focuses on the terminology used during the data collection process. | 1.1 | production | collection (32644) |
| NEAT: wiNd Energy tAxonomy of Topics | WETAXTOPICS | Controlled vocabularies such as taxonomies allow an accurate and controlled approach in describing datasets. One of such controlled vocabulary is Wind Energy Taxonomy of Topics. This taxonomy is the result of EERA JP WIND IRPWind Open Data initiative that took place in 2017 in which wind energy experts generated the first version of the taxonomy as an input for defining and structuring wind energy metadata. | v0.1.1 | production | - |

<a name="bioportal-category-yeast"></a>
## BioPortal Category "Yeast"
[Back to BioPortal Category list](#bioportal-categories-list)

| Name | ID | Description | Version | Status | Species Mentions |
|------|----|-------------|---------|--------|------------------|
| Semanticscience Integrated Ontology | SIO | The semanticscience integrated ontology (SIO) provides a simple, integrated upper level ontology (types, relations) for consistent knowledge representation across physical, processual and informational entities. It provides vocabulary for the Bio2RDF (http://bio2rdf.org) and SADI (http://sadiframework.org) projects. | 1.59 | production | SADI (GENE_OR_GENE_PRODUCT) |
| Cell Cycle Ontology | CCO | An application ontology integrating knowledge about the eukaryotic cell cycle. | 2.10 | beta | cell (CELL) |

